{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Create code to search similar words in the given article"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Step1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "import bs4 as bs\n",
    "import urllib.request\n",
    "import re\n",
    "import nltk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "scrapped_data = urllib.request.urlopen('https://en.wikipedia.org/wiki/Bioinformatics')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "article = scrapped_data.read()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<!DOCTYPE html>\\n<html class=\"client-nojs vector-feature-language-in-header-enabled vector-feature-language-in-main-page-header-disabled vector-feature-sticky-header-disabled vector-feature-page-tools-pinned-disabled vector-feature-toc-pinned-enabled vector-feature-main-menu-pinned-disabled vector-feature-limited-width-enabled vector-feature-limited-width-content-enabled vector-feature-zebra-design-disabled\" lang=\"en\" dir=\"ltr\">\\n<head>\\n<meta charset=\"UTF-8\">\\n<title>Bioinformatics - Wikipedia</title>\\n<script>document.documentElement.className=\"client-js vector-feature-language-in-header-enabled vector-feature-language-in-main-page-header-disabled vector-feature-sticky-header-disabled vector-feature-page-tools-pinned-disabled vector-feature-toc-pinned-enabled vector-feature-main-menu-pinned-disabled vector-feature-limited-width-enabled vector-feature-limited-width-content-enabled vector-feature-zebra-design-disabled\";(function(){var cookie=document.cookie.match(/(?:^|; )enwikimwclientprefs=([^;]+)/);if(cookie){var featureName=cookie[1];document.documentElement.className=document.documentElement.className.replace(featureName+\\'-enabled\\',featureName+\\'-disabled\\');}}());RLCONF={\"wgBreakFrames\":false,\"wgSeparatorTransformTable\":[\"\",\"\"],\"wgDigitTransformTable\":[\"\",\"\"],\"wgDefaultDateFormat\":\"dmy\",\"wgMonthNames\":[\"\",\"January\",\"February\",\"March\",\"April\",\"May\",\"June\",\"July\",\"August\",\"September\",\"October\",\"November\",\"December\"],\"wgRequestId\":\"14145525-ac8f-4730-9a49-968972ac95ad\",\"wgCSPNonce\":false,\\n\"wgCanonicalNamespace\":\"\",\"wgCanonicalSpecialPageName\":false,\"wgNamespaceNumber\":0,\"wgPageName\":\"Bioinformatics\",\"wgTitle\":\"Bioinformatics\",\"wgCurRevisionId\":1163899643,\"wgRevisionId\":1163899643,\"wgArticleId\":4214,\"wgIsArticle\":true,\"wgIsRedirect\":false,\"wgAction\":\"view\",\"wgUserName\":null,\"wgUserGroups\":[\"*\"],\"wgCategories\":[\"Articles with short description\",\"Short description matches Wikidata\",\"Use dmy dates from September 2020\",\"Articles with hAudio microformats\",\"Pages including recorded pronunciations\",\"All articles with vague or ambiguous time\",\"Vague or ambiguous time from June 2023\",\"All articles with specifically marked weasel-worded phrases\",\"Articles with specifically marked weasel-worded phrases from June 2020\",\"All articles with unsourced statements\",\"Articles with unsourced statements from June 2023\",\"Articles with unsourced statements from July 2015\",\"Wikipedia articles needing clarification from June 2023\",\"Webarchive template wayback links\",\"Spoken articles\",\\n\"Commons category link from Wikidata\",\"Articles with BNF identifiers\",\"Articles with BNFdata identifiers\",\"Articles with GND identifiers\",\"Articles with J9U identifiers\",\"Articles with LCCN identifiers\",\"Articles with NDL identifiers\",\"Articles with NKC identifiers\",\"Bioinformatics\"],\"wgPageViewLanguage\":\"en\",\"wgPageContentLanguage\":\"en\",\"wgPageContentModel\":\"wikitext\",\"wgRelevantPageName\":\"Bioinformatics\",\"wgRelevantArticleId\":4214,\"wgIsProbablyEditable\":true,\"wgRelevantPageIsProbablyEditable\":true,\"wgRestrictionEdit\":[],\"wgRestrictionMove\":[],\"wgFlaggedRevsParams\":{\"tags\":{\"status\":{\"levels\":1}}},\"wgVisualEditor\":{\"pageLanguageCode\":\"en\",\"pageLanguageDir\":\"ltr\",\"pageVariantFallbacks\":\"en\"},\"wgMFDisplayWikibaseDescriptions\":{\"search\":true,\"watchlist\":true,\"tagline\":false,\"nearby\":true},\"wgWMESchemaEditAttemptStepOversample\":false,\"wgWMEPageLength\":100000,\"wgNoticeProject\":\"wikipedia\",\"wgMediaViewerOnClick\":true,\"wgMediaViewerEnabledByDefault\":true,\"wgPopupsFlags\":10,\\n\"wgULSCurrentAutonym\":\"English\",\"wgEditSubmitButtonLabelPublish\":true,\"wgCentralAuthMobileDomain\":false,\"wgULSPosition\":\"interlanguage\",\"wgULSisCompactLinksEnabled\":true,\"wgULSisLanguageSelectorEmpty\":false,\"wgWikibaseItemId\":\"Q128570\",\"GEHomepageSuggestedEditsEnableTopics\":true,\"wgGETopicsMatchModeEnabled\":false,\"wgGEStructuredTaskRejectionReasonTextInputEnabled\":false,\"wgGELevelingUpEnabledForUser\":false};RLSTATE={\"skins.vector.user.styles\":\"ready\",\"ext.globalCssJs.user.styles\":\"ready\",\"site.styles\":\"ready\",\"user.styles\":\"ready\",\"skins.vector.user\":\"ready\",\"ext.globalCssJs.user\":\"ready\",\"user\":\"ready\",\"user.options\":\"loading\",\"ext.cite.styles\":\"ready\",\"ext.tmh.player.styles\":\"ready\",\"codex-search-styles\":\"ready\",\"skins.vector.styles\":\"ready\",\"skins.vector.icons\":\"ready\",\"jquery.makeCollapsible.styles\":\"ready\",\"ext.visualEditor.desktopArticleTarget.noscript\":\"ready\",\"ext.wikimediaBadges\":\"ready\",\"ext.uls.interlanguage\":\"ready\",\"wikibase.client.init\":\"ready\"};RLPAGEMODULES=[\\n\"ext.cite.ux-enhancements\",\"ext.tmh.player\",\"ext.scribunto.logs\",\"site\",\"mediawiki.page.ready\",\"jquery.makeCollapsible\",\"mediawiki.toc\",\"skins.vector.js\",\"mmv.head\",\"mmv.bootstrap.autostart\",\"ext.visualEditor.desktopArticleTarget.init\",\"ext.visualEditor.targetLoader\",\"ext.eventLogging\",\"ext.wikimediaEvents\",\"ext.navigationTiming\",\"ext.cx.eventlogging.campaigns\",\"ext.quicksurveys.init\",\"ext.centralNotice.geoIP\",\"ext.centralNotice.startUp\",\"ext.gadget.ReferenceTooltips\",\"ext.gadget.charinsert\",\"ext.gadget.extra-toolbar-buttons\",\"ext.gadget.switcher\",\"ext.centralauth.centralautologin\",\"ext.popups\",\"ext.echo.centralauth\",\"ext.uls.compactlinks\",\"ext.uls.interface\",\"ext.cx.uls.quick.actions\",\"wikibase.client.vector-2022\",\"ext.growthExperiments.SuggestedEditSession\"];</script>\\n<script>(RLQ=window.RLQ||[]).push(function(){mw.loader.implement(\"user.options@12s5i\",function($,jQuery,require,module){mw.user.tokens.set({\"patrolToken\":\"+\\\\\\\\\",\"watchToken\":\"+\\\\\\\\\",\"csrfToken\":\"+\\\\\\\\\"});});});</script>\\n<link rel=\"stylesheet\" href=\"/w/load.php?lang=en&amp;modules=codex-search-styles%7Cext.cite.styles%7Cext.tmh.player.styles%7Cext.uls.interlanguage%7Cext.visualEditor.desktopArticleTarget.noscript%7Cext.wikimediaBadges%7Cjquery.makeCollapsible.styles%7Cskins.vector.icons%2Cstyles%7Cwikibase.client.init&amp;only=styles&amp;skin=vector-2022\">\\n<script async=\"\" src=\"/w/load.php?lang=en&amp;modules=startup&amp;only=scripts&amp;raw=1&amp;skin=vector-2022\"></script>\\n<meta name=\"ResourceLoaderDynamicStyles\" content=\"\">\\n<link rel=\"stylesheet\" href=\"/w/load.php?lang=en&amp;modules=site.styles&amp;only=styles&amp;skin=vector-2022\">\\n<meta name=\"generator\" content=\"MediaWiki 1.41.0-wmf.16\">\\n<meta name=\"referrer\" content=\"origin\">\\n<meta name=\"referrer\" content=\"origin-when-crossorigin\">\\n<meta name=\"referrer\" content=\"origin-when-cross-origin\">\\n<meta name=\"robots\" content=\"max-image-preview:standard\">\\n<meta name=\"format-detection\" content=\"telephone=no\">\\n<meta property=\"og:image\" content=\"https://upload.wikimedia.org/wikipedia/commons/4/43/Genome_viewer_screenshot_small.png\">\\n<meta property=\"og:image:width\" content=\"1200\">\\n<meta property=\"og:image:height\" content=\"1322\">\\n<meta property=\"og:image\" content=\"https://upload.wikimedia.org/wikipedia/commons/4/43/Genome_viewer_screenshot_small.png\">\\n<meta property=\"og:image:width\" content=\"800\">\\n<meta property=\"og:image:height\" content=\"882\">\\n<meta property=\"og:image:width\" content=\"640\">\\n<meta property=\"og:image:height\" content=\"705\">\\n<meta name=\"viewport\" content=\"width=1000\">\\n<meta property=\"og:title\" content=\"Bioinformatics - Wikipedia\">\\n<meta property=\"og:type\" content=\"website\">\\n<link rel=\"preconnect\" href=\"//upload.wikimedia.org\">\\n<link rel=\"alternate\" media=\"only screen and (max-width: 720px)\" href=\"//en.m.wikipedia.org/wiki/Bioinformatics\">\\n<link rel=\"alternate\" type=\"application/x-wiki\" title=\"Edit this page\" href=\"/w/index.php?title=Bioinformatics&amp;action=edit\">\\n<link rel=\"apple-touch-icon\" href=\"/static/apple-touch/wikipedia.png\">\\n<link rel=\"icon\" href=\"/static/favicon/wikipedia.ico\">\\n<link rel=\"search\" type=\"application/opensearchdescription+xml\" href=\"/w/opensearch_desc.php\" title=\"Wikipedia (en)\">\\n<link rel=\"EditURI\" type=\"application/rsd+xml\" href=\"//en.wikipedia.org/w/api.php?action=rsd\">\\n<link rel=\"canonical\" href=\"https://en.wikipedia.org/wiki/Bioinformatics\">\\n<link rel=\"license\" href=\"https://creativecommons.org/licenses/by-sa/4.0/deed.en\">\\n<link rel=\"alternate\" type=\"application/atom+xml\" title=\"Wikipedia Atom feed\" href=\"/w/index.php?title=Special:RecentChanges&amp;feed=atom\">\\n<link rel=\"dns-prefetch\" href=\"//meta.wikimedia.org\" />\\n<link rel=\"dns-prefetch\" href=\"//login.wikimedia.org\">\\n</head>\\n<body class=\"skin-vector skin-vector-search-vue mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject mw-editable page-Bioinformatics rootpage-Bioinformatics skin-vector-2022 action-view\"><a class=\"mw-jump-link\" href=\"#bodyContent\">Jump to content</a>\\n<div class=\"vector-header-container\">\\n\\t<header class=\"vector-header mw-header\">\\n\\t\\t<div class=\"vector-header-start\">\\n\\t\\t\\t<nav class=\"vector-main-menu-landmark\" aria-label=\"Site\" role=\"navigation\">\\n\\t\\t\\t\\t\\n<div id=\"vector-main-menu-dropdown\" class=\"vector-dropdown vector-main-menu-dropdown vector-button-flush-left vector-button-flush-right\"  >\\n\\t<input type=\"checkbox\"\\n\\t\\tid=\"vector-main-menu-dropdown-checkbox\"\\n\\t\\trole=\"button\"\\n\\t\\taria-haspopup=\"true\"\\n\\t\\tdata-event-name=\"ui.dropdown-vector-main-menu-dropdown\"\\n\\t\\tclass=\"vector-dropdown-checkbox \"\\n\\t\\t\\n\\t\\taria-label=\"Main menu\"\\n\\t\\t\\n\\t/>\\n\\t<label\\n\\t\\tid=\"vector-main-menu-dropdown-label\"\\n\\t\\tfor=\"vector-main-menu-dropdown-checkbox\"\\n\\t\\tclass=\"vector-dropdown-label cdx-button cdx-button--fake-button cdx-button--fake-button--enabled cdx-button--weight-quiet cdx-button--icon-only \"\\n\\t\\taria-hidden=\"true\"\\n\\t\\t\\n\\t>\\n\\t\\t<span class=\"vector-icon mw-ui-icon-menu mw-ui-icon-wikimedia-menu\"></span>\\n\\n\\n\\t\\t<span class=\"vector-dropdown-label-text\">Main menu</span>\\n\\t</label>\\n\\t<div class=\"vector-dropdown-content\">\\n\\n\\n\\t\\t\\t\\t<div id=\"vector-main-menu-unpinned-container\" class=\"vector-unpinned-container\">\\n\\t\\t\\n<div id=\"vector-main-menu\" class=\"vector-main-menu vector-pinnable-element\">\\n\\t<div\\n\\tclass=\"vector-pinnable-header vector-main-menu-pinnable-header vector-pinnable-header-unpinned\"\\n\\tdata-feature-name=\"main-menu-pinned\"\\n\\tdata-pinnable-element-id=\"vector-main-menu\"\\n\\tdata-pinned-container-id=\"vector-main-menu-pinned-container\"\\n\\tdata-unpinned-container-id=\"vector-main-menu-unpinned-container\"\\n>\\n\\t<div class=\"vector-pinnable-header-label\">Main menu</div>\\n\\t<button class=\"vector-pinnable-header-toggle-button vector-pinnable-header-pin-button\" data-event-name=\"pinnable-header.vector-main-menu.pin\">move to sidebar</button>\\n\\t<button class=\"vector-pinnable-header-toggle-button vector-pinnable-header-unpin-button\" data-event-name=\"pinnable-header.vector-main-menu.unpin\">hide</button>\\n</div>\\n\\n\\t\\n<div id=\"p-navigation\" class=\"vector-menu mw-portlet mw-portlet-navigation\"  >\\n\\t<div class=\"vector-menu-heading\">\\n\\t\\tNavigation\\n\\t</div>\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\"><li id=\"n-mainpage-description\" class=\"mw-list-item\"><a href=\"/wiki/Main_Page\" title=\"Visit the main page [z]\" accesskey=\"z\"><span>Main page</span></a></li><li id=\"n-contents\" class=\"mw-list-item\"><a href=\"/wiki/Wikipedia:Contents\" title=\"Guides to browsing Wikipedia\"><span>Contents</span></a></li><li id=\"n-currentevents\" class=\"mw-list-item\"><a href=\"/wiki/Portal:Current_events\" title=\"Articles related to current events\"><span>Current events</span></a></li><li id=\"n-randompage\" class=\"mw-list-item\"><a href=\"/wiki/Special:Random\" title=\"Visit a randomly selected article [x]\" accesskey=\"x\"><span>Random article</span></a></li><li id=\"n-aboutsite\" class=\"mw-list-item\"><a href=\"/wiki/Wikipedia:About\" title=\"Learn about Wikipedia and how it works\"><span>About Wikipedia</span></a></li><li id=\"n-contactpage\" class=\"mw-list-item\"><a href=\"//en.wikipedia.org/wiki/Wikipedia:Contact_us\" title=\"How to contact Wikipedia\"><span>Contact us</span></a></li><li id=\"n-sitesupport\" class=\"mw-list-item\"><a href=\"https://donate.wikimedia.org/wiki/Special:FundraiserRedirector?utm_source=donate&amp;utm_medium=sidebar&amp;utm_campaign=C13_en.wikipedia.org&amp;uselang=en\" title=\"Support us by donating to the Wikimedia Foundation\"><span>Donate</span></a></li></ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n\\t\\n\\t\\n<div id=\"p-interaction\" class=\"vector-menu mw-portlet mw-portlet-interaction\"  >\\n\\t<div class=\"vector-menu-heading\">\\n\\t\\tContribute\\n\\t</div>\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\"><li id=\"n-help\" class=\"mw-list-item\"><a href=\"/wiki/Help:Contents\" title=\"Guidance on how to use and edit Wikipedia\"><span>Help</span></a></li><li id=\"n-introduction\" class=\"mw-list-item\"><a href=\"/wiki/Help:Introduction\" title=\"Learn how to edit Wikipedia\"><span>Learn to edit</span></a></li><li id=\"n-portal\" class=\"mw-list-item\"><a href=\"/wiki/Wikipedia:Community_portal\" title=\"The hub for editors\"><span>Community portal</span></a></li><li id=\"n-recentchanges\" class=\"mw-list-item\"><a href=\"/wiki/Special:RecentChanges\" title=\"A list of recent changes to Wikipedia [r]\" accesskey=\"r\"><span>Recent changes</span></a></li><li id=\"n-upload\" class=\"mw-list-item\"><a href=\"/wiki/Wikipedia:File_upload_wizard\" title=\"Add images or other media for use on Wikipedia\"><span>Upload file</span></a></li></ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n\\t\\n<div class=\"vector-main-menu-action vector-main-menu-action-lang-alert\">\\n\\t<div class=\"vector-main-menu-action-item\">\\n\\t\\t<div class=\"vector-main-menu-action-heading vector-menu-heading\">Languages</div>\\n\\t\\t<div class=\"vector-main-menu-action-content vector-menu-content\">\\n\\t\\t\\t<div class=\"mw-message-box cdx-message cdx-message--block mw-message-box-notice cdx-message--notice vector-language-sidebar-alert\"><span class=\"cdx-message__icon\"></span><div class=\"cdx-message__content\">Language links are at the top of the page across from the title.</div></div>\\n\\t\\t</div>\\n\\t</div>\\n</div>\\n\\n</div>\\n\\n\\t\\t\\t\\t</div>\\n\\n\\t</div>\\n</div>\\n\\t\\t</nav>\\n\\t\\t\\t\\n<a href=\"/wiki/Main_Page\" class=\"mw-logo\">\\n\\t<img class=\"mw-logo-icon\" src=\"/static/images/icons/wikipedia.png\" alt=\"\"\\n\\t\\taria-hidden=\"true\" height=\"50\" width=\"50\">\\n\\t<span class=\"mw-logo-container\">\\n\\t\\t<img class=\"mw-logo-wordmark\" alt=\"Wikipedia\"\\n\\t\\t\\tsrc=\"/static/images/mobile/copyright/wikipedia-wordmark-en.svg\" style=\"width: 7.5em; height: 1.125em;\">\\n\\t\\t<img class=\"mw-logo-tagline\"\\n\\t\\t\\talt=\"The Free Encyclopedia\"\\n\\t\\t\\tsrc=\"/static/images/mobile/copyright/wikipedia-tagline-en.svg\" width=\"117\" height=\"13\" style=\"width: 7.3125em; height: 0.8125em;\">\\n\\t</span>\\n</a>\\n\\n\\t\\t</div>\\n\\t\\t<div class=\"vector-header-end\">\\n\\t\\t\\t\\n<div id=\"p-search\" role=\"search\" class=\"vector-search-box-vue  vector-search-box-collapses vector-search-box-show-thumbnail vector-search-box-auto-expand-width vector-search-box\">\\n\\t<a href=\"/wiki/Special:Search\"\\n\\t\\n\\t\\tid=\"\"\\n\\t\\tclass=\"cdx-button cdx-button--fake-button cdx-button--fake-button--enabled cdx-button--weight-quiet cdx-button--icon-only search-toggle\"\\n\\t\\ttitle=\"Search Wikipedia [f]\"accesskey=\"f\"><span class=\"vector-icon mw-ui-icon-search mw-ui-icon-wikimedia-search\"></span>\\n\\n<span>Search</span>\\n\\t</a>\\n\\t\\n\\t<div class=\"vector-typeahead-search-container\">\\n\\t\\t<div class=\"cdx-typeahead-search cdx-typeahead-search--show-thumbnail cdx-typeahead-search--auto-expand-width\">\\n\\t\\t\\t<form action=\"/w/index.php\" id=\"searchform\" class=\"cdx-search-input cdx-search-input--has-end-button\">\\n\\t\\t\\t\\t<div id=\"simpleSearch\" class=\"cdx-search-input__input-wrapper\"  data-search-loc=\"header-moved\">\\n\\t\\t\\t\\t\\t<div class=\"cdx-text-input cdx-text-input--has-start-icon\">\\n\\t\\t\\t\\t\\t\\t<input\\n\\t\\t\\t\\t\\t\\t\\tclass=\"cdx-text-input__input\"\\n\\t\\t\\t\\t\\t\\t\\t type=\"search\" name=\"search\" placeholder=\"Search Wikipedia\" aria-label=\"Search Wikipedia\" autocapitalize=\"sentences\" title=\"Search Wikipedia [f]\" accesskey=\"f\" id=\"searchInput\"\\n\\t\\t\\t\\t\\t\\t\\t\\n\\t\\t\\t\\t\\t\\t>\\n\\t\\t\\t\\t\\t\\t<span class=\"cdx-text-input__icon cdx-text-input__start-icon\"></span>\\n\\t\\t\\t\\t\\t</div>\\n\\t\\t\\t\\t\\t<input type=\"hidden\" name=\"title\" value=\"Special:Search\">\\n\\t\\t\\t\\t</div>\\n\\t\\t\\t\\t<button class=\"cdx-button cdx-search-input__end-button\">Search</button>\\n\\t\\t\\t</form>\\n\\t\\t</div>\\n\\t</div>\\n</div>\\n\\n\\t\\t\\t<nav class=\"vector-user-links\" aria-label=\"Personal tools\" role=\"navigation\" >\\n\\t\\n<div id=\"p-vector-user-menu-overflow\" class=\"vector-menu mw-portlet mw-portlet-vector-user-menu-overflow\"  >\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\"><li id=\"pt-createaccount-2\" class=\"user-links-collapsible-item mw-list-item\"><a href=\"/w/index.php?title=Special:CreateAccount&amp;returnto=Bioinformatics\" title=\"You are encouraged to create an account and log in; however, it is not mandatory\"><span>Create account</span></a></li><li id=\"pt-login-2\" class=\"user-links-collapsible-item mw-list-item\"><a href=\"/w/index.php?title=Special:UserLogin&amp;returnto=Bioinformatics\" title=\"You&#039;re encouraged to log in; however, it&#039;s not mandatory. [o]\" accesskey=\"o\"><span>Log in</span></a></li></ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n\\t\\n<div id=\"vector-user-links-dropdown\" class=\"vector-dropdown vector-user-menu vector-button-flush-right vector-user-menu-logged-out\"  title=\"Log in and more options\" >\\n\\t<input type=\"checkbox\"\\n\\t\\tid=\"vector-user-links-dropdown-checkbox\"\\n\\t\\trole=\"button\"\\n\\t\\taria-haspopup=\"true\"\\n\\t\\tdata-event-name=\"ui.dropdown-vector-user-links-dropdown\"\\n\\t\\tclass=\"vector-dropdown-checkbox \"\\n\\t\\t\\n\\t\\taria-label=\"Personal tools\"\\n\\t\\t\\n\\t/>\\n\\t<label\\n\\t\\tid=\"vector-user-links-dropdown-label\"\\n\\t\\tfor=\"vector-user-links-dropdown-checkbox\"\\n\\t\\tclass=\"vector-dropdown-label cdx-button cdx-button--fake-button cdx-button--fake-button--enabled cdx-button--weight-quiet cdx-button--icon-only \"\\n\\t\\taria-hidden=\"true\"\\n\\t\\t\\n\\t>\\n\\t\\t<span class=\"vector-icon mw-ui-icon-ellipsis mw-ui-icon-wikimedia-ellipsis\"></span>\\n\\n\\n\\t\\t<span class=\"vector-dropdown-label-text\">Personal tools</span>\\n\\t</label>\\n\\t<div class=\"vector-dropdown-content\">\\n\\n\\n\\t\\t\\n<div id=\"p-personal\" class=\"vector-menu mw-portlet mw-portlet-personal user-links-collapsible-item\"  title=\"User menu\" >\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\"><li id=\"pt-createaccount\" class=\"user-links-collapsible-item mw-list-item\"><a href=\"/w/index.php?title=Special:CreateAccount&amp;returnto=Bioinformatics\" title=\"You are encouraged to create an account and log in; however, it is not mandatory\"><span class=\"vector-icon mw-ui-icon-userAdd mw-ui-icon-wikimedia-userAdd\"></span> <span>Create account</span></a></li><li id=\"pt-login\" class=\"user-links-collapsible-item mw-list-item\"><a href=\"/w/index.php?title=Special:UserLogin&amp;returnto=Bioinformatics\" title=\"You&#039;re encouraged to log in; however, it&#039;s not mandatory. [o]\" accesskey=\"o\"><span class=\"vector-icon mw-ui-icon-logIn mw-ui-icon-wikimedia-logIn\"></span> <span>Log in</span></a></li></ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n<div id=\"p-user-menu-anon-editor\" class=\"vector-menu mw-portlet mw-portlet-user-menu-anon-editor\"  >\\n\\t<div class=\"vector-menu-heading\">\\n\\t\\tPages for logged out editors <a href=\"/wiki/Help:Introduction\" aria-label=\"Learn more about editing\"><span>learn more</span></a>\\n\\t</div>\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\"><li id=\"pt-anoncontribs\" class=\"mw-list-item\"><a href=\"/wiki/Special:MyContributions\" title=\"A list of edits made from this IP address [y]\" accesskey=\"y\"><span>Contributions</span></a></li><li id=\"pt-anontalk\" class=\"mw-list-item\"><a href=\"/wiki/Special:MyTalk\" title=\"Discussion about edits from this IP address [n]\" accesskey=\"n\"><span>Talk</span></a></li></ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n\\t\\n\\t</div>\\n</div>\\n</nav>\\n\\n\\t\\t</div>\\n\\t</header>\\n</div>\\n<div class=\"mw-page-container\">\\n\\t<div class=\"mw-page-container-inner\">\\n\\t\\t<div class=\"vector-main-menu-container \">\\n\\t\\t\\t<div id=\"mw-navigation\">\\n\\t\\t\\t\\t<nav id=\"mw-panel\" class=\"vector-main-menu-landmark\" aria-label=\"Site\" role=\"navigation\">\\n\\t\\t\\t\\t\\t<div id=\"vector-main-menu-pinned-container\" class=\"vector-pinned-container\">\\n\\t\\t\\t\\t\\n\\t\\t\\t\\t\\t</div>\\n\\t\\t</nav>\\n\\t\\t\\t</div>\\n\\t\\t</div>\\n\\t\\t<div class=\"vector-sitenotice-container\">\\n\\t\\t\\t<div id=\"siteNotice\"><!-- CentralNotice --></div>\\n\\t\\t</div>\\n\\t\\t<input type=\"checkbox\" id=\"vector-toc-collapsed-checkbox\" class=\"vector-menu-checkbox\">\\n\\t\\t<nav id=\"mw-panel-toc\" role=\"navigation\" aria-label=\"Contents\" data-event-name=\"ui.sidebar-toc\" class=\"mw-table-of-contents-container vector-toc-landmark vector-sticky-pinned-container\">\\n\\t\\t\\t<div id=\"vector-toc-pinned-container\" class=\"vector-pinned-container\">\\n\\t\\t\\t\\t\\t<div id=\"vector-toc\" class=\"vector-toc vector-pinnable-element\">\\n\\t<div\\n\\tclass=\"vector-pinnable-header vector-toc-pinnable-header vector-pinnable-header-pinned\"\\n\\tdata-feature-name=\"toc-pinned\"\\n\\tdata-pinnable-element-id=\"vector-toc\"\\n\\t\\n\\t\\n>\\n\\t<h2 class=\"vector-pinnable-header-label\">Contents</h2>\\n\\t<button class=\"vector-pinnable-header-toggle-button vector-pinnable-header-pin-button\" data-event-name=\"pinnable-header.vector-toc.pin\">move to sidebar</button>\\n\\t<button class=\"vector-pinnable-header-toggle-button vector-pinnable-header-unpin-button\" data-event-name=\"pinnable-header.vector-toc.unpin\">hide</button>\\n</div>\\n\\n\\n\\t<ul class=\"vector-toc-contents\" id=\"mw-panel-toc-list\">\\n\\t\\t<li id=\"toc-mw-content-text\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t\\t<a href=\"#\" class=\"vector-toc-link\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">(Top)</div>\\n\\t\\t\\t</a>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-History\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#History\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">1</span>History</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t\\t<button aria-controls=\"toc-History-sublist\" class=\"cdx-button cdx-button--weight-quiet cdx-button--icon-only vector-toc-toggle\">\\n\\t\\t\\t\\t<span class=\"vector-icon vector-icon--x-small mw-ui-icon-wikimedia-expand\"></span>\\n\\t\\t\\t\\t<span>Toggle History subsection</span>\\n\\t\\t\\t</button>\\n\\t\\t\\n\\t\\t<ul id=\"toc-History-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t<li id=\"toc-Sequences\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Sequences\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">1.1</span>Sequences</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Sequences-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Goals\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Goals\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">2</span>Goals</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Goals-sublist\" class=\"vector-toc-list\">\\n\\t\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Sequence_analysis\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Sequence_analysis\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">3</span>Sequence analysis</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t\\t<button aria-controls=\"toc-Sequence_analysis-sublist\" class=\"cdx-button cdx-button--weight-quiet cdx-button--icon-only vector-toc-toggle\">\\n\\t\\t\\t\\t<span class=\"vector-icon vector-icon--x-small mw-ui-icon-wikimedia-expand\"></span>\\n\\t\\t\\t\\t<span>Toggle Sequence analysis subsection</span>\\n\\t\\t\\t</button>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Sequence_analysis-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t<li id=\"toc-DNA_sequencing\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#DNA_sequencing\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">3.1</span>DNA sequencing</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-DNA_sequencing-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Sequence_assembly\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Sequence_assembly\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">3.2</span>Sequence assembly</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Sequence_assembly-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Genome_annotation\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Genome_annotation\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">3.3</span>Genome annotation</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Genome_annotation-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t\\t<li id=\"toc-Gene_function_prediction\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-3\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Gene_function_prediction\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">3.3.1</span>Gene function prediction</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Gene_function_prediction-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Computational_evolutionary_biology\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Computational_evolutionary_biology\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">3.4</span>Computational evolutionary biology</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Computational_evolutionary_biology-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Comparative_genomics\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Comparative_genomics\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">3.5</span>Comparative genomics</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Comparative_genomics-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Pan_genomics\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Pan_genomics\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">3.6</span>Pan genomics</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Pan_genomics-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Genetics_of_disease\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Genetics_of_disease\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">3.7</span>Genetics of disease</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Genetics_of_disease-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Analysis_of_mutations_in_cancer\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Analysis_of_mutations_in_cancer\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">3.8</span>Analysis of mutations in cancer</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Analysis_of_mutations_in_cancer-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Gene_and_protein_expression\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Gene_and_protein_expression\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">4</span>Gene and protein expression</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t\\t<button aria-controls=\"toc-Gene_and_protein_expression-sublist\" class=\"cdx-button cdx-button--weight-quiet cdx-button--icon-only vector-toc-toggle\">\\n\\t\\t\\t\\t<span class=\"vector-icon vector-icon--x-small mw-ui-icon-wikimedia-expand\"></span>\\n\\t\\t\\t\\t<span>Toggle Gene and protein expression subsection</span>\\n\\t\\t\\t</button>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Gene_and_protein_expression-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t<li id=\"toc-Analysis_of_gene_expression\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Analysis_of_gene_expression\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">4.1</span>Analysis of gene expression</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Analysis_of_gene_expression-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Analysis_of_protein_expression\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Analysis_of_protein_expression\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">4.2</span>Analysis of protein expression</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Analysis_of_protein_expression-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Analysis_of_regulation\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Analysis_of_regulation\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">4.3</span>Analysis of regulation</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Analysis_of_regulation-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Analysis_of_cellular_organization\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Analysis_of_cellular_organization\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">5</span>Analysis of cellular organization</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t\\t<button aria-controls=\"toc-Analysis_of_cellular_organization-sublist\" class=\"cdx-button cdx-button--weight-quiet cdx-button--icon-only vector-toc-toggle\">\\n\\t\\t\\t\\t<span class=\"vector-icon vector-icon--x-small mw-ui-icon-wikimedia-expand\"></span>\\n\\t\\t\\t\\t<span>Toggle Analysis of cellular organization subsection</span>\\n\\t\\t\\t</button>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Analysis_of_cellular_organization-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t<li id=\"toc-Microscopy_and_image_analysis\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Microscopy_and_image_analysis\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">5.1</span>Microscopy and image analysis</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Microscopy_and_image_analysis-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Protein_localization\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Protein_localization\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">5.2</span>Protein localization</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Protein_localization-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Nuclear_organization_of_chromatin\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Nuclear_organization_of_chromatin\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">5.3</span>Nuclear organization of chromatin</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Nuclear_organization_of_chromatin-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Structural_bioinformatics\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Structural_bioinformatics\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">6</span>Structural bioinformatics</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t\\t<button aria-controls=\"toc-Structural_bioinformatics-sublist\" class=\"cdx-button cdx-button--weight-quiet cdx-button--icon-only vector-toc-toggle\">\\n\\t\\t\\t\\t<span class=\"vector-icon vector-icon--x-small mw-ui-icon-wikimedia-expand\"></span>\\n\\t\\t\\t\\t<span>Toggle Structural bioinformatics subsection</span>\\n\\t\\t\\t</button>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Structural_bioinformatics-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t<li id=\"toc-Amino_acid_sequence\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Amino_acid_sequence\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">6.1</span>Amino acid sequence</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Amino_acid_sequence-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Homology\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Homology\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">6.2</span>Homology</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Homology-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Network_and_systems_biology\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Network_and_systems_biology\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">7</span>Network and systems biology</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t\\t<button aria-controls=\"toc-Network_and_systems_biology-sublist\" class=\"cdx-button cdx-button--weight-quiet cdx-button--icon-only vector-toc-toggle\">\\n\\t\\t\\t\\t<span class=\"vector-icon vector-icon--x-small mw-ui-icon-wikimedia-expand\"></span>\\n\\t\\t\\t\\t<span>Toggle Network and systems biology subsection</span>\\n\\t\\t\\t</button>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Network_and_systems_biology-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t<li id=\"toc-Molecular_interaction_networks\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Molecular_interaction_networks\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">7.1</span>Molecular interaction networks</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Molecular_interaction_networks-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Others\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Others\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">8</span>Others</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t\\t<button aria-controls=\"toc-Others-sublist\" class=\"cdx-button cdx-button--weight-quiet cdx-button--icon-only vector-toc-toggle\">\\n\\t\\t\\t\\t<span class=\"vector-icon vector-icon--x-small mw-ui-icon-wikimedia-expand\"></span>\\n\\t\\t\\t\\t<span>Toggle Others subsection</span>\\n\\t\\t\\t</button>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Others-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t<li id=\"toc-Literature_analysis\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Literature_analysis\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">8.1</span>Literature analysis</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Literature_analysis-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-High-throughput_image_analysis\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#High-throughput_image_analysis\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">8.2</span>High-throughput image analysis</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-High-throughput_image_analysis-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-High-throughput_single_cell_data_analysis\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#High-throughput_single_cell_data_analysis\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">8.3</span>High-throughput single cell data analysis</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-High-throughput_single_cell_data_analysis-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Biodiversity_informatics\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Biodiversity_informatics\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">8.4</span>Biodiversity informatics</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Biodiversity_informatics-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Ontologies_and_data_integration\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Ontologies_and_data_integration\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">8.5</span>Ontologies and data integration</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Ontologies_and_data_integration-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Databases\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Databases\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">9</span>Databases</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Databases-sublist\" class=\"vector-toc-list\">\\n\\t\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Software_and_tools\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Software_and_tools\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">10</span>Software and tools</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t\\t<button aria-controls=\"toc-Software_and_tools-sublist\" class=\"cdx-button cdx-button--weight-quiet cdx-button--icon-only vector-toc-toggle\">\\n\\t\\t\\t\\t<span class=\"vector-icon vector-icon--x-small mw-ui-icon-wikimedia-expand\"></span>\\n\\t\\t\\t\\t<span>Toggle Software and tools subsection</span>\\n\\t\\t\\t</button>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Software_and_tools-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t<li id=\"toc-Open-source_bioinformatics_software\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Open-source_bioinformatics_software\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">10.1</span>Open-source bioinformatics software</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Open-source_bioinformatics_software-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-Web_services_in_bioinformatics\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Web_services_in_bioinformatics\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">10.2</span>Web services in bioinformatics</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Web_services_in_bioinformatics-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t\\t<li id=\"toc-Bioinformatics_workflow_management_systems\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-3\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#Bioinformatics_workflow_management_systems\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">10.2.1</span>Bioinformatics workflow management systems</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-Bioinformatics_workflow_management_systems-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t</ul>\\n\\t\\t</li>\\n\\t\\t<li id=\"toc-BioCompute_and_BioCompute_Objects\"\\n\\t\\t\\tclass=\"vector-toc-list-item vector-toc-level-2\">\\n\\t\\t\\t<a class=\"vector-toc-link\" href=\"#BioCompute_and_BioCompute_Objects\">\\n\\t\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t\\t<span class=\"vector-toc-numb\">10.3</span>BioCompute and BioCompute Objects</div>\\n\\t\\t\\t</a>\\n\\t\\t\\t\\n\\t\\t\\t<ul id=\"toc-BioCompute_and_BioCompute_Objects-sublist\" class=\"vector-toc-list\">\\n\\t\\t\\t</ul>\\n\\t\\t</li>\\n\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Education_platforms\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Education_platforms\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">11</span>Education platforms</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Education_platforms-sublist\" class=\"vector-toc-list\">\\n\\t\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Conferences\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Conferences\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">12</span>Conferences</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Conferences-sublist\" class=\"vector-toc-list\">\\n\\t\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-See_also\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#See_also\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">13</span>See also</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t<ul id=\"toc-See_also-sublist\" class=\"vector-toc-list\">\\n\\t\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-References\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#References\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">14</span>References</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t<ul id=\"toc-References-sublist\" class=\"vector-toc-list\">\\n\\t\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-Further_reading\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#Further_reading\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">15</span>Further reading</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t<ul id=\"toc-Further_reading-sublist\" class=\"vector-toc-list\">\\n\\t\\t</ul>\\n\\t</li>\\n\\t<li id=\"toc-External_links\"\\n\\t\\tclass=\"vector-toc-list-item vector-toc-level-1\">\\n\\t\\t<a class=\"vector-toc-link\" href=\"#External_links\">\\n\\t\\t\\t<div class=\"vector-toc-text\">\\n\\t\\t\\t<span class=\"vector-toc-numb\">16</span>External links</div>\\n\\t\\t</a>\\n\\t\\t\\n\\t\\t<ul id=\"toc-External_links-sublist\" class=\"vector-toc-list\">\\n\\t\\t</ul>\\n\\t</li>\\n</ul>\\n</div>\\n\\n\\t\\t\\t</div>\\n\\t\\t</nav>\\n\\t\\t<div class=\"mw-content-container\">\\n\\t\\t\\t<main id=\"content\" class=\"mw-body\" role=\"main\">\\n\\t\\t\\t\\t<header class=\"mw-body-header vector-page-titlebar\">\\n\\t\\t\\t\\t\\t<label\\n\\t\\t\\t\\t\\t\\tid=\"vector-toc-collapsed-button\"\\n\\t\\t\\t\\t\\t\\tclass=\"cdx-button cdx-button--fake-button cdx-button--fake-button--enabled cdx-button--weight-quiet vector-button-flush-left cdx-button--icon-only\"\\n\\t\\t\\t\\t\\t\\tfor=\"vector-toc-collapsed-checkbox\"\\n\\t\\t\\t\\t\\t\\trole=\"button\"\\n\\t\\t\\t\\t\\t\\taria-controls=\"vector-toc\"\\n\\t\\t\\t\\t\\t\\ttabindex=\"0\"\\n\\t\\t\\t\\t\\t\\ttitle=\"Table of Contents\">\\n\\t\\t\\t\\t\\t\\t<span class=\"vector-icon mw-ui-icon-wikimedia-listBullet\"></span>\\n\\t\\t\\t\\t\\t\\t<span>Toggle the table of contents</span>\\n\\t\\t\\t\\t\\t</label>\\n\\t\\t\\t\\t\\n\\t\\t\\t\\t\\t<nav role=\"navigation\" aria-label=\"Contents\" class=\"vector-toc-landmark\">\\n\\t\\t\\t\\t\\t\\t\\n<div id=\"vector-page-titlebar-toc\" class=\"vector-dropdown vector-page-titlebar-toc vector-button-flush-left\"  >\\n\\t<input type=\"checkbox\"\\n\\t\\tid=\"vector-page-titlebar-toc-checkbox\"\\n\\t\\trole=\"button\"\\n\\t\\taria-haspopup=\"true\"\\n\\t\\tdata-event-name=\"ui.dropdown-vector-page-titlebar-toc\"\\n\\t\\tclass=\"vector-dropdown-checkbox \"\\n\\t\\t\\n\\t\\taria-label=\"Toggle the table of contents\"\\n\\t\\t\\n\\t/>\\n\\t<label\\n\\t\\tid=\"vector-page-titlebar-toc-label\"\\n\\t\\tfor=\"vector-page-titlebar-toc-checkbox\"\\n\\t\\tclass=\"vector-dropdown-label cdx-button cdx-button--fake-button cdx-button--fake-button--enabled cdx-button--weight-quiet cdx-button--icon-only \"\\n\\t\\taria-hidden=\"true\"\\n\\t\\t\\n\\t>\\n\\t\\t<span class=\"vector-icon mw-ui-icon-listBullet mw-ui-icon-wikimedia-listBullet\"></span>\\n\\n\\n\\t\\t<span class=\"vector-dropdown-label-text\">Toggle the table of contents</span>\\n\\t</label>\\n\\t<div class=\"vector-dropdown-content\">\\n\\n\\n\\t\\t\\t\\t\\t\\t\\t<div id=\"vector-page-titlebar-toc-unpinned-container\" class=\"vector-unpinned-container\">\\n\\t\\t\\t</div>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\t\\t\\t\\t\\t</nav>\\n\\t\\t\\t\\t\\t<h1 id=\"firstHeading\" class=\"firstHeading mw-first-heading\"><span class=\"mw-page-title-main\">Bioinformatics</span></h1>\\n\\t\\t\\t\\t\\n\\t\\t\\t\\t\\t\\t\\t\\n<div id=\"p-lang-btn\" class=\"vector-dropdown mw-portlet mw-portlet-lang\"  >\\n\\t<input type=\"checkbox\"\\n\\t\\tid=\"p-lang-btn-checkbox\"\\n\\t\\trole=\"button\"\\n\\t\\taria-haspopup=\"true\"\\n\\t\\tdata-event-name=\"ui.dropdown-p-lang-btn\"\\n\\t\\tclass=\"vector-dropdown-checkbox mw-interlanguage-selector\"\\n\\t\\taria-label=\"Go to an article in another language. Available in 76 languages\"\\n\\t\\t\\n\\t\\t\\n\\t/>\\n\\t<label\\n\\t\\tid=\"p-lang-btn-label\"\\n\\t\\tfor=\"p-lang-btn-checkbox\"\\n\\t\\tclass=\"vector-dropdown-label cdx-button cdx-button--fake-button cdx-button--fake-button--enabled cdx-button--weight-quiet cdx-button--action-progressive mw-portlet-lang-heading-76\"\\n\\t\\taria-hidden=\"true\"\\n\\t\\t\\n\\t>\\n\\t\\t<span class=\"vector-icon mw-ui-icon-language-progressive mw-ui-icon-wikimedia-language-progressive\"></span>\\n\\n\\n\\t\\t<span class=\"vector-dropdown-label-text\">76 languages</span>\\n\\t</label>\\n\\t<div class=\"vector-dropdown-content\">\\n\\n\\t\\t<div class=\"vector-menu-content\">\\n\\t\\t\\t\\n\\t\\t\\t<ul class=\"vector-menu-content-list\"><li class=\"interlanguage-link interwiki-ar mw-list-item\"><a href=\"https://ar.wikipedia.org/wiki/%D9%85%D8%B9%D9%84%D9%88%D9%85%D8%A7%D8%AA%D9%8A%D8%A9_%D8%AD%D9%8A%D9%88%D9%8A%D8%A9\" title=\"\\xd9\\x85\\xd8\\xb9\\xd9\\x84\\xd9\\x88\\xd9\\x85\\xd8\\xa7\\xd8\\xaa\\xd9\\x8a\\xd8\\xa9 \\xd8\\xad\\xd9\\x8a\\xd9\\x88\\xd9\\x8a\\xd8\\xa9 \\xe2\\x80\\x93 Arabic\" lang=\"ar\" hreflang=\"ar\" class=\"interlanguage-link-target\"><span>\\xd8\\xa7\\xd9\\x84\\xd8\\xb9\\xd8\\xb1\\xd8\\xa8\\xd9\\x8a\\xd8\\xa9</span></a></li><li class=\"interlanguage-link interwiki-az mw-list-item\"><a href=\"https://az.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Azerbaijani\" lang=\"az\" hreflang=\"az\" class=\"interlanguage-link-target\"><span>Az\\xc9\\x99rbaycanca</span></a></li><li class=\"interlanguage-link interwiki-bn mw-list-item\"><a href=\"https://bn.wikipedia.org/wiki/%E0%A6%9C%E0%A7%88%E0%A6%AC_%E0%A6%A4%E0%A6%A5%E0%A7%8D%E0%A6%AF%E0%A6%AC%E0%A6%BF%E0%A6%9C%E0%A7%8D%E0%A6%9E%E0%A6%BE%E0%A6%A8\" title=\"\\xe0\\xa6\\x9c\\xe0\\xa7\\x88\\xe0\\xa6\\xac \\xe0\\xa6\\xa4\\xe0\\xa6\\xa5\\xe0\\xa7\\x8d\\xe0\\xa6\\xaf\\xe0\\xa6\\xac\\xe0\\xa6\\xbf\\xe0\\xa6\\x9c\\xe0\\xa7\\x8d\\xe0\\xa6\\x9e\\xe0\\xa6\\xbe\\xe0\\xa6\\xa8 \\xe2\\x80\\x93 Bangla\" lang=\"bn\" hreflang=\"bn\" class=\"interlanguage-link-target\"><span>\\xe0\\xa6\\xac\\xe0\\xa6\\xbe\\xe0\\xa6\\x82\\xe0\\xa6\\xb2\\xe0\\xa6\\xbe</span></a></li><li class=\"interlanguage-link interwiki-zh-min-nan mw-list-item\"><a href=\"https://zh-min-nan.wikipedia.org/wiki/Seng-bu%CC%8Dt_chu-s%C3%ACn-ha%CC%8Dk\" title=\"Seng-bu\\xcc\\x8dt chu-s\\xc3\\xacn-ha\\xcc\\x8dk \\xe2\\x80\\x93 Min Nan Chinese\" lang=\"nan\" hreflang=\"nan\" class=\"interlanguage-link-target\"><span>B\\xc3\\xa2n-l\\xc3\\xa2m-g\\xc3\\xba</span></a></li><li class=\"interlanguage-link interwiki-be mw-list-item\"><a href=\"https://be.wikipedia.org/wiki/%D0%91%D1%96%D1%8F%D1%96%D0%BD%D1%84%D0%B0%D1%80%D0%BC%D0%B0%D1%82%D1%8B%D0%BA%D0%B0\" title=\"\\xd0\\x91\\xd1\\x96\\xd1\\x8f\\xd1\\x96\\xd0\\xbd\\xd1\\x84\\xd0\\xb0\\xd1\\x80\\xd0\\xbc\\xd0\\xb0\\xd1\\x82\\xd1\\x8b\\xd0\\xba\\xd0\\xb0 \\xe2\\x80\\x93 Belarusian\" lang=\"be\" hreflang=\"be\" class=\"interlanguage-link-target\"><span>\\xd0\\x91\\xd0\\xb5\\xd0\\xbb\\xd0\\xb0\\xd1\\x80\\xd1\\x83\\xd1\\x81\\xd0\\xba\\xd0\\xb0\\xd1\\x8f</span></a></li><li class=\"interlanguage-link interwiki-bg mw-list-item\"><a href=\"https://bg.wikipedia.org/wiki/%D0%91%D0%B8%D0%BE%D0%B8%D0%BD%D1%84%D0%BE%D1%80%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0\" title=\"\\xd0\\x91\\xd0\\xb8\\xd0\\xbe\\xd0\\xb8\\xd0\\xbd\\xd1\\x84\\xd0\\xbe\\xd1\\x80\\xd0\\xbc\\xd0\\xb0\\xd1\\x82\\xd0\\xb8\\xd0\\xba\\xd0\\xb0 \\xe2\\x80\\x93 Bulgarian\" lang=\"bg\" hreflang=\"bg\" class=\"interlanguage-link-target\"><span>\\xd0\\x91\\xd1\\x8a\\xd0\\xbb\\xd0\\xb3\\xd0\\xb0\\xd1\\x80\\xd1\\x81\\xd0\\xba\\xd0\\xb8</span></a></li><li class=\"interlanguage-link interwiki-bs mw-list-item\"><a href=\"https://bs.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Bosnian\" lang=\"bs\" hreflang=\"bs\" class=\"interlanguage-link-target\"><span>Bosanski</span></a></li><li class=\"interlanguage-link interwiki-ca badge-Q70893996 mw-list-item\" title=\"\"><a href=\"https://ca.wikipedia.org/wiki/Bioinform%C3%A0tica\" title=\"Bioinform\\xc3\\xa0tica \\xe2\\x80\\x93 Catalan\" lang=\"ca\" hreflang=\"ca\" class=\"interlanguage-link-target\"><span>Catal\\xc3\\xa0</span></a></li><li class=\"interlanguage-link interwiki-cs mw-list-item\"><a href=\"https://cs.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Czech\" lang=\"cs\" hreflang=\"cs\" class=\"interlanguage-link-target\"><span>\\xc4\\x8ce\\xc5\\xa1tina</span></a></li><li class=\"interlanguage-link interwiki-da mw-list-item\"><a href=\"https://da.wikipedia.org/wiki/Bioinformatik\" title=\"Bioinformatik \\xe2\\x80\\x93 Danish\" lang=\"da\" hreflang=\"da\" class=\"interlanguage-link-target\"><span>Dansk</span></a></li><li class=\"interlanguage-link interwiki-de mw-list-item\"><a href=\"https://de.wikipedia.org/wiki/Bioinformatik\" title=\"Bioinformatik \\xe2\\x80\\x93 German\" lang=\"de\" hreflang=\"de\" class=\"interlanguage-link-target\"><span>Deutsch</span></a></li><li class=\"interlanguage-link interwiki-et mw-list-item\"><a href=\"https://et.wikipedia.org/wiki/Bioinformaatika\" title=\"Bioinformaatika \\xe2\\x80\\x93 Estonian\" lang=\"et\" hreflang=\"et\" class=\"interlanguage-link-target\"><span>Eesti</span></a></li><li class=\"interlanguage-link interwiki-el mw-list-item\"><a href=\"https://el.wikipedia.org/wiki/%CE%92%CE%B9%CE%BF%CF%80%CE%BB%CE%B7%CF%81%CE%BF%CF%86%CE%BF%CF%81%CE%B9%CE%BA%CE%AE\" title=\"\\xce\\x92\\xce\\xb9\\xce\\xbf\\xcf\\x80\\xce\\xbb\\xce\\xb7\\xcf\\x81\\xce\\xbf\\xcf\\x86\\xce\\xbf\\xcf\\x81\\xce\\xb9\\xce\\xba\\xce\\xae \\xe2\\x80\\x93 Greek\" lang=\"el\" hreflang=\"el\" class=\"interlanguage-link-target\"><span>\\xce\\x95\\xce\\xbb\\xce\\xbb\\xce\\xb7\\xce\\xbd\\xce\\xb9\\xce\\xba\\xce\\xac</span></a></li><li class=\"interlanguage-link interwiki-es mw-list-item\"><a href=\"https://es.wikipedia.org/wiki/Bioinform%C3%A1tica\" title=\"Bioinform\\xc3\\xa1tica \\xe2\\x80\\x93 Spanish\" lang=\"es\" hreflang=\"es\" class=\"interlanguage-link-target\"><span>Espa\\xc3\\xb1ol</span></a></li><li class=\"interlanguage-link interwiki-eo mw-list-item\"><a href=\"https://eo.wikipedia.org/wiki/Biokomputiko\" title=\"Biokomputiko \\xe2\\x80\\x93 Esperanto\" lang=\"eo\" hreflang=\"eo\" class=\"interlanguage-link-target\"><span>Esperanto</span></a></li><li class=\"interlanguage-link interwiki-eu mw-list-item\"><a href=\"https://eu.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Basque\" lang=\"eu\" hreflang=\"eu\" class=\"interlanguage-link-target\"><span>Euskara</span></a></li><li class=\"interlanguage-link interwiki-fa mw-list-item\"><a href=\"https://fa.wikipedia.org/wiki/%D8%A8%DB%8C%D9%88%D8%A7%D9%86%D9%81%D9%88%D8%B1%D9%85%D8%A7%D8%AA%DB%8C%DA%A9\" title=\"\\xd8\\xa8\\xdb\\x8c\\xd9\\x88\\xd8\\xa7\\xd9\\x86\\xd9\\x81\\xd9\\x88\\xd8\\xb1\\xd9\\x85\\xd8\\xa7\\xd8\\xaa\\xdb\\x8c\\xda\\xa9 \\xe2\\x80\\x93 Persian\" lang=\"fa\" hreflang=\"fa\" class=\"interlanguage-link-target\"><span>\\xd9\\x81\\xd8\\xa7\\xd8\\xb1\\xd8\\xb3\\xdb\\x8c</span></a></li><li class=\"interlanguage-link interwiki-fr mw-list-item\"><a href=\"https://fr.wikipedia.org/wiki/Bio-informatique\" title=\"Bio-informatique \\xe2\\x80\\x93 French\" lang=\"fr\" hreflang=\"fr\" class=\"interlanguage-link-target\"><span>Fran\\xc3\\xa7ais</span></a></li><li class=\"interlanguage-link interwiki-gl mw-list-item\"><a href=\"https://gl.wikipedia.org/wiki/Bioinform%C3%A1tica\" title=\"Bioinform\\xc3\\xa1tica \\xe2\\x80\\x93 Galician\" lang=\"gl\" hreflang=\"gl\" class=\"interlanguage-link-target\"><span>Galego</span></a></li><li class=\"interlanguage-link interwiki-ki mw-list-item\"><a href=\"https://ki.wikipedia.org/wiki/Bioinformatics\" title=\"Bioinformatics \\xe2\\x80\\x93 Kikuyu\" lang=\"ki\" hreflang=\"ki\" class=\"interlanguage-link-target\"><span>G\\xc4\\xa9k\\xc5\\xa9y\\xc5\\xa9</span></a></li><li class=\"interlanguage-link interwiki-ko mw-list-item\"><a href=\"https://ko.wikipedia.org/wiki/%EC%83%9D%EB%AC%BC%EC%A0%95%EB%B3%B4%ED%95%99\" title=\"\\xec\\x83\\x9d\\xeb\\xac\\xbc\\xec\\xa0\\x95\\xeb\\xb3\\xb4\\xed\\x95\\x99 \\xe2\\x80\\x93 Korean\" lang=\"ko\" hreflang=\"ko\" class=\"interlanguage-link-target\"><span>\\xed\\x95\\x9c\\xea\\xb5\\xad\\xec\\x96\\xb4</span></a></li><li class=\"interlanguage-link interwiki-hy mw-list-item\"><a href=\"https://hy.wikipedia.org/wiki/%D4%BF%D5%A5%D5%B6%D5%BD%D5%A1%D5%AB%D5%B6%D6%86%D5%B8%D6%80%D5%B4%D5%A1%D5%BF%D5%AB%D5%AF%D5%A1\" title=\"\\xd4\\xbf\\xd5\\xa5\\xd5\\xb6\\xd5\\xbd\\xd5\\xa1\\xd5\\xab\\xd5\\xb6\\xd6\\x86\\xd5\\xb8\\xd6\\x80\\xd5\\xb4\\xd5\\xa1\\xd5\\xbf\\xd5\\xab\\xd5\\xaf\\xd5\\xa1 \\xe2\\x80\\x93 Armenian\" lang=\"hy\" hreflang=\"hy\" class=\"interlanguage-link-target\"><span>\\xd5\\x80\\xd5\\xa1\\xd5\\xb5\\xd5\\xa5\\xd6\\x80\\xd5\\xa5\\xd5\\xb6</span></a></li><li class=\"interlanguage-link interwiki-hi mw-list-item\"><a href=\"https://hi.wikipedia.org/wiki/%E0%A4%9C%E0%A5%88%E0%A4%B5_%E0%A4%B8%E0%A5%82%E0%A4%9A%E0%A4%A8%E0%A4%BE_%E0%A4%B5%E0%A4%BF%E0%A4%9C%E0%A5%8D%E0%A4%9E%E0%A4%BE%E0%A4%A8\" title=\"\\xe0\\xa4\\x9c\\xe0\\xa5\\x88\\xe0\\xa4\\xb5 \\xe0\\xa4\\xb8\\xe0\\xa5\\x82\\xe0\\xa4\\x9a\\xe0\\xa4\\xa8\\xe0\\xa4\\xbe \\xe0\\xa4\\xb5\\xe0\\xa4\\xbf\\xe0\\xa4\\x9c\\xe0\\xa5\\x8d\\xe0\\xa4\\x9e\\xe0\\xa4\\xbe\\xe0\\xa4\\xa8 \\xe2\\x80\\x93 Hindi\" lang=\"hi\" hreflang=\"hi\" class=\"interlanguage-link-target\"><span>\\xe0\\xa4\\xb9\\xe0\\xa4\\xbf\\xe0\\xa4\\xa8\\xe0\\xa5\\x8d\\xe0\\xa4\\xa6\\xe0\\xa5\\x80</span></a></li><li class=\"interlanguage-link interwiki-hr mw-list-item\"><a href=\"https://hr.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Croatian\" lang=\"hr\" hreflang=\"hr\" class=\"interlanguage-link-target\"><span>Hrvatski</span></a></li><li class=\"interlanguage-link interwiki-io mw-list-item\"><a href=\"https://io.wikipedia.org/wiki/Bioinformatiko\" title=\"Bioinformatiko \\xe2\\x80\\x93 Ido\" lang=\"io\" hreflang=\"io\" class=\"interlanguage-link-target\"><span>Ido</span></a></li><li class=\"interlanguage-link interwiki-id mw-list-item\"><a href=\"https://id.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Indonesian\" lang=\"id\" hreflang=\"id\" class=\"interlanguage-link-target\"><span>Bahasa Indonesia</span></a></li><li class=\"interlanguage-link interwiki-is mw-list-item\"><a href=\"https://is.wikipedia.org/wiki/L%C3%ADfuppl%C3%BDsingafr%C3%A6%C3%B0i\" title=\"L\\xc3\\xadfuppl\\xc3\\xbdsingafr\\xc3\\xa6\\xc3\\xb0i \\xe2\\x80\\x93 Icelandic\" lang=\"is\" hreflang=\"is\" class=\"interlanguage-link-target\"><span>\\xc3\\x8dslenska</span></a></li><li class=\"interlanguage-link interwiki-it mw-list-item\"><a href=\"https://it.wikipedia.org/wiki/Bioinformatica\" title=\"Bioinformatica \\xe2\\x80\\x93 Italian\" lang=\"it\" hreflang=\"it\" class=\"interlanguage-link-target\"><span>Italiano</span></a></li><li class=\"interlanguage-link interwiki-he mw-list-item\"><a href=\"https://he.wikipedia.org/wiki/%D7%91%D7%99%D7%95%D7%90%D7%99%D7%A0%D7%A4%D7%95%D7%A8%D7%9E%D7%98%D7%99%D7%A7%D7%94\" title=\"\\xd7\\x91\\xd7\\x99\\xd7\\x95\\xd7\\x90\\xd7\\x99\\xd7\\xa0\\xd7\\xa4\\xd7\\x95\\xd7\\xa8\\xd7\\x9e\\xd7\\x98\\xd7\\x99\\xd7\\xa7\\xd7\\x94 \\xe2\\x80\\x93 Hebrew\" lang=\"he\" hreflang=\"he\" class=\"interlanguage-link-target\"><span>\\xd7\\xa2\\xd7\\x91\\xd7\\xa8\\xd7\\x99\\xd7\\xaa</span></a></li><li class=\"interlanguage-link interwiki-jv mw-list-item\"><a href=\"https://jv.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Javanese\" lang=\"jv\" hreflang=\"jv\" class=\"interlanguage-link-target\"><span>Jawa</span></a></li><li class=\"interlanguage-link interwiki-kn mw-list-item\"><a href=\"https://kn.wikipedia.org/wiki/%E0%B2%AC%E0%B2%AF%E0%B3%8B%E0%B2%87%E0%B2%A8%E0%B3%8D%E2%80%8C%E0%B2%AB%E0%B2%B0%E0%B3%8D%E0%B2%AE%E0%B3%8D%E0%B2%AF%E0%B2%BE%E0%B2%9F%E0%B2%BF%E0%B2%95%E0%B3%8D%E0%B2%B8%E0%B3%8D%E2%80%8C\" title=\"\\xe0\\xb2\\xac\\xe0\\xb2\\xaf\\xe0\\xb3\\x8b\\xe0\\xb2\\x87\\xe0\\xb2\\xa8\\xe0\\xb3\\x8d\\xe2\\x80\\x8c\\xe0\\xb2\\xab\\xe0\\xb2\\xb0\\xe0\\xb3\\x8d\\xe0\\xb2\\xae\\xe0\\xb3\\x8d\\xe0\\xb2\\xaf\\xe0\\xb2\\xbe\\xe0\\xb2\\x9f\\xe0\\xb2\\xbf\\xe0\\xb2\\x95\\xe0\\xb3\\x8d\\xe0\\xb2\\xb8\\xe0\\xb3\\x8d\\xe2\\x80\\x8c \\xe2\\x80\\x93 Kannada\" lang=\"kn\" hreflang=\"kn\" class=\"interlanguage-link-target\"><span>\\xe0\\xb2\\x95\\xe0\\xb2\\xa8\\xe0\\xb3\\x8d\\xe0\\xb2\\xa8\\xe0\\xb2\\xa1</span></a></li><li class=\"interlanguage-link interwiki-kk mw-list-item\"><a href=\"https://kk.wikipedia.org/wiki/%D0%91%D0%B8%D0%BE%D0%B8%D0%BD%D1%84%D0%BE%D1%80%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0\" title=\"\\xd0\\x91\\xd0\\xb8\\xd0\\xbe\\xd0\\xb8\\xd0\\xbd\\xd1\\x84\\xd0\\xbe\\xd1\\x80\\xd0\\xbc\\xd0\\xb0\\xd1\\x82\\xd0\\xb8\\xd0\\xba\\xd0\\xb0 \\xe2\\x80\\x93 Kazakh\" lang=\"kk\" hreflang=\"kk\" class=\"interlanguage-link-target\"><span>\\xd2\\x9a\\xd0\\xb0\\xd0\\xb7\\xd0\\xb0\\xd2\\x9b\\xd1\\x88\\xd0\\xb0</span></a></li><li class=\"interlanguage-link interwiki-la mw-list-item\"><a href=\"https://la.wikipedia.org/wiki/Informatio_genetica\" title=\"Informatio genetica \\xe2\\x80\\x93 Latin\" lang=\"la\" hreflang=\"la\" class=\"interlanguage-link-target\"><span>Latina</span></a></li><li class=\"interlanguage-link interwiki-lv mw-list-item\"><a href=\"https://lv.wikipedia.org/wiki/Bioinform%C4%81tika\" title=\"Bioinform\\xc4\\x81tika \\xe2\\x80\\x93 Latvian\" lang=\"lv\" hreflang=\"lv\" class=\"interlanguage-link-target\"><span>Latvie\\xc5\\xa1u</span></a></li><li class=\"interlanguage-link interwiki-lb mw-list-item\"><a href=\"https://lb.wikipedia.org/wiki/Bioinformatik\" title=\"Bioinformatik \\xe2\\x80\\x93 Luxembourgish\" lang=\"lb\" hreflang=\"lb\" class=\"interlanguage-link-target\"><span>L\\xc3\\xabtzebuergesch</span></a></li><li class=\"interlanguage-link interwiki-li mw-list-item\"><a href=\"https://li.wikipedia.org/wiki/Bioinformatica\" title=\"Bioinformatica \\xe2\\x80\\x93 Limburgish\" lang=\"li\" hreflang=\"li\" class=\"interlanguage-link-target\"><span>Limburgs</span></a></li><li class=\"interlanguage-link interwiki-lfn mw-list-item\"><a href=\"https://lfn.wikipedia.org/wiki/Bioinformatica\" title=\"Bioinformatica \\xe2\\x80\\x93 Lingua Franca Nova\" lang=\"lfn\" hreflang=\"lfn\" class=\"interlanguage-link-target\"><span>Lingua Franca Nova</span></a></li><li class=\"interlanguage-link interwiki-hu mw-list-item\"><a href=\"https://hu.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Hungarian\" lang=\"hu\" hreflang=\"hu\" class=\"interlanguage-link-target\"><span>Magyar</span></a></li><li class=\"interlanguage-link interwiki-mk mw-list-item\"><a href=\"https://mk.wikipedia.org/wiki/%D0%91%D0%B8%D0%BE%D0%B8%D0%BD%D1%84%D0%BE%D1%80%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0\" title=\"\\xd0\\x91\\xd0\\xb8\\xd0\\xbe\\xd0\\xb8\\xd0\\xbd\\xd1\\x84\\xd0\\xbe\\xd1\\x80\\xd0\\xbc\\xd0\\xb0\\xd1\\x82\\xd0\\xb8\\xd0\\xba\\xd0\\xb0 \\xe2\\x80\\x93 Macedonian\" lang=\"mk\" hreflang=\"mk\" class=\"interlanguage-link-target\"><span>\\xd0\\x9c\\xd0\\xb0\\xd0\\xba\\xd0\\xb5\\xd0\\xb4\\xd0\\xbe\\xd0\\xbd\\xd1\\x81\\xd0\\xba\\xd0\\xb8</span></a></li><li class=\"interlanguage-link interwiki-ml badge-Q70893996 mw-list-item\" title=\"\"><a href=\"https://ml.wikipedia.org/wiki/%E0%B4%AC%E0%B4%AF%E0%B5%8B_%E0%B4%87%E0%B5%BB%E0%B4%AB%E0%B5%BC%E0%B4%AE%E0%B4%BE%E0%B4%B1%E0%B5%8D%E0%B4%B1%E0%B4%BF%E0%B4%95%E0%B5%8D%E0%B4%B8%E0%B5%8D\" title=\"\\xe0\\xb4\\xac\\xe0\\xb4\\xaf\\xe0\\xb5\\x8b \\xe0\\xb4\\x87\\xe0\\xb5\\xbb\\xe0\\xb4\\xab\\xe0\\xb5\\xbc\\xe0\\xb4\\xae\\xe0\\xb4\\xbe\\xe0\\xb4\\xb1\\xe0\\xb5\\x8d\\xe0\\xb4\\xb1\\xe0\\xb4\\xbf\\xe0\\xb4\\x95\\xe0\\xb5\\x8d\\xe0\\xb4\\xb8\\xe0\\xb5\\x8d \\xe2\\x80\\x93 Malayalam\" lang=\"ml\" hreflang=\"ml\" class=\"interlanguage-link-target\"><span>\\xe0\\xb4\\xae\\xe0\\xb4\\xb2\\xe0\\xb4\\xaf\\xe0\\xb4\\xbe\\xe0\\xb4\\xb3\\xe0\\xb4\\x82</span></a></li><li class=\"interlanguage-link interwiki-xmf mw-list-item\"><a href=\"https://xmf.wikipedia.org/wiki/%E1%83%91%E1%83%98%E1%83%9D%E1%83%98%E1%83%9C%E1%83%A4%E1%83%9D%E1%83%A0%E1%83%9B%E1%83%90%E1%83%A2%E1%83%98%E1%83%99%E1%83%90\" title=\"\\xe1\\x83\\x91\\xe1\\x83\\x98\\xe1\\x83\\x9d\\xe1\\x83\\x98\\xe1\\x83\\x9c\\xe1\\x83\\xa4\\xe1\\x83\\x9d\\xe1\\x83\\xa0\\xe1\\x83\\x9b\\xe1\\x83\\x90\\xe1\\x83\\xa2\\xe1\\x83\\x98\\xe1\\x83\\x99\\xe1\\x83\\x90 \\xe2\\x80\\x93 Mingrelian\" lang=\"xmf\" hreflang=\"xmf\" class=\"interlanguage-link-target\"><span>\\xe1\\x83\\x9b\\xe1\\x83\\x90\\xe1\\x83\\xa0\\xe1\\x83\\x92\\xe1\\x83\\x90\\xe1\\x83\\x9a\\xe1\\x83\\xa3\\xe1\\x83\\xa0\\xe1\\x83\\x98</span></a></li><li class=\"interlanguage-link interwiki-arz mw-list-item\"><a href=\"https://arz.wikipedia.org/wiki/%D9%85%D8%B9%D9%84%D9%88%D9%85%D8%A7%D8%AA%D9%8A%D9%87_%D8%AD%D9%8A%D9%88%D9%8A%D9%87\" title=\"\\xd9\\x85\\xd8\\xb9\\xd9\\x84\\xd9\\x88\\xd9\\x85\\xd8\\xa7\\xd8\\xaa\\xd9\\x8a\\xd9\\x87 \\xd8\\xad\\xd9\\x8a\\xd9\\x88\\xd9\\x8a\\xd9\\x87 \\xe2\\x80\\x93 Egyptian Arabic\" lang=\"arz\" hreflang=\"arz\" class=\"interlanguage-link-target\"><span>\\xd9\\x85\\xd8\\xb5\\xd8\\xb1\\xd9\\x89</span></a></li><li class=\"interlanguage-link interwiki-ms mw-list-item\"><a href=\"https://ms.wikipedia.org/wiki/Bioinformasi\" title=\"Bioinformasi \\xe2\\x80\\x93 Malay\" lang=\"ms\" hreflang=\"ms\" class=\"interlanguage-link-target\"><span>Bahasa Melayu</span></a></li><li class=\"interlanguage-link interwiki-mn mw-list-item\"><a href=\"https://mn.wikipedia.org/wiki/%D0%91%D0%B8%D0%BE%D0%B8%D0%BD%D1%84%D0%BE%D1%80%D0%BC%D0%B0%D1%82%D0%B8%D0%BA\" title=\"\\xd0\\x91\\xd0\\xb8\\xd0\\xbe\\xd0\\xb8\\xd0\\xbd\\xd1\\x84\\xd0\\xbe\\xd1\\x80\\xd0\\xbc\\xd0\\xb0\\xd1\\x82\\xd0\\xb8\\xd0\\xba \\xe2\\x80\\x93 Mongolian\" lang=\"mn\" hreflang=\"mn\" class=\"interlanguage-link-target\"><span>\\xd0\\x9c\\xd0\\xbe\\xd0\\xbd\\xd0\\xb3\\xd0\\xbe\\xd0\\xbb</span></a></li><li class=\"interlanguage-link interwiki-nl mw-list-item\"><a href=\"https://nl.wikipedia.org/wiki/Bio-informatica\" title=\"Bio-informatica \\xe2\\x80\\x93 Dutch\" lang=\"nl\" hreflang=\"nl\" class=\"interlanguage-link-target\"><span>Nederlands</span></a></li><li class=\"interlanguage-link interwiki-ja mw-list-item\"><a href=\"https://ja.wikipedia.org/wiki/%E3%83%90%E3%82%A4%E3%82%AA%E3%82%A4%E3%83%B3%E3%83%95%E3%82%A9%E3%83%9E%E3%83%86%E3%82%A3%E3%82%AF%E3%82%B9\" title=\"\\xe3\\x83\\x90\\xe3\\x82\\xa4\\xe3\\x82\\xaa\\xe3\\x82\\xa4\\xe3\\x83\\xb3\\xe3\\x83\\x95\\xe3\\x82\\xa9\\xe3\\x83\\x9e\\xe3\\x83\\x86\\xe3\\x82\\xa3\\xe3\\x82\\xaf\\xe3\\x82\\xb9 \\xe2\\x80\\x93 Japanese\" lang=\"ja\" hreflang=\"ja\" class=\"interlanguage-link-target\"><span>\\xe6\\x97\\xa5\\xe6\\x9c\\xac\\xe8\\xaa\\x9e</span></a></li><li class=\"interlanguage-link interwiki-no mw-list-item\"><a href=\"https://no.wikipedia.org/wiki/Bioinformatikk\" title=\"Bioinformatikk \\xe2\\x80\\x93 Norwegian Bokm\\xc3\\xa5l\" lang=\"nb\" hreflang=\"nb\" class=\"interlanguage-link-target\"><span>Norsk bokm\\xc3\\xa5l</span></a></li><li class=\"interlanguage-link interwiki-nn mw-list-item\"><a href=\"https://nn.wikipedia.org/wiki/Bioinformatikk\" title=\"Bioinformatikk \\xe2\\x80\\x93 Norwegian Nynorsk\" lang=\"nn\" hreflang=\"nn\" class=\"interlanguage-link-target\"><span>Norsk nynorsk</span></a></li><li class=\"interlanguage-link interwiki-nov mw-list-item\"><a href=\"https://nov.wikipedia.org/wiki/Bioinformatike\" title=\"Bioinformatike \\xe2\\x80\\x93 Novial\" lang=\"nov\" hreflang=\"nov\" class=\"interlanguage-link-target\"><span>Novial</span></a></li><li class=\"interlanguage-link interwiki-oc mw-list-item\"><a href=\"https://oc.wikipedia.org/wiki/Bioinformatica\" title=\"Bioinformatica \\xe2\\x80\\x93 Occitan\" lang=\"oc\" hreflang=\"oc\" class=\"interlanguage-link-target\"><span>Occitan</span></a></li><li class=\"interlanguage-link interwiki-ps mw-list-item\"><a href=\"https://ps.wikipedia.org/wiki/%D8%A8%D8%A7%D9%8A%D9%88%D8%A7%D9%86%D9%81%D8%A7%D8%B1%D9%85%D9%8A%D9%BC%D9%8A%DA%A9%D8%B3\" title=\"\\xd8\\xa8\\xd8\\xa7\\xd9\\x8a\\xd9\\x88\\xd8\\xa7\\xd9\\x86\\xd9\\x81\\xd8\\xa7\\xd8\\xb1\\xd9\\x85\\xd9\\x8a\\xd9\\xbc\\xd9\\x8a\\xda\\xa9\\xd8\\xb3 \\xe2\\x80\\x93 Pashto\" lang=\"ps\" hreflang=\"ps\" class=\"interlanguage-link-target\"><span>\\xd9\\xbe\\xda\\x9a\\xd8\\xaa\\xd9\\x88</span></a></li><li class=\"interlanguage-link interwiki-pl mw-list-item\"><a href=\"https://pl.wikipedia.org/wiki/Bioinformatyka\" title=\"Bioinformatyka \\xe2\\x80\\x93 Polish\" lang=\"pl\" hreflang=\"pl\" class=\"interlanguage-link-target\"><span>Polski</span></a></li><li class=\"interlanguage-link interwiki-pt mw-list-item\"><a href=\"https://pt.wikipedia.org/wiki/Bioinform%C3%A1tica\" title=\"Bioinform\\xc3\\xa1tica \\xe2\\x80\\x93 Portuguese\" lang=\"pt\" hreflang=\"pt\" class=\"interlanguage-link-target\"><span>Portugu\\xc3\\xaas</span></a></li><li class=\"interlanguage-link interwiki-ro mw-list-item\"><a href=\"https://ro.wikipedia.org/wiki/Bioinformatic%C4%83\" title=\"Bioinformatic\\xc4\\x83 \\xe2\\x80\\x93 Romanian\" lang=\"ro\" hreflang=\"ro\" class=\"interlanguage-link-target\"><span>Rom\\xc3\\xa2n\\xc4\\x83</span></a></li><li class=\"interlanguage-link interwiki-ru mw-list-item\"><a href=\"https://ru.wikipedia.org/wiki/%D0%91%D0%B8%D0%BE%D0%B8%D0%BD%D1%84%D0%BE%D1%80%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0\" title=\"\\xd0\\x91\\xd0\\xb8\\xd0\\xbe\\xd0\\xb8\\xd0\\xbd\\xd1\\x84\\xd0\\xbe\\xd1\\x80\\xd0\\xbc\\xd0\\xb0\\xd1\\x82\\xd0\\xb8\\xd0\\xba\\xd0\\xb0 \\xe2\\x80\\x93 Russian\" lang=\"ru\" hreflang=\"ru\" class=\"interlanguage-link-target\"><span>\\xd0\\xa0\\xd1\\x83\\xd1\\x81\\xd1\\x81\\xd0\\xba\\xd0\\xb8\\xd0\\xb9</span></a></li><li class=\"interlanguage-link interwiki-sq mw-list-item\"><a href=\"https://sq.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Albanian\" lang=\"sq\" hreflang=\"sq\" class=\"interlanguage-link-target\"><span>Shqip</span></a></li><li class=\"interlanguage-link interwiki-simple mw-list-item\"><a href=\"https://simple.wikipedia.org/wiki/Bioinformatics\" title=\"Bioinformatics \\xe2\\x80\\x93 Simple English\" lang=\"en-simple\" hreflang=\"en-simple\" class=\"interlanguage-link-target\"><span>Simple English</span></a></li><li class=\"interlanguage-link interwiki-sk mw-list-item\"><a href=\"https://sk.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Slovak\" lang=\"sk\" hreflang=\"sk\" class=\"interlanguage-link-target\"><span>Sloven\\xc4\\x8dina</span></a></li><li class=\"interlanguage-link interwiki-sl mw-list-item\"><a href=\"https://sl.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Slovenian\" lang=\"sl\" hreflang=\"sl\" class=\"interlanguage-link-target\"><span>Sloven\\xc5\\xa1\\xc4\\x8dina</span></a></li><li class=\"interlanguage-link interwiki-ckb mw-list-item\"><a href=\"https://ckb.wikipedia.org/wiki/%D8%B2%DB%8C%D9%86%D8%AF%DB%95%D8%B2%D8%A7%D9%86%DB%8C%D8%A7%D8%B1%DB%8C%DB%8C%DB%95%DA%A9%D8%A7%D9%86\" title=\"\\xd8\\xb2\\xdb\\x8c\\xd9\\x86\\xd8\\xaf\\xdb\\x95\\xd8\\xb2\\xd8\\xa7\\xd9\\x86\\xdb\\x8c\\xd8\\xa7\\xd8\\xb1\\xdb\\x8c\\xdb\\x8c\\xdb\\x95\\xda\\xa9\\xd8\\xa7\\xd9\\x86 \\xe2\\x80\\x93 Central Kurdish\" lang=\"ckb\" hreflang=\"ckb\" class=\"interlanguage-link-target\"><span>\\xda\\xa9\\xd9\\x88\\xd8\\xb1\\xd8\\xaf\\xdb\\x8c</span></a></li><li class=\"interlanguage-link interwiki-sr mw-list-item\"><a href=\"https://sr.wikipedia.org/wiki/%D0%91%D0%B8%D0%BE%D0%B8%D0%BD%D1%84%D0%BE%D1%80%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0\" title=\"\\xd0\\x91\\xd0\\xb8\\xd0\\xbe\\xd0\\xb8\\xd0\\xbd\\xd1\\x84\\xd0\\xbe\\xd1\\x80\\xd0\\xbc\\xd0\\xb0\\xd1\\x82\\xd0\\xb8\\xd0\\xba\\xd0\\xb0 \\xe2\\x80\\x93 Serbian\" lang=\"sr\" hreflang=\"sr\" class=\"interlanguage-link-target\"><span>\\xd0\\xa1\\xd1\\x80\\xd0\\xbf\\xd1\\x81\\xd0\\xba\\xd0\\xb8 / srpski</span></a></li><li class=\"interlanguage-link interwiki-sh mw-list-item\"><a href=\"https://sh.wikipedia.org/wiki/Bioinformatika\" title=\"Bioinformatika \\xe2\\x80\\x93 Serbo-Croatian\" lang=\"sh\" hreflang=\"sh\" class=\"interlanguage-link-target\"><span>Srpskohrvatski / \\xd1\\x81\\xd1\\x80\\xd0\\xbf\\xd1\\x81\\xd0\\xba\\xd0\\xbe\\xd1\\x85\\xd1\\x80\\xd0\\xb2\\xd0\\xb0\\xd1\\x82\\xd1\\x81\\xd0\\xba\\xd0\\xb8</span></a></li><li class=\"interlanguage-link interwiki-fi mw-list-item\"><a href=\"https://fi.wikipedia.org/wiki/Bioinformatiikka\" title=\"Bioinformatiikka \\xe2\\x80\\x93 Finnish\" lang=\"fi\" hreflang=\"fi\" class=\"interlanguage-link-target\"><span>Suomi</span></a></li><li class=\"interlanguage-link interwiki-sv mw-list-item\"><a href=\"https://sv.wikipedia.org/wiki/Bioinformatik\" title=\"Bioinformatik \\xe2\\x80\\x93 Swedish\" lang=\"sv\" hreflang=\"sv\" class=\"interlanguage-link-target\"><span>Svenska</span></a></li><li class=\"interlanguage-link interwiki-tl mw-list-item\"><a href=\"https://tl.wikipedia.org/wiki/Biyoimpormatika\" title=\"Biyoimpormatika \\xe2\\x80\\x93 Tagalog\" lang=\"tl\" hreflang=\"tl\" class=\"interlanguage-link-target\"><span>Tagalog</span></a></li><li class=\"interlanguage-link interwiki-ta mw-list-item\"><a href=\"https://ta.wikipedia.org/wiki/%E0%AE%89%E0%AE%AF%E0%AE%BF%E0%AE%B0%E0%AF%8D_%E0%AE%A4%E0%AE%95%E0%AE%B5%E0%AE%B2%E0%AE%BF%E0%AE%AF%E0%AE%B2%E0%AF%8D\" title=\"\\xe0\\xae\\x89\\xe0\\xae\\xaf\\xe0\\xae\\xbf\\xe0\\xae\\xb0\\xe0\\xaf\\x8d \\xe0\\xae\\xa4\\xe0\\xae\\x95\\xe0\\xae\\xb5\\xe0\\xae\\xb2\\xe0\\xae\\xbf\\xe0\\xae\\xaf\\xe0\\xae\\xb2\\xe0\\xaf\\x8d \\xe2\\x80\\x93 Tamil\" lang=\"ta\" hreflang=\"ta\" class=\"interlanguage-link-target\"><span>\\xe0\\xae\\xa4\\xe0\\xae\\xae\\xe0\\xae\\xbf\\xe0\\xae\\xb4\\xe0\\xaf\\x8d</span></a></li><li class=\"interlanguage-link interwiki-th mw-list-item\"><a href=\"https://th.wikipedia.org/wiki/%E0%B8%8A%E0%B8%B5%E0%B8%A7%E0%B8%AA%E0%B8%B2%E0%B8%A3%E0%B8%AA%E0%B8%99%E0%B9%80%E0%B8%97%E0%B8%A8%E0%B8%A8%E0%B8%B2%E0%B8%AA%E0%B8%95%E0%B8%A3%E0%B9%8C\" title=\"\\xe0\\xb8\\x8a\\xe0\\xb8\\xb5\\xe0\\xb8\\xa7\\xe0\\xb8\\xaa\\xe0\\xb8\\xb2\\xe0\\xb8\\xa3\\xe0\\xb8\\xaa\\xe0\\xb8\\x99\\xe0\\xb9\\x80\\xe0\\xb8\\x97\\xe0\\xb8\\xa8\\xe0\\xb8\\xa8\\xe0\\xb8\\xb2\\xe0\\xb8\\xaa\\xe0\\xb8\\x95\\xe0\\xb8\\xa3\\xe0\\xb9\\x8c \\xe2\\x80\\x93 Thai\" lang=\"th\" hreflang=\"th\" class=\"interlanguage-link-target\"><span>\\xe0\\xb9\\x84\\xe0\\xb8\\x97\\xe0\\xb8\\xa2</span></a></li><li class=\"interlanguage-link interwiki-tr mw-list-item\"><a href=\"https://tr.wikipedia.org/wiki/Biyoenformatik\" title=\"Biyoenformatik \\xe2\\x80\\x93 Turkish\" lang=\"tr\" hreflang=\"tr\" class=\"interlanguage-link-target\"><span>T\\xc3\\xbcrk\\xc3\\xa7e</span></a></li><li class=\"interlanguage-link interwiki-uk mw-list-item\"><a href=\"https://uk.wikipedia.org/wiki/%D0%91%D1%96%D0%BE%D1%96%D0%BD%D1%84%D0%BE%D1%80%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0\" title=\"\\xd0\\x91\\xd1\\x96\\xd0\\xbe\\xd1\\x96\\xd0\\xbd\\xd1\\x84\\xd0\\xbe\\xd1\\x80\\xd0\\xbc\\xd0\\xb0\\xd1\\x82\\xd0\\xb8\\xd0\\xba\\xd0\\xb0 \\xe2\\x80\\x93 Ukrainian\" lang=\"uk\" hreflang=\"uk\" class=\"interlanguage-link-target\"><span>\\xd0\\xa3\\xd0\\xba\\xd1\\x80\\xd0\\xb0\\xd1\\x97\\xd0\\xbd\\xd1\\x81\\xd1\\x8c\\xd0\\xba\\xd0\\xb0</span></a></li><li class=\"interlanguage-link interwiki-ur mw-list-item\"><a href=\"https://ur.wikipedia.org/wiki/%D8%AD%DB%8C%D8%A7%D8%AA%DB%8C_%D8%A7%D8%B7%D9%84%D8%A7%D8%B9%DB%8C%D8%A7%D8%AA\" title=\"\\xd8\\xad\\xdb\\x8c\\xd8\\xa7\\xd8\\xaa\\xdb\\x8c \\xd8\\xa7\\xd8\\xb7\\xd9\\x84\\xd8\\xa7\\xd8\\xb9\\xdb\\x8c\\xd8\\xa7\\xd8\\xaa \\xe2\\x80\\x93 Urdu\" lang=\"ur\" hreflang=\"ur\" class=\"interlanguage-link-target\"><span>\\xd8\\xa7\\xd8\\xb1\\xd8\\xaf\\xd9\\x88</span></a></li><li class=\"interlanguage-link interwiki-vi mw-list-item\"><a href=\"https://vi.wikipedia.org/wiki/Tin_sinh_h%E1%BB%8Dc\" title=\"Tin sinh h\\xe1\\xbb\\x8dc \\xe2\\x80\\x93 Vietnamese\" lang=\"vi\" hreflang=\"vi\" class=\"interlanguage-link-target\"><span>Ti\\xe1\\xba\\xbfng Vi\\xe1\\xbb\\x87t</span></a></li><li class=\"interlanguage-link interwiki-war mw-list-item\"><a href=\"https://war.wikipedia.org/wiki/Biyoimpormatika\" title=\"Biyoimpormatika \\xe2\\x80\\x93 Waray\" lang=\"war\" hreflang=\"war\" class=\"interlanguage-link-target\"><span>Winaray</span></a></li><li class=\"interlanguage-link interwiki-wuu mw-list-item\"><a href=\"https://wuu.wikipedia.org/wiki/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6\" title=\"\\xe7\\x94\\x9f\\xe7\\x89\\xa9\\xe4\\xbf\\xa1\\xe6\\x81\\xaf\\xe5\\xad\\xa6 \\xe2\\x80\\x93 Wu Chinese\" lang=\"wuu\" hreflang=\"wuu\" class=\"interlanguage-link-target\"><span>\\xe5\\x90\\xb4\\xe8\\xaf\\xad</span></a></li><li class=\"interlanguage-link interwiki-yi mw-list-item\"><a href=\"https://yi.wikipedia.org/wiki/%D7%91%D7%99%D7%90%D7%90%D7%99%D7%A0%D7%A4%D7%90%D7%A8%D7%9E%D7%90%D7%98%D7%99%D7%A7\" title=\"\\xd7\\x91\\xd7\\x99\\xd7\\x90\\xd7\\x90\\xd7\\x99\\xd7\\xa0\\xd7\\xa4\\xd7\\x90\\xd7\\xa8\\xd7\\x9e\\xd7\\x90\\xd7\\x98\\xd7\\x99\\xd7\\xa7 \\xe2\\x80\\x93 Yiddish\" lang=\"yi\" hreflang=\"yi\" class=\"interlanguage-link-target\"><span>\\xd7\\x99\\xd7\\x99\\xd6\\xb4\\xd7\\x93\\xd7\\x99\\xd7\\xa9</span></a></li><li class=\"interlanguage-link interwiki-zh-yue mw-list-item\"><a href=\"https://zh-yue.wikipedia.org/wiki/%E7%94%9F%E7%89%A9%E8%B3%87%E8%A8%8A%E7%A7%91%E5%AD%B8\" title=\"\\xe7\\x94\\x9f\\xe7\\x89\\xa9\\xe8\\xb3\\x87\\xe8\\xa8\\x8a\\xe7\\xa7\\x91\\xe5\\xad\\xb8 \\xe2\\x80\\x93 Cantonese\" lang=\"yue\" hreflang=\"yue\" class=\"interlanguage-link-target\"><span>\\xe7\\xb2\\xb5\\xe8\\xaa\\x9e</span></a></li><li class=\"interlanguage-link interwiki-zh mw-list-item\"><a href=\"https://zh.wikipedia.org/wiki/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6\" title=\"\\xe7\\x94\\x9f\\xe7\\x89\\xa9\\xe4\\xbf\\xa1\\xe6\\x81\\xaf\\xe5\\xad\\xa6 \\xe2\\x80\\x93 Chinese\" lang=\"zh\" hreflang=\"zh\" class=\"interlanguage-link-target\"><span>\\xe4\\xb8\\xad\\xe6\\x96\\x87</span></a></li></ul>\\n\\t\\t\\t<div class=\"after-portlet after-portlet-lang\"><span class=\"wb-langlinks-edit wb-langlinks-link\"><a href=\"https://www.wikidata.org/wiki/Special:EntityPage/Q128570#sitelinks-wikipedia\" title=\"Edit interlanguage links\" class=\"wbc-editpage\">Edit links</a></span></div>\\n\\t\\t</div>\\n\\n\\t</div>\\n</div>\\n\\t\\t\\t\\t</header>\\n\\t\\t\\t\\t<div class=\"vector-page-toolbar\">\\n\\t\\t\\t\\t\\t<div class=\"vector-page-toolbar-container\">\\n\\t\\t\\t\\t\\t\\t<div id=\"left-navigation\">\\n\\t\\t\\t\\t\\t\\t\\t<nav aria-label=\"Namespaces\">\\n\\t\\t\\t\\t\\t\\t\\t\\t\\n<div id=\"p-associated-pages\" class=\"vector-menu vector-menu-tabs mw-portlet mw-portlet-associated-pages\"  >\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\">\\n\\t\\t\\t<li id=\"ca-nstab-main\" class=\"selected vector-tab-noicon mw-list-item\"><a data-mw=\"interface\" href=\"/wiki/Bioinformatics\" title=\"View the content page [c]\" accesskey=\"c\" class=\"\"><span>Article</span></a>\\n</li>\\n<li id=\"ca-talk\" class=\"vector-tab-noicon mw-list-item\"><a data-mw=\"interface\" href=\"/wiki/Talk:Bioinformatics\" rel=\"discussion\" title=\"Discuss improvements to the content page [t]\" accesskey=\"t\" class=\"\"><span>Talk</span></a>\\n</li>\\n\\n\\t\\t</ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n\\t\\t\\t\\t\\t\\t\\t\\t\\n\\n<div id=\"p-variants\" class=\"vector-dropdown mw-portlet mw-portlet-variants emptyPortlet\"  >\\n\\t<input type=\"checkbox\"\\n\\t\\tid=\"p-variants-checkbox\"\\n\\t\\trole=\"button\"\\n\\t\\taria-haspopup=\"true\"\\n\\t\\tdata-event-name=\"ui.dropdown-p-variants\"\\n\\t\\tclass=\"vector-dropdown-checkbox\"\\n\\t\\taria-label=\"Change language variant\"\\n\\t\\t\\n\\t\\t\\n\\t/>\\n\\t<label\\n\\t\\tid=\"p-variants-label\"\\n\\t\\tfor=\"p-variants-checkbox\"\\n\\t\\tclass=\"vector-dropdown-label \"\\n\\t\\taria-hidden=\"true\"\\n\\t\\t\\n\\t>\\n\\t\\t\\n\\t\\t<span class=\"vector-dropdown-label-text\">English</span>\\n\\t</label>\\n\\t<div class=\"vector-dropdown-content\">\\n\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\"></ul>\\n\\t\\t\\n\\t</div>\\n\\n\\t</div>\\n</div>\\n\\t\\t\\t\\t\\t\\t\\t</nav>\\n\\t\\t\\t\\t\\t\\t</div>\\n\\t\\t\\t\\t\\t\\t<div id=\"right-navigation\" class=\"vector-collapsible\">\\n\\t\\t\\t\\t\\t\\t\\t<nav aria-label=\"Views\">\\n\\t\\t\\t\\t\\t\\t\\t\\t\\n<div id=\"p-views\" class=\"vector-menu vector-menu-tabs mw-portlet mw-portlet-views\"  >\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\">\\n\\t\\t\\t<li id=\"ca-view\" class=\"selected vector-tab-noicon mw-list-item\"><a data-mw=\"interface\" href=\"/wiki/Bioinformatics\" class=\"\"><span>Read</span></a>\\n</li>\\n<li id=\"ca-edit\" class=\"vector-tab-noicon mw-list-item\"><a data-mw=\"interface\" href=\"/w/index.php?title=Bioinformatics&amp;action=edit\" title=\"Edit this page [e]\" accesskey=\"e\" class=\"\"><span>Edit</span></a>\\n</li>\\n<li id=\"ca-history\" class=\"vector-tab-noicon mw-list-item\"><a data-mw=\"interface\" href=\"/w/index.php?title=Bioinformatics&amp;action=history\" title=\"Past revisions of this page [h]\" accesskey=\"h\" class=\"\"><span>View history</span></a>\\n</li>\\n\\n\\t\\t</ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n\\t\\t\\t\\t\\t\\t\\t</nav>\\n\\t\\t\\t\\t\\n\\t\\t\\t\\t\\t\\t\\t<nav class=\"vector-page-tools-landmark\" aria-label=\"More options\">\\n\\t\\t\\t\\t\\t\\t\\t\\t\\n<div id=\"vector-page-tools-dropdown\" class=\"vector-dropdown vector-page-tools-dropdown\"  >\\n\\t<input type=\"checkbox\"\\n\\t\\tid=\"vector-page-tools-dropdown-checkbox\"\\n\\t\\trole=\"button\"\\n\\t\\taria-haspopup=\"true\"\\n\\t\\tdata-event-name=\"ui.dropdown-vector-page-tools-dropdown\"\\n\\t\\tclass=\"vector-dropdown-checkbox \"\\n\\t\\t\\n\\t\\taria-label=\"Tools\"\\n\\t\\t\\n\\t/>\\n\\t<label\\n\\t\\tid=\"vector-page-tools-dropdown-label\"\\n\\t\\tfor=\"vector-page-tools-dropdown-checkbox\"\\n\\t\\tclass=\"vector-dropdown-label cdx-button cdx-button--fake-button cdx-button--fake-button--enabled cdx-button--weight-quiet\"\\n\\t\\taria-hidden=\"true\"\\n\\t\\t\\n\\t>\\n\\t\\t\\n\\t\\t<span class=\"vector-dropdown-label-text\">Tools</span>\\n\\t</label>\\n\\t<div class=\"vector-dropdown-content\">\\n\\n\\n\\t\\t\\t\\t\\t\\t\\t\\t\\t<div id=\"vector-page-tools-unpinned-container\" class=\"vector-unpinned-container\">\\n\\t\\t\\t\\t\\t\\t\\n<div id=\"vector-page-tools\" class=\"vector-page-tools vector-pinnable-element\">\\n\\t<div\\n\\tclass=\"vector-pinnable-header vector-page-tools-pinnable-header vector-pinnable-header-unpinned\"\\n\\tdata-feature-name=\"page-tools-pinned\"\\n\\tdata-pinnable-element-id=\"vector-page-tools\"\\n\\tdata-pinned-container-id=\"vector-page-tools-pinned-container\"\\n\\tdata-unpinned-container-id=\"vector-page-tools-unpinned-container\"\\n>\\n\\t<div class=\"vector-pinnable-header-label\">Tools</div>\\n\\t<button class=\"vector-pinnable-header-toggle-button vector-pinnable-header-pin-button\" data-event-name=\"pinnable-header.vector-page-tools.pin\">move to sidebar</button>\\n\\t<button class=\"vector-pinnable-header-toggle-button vector-pinnable-header-unpin-button\" data-event-name=\"pinnable-header.vector-page-tools.unpin\">hide</button>\\n</div>\\n\\n\\t\\n<div id=\"p-cactions\" class=\"vector-menu mw-portlet mw-portlet-cactions emptyPortlet vector-has-collapsible-items\"  title=\"More options\" >\\n\\t<div class=\"vector-menu-heading\">\\n\\t\\tActions\\n\\t</div>\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\"><li id=\"ca-more-view\" class=\"selected vector-more-collapsible-item mw-list-item\"><a href=\"/wiki/Bioinformatics\"><span>Read</span></a></li><li id=\"ca-more-edit\" class=\"vector-more-collapsible-item mw-list-item\"><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit\"><span>Edit</span></a></li><li id=\"ca-more-history\" class=\"vector-more-collapsible-item mw-list-item\"><a href=\"/w/index.php?title=Bioinformatics&amp;action=history\"><span>View history</span></a></li></ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n<div id=\"p-tb\" class=\"vector-menu mw-portlet mw-portlet-tb\"  >\\n\\t<div class=\"vector-menu-heading\">\\n\\t\\tGeneral\\n\\t</div>\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\"><li id=\"t-whatlinkshere\" class=\"mw-list-item\"><a href=\"/wiki/Special:WhatLinksHere/Bioinformatics\" title=\"List of all English Wikipedia pages containing links to this page [j]\" accesskey=\"j\"><span>What links here</span></a></li><li id=\"t-recentchangeslinked\" class=\"mw-list-item\"><a href=\"/wiki/Special:RecentChangesLinked/Bioinformatics\" rel=\"nofollow\" title=\"Recent changes in pages linked from this page [k]\" accesskey=\"k\"><span>Related changes</span></a></li><li id=\"t-upload\" class=\"mw-list-item\"><a href=\"/wiki/Wikipedia:File_Upload_Wizard\" title=\"Upload files [u]\" accesskey=\"u\"><span>Upload file</span></a></li><li id=\"t-specialpages\" class=\"mw-list-item\"><a href=\"/wiki/Special:SpecialPages\" title=\"A list of all special pages [q]\" accesskey=\"q\"><span>Special pages</span></a></li><li id=\"t-permalink\" class=\"mw-list-item\"><a href=\"/w/index.php?title=Bioinformatics&amp;oldid=1163899643\" title=\"Permanent link to this revision of this page\"><span>Permanent link</span></a></li><li id=\"t-info\" class=\"mw-list-item\"><a href=\"/w/index.php?title=Bioinformatics&amp;action=info\" title=\"More information about this page\"><span>Page information</span></a></li><li id=\"t-cite\" class=\"mw-list-item\"><a href=\"/w/index.php?title=Special:CiteThisPage&amp;page=Bioinformatics&amp;id=1163899643&amp;wpFormIdentifier=titleform\" title=\"Information on how to cite this page\"><span>Cite this page</span></a></li><li id=\"t-wikibase\" class=\"mw-list-item\"><a href=\"https://www.wikidata.org/wiki/Special:EntityPage/Q128570\" title=\"Structured data on this page hosted by Wikidata [g]\" accesskey=\"g\"><span>Wikidata item</span></a></li></ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n<div id=\"p-coll-print_export\" class=\"vector-menu mw-portlet mw-portlet-coll-print_export\"  >\\n\\t<div class=\"vector-menu-heading\">\\n\\t\\tPrint/export\\n\\t</div>\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\"><li id=\"coll-download-as-rl\" class=\"mw-list-item\"><a href=\"/w/index.php?title=Special:DownloadAsPdf&amp;page=Bioinformatics&amp;action=show-download-screen\" title=\"Download this page as a PDF file\"><span>Download as PDF</span></a></li><li id=\"t-print\" class=\"mw-list-item\"><a href=\"/w/index.php?title=Bioinformatics&amp;printable=yes\" title=\"Printable version of this page [p]\" accesskey=\"p\"><span>Printable version</span></a></li></ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n<div id=\"p-wikibase-otherprojects\" class=\"vector-menu mw-portlet mw-portlet-wikibase-otherprojects\"  >\\n\\t<div class=\"vector-menu-heading\">\\n\\t\\tIn other projects\\n\\t</div>\\n\\t<div class=\"vector-menu-content\">\\n\\t\\t\\n\\t\\t<ul class=\"vector-menu-content-list\"><li class=\"wb-otherproject-link wb-otherproject-commons mw-list-item\"><a href=\"https://commons.wikimedia.org/wiki/Category:Bioinformatics\" hreflang=\"en\"><span>Wikimedia Commons</span></a></li><li class=\"wb-otherproject-link wb-otherproject-wikibooks mw-list-item\"><a href=\"https://en.wikibooks.org/wiki/Bioinformatics\" hreflang=\"en\"><span>Wikibooks</span></a></li><li class=\"wb-otherproject-link wb-otherproject-wikiversity mw-list-item\"><a href=\"https://en.wikiversity.org/wiki/Bioinformatics\" hreflang=\"en\"><span>Wikiversity</span></a></li></ul>\\n\\t\\t\\n\\t</div>\\n</div>\\n\\n</div>\\n\\n\\t\\t\\t\\t\\t\\t\\t\\t\\t</div>\\n\\t\\t\\t\\t\\n\\t</div>\\n</div>\\n\\t\\t\\t\\t\\t\\t\\t</nav>\\n\\t\\t\\t\\t\\t\\t</div>\\n\\t\\t\\t\\t\\t</div>\\n\\t\\t\\t\\t</div>\\n\\t\\t\\t\\t<div class=\"vector-column-end\">\\n\\t\\t\\t\\t\\t<nav class=\"vector-page-tools-landmark vector-sticky-pinned-container\" aria-label=\"More options\">\\n\\t\\t\\t\\t\\t\\t<div id=\"vector-page-tools-pinned-container\" class=\"vector-pinned-container\">\\n\\t\\t\\t\\n\\t\\t\\t\\t\\t\\t</div>\\n\\t</nav>\\n\\t\\t\\t\\t</div>\\n\\t\\t\\t\\t<div id=\"bodyContent\" class=\"vector-body\" aria-labelledby=\"firstHeading\" data-mw-ve-target-container>\\n\\t\\t\\t\\t\\t<div class=\"vector-body-before-content\">\\n\\t\\t\\t\\t\\t\\t\\t<div class=\"mw-indicators\">\\n\\t\\t<div id=\"mw-indicator-spoken-icon\" class=\"mw-indicator\"><div class=\"mw-parser-output\"><a href=\"/wiki/File:En-Bioinformatics.ogg\" title=\"Listen to this article\"><img alt=\"Listen to this article\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/4/47/Sound-icon.svg/20px-Sound-icon.svg.png\" decoding=\"async\" width=\"20\" height=\"15\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/4/47/Sound-icon.svg/30px-Sound-icon.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/4/47/Sound-icon.svg/40px-Sound-icon.svg.png 2x\" data-file-width=\"128\" data-file-height=\"96\" /></a></div></div>\\n\\t\\t</div>\\n\\n\\t\\t\\t\\t\\t\\t<div id=\"siteSub\" class=\"noprint\">From Wikipedia, the free encyclopedia</div>\\n\\t\\t\\t\\t\\t</div>\\n\\t\\t\\t\\t\\t<div id=\"contentSub\"><div id=\"mw-content-subtitle\"></div></div>\\n\\t\\t\\t\\t\\t\\n\\t\\t\\t\\t\\t\\n\\t\\t\\t\\t\\t<div id=\"mw-content-text\" class=\"mw-body-content mw-content-ltr\" lang=\"en\" dir=\"ltr\"><div class=\"mw-parser-output\"><div class=\"shortdescription nomobile noexcerpt noprint searchaux\" style=\"display:none\">Computational analysis of large, complex sets of biological data</div>\\n<style data-mw-deduplicate=\"TemplateStyles:r1033289096\">.mw-parser-output .hatnote{font-style:italic}.mw-parser-output div.hatnote{padding-left:1.6em;margin-bottom:0.5em}.mw-parser-output .hatnote i{font-style:normal}.mw-parser-output .hatnote+link+.hatnote{margin-top:-0.5em}</style><div role=\"note\" class=\"hatnote navigation-not-searchable\">For the journal, see <a href=\"/wiki/Bioinformatics_(journal)\" title=\"Bioinformatics (journal)\">Bioinformatics (journal)</a>.</div>\\n<p class=\"mw-empty-elt\">\\n</p>\\n<div class=\"thumb tright\"><div class=\"thumbinner\" style=\"width:502px;\"><a href=\"/wiki/File:WPP_domain_alignment.PNG\" class=\"image\"><img src=\"//upload.wikimedia.org/wikipedia/commons/thumb/a/a7/WPP_domain_alignment.PNG/500px-WPP_domain_alignment.PNG\" decoding=\"async\" width=\"500\" height=\"116\" class=\"thumbimage\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/a/a7/WPP_domain_alignment.PNG/750px-WPP_domain_alignment.PNG 1.5x, //upload.wikimedia.org/wikipedia/commons/a/a7/WPP_domain_alignment.PNG 2x\" data-file-width=\"921\" data-file-height=\"214\" /></a>  <div class=\"thumbcaption\"><div class=\"magnify\"><a href=\"/wiki/File:WPP_domain_alignment.PNG\" class=\"internal\" title=\"Enlarge\"></a></div>Early bioinformatics\\xe2\\x80\\x94computational alignment of experimentally determined sequences of a class of related proteins; see <a href=\"#Sequence_analysis\">\\xc2\\xa7&#160;Sequence analysis</a> for further information.</div></div></div>\\n<div class=\"thumb tright\"><div class=\"thumbinner\" style=\"width:222px;\"><a href=\"/wiki/File:Genome_viewer_screenshot_small.png\" class=\"image\"><img src=\"//upload.wikimedia.org/wikipedia/commons/thumb/4/43/Genome_viewer_screenshot_small.png/220px-Genome_viewer_screenshot_small.png\" decoding=\"async\" width=\"220\" height=\"242\" class=\"thumbimage\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/4/43/Genome_viewer_screenshot_small.png/330px-Genome_viewer_screenshot_small.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/4/43/Genome_viewer_screenshot_small.png/440px-Genome_viewer_screenshot_small.png 2x\" data-file-width=\"490\" data-file-height=\"540\" /></a>  <div class=\"thumbcaption\"><div class=\"magnify\"><a href=\"/wiki/File:Genome_viewer_screenshot_small.png\" class=\"internal\" title=\"Enlarge\"></a></div>Map of the human X chromosome (from the <a href=\"/wiki/National_Center_for_Biotechnology_Information\" title=\"National Center for Biotechnology Information\">National Center for Biotechnology Information</a> website)</div></div></div><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Not to be confused with <a href=\"/wiki/Biological_computation\" title=\"Biological computation\">Biological computation</a> or <a href=\"/wiki/Genetic_algorithm\" title=\"Genetic algorithm\">Genetic algorithm</a>.</div>\\n<p><b>Bioinformatics</b> (<span class=\"rt-commentedText nowrap\"><span class=\"IPA nopopups noexcerpt\" lang=\"en-fonipa\"><a href=\"/wiki/Help:IPA/English\" title=\"Help:IPA/English\">/<span style=\"border-bottom:1px dotted\"><span title=\"/\\xcb\\x8c/: secondary stress follows\">\\xcb\\x8c</span><span title=\"&#39;b&#39; in &#39;buy&#39;\">b</span><span title=\"/a\\xc9\\xaa/: &#39;i&#39; in &#39;tide&#39;\">a\\xc9\\xaa</span><span title=\"/./: syllable break\">.</span><span title=\"/o\\xca\\x8a/: &#39;o&#39; in &#39;code&#39;\">o\\xca\\x8a</span><span title=\"/\\xcb\\x8c/: secondary stress follows\">\\xcb\\x8c</span><span title=\"/\\xc9\\xaa/: &#39;i&#39; in &#39;kit&#39;\">\\xc9\\xaa</span><span title=\"&#39;n&#39; in &#39;nigh&#39;\">n</span><span title=\"&#39;f&#39; in &#39;find&#39;\">f</span><span title=\"/\\xc9\\x99r/: &#39;er&#39; in &#39;letter&#39;\">\\xc9\\x99r</span><span title=\"/\\xcb\\x88/: primary stress follows\">\\xcb\\x88</span><span title=\"&#39;m&#39; in &#39;my&#39;\">m</span><span title=\"/\\xc3\\xa6/: &#39;a&#39; in &#39;bad&#39;\">\\xc3\\xa6</span><span title=\"&#39;t&#39; in &#39;tie&#39;\">t</span><span title=\"/\\xc9\\xaa/: &#39;i&#39; in &#39;kit&#39;\">\\xc9\\xaa</span><span title=\"&#39;k&#39; in &#39;kind&#39;\">k</span><span title=\"&#39;s&#39; in &#39;sigh&#39;\">s</span></span>/</a></span>&#32;<span class=\"nowrap\" style=\"font-size:85%\">(<span class=\"unicode haudio\"><span class=\"fn\"><span style=\"white-space:nowrap;margin-right:.25em;\"><a href=\"/wiki/File:En-us-bioinformatics.ogg\" title=\"About this sound\"><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/8/8a/Loudspeaker.svg/11px-Loudspeaker.svg.png\" decoding=\"async\" width=\"11\" height=\"11\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/8/8a/Loudspeaker.svg/17px-Loudspeaker.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/8/8a/Loudspeaker.svg/22px-Loudspeaker.svg.png 2x\" data-file-width=\"20\" data-file-height=\"20\" /></a></span><a href=\"//upload.wikimedia.org/wikipedia/commons/7/76/En-us-bioinformatics.ogg\" class=\"internal\" title=\"En-us-bioinformatics.ogg\">listen</a></span></span>)</span></span>) is an <a href=\"/wiki/Interdisciplinary\" class=\"mw-redirect\" title=\"Interdisciplinary\">interdisciplinary</a> field of <a href=\"/wiki/Science\" title=\"Science\">science</a> that develops methods and <a href=\"/wiki/Software_tool\" class=\"mw-redirect\" title=\"Software tool\">software tools</a> for understanding <a href=\"/wiki/Biology\" title=\"Biology\">biological</a> data, especially when the data sets are large and complex. Bioinformatics uses <a href=\"/wiki/Biology\" title=\"Biology\">biology</a>, <a href=\"/wiki/Chemistry\" title=\"Chemistry\">chemistry</a>, <a href=\"/wiki/Physics\" title=\"Physics\">physics</a>, <a href=\"/wiki/Computer_science\" title=\"Computer science\">computer science</a>, <a href=\"/wiki/Computer_programming\" title=\"Computer programming\">computer programming</a>, <a href=\"/wiki/Information_engineering_(field)\" title=\"Information engineering (field)\">information engineering</a>, <a href=\"/wiki/Mathematics\" title=\"Mathematics\">mathematics</a> and <a href=\"/wiki/Statistics\" title=\"Statistics\">statistics</a> to analyze and interpret <a href=\"/wiki/Biological_data\" title=\"Biological data\">biological data</a>. The subsequent process of analyzing and interpreting data is referred to as <a href=\"/wiki/Computational_biology\" title=\"Computational biology\">computational biology</a>. \\n</p><p>Computational, statistical, and computer programming techniques have been used  for <a href=\"/wiki/In_silico\" title=\"In silico\">computer simulation</a> analyses of biological queries. They include reused specific analysis \"pipelines\", particularly in the field of <a href=\"/wiki/Genomics\" title=\"Genomics\">genomics</a>, such as by the identification of <a href=\"/wiki/Gene\" title=\"Gene\">genes</a> and single <a href=\"/wiki/Nucleotide\" title=\"Nucleotide\">nucleotide</a> polymorphisms (<a href=\"/wiki/Single-nucleotide_polymorphism\" title=\"Single-nucleotide polymorphism\">SNPs</a>). These pipelines are used to better understand the genetic basis of disease, unique adaptations, desirable properties (esp. in agricultural species), or differences between populations. Bioinformatics also includes <a href=\"/wiki/Proteomics\" title=\"Proteomics\">proteomics</a>, which tries to understand the organizational principles within <a href=\"/wiki/Nucleic_acid\" title=\"Nucleic acid\">nucleic acid</a> and <a href=\"/wiki/Protein\" title=\"Protein\">protein</a> sequences.<sup id=\"cite_ref-1\" class=\"reference\"><a href=\"#cite_note-1\">&#91;1&#93;</a></sup>\\n</p><p>Image and <a href=\"/wiki/Signal_processing\" title=\"Signal processing\">signal processing</a> allow extraction of useful results from large amounts of raw data. In the field of genetics, it aids in sequencing and annotating genomes and their observed <a href=\"/wiki/Mutation\" title=\"Mutation\">mutations</a>. Bioinformatics includes <a href=\"/wiki/Text_mining\" title=\"Text mining\">text mining</a> of biological literature and the development of biological and gene <a href=\"/wiki/Ontology_(information_science)\" title=\"Ontology (information science)\">ontologies</a> to organize and query biological data. It also plays a role in the analysis of gene and protein expression and regulation. Bioinformatics tools aid in comparing, analyzing and interpreting genetic and genomic data and more generally in the understanding of evolutionary aspects of molecular biology. At a more integrative level, it helps analyze and catalogue the biological pathways and networks that are an important part of <a href=\"/wiki/Systems_biology\" title=\"Systems biology\">systems biology</a>. In <a href=\"/wiki/Structural_biology\" title=\"Structural biology\">structural biology</a>, it aids in the simulation and modeling of DNA,<sup id=\"cite_ref-:0_2-0\" class=\"reference\"><a href=\"#cite_note-:0-2\">&#91;2&#93;</a></sup> RNA,<sup id=\"cite_ref-:0_2-1\" class=\"reference\"><a href=\"#cite_note-:0-2\">&#91;2&#93;</a></sup><sup id=\"cite_ref-3\" class=\"reference\"><a href=\"#cite_note-3\">&#91;3&#93;</a></sup> proteins<sup id=\"cite_ref-4\" class=\"reference\"><a href=\"#cite_note-4\">&#91;4&#93;</a></sup> as well as biomolecular interactions.<sup id=\"cite_ref-5\" class=\"reference\"><a href=\"#cite_note-5\">&#91;5&#93;</a></sup><sup id=\"cite_ref-6\" class=\"reference\"><a href=\"#cite_note-6\">&#91;6&#93;</a></sup><sup id=\"cite_ref-7\" class=\"reference\"><a href=\"#cite_note-7\">&#91;7&#93;</a></sup><sup id=\"cite_ref-8\" class=\"reference\"><a href=\"#cite_note-8\">&#91;8&#93;</a></sup>\\n</p>\\n<meta property=\"mw:PageProp/toc\" />\\n<h2><span class=\"mw-headline\" id=\"History\">History</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=1\" title=\"Edit section: History\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<p>The first definition of the term <i>bioinformatics</i> was coined by <a href=\"/wiki/Paulien_Hogeweg\" title=\"Paulien Hogeweg\">Paulien Hogeweg</a> and <a href=\"/w/index.php?title=Ben_Hesper&amp;action=edit&amp;redlink=1\" class=\"new\" title=\"Ben Hesper (page does not exist)\">Ben Hesper</a> in 1970, to refer to the study of information processes in biotic systems.<sup id=\"cite_ref-9\" class=\"reference\"><a href=\"#cite_note-9\">&#91;9&#93;</a></sup><sup id=\"cite_ref-Hogeweg2011_10-0\" class=\"reference\"><a href=\"#cite_note-Hogeweg2011-10\">&#91;10&#93;</a></sup><sup id=\"cite_ref-11\" class=\"reference\"><a href=\"#cite_note-11\">&#91;11&#93;</a></sup><sup id=\"cite_ref-12\" class=\"reference\"><a href=\"#cite_note-12\">&#91;12&#93;</a></sup><sup id=\"cite_ref-13\" class=\"reference\"><a href=\"#cite_note-13\">&#91;13&#93;</a></sup> This definition placed bioinformatics as a field parallel to <a href=\"/wiki/Biochemistry\" title=\"Biochemistry\">biochemistry</a> (the study of chemical processes in biological systems).<sup id=\"cite_ref-Hogeweg2011_10-1\" class=\"reference\"><a href=\"#cite_note-Hogeweg2011-10\">&#91;10&#93;</a></sup>\\n</p><p>Bioinformatics and computational biology involved the analysis of biological data, particularly DNA, RNA, and protein sequences. The field of bioinformatics experienced explosive growth starting in the mid-1990s, driven largely by the <a href=\"/wiki/Human_Genome_Project\" title=\"Human Genome Project\">Human Genome Project</a> and by rapid advances in DNA sequencing technology.\\n</p><p>Analyzing biological data to produce meaningful information involves writing and running software programs that use <a href=\"/wiki/Algorithm\" title=\"Algorithm\">algorithms</a> from <a href=\"/wiki/Graph_theory\" title=\"Graph theory\">graph theory</a>, <a href=\"/wiki/Artificial_intelligence\" title=\"Artificial intelligence\">artificial intelligence</a>, <a href=\"/wiki/Soft_computing\" title=\"Soft computing\">soft computing</a>, <a href=\"/wiki/Data_mining\" title=\"Data mining\">data mining</a>, <a href=\"/wiki/Image_processing\" class=\"mw-redirect\" title=\"Image processing\">image processing</a>, and <a href=\"/wiki/Computer_simulation\" title=\"Computer simulation\">computer simulation</a>. The algorithms in turn depend on theoretical foundations such as <a href=\"/wiki/Discrete_mathematics\" title=\"Discrete mathematics\">discrete mathematics</a>, <a href=\"/wiki/Control_theory\" title=\"Control theory\">control theory</a>, <a href=\"/wiki/System_theory\" class=\"mw-redirect\" title=\"System theory\">system theory</a>, <a href=\"/wiki/Information_theory\" title=\"Information theory\">information theory</a>, and <a href=\"/wiki/Statistics\" title=\"Statistics\">statistics</a>.\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Sequences\">Sequences</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=2\" title=\"Edit section: Sequences\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<div class=\"thumb tright\"><div class=\"thumbinner\" style=\"width:222px;\"><a href=\"/wiki/File:Example_DNA_sequence.png\" class=\"image\"><img src=\"//upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Example_DNA_sequence.png/220px-Example_DNA_sequence.png\" decoding=\"async\" width=\"220\" height=\"63\" class=\"thumbimage\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Example_DNA_sequence.png/330px-Example_DNA_sequence.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Example_DNA_sequence.png/440px-Example_DNA_sequence.png 2x\" data-file-width=\"535\" data-file-height=\"153\" /></a>  <div class=\"thumbcaption\"><div class=\"magnify\"><a href=\"/wiki/File:Example_DNA_sequence.png\" class=\"internal\" title=\"Enlarge\"></a></div>Sequences of genetic material are frequently used in bioinformatics and are easier to manage using computers than manually.</div></div></div>\\n<p>There has been a tremendous advance in speed and cost reduction since the completion of the Human Genome Project, with some labs able to <a href=\"/wiki/Sequence\" title=\"Sequence\">sequence</a> over 100,000 billion bases each year, and a full genome can be sequenced for $1,000 or less.<sup id=\"cite_ref-14\" class=\"reference\"><a href=\"#cite_note-14\">&#91;14&#93;</a></sup> \\n</p><p>Computers became essential in molecular biology when <a href=\"/wiki/Protein_sequences\" class=\"mw-redirect\" title=\"Protein sequences\">protein sequences</a> became available after <a href=\"/wiki/Frederick_Sanger\" title=\"Frederick Sanger\">Frederick Sanger</a> determined the sequence of <a href=\"/wiki/Insulin\" title=\"Insulin\">insulin</a> in the early 1950s. Comparing multiple sequences manually turned out to be impractical. <a href=\"/wiki/Margaret_Oakley_Dayhoff\" title=\"Margaret Oakley Dayhoff\">Margaret Oakley Dayhoff</a>, a pioneer in the field,<sup id=\"cite_ref-15\" class=\"reference\"><a href=\"#cite_note-15\">&#91;15&#93;</a></sup> compiled one of the first protein sequence databases, initially published as books<sup id=\"cite_ref-16\" class=\"reference\"><a href=\"#cite_note-16\">&#91;16&#93;</a></sup> as well as methods of sequence alignment and <a href=\"/wiki/Molecular_evolution\" title=\"Molecular evolution\">molecular evolution</a>.<sup id=\"cite_ref-pmid17775169_17-0\" class=\"reference\"><a href=\"#cite_note-pmid17775169-17\">&#91;17&#93;</a></sup> Another early contributor to bioinformatics was <a href=\"/wiki/Elvin_A._Kabat\" title=\"Elvin A. Kabat\">Elvin A. Kabat</a>, who pioneered biological sequence analysis in 1970 with his comprehensive volumes of antibody sequences released online with Tai Te Wu between 1980 and 1991.<sup id=\"cite_ref-18\" class=\"reference\"><a href=\"#cite_note-18\">&#91;18&#93;</a></sup>\\n</p><p>In the 1970s, new techniques for sequencing DNA were applied to bacteriophage MS2 and \\xc3\\xb8X174, and the extended nucleotide sequences were then parsed with informational and statistical algorithms.  These studies illustrated that well known features, such as the coding segments and the triplet code, are revealed in straightforward statistical analyses and were the proof of the concept that bioinformatics would be insightful.<sup id=\"cite_ref-19\" class=\"reference\"><a href=\"#cite_note-19\">&#91;19&#93;</a></sup><sup id=\"cite_ref-20\" class=\"reference\"><a href=\"#cite_note-20\">&#91;20&#93;</a></sup>\\n</p>\\n<div class=\"thumb tright\"><div class=\"thumbinner\" style=\"width:371px;\"><a href=\"/wiki/File:Muscle_alignment_view.png\" class=\"image\"><img src=\"//upload.wikimedia.org/wikipedia/commons/thumb/9/98/Muscle_alignment_view.png/369px-Muscle_alignment_view.png\" decoding=\"async\" width=\"369\" height=\"126\" class=\"thumbimage\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/9/98/Muscle_alignment_view.png/554px-Muscle_alignment_view.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/98/Muscle_alignment_view.png/738px-Muscle_alignment_view.png 2x\" data-file-width=\"1782\" data-file-height=\"610\" /></a>  <div class=\"thumbcaption\"><div class=\"magnify\"><a href=\"/wiki/File:Muscle_alignment_view.png\" class=\"internal\" title=\"Enlarge\"></a></div>These are sequences being compared in a MUSCLE multiple sequence alignment (MSA). Each sequence name (leftmost column) is from various louse species, while the sequences themselves are in the second column.</div></div></div>\\n<h2><span class=\"mw-headline\" id=\"Goals\">Goals</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=3\" title=\"Edit section: Goals\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<p>In order to study how normal cellular activities are altered in different disease states, raw biological data must be combined to form a comprehensive picture of these activities. Therefore<sup class=\"noprint Inline-Template\" style=\"white-space:nowrap;\">&#91;<i><a href=\"/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Chronological_items\" title=\"Wikipedia:Manual of Style/Dates and numbers\"><span title=\"The time period mentioned near this tag is ambiguous. (June 2023)\">when?</span></a></i>&#93;</sup>, the field of bioinformatics has evolved such that the most pressing task now involves the analysis and interpretation of various types of data. This also includes nucleotide and <a href=\"/wiki/Amino_acid_sequence\" class=\"mw-redirect\" title=\"Amino acid sequence\">amino acid sequences</a>, <a href=\"/wiki/Protein_domain\" title=\"Protein domain\">protein domains</a>, and <a href=\"/wiki/Protein_structure\" title=\"Protein structure\">protein structures</a>.<sup id=\"cite_ref-21\" class=\"reference\"><a href=\"#cite_note-21\">&#91;21&#93;</a></sup> \\n</p><p>Important sub-disciplines within bioinformatics and <a href=\"/wiki/Computational_biology\" title=\"Computational biology\">computational biology</a> include:\\n</p>\\n<ul><li>Development and implementation of computer programs to efficiently access, manage, and use various types of information.</li>\\n<li>Development of new mathematical algorithms and statistical measures to assess relationships among members of large data sets. For example, there are methods to locate a <a href=\"/wiki/Gene\" title=\"Gene\">gene</a> within a sequence, to predict protein structure and/or function, and to <a href=\"/wiki/Cluster_analysis\" title=\"Cluster analysis\">cluster</a> protein sequences into families of related sequences.</li></ul>\\n<p>The primary goal of bioinformatics is to increase the understanding of biological processes. What sets it apart from other approaches is its focus on developing and applying computationally intensive techniques to achieve this goal. Examples include: <a href=\"/wiki/Pattern_recognition\" title=\"Pattern recognition\">pattern recognition</a>, <a href=\"/wiki/Data_mining\" title=\"Data mining\">data mining</a>, <a href=\"/wiki/Machine_learning\" title=\"Machine learning\">machine learning</a> algorithms, and <a href=\"/wiki/Biological_Data_Visualization\" class=\"mw-redirect\" title=\"Biological Data Visualization\">visualization</a>. Major research efforts in the field include <a href=\"/wiki/Sequence_alignment\" title=\"Sequence alignment\">sequence alignment</a>, <a href=\"/wiki/Gene_finding\" class=\"mw-redirect\" title=\"Gene finding\">gene finding</a>, <a href=\"/wiki/Genome_assembly\" class=\"mw-redirect\" title=\"Genome assembly\">genome assembly</a>, <a href=\"/wiki/Drug_design\" title=\"Drug design\">drug design</a>, <a href=\"/wiki/Drug_discovery\" title=\"Drug discovery\">drug discovery</a>, <a href=\"/wiki/Protein_structural_alignment\" class=\"mw-redirect\" title=\"Protein structural alignment\">protein structure alignment</a>, <a href=\"/wiki/Protein_structure_prediction\" title=\"Protein structure prediction\">protein structure prediction</a>, prediction of <a href=\"/wiki/Gene_expression\" title=\"Gene expression\">gene expression</a> and <a href=\"/wiki/Protein%E2%80%93protein_interaction\" title=\"Protein\\xe2\\x80\\x93protein interaction\">protein\\xe2\\x80\\x93protein interactions</a>, <a href=\"/wiki/Genome-wide_association_studies\" class=\"mw-redirect\" title=\"Genome-wide association studies\">genome-wide association studies</a>, the modeling of <a href=\"/wiki/Evolution\" title=\"Evolution\">evolution</a> and <a href=\"/wiki/Cellular_model\" title=\"Cellular model\">cell division/mitosis.</a>\\n</p><p>Bioinformatics entails the creation and advancement of databases, algorithms, computational and statistical techniques, and theory to solve formal and practical problems arising from the management and analysis of biological data.\\n</p><p>Over the past few decades, rapid developments in genomic and other molecular research technologies and developments in <a href=\"/wiki/Information_technologies\" class=\"mw-redirect\" title=\"Information technologies\">information technologies</a> have combined to produce a tremendous amount of information related to molecular biology. Bioinformatics is the name given to these mathematical and computing approaches used to glean understanding of biological processes.\\n</p><p>Common activities in bioinformatics include mapping and analyzing <a href=\"/wiki/DNA\" title=\"DNA\">DNA</a> and protein sequences, aligning DNA and protein sequences to compare them, and creating and viewing 3-D models of protein structures.\\n</p>\\n<h2><span class=\"mw-headline\" id=\"Sequence_analysis\">Sequence analysis</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=4\" title=\"Edit section: Sequence analysis\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main articles: <a href=\"/wiki/Sequence_alignment\" title=\"Sequence alignment\">Sequence alignment</a>, <a href=\"/wiki/Sequence_database\" title=\"Sequence database\">Sequence database</a>, and <a href=\"/wiki/Alignment-free_sequence_analysis\" title=\"Alignment-free sequence analysis\">Alignment-free sequence analysis</a></div>\\n<p>Since the bacteriophage <a href=\"/wiki/Phi_X_174\" title=\"Phi X 174\">Phage \\xce\\xa6-X174</a> was <a href=\"/wiki/Sequencing\" title=\"Sequencing\">sequenced</a> in 1977,<sup id=\"cite_ref-pmid870828_22-0\" class=\"reference\"><a href=\"#cite_note-pmid870828-22\">&#91;22&#93;</a></sup> the <a href=\"/wiki/DNA_sequence\" class=\"mw-redirect\" title=\"DNA sequence\">DNA sequences</a> of thousands of organisms have been decoded and stored in databases. This sequence information is analyzed to determine genes that encode <a href=\"/wiki/Protein\" title=\"Protein\">proteins</a>, RNA genes, regulatory sequences, structural motifs, and repetitive sequences. A comparison of genes within a <a href=\"/wiki/Species\" title=\"Species\">species</a> or between different species can show similarities between protein functions, or relations between species (the use of <a href=\"/wiki/Molecular_systematics\" class=\"mw-redirect\" title=\"Molecular systematics\">molecular systematics</a> to construct <a href=\"/wiki/Phylogenetic_tree\" title=\"Phylogenetic tree\">phylogenetic trees</a>). With the growing amount of data, it long ago became impractical to analyze DNA sequences manually. <a href=\"/wiki/Computer_program\" title=\"Computer program\">Computer programs</a> such as <a href=\"/wiki/BLAST\" class=\"mw-redirect\" title=\"BLAST\">BLAST</a> are used routinely to search sequences\\xe2\\x80\\x94as of 2008, from more than 260,000 organisms, containing over 190 billion <a href=\"/wiki/Nucleotide\" title=\"Nucleotide\">nucleotides</a>.<sup id=\"cite_ref-pmid18073190_23-0\" class=\"reference\"><a href=\"#cite_note-pmid18073190-23\">&#91;23&#93;</a></sup>\\n</p>\\n<div class=\"center\"><div class=\"floatnone\"><a href=\"/wiki/File:Sequencing_analysis_steps.png\" class=\"image\" title=\"Image: 450 pixels Sequencing analysis steps\"><img alt=\"Image: 450 pixels Sequencing analysis steps\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/e/e4/Sequencing_analysis_steps.png/600px-Sequencing_analysis_steps.png\" decoding=\"async\" width=\"600\" height=\"243\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/e/e4/Sequencing_analysis_steps.png/900px-Sequencing_analysis_steps.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/e/e4/Sequencing_analysis_steps.png/1200px-Sequencing_analysis_steps.png 2x\" data-file-width=\"1382\" data-file-height=\"560\" /></a></div></div>\\n<h3><span class=\"mw-headline\" id=\"DNA_sequencing\">DNA sequencing</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=5\" title=\"Edit section: DNA sequencing\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/DNA_sequencing\" title=\"DNA sequencing\">DNA sequencing</a></div>\\n<p>Before sequences can be analyzed, they are obtained from a data storage bank, such as Genbank. <a href=\"/wiki/DNA_sequencing\" title=\"DNA sequencing\">DNA sequencing</a> is still a non-trivial problem as the raw data may be noisy or affected by weak signals. <a href=\"/wiki/Algorithm\" title=\"Algorithm\">Algorithms</a> have been developed for <a href=\"/wiki/Base_calling\" title=\"Base calling\">base calling</a> for the various experimental approaches to DNA sequencing.\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Sequence_assembly\">Sequence assembly</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=6\" title=\"Edit section: Sequence assembly\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/Sequence_assembly\" title=\"Sequence assembly\">Sequence assembly</a></div>\\n<p>Most DNA sequencing techniques produce short fragments of sequence that need to be assembled to obtain complete gene or genome sequences. The <a href=\"/wiki/Shotgun_sequencing\" title=\"Shotgun sequencing\">shotgun sequencing</a> technique (used by <a href=\"/wiki/The_Institute_for_Genomic_Research\" class=\"mw-redirect\" title=\"The Institute for Genomic Research\">The Institute for Genomic Research</a> (TIGR) to sequence the first bacterial genome, <i><a href=\"/wiki/Haemophilus_influenzae\" title=\"Haemophilus influenzae\">Haemophilus influenzae</a></i>)<sup id=\"cite_ref-pmid7542800_24-0\" class=\"reference\"><a href=\"#cite_note-pmid7542800-24\">&#91;24&#93;</a></sup> generates the sequences of many thousands of small DNA fragments (ranging from 35 to 900 nucleotides long, depending on the sequencing technology). The ends of these fragments overlap and, when aligned properly by a genome assembly program, can be used to reconstruct the complete genome. Shotgun sequencing yields sequence data quickly, but the task of assembling the fragments can be quite complicated for larger genomes. For a genome as large as the <a href=\"/wiki/Human_genome\" title=\"Human genome\">human genome</a>, it may take many days of CPU time on large-memory, multiprocessor computers to assemble the fragments, and the resulting assembly usually contains numerous gaps that must be filled in later. Shotgun sequencing is the method of choice for virtually all genomes sequenced (rather than chain-termination or chemical degradation methods), and genome assembly algorithms are a critical area of bioinformatics research.\\n</p>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">See also: <a href=\"/wiki/Sequence_analysis\" title=\"Sequence analysis\">sequence analysis</a>, <a href=\"/wiki/Sequence_mining\" class=\"mw-redirect\" title=\"Sequence mining\">sequence mining</a>, <a href=\"/wiki/Sequence_profiling_tool\" title=\"Sequence profiling tool\">sequence profiling tool</a>, and <a href=\"/wiki/Sequence_motif\" title=\"Sequence motif\">sequence motif</a></div>\\n<h3><span class=\"mw-headline\" id=\"Genome_annotation\">Genome annotation</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=7\" title=\"Edit section: Genome annotation\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/Gene_prediction\" title=\"Gene prediction\">Gene prediction</a></div>\\n<p>In <a href=\"/wiki/Genomics\" title=\"Genomics\">genomics</a>, <a href=\"/wiki/Genome_project#Genome_annotation\" title=\"Genome project\">annotation</a> refers to the process of marking the stop and start regions of genes and other biological features in a sequenced DNA sequence. Many genomes are too large to be annotated by hand. As the rate of <a href=\"/wiki/DNA_sequencing\" title=\"DNA sequencing\">sequencing</a> exceeds the rate of genome annotation, genome annotation has become the new bottleneck in bioinformatics<sup class=\"noprint Inline-Template\" style=\"white-space:nowrap;\">&#91;<i><a href=\"/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Chronological_items\" title=\"Wikipedia:Manual of Style/Dates and numbers\"><span title=\"The time period mentioned near this tag is ambiguous. (June 2023)\">when?</span></a></i>&#93;</sup>. \\n</p><p>Genome annotation can be classified into three levels: the nucleotide, protein, and process levels.\\n</p><p>Gene finding is a chief aspect of nucleotide-level annotation. For complex genomes, a combination of <a href=\"/wiki/Ab_initio\" title=\"Ab initio\">ab initio</a> gene prediction and sequence comparison with expressed sequence databases and other organisms can be successful. Nucleotide-level annotation also allows the integration of genome sequence with other genetic and physical maps of the genome.\\n</p><p>The principal aim of protein-level annotation is to assign function to the <a href=\"/wiki/Protein\" title=\"Protein\">protein</a> products of the genome. Databases of protein sequences and functional domains and motifs are used for this type of annotation. About half of the predicted proteins in a new genome sequence tend to have no obvious function.\\n</p><p>Understanding the function of genes and their products in the context of cellular and organismal physiology is the goal of process-level annotation. An obstacle of process-level annotation has been the inconsistency of terms used by different model systems. The Gene Ontology Consortium is helping to solve this problem.<sup id=\"cite_ref-25\" class=\"reference\"><a href=\"#cite_note-25\">&#91;25&#93;</a></sup>\\n</p><p>The first description of a comprehensive annotation system was published in 1995<sup id=\"cite_ref-pmid7542800_24-1\" class=\"reference\"><a href=\"#cite_note-pmid7542800-24\">&#91;24&#93;</a></sup> by the <a href=\"/wiki/The_Institute_for_Genomic_Research\" class=\"mw-redirect\" title=\"The Institute for Genomic Research\">The Institute for Genomic Research</a>, which performed the first complete sequencing and analysis of the genome of a <a href=\"/wiki/Free-living\" class=\"mw-disambig\" title=\"Free-living\">free-living</a> organism, the bacterium <i><a href=\"/wiki/Haemophilus_influenzae\" title=\"Haemophilus influenzae\">Haemophilus influenzae</a></i>.<sup id=\"cite_ref-pmid7542800_24-2\" class=\"reference\"><a href=\"#cite_note-pmid7542800-24\">&#91;24&#93;</a></sup> The system identifies the genes encoding all proteins, transfer RNAs, ribosomal RNAs, in order to make initial functional assignments. The <a href=\"/wiki/GeneMark\" title=\"GeneMark\">GeneMark</a> program trained to find protein-coding genes in <i><a href=\"/wiki/Haemophilus_influenzae\" title=\"Haemophilus influenzae\">Haemophilus influenzae</a></i> is constantly changing and improving.\\n</p><p>Following the goals that the Human Genome Project left to achieve after its closure in 2003, the <a href=\"/wiki/ENCODE\" title=\"ENCODE\">ENCODE</a> project was developed by the <a href=\"/wiki/National_Human_Genome_Research_Institute\" title=\"National Human Genome Research Institute\">National Human Genome Research Institute</a>. This project is a collaborative data collection of the functional elements of the human genome that uses next-generation DNA-sequencing technologies and genomic tiling arrays, technologies able to automatically generate large amounts of data at a dramatically reduced per-base cost but with the same accuracy (base call error) and fidelity (assembly error).\\n</p>\\n<h4><span class=\"mw-headline\" id=\"Gene_function_prediction\">Gene function prediction</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=8\" title=\"Edit section: Gene function prediction\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h4>\\n<p>While genome annotation is primarily based on sequence similarity (and thus <a href=\"/wiki/Homology_(biology)\" title=\"Homology (biology)\">homology</a>), other properties of sequences can be used to predict the function of genes. In fact, most <i>gene</i> function prediction methods focus on <i>protein</i> sequences as they are more informative and more feature-rich. For instance, the distribution of hydrophobic <a href=\"/wiki/Amino_acid\" title=\"Amino acid\">amino acids</a> predicts <a href=\"/wiki/Transmembrane_domain\" title=\"Transmembrane domain\">transmembrane segments</a> in proteins. However, protein function prediction can also use external information such as gene (or protein) <a href=\"/wiki/Gene_expression\" title=\"Gene expression\">expression</a> data, <a href=\"/wiki/Protein_structure\" title=\"Protein structure\">protein structure</a>, or <a href=\"/wiki/Protein%E2%80%93protein_interaction\" title=\"Protein\\xe2\\x80\\x93protein interaction\">protein-protein interactions</a>.<sup id=\"cite_ref-26\" class=\"reference\"><a href=\"#cite_note-26\">&#91;26&#93;</a></sup>\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Computational_evolutionary_biology\">Computational evolutionary biology</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=9\" title=\"Edit section: Computational evolutionary biology\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Further information: <a href=\"/wiki/Computational_phylogenetics\" title=\"Computational phylogenetics\">Computational phylogenetics</a></div>\\n<p><a href=\"/wiki/Evolutionary_biology\" title=\"Evolutionary biology\">Evolutionary biology</a> is the study of the origin and descent of <a href=\"/wiki/Species\" title=\"Species\">species</a>, as well as their change over time. <a href=\"/wiki/Informatics_(academic_field)\" class=\"mw-redirect\" title=\"Informatics (academic field)\">Informatics</a> has assisted evolutionary biologists by enabling researchers to:\\n</p>\\n<ul><li>trace the evolution of a large number of organisms by measuring changes in their <a href=\"/wiki/DNA\" title=\"DNA\">DNA</a>, rather than through physical taxonomy or physiological observations alone,</li>\\n<li>compare entire <a href=\"/wiki/Genomes\" class=\"mw-redirect\" title=\"Genomes\">genomes</a>, which permits the study of more complex evolutionary events, such as <a href=\"/wiki/Gene_duplication\" title=\"Gene duplication\">gene duplication</a>, <a href=\"/wiki/Horizontal_gene_transfer\" title=\"Horizontal gene transfer\">horizontal gene transfer</a>, and the prediction of factors important in bacterial <a href=\"/wiki/Speciation\" title=\"Speciation\">speciation</a>,</li>\\n<li>build complex computational <a href=\"/wiki/Population_genetics\" title=\"Population genetics\">population genetics</a> models to predict the outcome of the system over time<sup id=\"cite_ref-27\" class=\"reference\"><a href=\"#cite_note-27\">&#91;27&#93;</a></sup></li>\\n<li>track and share information on an increasingly large number of species and organisms</li></ul>\\n<p>Future work endeavours to reconstruct the now more complex <a href=\"/wiki/Evolutionary_tree\" class=\"mw-redirect\" title=\"Evolutionary tree\">tree of life</a>.<sup class=\"noprint Inline-Template\" style=\"margin-left:0.1em; white-space:nowrap;\">&#91;<i><a href=\"/wiki/Wikipedia:Manual_of_Style/Words_to_watch#Unsupported_attributions\" title=\"Wikipedia:Manual of Style/Words to watch\"><span title=\"The material near this tag may use weasel words or too-vague attribution. (June 2020)\">according to whom?</span></a></i>&#93;</sup>\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Comparative_genomics\">Comparative genomics</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=10\" title=\"Edit section: Comparative genomics\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/Comparative_genomics\" title=\"Comparative genomics\">Comparative genomics</a></div>\\n<p>The core of comparative genome analysis is the establishment of the correspondence between <a href=\"/wiki/Genes\" class=\"mw-redirect\" title=\"Genes\">genes</a> (<a href=\"/wiki/Homology_(biology)#Orthology\" title=\"Homology (biology)\">orthology</a> analysis) or other genomic features in different organisms. Intergenomic maps are made to trace the evolutionary processes responsible for the divergence of two genomes. A multitude of evolutionary events acting at various organizational levels shape genome evolution. At the lowest level, point mutations affect individual nucleotides. At a higher level, large chromosomal segments undergo duplication, lateral transfer, inversion, transposition, deletion and insertion.<sup id=\"cite_ref-28\" class=\"reference\"><a href=\"#cite_note-28\">&#91;28&#93;</a></sup> Entire genomes are involved in processes of hybridization, polyploidization and <a href=\"/wiki/Endosymbiosis\" class=\"mw-redirect\" title=\"Endosymbiosis\">endosymbiosis</a> that lead to rapid speciation. The complexity of genome evolution poses many exciting challenges to developers of mathematical models and algorithms, who have recourse to a spectrum of algorithmic, statistical and mathematical techniques, ranging from exact, <a href=\"/wiki/Heuristics\" class=\"mw-redirect\" title=\"Heuristics\">heuristics</a>, fixed parameter and <a href=\"/wiki/Approximation_algorithms\" class=\"mw-redirect\" title=\"Approximation algorithms\">approximation algorithms</a> for problems based on parsimony models to <a href=\"/wiki/Markov_chain_Monte_Carlo\" title=\"Markov chain Monte Carlo\">Markov chain Monte Carlo</a> algorithms for <a href=\"/wiki/Bayesian_analysis\" class=\"mw-redirect\" title=\"Bayesian analysis\">Bayesian analysis</a> of problems based on probabilistic models.\\n</p><p>Many of these studies are based on the detection of <a href=\"/wiki/Sequence_homology\" title=\"Sequence homology\">sequence homology</a> to assign sequences to <a href=\"/wiki/Protein_family\" title=\"Protein family\">protein families</a>.<sup id=\"cite_ref-29\" class=\"reference\"><a href=\"#cite_note-29\">&#91;29&#93;</a></sup>\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Pan_genomics\">Pan genomics</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=11\" title=\"Edit section: Pan genomics\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/Pan-genome\" title=\"Pan-genome\">Pan-genome</a></div>\\n<p>Pan genomics is a concept introduced in 2005 by Tettelin and Medini. Pan genome is the complete gene repertoire of a particular <a href=\"/wiki/Monophyly\" title=\"Monophyly\">monophyletic</a> taxonomic group. Although initially applied to closely related strains of a species, it can be applied to a larger context like genus, phylum, etc. It is divided in two parts: the Core genome, a set of genes common to all the genomes under study (often housekeeping genes vital for survival), and the Dispensable/Flexible genome: a set of genes not present in all but one or some genomes under study. A bioinformatics tool BPGA can be used to characterize the Pan Genome of bacterial species.<sup id=\"cite_ref-30\" class=\"reference\"><a href=\"#cite_note-30\">&#91;30&#93;</a></sup>\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Genetics_of_disease\">Genetics of disease</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=12\" title=\"Edit section: Genetics of disease\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/Genome-wide_association_studies\" class=\"mw-redirect\" title=\"Genome-wide association studies\">Genome-wide association studies</a></div>\\n<p>As of 2013, the existence of efficient high-throughput next-generation sequencing technology allows for the identification of cause many different human disorders. Simple <a href=\"/wiki/Mendelian_inheritance\" title=\"Mendelian inheritance\">Mendelian inheritance</a> has been observed for over 3,000 disorders that have been identified at the <a href=\"/wiki/Online_Mendelian_Inheritance_in_Man\" title=\"Online Mendelian Inheritance in Man\">Online Mendelian Inheritance in Man</a> database, but complex diseases are more difficult. Association studies have found many individual genetic regions that individually are weakly associated with complex diseases (such as <a href=\"/wiki/Infertility\" title=\"Infertility\">infertility</a>,<sup id=\"cite_ref-Demerec1945_31-0\" class=\"reference\"><a href=\"#cite_note-Demerec1945-31\">&#91;31&#93;</a></sup> <a href=\"/wiki/Breast_cancer\" title=\"Breast cancer\">breast cancer</a><sup id=\"cite_ref-V\\xc3\\xa9ron2013_32-0\" class=\"reference\"><a href=\"#cite_note-V\\xc3\\xa9ron2013-32\">&#91;32&#93;</a></sup> and <a href=\"/wiki/Alzheimer%27s_disease\" title=\"Alzheimer&#39;s disease\">Alzheimer\\'s disease</a><sup id=\"cite_ref-Tosto2013_33-0\" class=\"reference\"><a href=\"#cite_note-Tosto2013-33\">&#91;33&#93;</a></sup>), rather than a single cause.<sup id=\"cite_ref-Londin2013_34-0\" class=\"reference\"><a href=\"#cite_note-Londin2013-34\">&#91;34&#93;</a></sup><sup id=\"cite_ref-35\" class=\"reference\"><a href=\"#cite_note-35\">&#91;35&#93;</a></sup> There are currently many challenges to using genes for diagnosis and treatment, such as how we don\\'t know which genes are important, or how stable the choices an algorithm provides. <sup id=\"cite_ref-36\" class=\"reference\"><a href=\"#cite_note-36\">&#91;36&#93;</a></sup>\\n</p><p>Genome-wide association studies have successfully identified thousands of common genetic variants for complex diseases and traits; however, these common variants only explain a small fraction of heritability.<sup id=\"cite_ref-37\" class=\"reference\"><a href=\"#cite_note-37\">&#91;37&#93;</a></sup> <a href=\"/wiki/Rare_functional_variant\" class=\"mw-redirect\" title=\"Rare functional variant\">Rare variants</a> may account for some of the <a href=\"/wiki/Missing_heritability_problem\" title=\"Missing heritability problem\">missing heritability</a>.<sup id=\"cite_ref-38\" class=\"reference\"><a href=\"#cite_note-38\">&#91;38&#93;</a></sup> Large-scale <a href=\"/wiki/Whole_genome_sequencing\" title=\"Whole genome sequencing\">whole genome sequencing</a> studies have rapidly sequenced millions of whole genomes, and such studies have identified hundreds of millions of <a href=\"/wiki/Rare_functional_variant\" class=\"mw-redirect\" title=\"Rare functional variant\">rare variants</a>.<sup id=\"cite_ref-39\" class=\"reference\"><a href=\"#cite_note-39\">&#91;39&#93;</a></sup> <a href=\"/wiki/SNP_annotation\" title=\"SNP annotation\">Functional annotations</a> predict the effect or function of a genetic variant and help to prioritize rare functional variants, and incorporating these annotations can effectively boost the power of genetic association of rare variants analysis of whole genome sequencing studies.<sup id=\"cite_ref-40\" class=\"reference\"><a href=\"#cite_note-40\">&#91;40&#93;</a></sup> Some tools have been developed to provide all-in-one rare variant association analysis for whole-genome sequencing data, including integration of genotype data and their functional annotations, association analysis, result summary and visualization.<sup id=\"cite_ref-41\" class=\"reference\"><a href=\"#cite_note-41\">&#91;41&#93;</a></sup><sup id=\"cite_ref-42\" class=\"reference\"><a href=\"#cite_note-42\">&#91;42&#93;</a></sup> Meta-analysis of whole genome sequencing studies provides an attractive solution to the problem of collecting large sample sizes for discovering rare variants associated with complex phenotypes.<sup id=\"cite_ref-43\" class=\"reference\"><a href=\"#cite_note-43\">&#91;43&#93;</a></sup>\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Analysis_of_mutations_in_cancer\">Analysis of mutations in cancer</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=13\" title=\"Edit section: Analysis of mutations in cancer\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/Oncogenomics\" title=\"Oncogenomics\">Oncogenomics</a></div>\\n<p>In <a href=\"/wiki/Cancer\" title=\"Cancer\">cancer</a>, the genomes of affected cells are rearranged in complex or unpredictable ways. In addition to <a href=\"/wiki/Single-nucleotide_polymorphism\" title=\"Single-nucleotide polymorphism\">single-nucleotide polymorphism</a> arrays identifying <a href=\"/wiki/Point_mutation\" title=\"Point mutation\">point mutations</a> that cause cancer, <a href=\"/wiki/Oligonucleotide\" title=\"Oligonucleotide\">oligonucleotide</a> microarrays can be used to identify chromosomal gains and losses (called <a href=\"/wiki/Comparative_genomic_hybridization\" title=\"Comparative genomic hybridization\">comparative genomic hybridization</a>). These detection methods generate <a href=\"/wiki/Terabyte\" class=\"mw-redirect\" title=\"Terabyte\">terabytes</a> of data per experiment. The data is often found to contain considerable variability, or <a href=\"/wiki/Noise\" title=\"Noise\">noise</a>, and thus <a href=\"/wiki/Hidden_Markov_model\" title=\"Hidden Markov model\">Hidden Markov model</a> and change-point analysis methods are being developed to infer real <a href=\"/wiki/Copy_number_variation\" title=\"Copy number variation\">copy number</a> changes.<sup class=\"noprint Inline-Template Template-Fact\" style=\"white-space:nowrap;\">&#91;<i><a href=\"/wiki/Wikipedia:Citation_needed\" title=\"Wikipedia:Citation needed\"><span title=\"This claim needs references to reliable sources. (June 2023)\">citation needed</span></a></i>&#93;</sup>\\n</p><p>Two important principles can be used to identify cancer by mutations in the <a href=\"/wiki/Exome\" title=\"Exome\">exome</a>. First, cancer is a disease of accumulated somatic mutations in genes. Second, cancer contains driver mutations which need to be distinguished from passengers.<sup id=\"cite_ref-44\" class=\"reference\"><a href=\"#cite_note-44\">&#91;44&#93;</a></sup>\\n</p><p>Further improvements in bioinformatics could allow for classifying types of cancer by analysis of cancer driven mutations in the genome. Furthermore, tracking of patients while the disease progresses may be possible in the future with the sequence of cancer samples. Another type of data that requires novel informatics development is the analysis of <a href=\"/wiki/Lesion\" title=\"Lesion\">lesions</a> found to be recurrent among many tumors.<sup id=\"cite_ref-45\" class=\"reference\"><a href=\"#cite_note-45\">&#91;45&#93;</a></sup>\\n</p>\\n<h2><span class=\"mw-headline\" id=\"Gene_and_protein_expression\">Gene and protein expression</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=14\" title=\"Edit section: Gene and protein expression\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<h3><span class=\"mw-headline\" id=\"Analysis_of_gene_expression\">Analysis of gene expression</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=15\" title=\"Edit section: Analysis of gene expression\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p>The <a href=\"/wiki/Gene_expression\" title=\"Gene expression\">expression</a> of many genes can be determined by measuring <a href=\"/wiki/Messenger_RNA\" title=\"Messenger RNA\">mRNA</a> levels with multiple techniques including <a href=\"/wiki/DNA_microarray\" title=\"DNA microarray\">microarrays</a>, <a href=\"/wiki/Expressed_sequence_tag\" title=\"Expressed sequence tag\">expressed cDNA sequence tag</a> (EST) sequencing, <a href=\"/wiki/Serial_analysis_of_gene_expression\" title=\"Serial analysis of gene expression\">serial analysis of gene expression</a> (SAGE) tag sequencing, <a href=\"/wiki/Massively_parallel_signature_sequencing\" title=\"Massively parallel signature sequencing\">massively parallel signature sequencing</a> (MPSS), <a href=\"/wiki/RNA-Seq\" title=\"RNA-Seq\">RNA-Seq</a>, also known as \"Whole Transcriptome Shotgun Sequencing\" (WTSS), or various applications of multiplexed in-situ hybridization. All of these techniques are extremely noise-prone and/or subject to bias in the biological measurement, and a major research area in computational biology involves developing statistical tools to separate <a href=\"/wiki/Signal_(information_theory)\" class=\"mw-redirect\" title=\"Signal (information theory)\">signal</a> from <a href=\"/wiki/Noise\" title=\"Noise\">noise</a> in high-throughput gene expression studies.<sup id=\"cite_ref-46\" class=\"reference\"><a href=\"#cite_note-46\">&#91;46&#93;</a></sup> Such studies are often used to determine the genes implicated in a disorder: one might compare microarray data from cancerous <a href=\"/wiki/Epithelial\" class=\"mw-redirect\" title=\"Epithelial\">epithelial</a> cells to data from non-cancerous cells to determine the transcripts that are up-regulated and down-regulated in a particular population of cancer cells.\\n</p>\\n<div class=\"center\"><div class=\"thumb tnone\"><div class=\"thumbinner\" style=\"width:402px;\"><a href=\"/wiki/File:MIcroarray_vs_RNA-Seq.png\" class=\"image\"><img src=\"//upload.wikimedia.org/wikipedia/commons/thumb/6/68/MIcroarray_vs_RNA-Seq.png/400px-MIcroarray_vs_RNA-Seq.png\" decoding=\"async\" width=\"400\" height=\"312\" class=\"thumbimage\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/6/68/MIcroarray_vs_RNA-Seq.png/600px-MIcroarray_vs_RNA-Seq.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/68/MIcroarray_vs_RNA-Seq.png/800px-MIcroarray_vs_RNA-Seq.png 2x\" data-file-width=\"1200\" data-file-height=\"936\" /></a>  <div class=\"thumbcaption\"><div class=\"magnify\"><a href=\"/wiki/File:MIcroarray_vs_RNA-Seq.png\" class=\"internal\" title=\"Enlarge\"></a></div>MIcroarray vs RNA-Seq</div></div></div></div>\\n<h3><span class=\"mw-headline\" id=\"Analysis_of_protein_expression\">Analysis of protein expression</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=16\" title=\"Edit section: Analysis of protein expression\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p><a href=\"/wiki/Protein_microarray\" title=\"Protein microarray\">Protein microarrays</a> and high throughput (HT) <a href=\"/wiki/Mass_spectrometry\" title=\"Mass spectrometry\">mass spectrometry</a> (MS) can provide a snapshot of the proteins present in a biological sample. The former approach faces similar problems as with microarrays targeted at mRNA, the latter involves the problem of matching large amounts of mass data against predicted masses from protein sequence databases, and the complicated statistical analysis of samples when multiple incomplete peptides from each protein are detected. Cellular protein localization in a tissue context can be achieved through affinity <a href=\"/wiki/Proteomics\" title=\"Proteomics\">proteomics</a> displayed as spatial data based on <a href=\"/wiki/Immunohistochemistry\" title=\"Immunohistochemistry\">immunohistochemistry</a> and <a href=\"/wiki/Tissue_microarray\" title=\"Tissue microarray\">tissue microarrays</a>.<sup id=\"cite_ref-47\" class=\"reference\"><a href=\"#cite_note-47\">&#91;47&#93;</a></sup>\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Analysis_of_regulation\">Analysis of regulation</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=17\" title=\"Edit section: Analysis of regulation\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p><a href=\"/wiki/Regulation_of_gene_expression\" title=\"Regulation of gene expression\">Gene regulation</a> is a complex process where a signal, such as an extracellular signal such as a <a href=\"/wiki/Hormone\" title=\"Hormone\">hormone</a>, eventually leads to an increase or decrease in the activity of one or more <a href=\"/wiki/Protein\" title=\"Protein\">proteins</a>. Bioinformatics techniques have been applied to explore various steps in this process.\\n</p><p>For example, gene expression can be regulated by nearby elements in the genome. Promoter analysis involves the identification and study of <a href=\"/wiki/Sequence_motif\" title=\"Sequence motif\">sequence motifs</a> in the DNA surrounding the protein-coding region of a gene. These motifs influence the extent to which that region is transcribed into mRNA. <a href=\"/wiki/Enhancer_(genetics)\" title=\"Enhancer (genetics)\">Enhancer</a> elements far away from the promoter can also regulate gene expression, through three-dimensional looping interactions. These interactions can be determined by bioinformatic analysis of <a href=\"/wiki/Chromosome_conformation_capture\" title=\"Chromosome conformation capture\">chromosome conformation capture</a> experiments.\\n</p><p>Expression data can be used to infer gene regulation: one might compare <a href=\"/wiki/Microarray\" title=\"Microarray\">microarray</a> data from a wide variety of states of an organism to form hypotheses about the genes involved in each state. In a single-cell organism, one might compare stages of the <a href=\"/wiki/Cell_cycle\" title=\"Cell cycle\">cell cycle</a>, along with various stress conditions (heat shock, starvation, etc.). <a href=\"/wiki/Cluster_analysis\" title=\"Cluster analysis\">Clustering algorithms</a> can be then applied to expression data to determine which genes are co-expressed. For example, the upstream regions (promoters) of co-expressed genes can be searched for over-represented <a href=\"/wiki/Regulatory_elements\" class=\"mw-redirect\" title=\"Regulatory elements\">regulatory elements</a>. Examples of clustering algorithms applied in gene clustering are <a href=\"/wiki/K-means_clustering\" title=\"K-means clustering\">k-means clustering</a>, <a href=\"/wiki/Self-organizing_map\" title=\"Self-organizing map\">self-organizing maps</a> (SOMs), <a href=\"/wiki/Hierarchical_clustering\" title=\"Hierarchical clustering\">hierarchical clustering</a>, and <a href=\"/wiki/Consensus_clustering\" title=\"Consensus clustering\">consensus clustering</a> methods.\\n</p>\\n<h2><span class=\"mw-headline\" id=\"Analysis_of_cellular_organization\">Analysis of cellular organization</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=18\" title=\"Edit section: Analysis of cellular organization\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<p>Several approaches have been developed to analyze the location of organelles, genes, proteins, and other components within cells. A <a href=\"/wiki/Gene_ontology\" class=\"mw-redirect\" title=\"Gene ontology\">gene ontology</a> category, <i>cellular component</i>, has been devised to capture subcellular localization in many <a href=\"/wiki/Biological_database\" title=\"Biological database\">biological databases</a>.\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Microscopy_and_image_analysis\">Microscopy and image analysis</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=19\" title=\"Edit section: Microscopy and image analysis\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p>Microscopic pictures allow for the location of <a href=\"/wiki/Organelle\" title=\"Organelle\">organelles</a> as well as molecules, which may be the source of abnormalities in diseases.\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Protein_localization\">Protein localization</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=20\" title=\"Edit section: Protein localization\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p>Finding the location of proteins allows us to predict what they do. This is called <a href=\"/wiki/Protein_function_prediction\" title=\"Protein function prediction\">protein function prediction</a>. For instance, if a protein is found in the <a href=\"/wiki/Cell_nucleus\" title=\"Cell nucleus\">nucleus</a> it may be involved in <a href=\"/wiki/Regulation_of_gene_expression\" title=\"Regulation of gene expression\">gene regulation</a> or <a href=\"/wiki/RNA_splicing\" title=\"RNA splicing\">splicing</a>. By contrast, if a protein is found in <a href=\"/wiki/Mitochondrion\" title=\"Mitochondrion\">mitochondria</a>, it may be involved in <a href=\"/wiki/Cellular_respiration\" title=\"Cellular respiration\">respiration</a> or other <a href=\"/wiki/Metabolism\" title=\"Metabolism\">metabolic processes</a>. There are well developed <a href=\"/wiki/Protein_subcellular_localization_prediction\" title=\"Protein subcellular localization prediction\">protein subcellular localization prediction</a> resources available, including protein subcellular location databases, and prediction tools.<sup id=\"cite_ref-48\" class=\"reference\"><a href=\"#cite_note-48\">&#91;48&#93;</a></sup><sup id=\"cite_ref-49\" class=\"reference\"><a href=\"#cite_note-49\">&#91;49&#93;</a></sup>\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Nuclear_organization_of_chromatin\">Nuclear organization of chromatin</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=21\" title=\"Edit section: Nuclear organization of chromatin\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/Nuclear_organization\" title=\"Nuclear organization\">Nuclear organization</a></div>\\n<p>Data from high-throughput <a href=\"/wiki/Chromosome_conformation_capture\" title=\"Chromosome conformation capture\">chromosome conformation capture</a> experiments, such as <a href=\"/wiki/Hi-C_(genomic_analysis_technique)\" title=\"Hi-C (genomic analysis technique)\">Hi-C (experiment)</a> and <a href=\"/wiki/ChIA-PET\" title=\"ChIA-PET\">ChIA-PET</a>, can provide information on the three-dimensional structure and <a href=\"/wiki/Nuclear_organization\" title=\"Nuclear organization\">nuclear organization</a> of <a href=\"/wiki/Chromatin\" title=\"Chromatin\">chromatin</a>. Bioinformatic challenges in this field include partitioning the genome into domains, such as <a href=\"/wiki/Topologically_Associating_Domain\" class=\"mw-redirect\" title=\"Topologically Associating Domain\">Topologically Associating Domains</a> (TADs), that are organised together in three-dimensional space.<sup id=\"cite_ref-50\" class=\"reference\"><a href=\"#cite_note-50\">&#91;50&#93;</a></sup>\\n</p>\\n<h2><span class=\"mw-headline\" id=\"Structural_bioinformatics\">Structural bioinformatics</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=22\" title=\"Edit section: Structural bioinformatics\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main articles: <a href=\"/wiki/Structural_bioinformatics\" title=\"Structural bioinformatics\">Structural bioinformatics</a> and <a href=\"/wiki/Protein_structure_prediction\" title=\"Protein structure prediction\">Protein structure prediction</a></div>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">See also: <a href=\"/wiki/Structural_motif\" title=\"Structural motif\">Structural motif</a> and <a href=\"/wiki/Structural_domain\" class=\"mw-redirect\" title=\"Structural domain\">Structural domain</a></div>\\n<div class=\"thumb tleft\"><div class=\"thumbinner\" style=\"width:222px;\"><a href=\"/wiki/File:1kqf_opm.png\" class=\"image\"><img src=\"//upload.wikimedia.org/wikipedia/commons/thumb/5/50/1kqf_opm.png/220px-1kqf_opm.png\" decoding=\"async\" width=\"220\" height=\"262\" class=\"thumbimage\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/5/50/1kqf_opm.png/330px-1kqf_opm.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/5/50/1kqf_opm.png/440px-1kqf_opm.png 2x\" data-file-width=\"626\" data-file-height=\"745\" /></a>  <div class=\"thumbcaption\"><div class=\"magnify\"><a href=\"/wiki/File:1kqf_opm.png\" class=\"internal\" title=\"Enlarge\"></a></div>3-dimensional protein structures such as this one are common subjects in bioinformatic analyses.</div></div></div>\\n<p>Finding the structure of proteins important application of bioinformatics. The Critical Assessment of Protein Structure Prediction (CASP) is an open competition where worldwide research groups submit protein models for evaluating unknown protein models.<sup id=\"cite_ref-51\" class=\"reference\"><a href=\"#cite_note-51\">&#91;51&#93;</a></sup><sup id=\"cite_ref-52\" class=\"reference\"><a href=\"#cite_note-52\">&#91;52&#93;</a></sup>\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Amino_acid_sequence\">Amino acid sequence</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=23\" title=\"Edit section: Amino acid sequence\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p>The linear <a href=\"/wiki/Amino_acid\" title=\"Amino acid\">amino acid</a> sequence of a protein is called the <a href=\"/wiki/Primary_structure\" class=\"mw-redirect\" title=\"Primary structure\">primary structure</a>, can be easily determined from the sequence of <a href=\"/wiki/Codons\" class=\"mw-redirect\" title=\"Codons\">codons</a> on the DNA gene that codes for it. In most proteins, the primary structure uniquely determines the 3-dimensional structure of a protein in its native environment. An exception is the <a href=\"/wiki/Prion\" title=\"Prion\">misfolded protein</a> involved in <a href=\"/wiki/Bovine_spongiform_encephalopathy\" title=\"Bovine spongiform encephalopathy\">bovine spongiform encephalopathy</a>. This structure is linked to the function of the protein. Additional structural information includes the <i><a href=\"/wiki/Secondary_structure\" class=\"mw-redirect\" title=\"Secondary structure\">secondary</a></i>, <i><a href=\"/wiki/Tertiary_structure\" class=\"mw-redirect\" title=\"Tertiary structure\">tertiary</a></i> and <i><a href=\"/wiki/Quaternary_structure\" class=\"mw-redirect\" title=\"Quaternary structure\">quaternary</a></i> structure. A viable general solution to the prediction of the function of a protein remains an open problem. Most efforts have so far been directed towards heuristics that work most of the time.<sup class=\"noprint Inline-Template Template-Fact\" style=\"white-space:nowrap;\">&#91;<i><a href=\"/wiki/Wikipedia:Citation_needed\" title=\"Wikipedia:Citation needed\"><span title=\"This claim needs references to reliable sources. (July 2015)\">citation needed</span></a></i>&#93;</sup>\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Homology\">Homology</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=24\" title=\"Edit section: Homology\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p>In the genomic branch of bioinformatics, homology is used to predict the function of a gene: if the sequence of gene <i>A</i>, whose function is known, is homologous to the sequence of gene <i>B,</i> whose function is unknown, one could infer that B may share A\\'s function. In structural bioinformatics, homology is used to determine which parts of a protein are important in structure formation and interaction with other proteins. <a href=\"/wiki/Homology_modeling\" title=\"Homology modeling\">Homology modeling</a> is used to predict the structure of an unknown protein from existing homologous proteins. \\n</p><p>One example of this is hemoglobin in humans and the hemoglobin in legumes (<a href=\"/wiki/Leghemoglobin\" title=\"Leghemoglobin\">leghemoglobin</a>), which are distant relatives from the same <a href=\"/wiki/Protein_superfamily\" title=\"Protein superfamily\">protein superfamily</a>. Both serve the same purpose of transporting oxygen in the organism. Although both of these proteins have completely different amino acid sequences, their protein structures are virtually identical, which reflects their near identical purposes and shared ancestor.<sup id=\"cite_ref-53\" class=\"reference\"><a href=\"#cite_note-53\">&#91;53&#93;</a></sup>\\n</p><p>Other techniques for predicting protein structure include protein threading and <i>de novo</i> (from scratch) physics-based modeling.\\n</p><p>Another aspect of structural bioinformatics include the use of protein structures for <a href=\"/wiki/Virtual_screening\" title=\"Virtual screening\">Virtual Screening</a> models such as <a href=\"/wiki/QSAR\" class=\"mw-redirect\" title=\"QSAR\">Quantitative Structure-Activity Relationship</a> models and proteochemometric models (PCM). Furthermore, a protein\\'s crystal structure can be used in simulation of for example ligand-binding studies and <i>in silico</i> mutagenesis studies.\\n</p><p>A 2021 <a href=\"/wiki/Deep-learning\" class=\"mw-redirect\" title=\"Deep-learning\">deep-learning</a> algorithms-based software called <a href=\"/wiki/AlphaFold\" title=\"AlphaFold\">AlphaFold</a>, developed by Google\\'s <a href=\"/wiki/DeepMind\" class=\"mw-redirect\" title=\"DeepMind\">DeepMind</a>, greatly outperforms all other prediction software methods<sup id=\"cite_ref-54\" class=\"reference\"><a href=\"#cite_note-54\">&#91;54&#93;</a></sup><sup class=\"noprint Inline-Template\" style=\"white-space:nowrap;\">&#91;<i><a href=\"/wiki/Wikipedia:Please_clarify\" title=\"Wikipedia:Please clarify\"><span title=\"Please clarify the preceding statement or statements with a good explanation from a reliable source. (June 2023)\">how?</span></a></i>&#93;</sup>, and has released predicted structures for hundreds of millions of proteins in the AlphaFold protein structure database.<sup id=\"cite_ref-55\" class=\"reference\"><a href=\"#cite_note-55\">&#91;55&#93;</a></sup>\\n</p>\\n<h2><span class=\"mw-headline\" id=\"Network_and_systems_biology\">Network and systems biology</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=25\" title=\"Edit section: Network and systems biology\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main articles: <a href=\"/wiki/Computational_systems_biology\" class=\"mw-redirect\" title=\"Computational systems biology\">Computational systems biology</a>, <a href=\"/wiki/Biological_network\" title=\"Biological network\">Biological network</a>, and <a href=\"/wiki/Interactome\" title=\"Interactome\">Interactome</a></div>\\n<p><i>Network analysis</i> seeks to understand the relationships within <a href=\"/wiki/Biological_network\" title=\"Biological network\">biological networks</a> such as <a href=\"/wiki/Metabolic_network\" title=\"Metabolic network\">metabolic</a> or <a href=\"/wiki/Interactome\" title=\"Interactome\">protein\\xe2\\x80\\x93protein interaction networks</a>. Although biological networks can be constructed from a single type of molecule or entity (such as genes), network biology often attempts to integrate many different data types, such as proteins, small molecules, gene expression data, and others, which are all connected physically, functionally, or both.\\n</p><p><i>Systems biology</i> involves the use of <a href=\"/wiki/Computer_simulation\" title=\"Computer simulation\">computer simulations</a> of <a href=\"/wiki/Cell_(biology)\" title=\"Cell (biology)\">cellular</a> subsystems (such as the <a href=\"/wiki/Metabolic_network\" title=\"Metabolic network\">networks of metabolites</a> and <a href=\"/wiki/Enzyme\" title=\"Enzyme\">enzymes</a> that comprise <a href=\"/wiki/Metabolism\" title=\"Metabolism\">metabolism</a>, <a href=\"/wiki/Signal_transduction\" title=\"Signal transduction\">signal transduction</a> pathways and <a href=\"/wiki/Gene_regulatory_network\" title=\"Gene regulatory network\">gene regulatory networks</a>) to both analyze and visualize the complex connections of these cellular processes. <a href=\"/wiki/Artificial_life\" title=\"Artificial life\">Artificial life</a> or virtual evolution attempts to understand evolutionary processes via the computer simulation of simple (artificial) life forms.\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Molecular_interaction_networks\">Molecular interaction networks</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=26\" title=\"Edit section: Molecular interaction networks\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<div class=\"thumb tright\"><div class=\"thumbinner\" style=\"width:202px;\"><a href=\"/wiki/File:The_protein_interaction_network_of_Treponema_pallidum.png\" class=\"image\"><img src=\"//upload.wikimedia.org/wikipedia/commons/thumb/b/b4/The_protein_interaction_network_of_Treponema_pallidum.png/200px-The_protein_interaction_network_of_Treponema_pallidum.png\" decoding=\"async\" width=\"200\" height=\"188\" class=\"thumbimage\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/b/b4/The_protein_interaction_network_of_Treponema_pallidum.png/300px-The_protein_interaction_network_of_Treponema_pallidum.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/b/b4/The_protein_interaction_network_of_Treponema_pallidum.png/400px-The_protein_interaction_network_of_Treponema_pallidum.png 2x\" data-file-width=\"1534\" data-file-height=\"1444\" /></a>  <div class=\"thumbcaption\"><div class=\"magnify\"><a href=\"/wiki/File:The_protein_interaction_network_of_Treponema_pallidum.png\" class=\"internal\" title=\"Enlarge\"></a></div>Interactions between proteins are frequently visualized and analyzed using networks. This network is made up of protein\\xe2\\x80\\x93protein interactions from <i><a href=\"/wiki/Treponema_pallidum\" title=\"Treponema pallidum\">Treponema pallidum</a></i>, the causative agent of <a href=\"/wiki/Syphilis\" title=\"Syphilis\">syphilis</a> and other diseases.<sup id=\"cite_ref-56\" class=\"reference\"><a href=\"#cite_note-56\">&#91;56&#93;</a></sup></div></div></div>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main articles: <a href=\"/wiki/Protein%E2%80%93protein_interaction_prediction\" title=\"Protein\\xe2\\x80\\x93protein interaction prediction\">Protein\\xe2\\x80\\x93protein interaction prediction</a> and <a href=\"/wiki/Interactome\" title=\"Interactome\">interactome</a></div>\\n<p>Tens of thousands of three-dimensional protein structures have been determined by <a href=\"/wiki/X-ray_crystallography\" title=\"X-ray crystallography\">X-ray crystallography</a> and <a href=\"/wiki/Protein_nuclear_magnetic_resonance_spectroscopy\" class=\"mw-redirect\" title=\"Protein nuclear magnetic resonance spectroscopy\">protein nuclear magnetic resonance spectroscopy</a> (protein NMR) and a central question in structural bioinformatics is whether it is practical to predict possible protein\\xe2\\x80\\x93protein interactions only based on these 3D shapes, without performing <a href=\"/wiki/Protein%E2%80%93protein_interaction\" title=\"Protein\\xe2\\x80\\x93protein interaction\">protein\\xe2\\x80\\x93protein interaction</a> experiments. A variety of methods have been developed to tackle the <a href=\"/wiki/Protein%E2%80%93protein_docking\" class=\"mw-redirect\" title=\"Protein\\xe2\\x80\\x93protein docking\">protein\\xe2\\x80\\x93protein docking</a> problem, though it seems that there is still much work to be done in this field.\\n</p><p>Other interactions encountered in the field include Protein\\xe2\\x80\\x93ligand (including drug) and <a href=\"/w/index.php?title=Protein%E2%80%93peptide&amp;action=edit&amp;redlink=1\" class=\"new\" title=\"Protein\\xe2\\x80\\x93peptide (page does not exist)\">protein\\xe2\\x80\\x93peptide</a>. Molecular dynamic simulation of movement of atoms about rotatable bonds is the fundamental principle behind computational <a href=\"/wiki/Algorithm\" title=\"Algorithm\">algorithms</a>, termed docking algorithms, for studying <a href=\"/wiki/Interactome\" title=\"Interactome\">molecular interactions</a>.\\n</p>\\n<h2><span class=\"mw-headline\" id=\"Others\">Others</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=27\" title=\"Edit section: Others\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<h3><span class=\"mw-headline\" id=\"Literature_analysis\">Literature analysis</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=28\" title=\"Edit section: Literature analysis\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main articles: <a href=\"/wiki/Text_mining\" title=\"Text mining\">Text mining</a> and <a href=\"/wiki/Biomedical_text_mining\" title=\"Biomedical text mining\">Biomedical text mining</a></div>\\n<p>The enormous number of published literature makes it virtually impossible for individuals to read every paper, resulting in disjointed sub-fields of research. Literature analysis aims to employ computational and statistical linguistics to mine this growing library of text resources. For example:\\n</p>\\n<ul><li>Abbreviation recognition \\xe2\\x80\\x93 identify the long-form and abbreviation of biological terms</li>\\n<li><a href=\"/wiki/Named-entity_recognition\" title=\"Named-entity recognition\">Named-entity recognition</a> \\xe2\\x80\\x93 recognizing biological terms such as gene names</li>\\n<li>Protein\\xe2\\x80\\x93protein interaction \\xe2\\x80\\x93 identify which <a href=\"/wiki/Protein\" title=\"Protein\">proteins</a> interact with which proteins from text</li></ul>\\n<p>The area of research draws from <a href=\"/wiki/Statistics\" title=\"Statistics\">statistics</a> and <a href=\"/wiki/Computational_linguistics\" title=\"Computational linguistics\">computational linguistics</a>.\\n</p>\\n<h3><span class=\"mw-headline\" id=\"High-throughput_image_analysis\">High-throughput image analysis</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=29\" title=\"Edit section: High-throughput image analysis\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p>Computational technologies are used to automate the processing, quantification and analysis of large amounts of high-information-content <a href=\"/wiki/Medical_imaging\" title=\"Medical imaging\">biomedical imagery</a>. Modern <a href=\"/wiki/Image_analysis\" title=\"Image analysis\">image analysis</a> systems can improve an observer\\'s <a href=\"/wiki/Accuracy\" class=\"mw-redirect\" title=\"Accuracy\">accuracy</a>, <a href=\"/wiki/Objectivity_(science)\" title=\"Objectivity (science)\">objectivity</a>, or speed. Image analysis is important for both <a href=\"/wiki/Diagnostics\" class=\"mw-redirect\" title=\"Diagnostics\">diagnostics</a> and research. Some examples are:\\n</p>\\n<ul><li>high-throughput and high-fidelity quantification and sub-cellular localization (<a href=\"/wiki/High-content_screening\" title=\"High-content screening\">high-content screening</a>, cytohistopathology, <a href=\"/wiki/Bioimage_informatics\" title=\"Bioimage informatics\">Bioimage informatics</a>)</li>\\n<li><a href=\"/wiki/Morphometrics\" title=\"Morphometrics\">morphometrics</a></li>\\n<li>clinical image analysis and visualization</li>\\n<li>determining the real-time air-flow patterns in breathing lungs of living animals</li>\\n<li>quantifying occlusion size in real-time imagery from the development of and recovery during arterial injury</li>\\n<li>making behavioral observations from extended video recordings of laboratory animals</li>\\n<li>infrared measurements for metabolic activity determination</li>\\n<li>inferring clone overlaps in <a href=\"/wiki/Gene_mapping\" title=\"Gene mapping\">DNA mapping</a>, e.g. the <a href=\"/wiki/Sulston_score\" title=\"Sulston score\">Sulston score</a></li></ul>\\n<h3><span class=\"mw-headline\" id=\"High-throughput_single_cell_data_analysis\">High-throughput single cell data analysis</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=30\" title=\"Edit section: High-throughput single cell data analysis\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/Flow_cytometry_bioinformatics\" title=\"Flow cytometry bioinformatics\">Flow cytometry bioinformatics</a></div>\\n<p>Computational techniques are used to analyse high-throughput, low-measurement single cell data, such as that obtained from <a href=\"/wiki/Flow_cytometry\" title=\"Flow cytometry\">flow cytometry</a>. These methods typically involve finding populations of cells that are relevant to a particular disease state or experimental condition.\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Biodiversity_informatics\">Biodiversity informatics</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=31\" title=\"Edit section: Biodiversity informatics\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/Biodiversity_informatics\" title=\"Biodiversity informatics\">Biodiversity informatics</a></div>\\n<p>Biodiversity informatics deals with the collection and analysis of <a href=\"/wiki/Biodiversity\" title=\"Biodiversity\">biodiversity</a> data, such as <a href=\"/wiki/Taxonomic_database\" title=\"Taxonomic database\">taxonomic databases</a>, or <a href=\"/wiki/Microbiome\" title=\"Microbiome\">microbiome</a> data. Examples of such analyses include <a href=\"/wiki/Phylogenetics\" title=\"Phylogenetics\">phylogenetics</a>, <a href=\"/wiki/Niche_modelling\" class=\"mw-redirect\" title=\"Niche modelling\">niche modelling</a>, <a href=\"/wiki/Species_richness\" title=\"Species richness\">species richness</a> mapping, <a href=\"/wiki/DNA_barcoding\" title=\"DNA barcoding\">DNA barcoding</a>, or <a href=\"/wiki/Speciesism\" title=\"Speciesism\">species</a> identification tools.\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Ontologies_and_data_integration\">Ontologies and data integration</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=32\" title=\"Edit section: Ontologies and data integration\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p>Biological ontologies are <a href=\"/wiki/Directed_acyclic_graph\" title=\"Directed acyclic graph\">directed acyclic graphs</a> of <a href=\"/wiki/Controlled_vocabulary\" title=\"Controlled vocabulary\">controlled vocabularies</a>. They create categories for biological concepts and descriptions so they can be easily analyzed with computers. When categorised in this way, it is possible to gain added value from holistic and integrated analysis.<sup class=\"noprint Inline-Template Template-Fact\" style=\"white-space:nowrap;\">&#91;<i><a href=\"/wiki/Wikipedia:Citation_needed\" title=\"Wikipedia:Citation needed\"><span title=\"This claim needs references to reliable sources. (June 2023)\">citation needed</span></a></i>&#93;</sup>\\n</p><p>The <a href=\"/wiki/OBO_Foundry\" title=\"OBO Foundry\">OBO Foundry</a> was an effort to standardise certain ontologies. One of the most widespread is the <a href=\"/wiki/Gene_ontology\" class=\"mw-redirect\" title=\"Gene ontology\">Gene ontology</a> which describes gene function. There are also ontologies which describe phenotypes.\\n</p>\\n<h2><span class=\"mw-headline\" id=\"Databases\">Databases</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=33\" title=\"Edit section: Databases\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main articles: <a href=\"/wiki/List_of_biological_databases\" title=\"List of biological databases\">List of biological databases</a> and <a href=\"/wiki/Biological_database\" title=\"Biological database\">Biological database</a></div>\\n<p>Databases are essential for bioinformatics research and applications. Databases exist for many different information types, including DNA and protein sequences, molecular structures, phenotypes and biodiversity. Databases can contain both empirical data (obtained directly from experiments) and predicted data (obtained from analysis of existing data). They may be specific to a particular organism, pathway or molecule of interest. Alternatively, they can incorporate data compiled from multiple other databases. Databases can have different formats, access mechanisms, and be public or private.\\n</p><p>Some of the most commonly used databases are listed below:\\n</p>\\n<ul><li>Used in biological sequence analysis: <a href=\"/wiki/Genbank\" class=\"mw-redirect\" title=\"Genbank\">Genbank</a>, <a href=\"/wiki/UniProt\" title=\"UniProt\">UniProt</a></li>\\n<li>Used in structure analysis: <a href=\"/wiki/Protein_Data_Bank\" title=\"Protein Data Bank\">Protein Data Bank</a> (PDB)</li>\\n<li>Used in finding Protein Families and <a href=\"/wiki/Sequence_motif\" title=\"Sequence motif\">Motif</a> Finding: <a href=\"/wiki/InterPro\" title=\"InterPro\">InterPro</a>, <a href=\"/wiki/Pfam\" title=\"Pfam\">Pfam</a></li>\\n<li>Used for Next Generation Sequencing: <a href=\"/wiki/Sequence_Read_Archive\" title=\"Sequence Read Archive\">Sequence Read Archive</a></li>\\n<li>Used in Network Analysis: Metabolic Pathway Databases (<a href=\"/wiki/KEGG\" title=\"KEGG\">KEGG</a>, <a href=\"/wiki/BioCyc_database_collection\" title=\"BioCyc database collection\">BioCyc</a>), Interaction Analysis Databases, Functional Networks</li>\\n<li>Used in design of synthetic genetic circuits: <a href=\"/wiki/GenoCAD\" title=\"GenoCAD\">GenoCAD</a><sup class=\"noprint Inline-Template Template-Fact\" style=\"white-space:nowrap;\">&#91;<i><a href=\"/wiki/Wikipedia:Citation_needed\" title=\"Wikipedia:Citation needed\"><span title=\"This claim needs references to reliable sources. (June 2023)\">citation needed</span></a></i>&#93;</sup></li></ul>\\n<h2><span class=\"mw-headline\" id=\"Software_and_tools\">Software and tools</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=34\" title=\"Edit section: Software and tools\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<p><a href=\"/wiki/List_of_bioinformatics_software\" title=\"List of bioinformatics software\">Software tools for bioinformatics</a> include simple command-line tools, more complex graphical programs, and standalone web-services. They are made by <a href=\"/wiki/List_of_bioinformatics_companies\" title=\"List of bioinformatics companies\">bioinformatics companies</a> or by public institutions.\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Open-source_bioinformatics_software\">Open-source bioinformatics software</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=35\" title=\"Edit section: Open-source bioinformatics software\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/List_of_bioinformatics_software\" title=\"List of bioinformatics software\">List of bioinformatics software</a></div>\\n<p>Many <a href=\"/wiki/Free_and_open-source_software\" title=\"Free and open-source software\">free and open-source software</a> tools have existed and continued to grow since the 1980s.<sup id=\"cite_ref-obf-main_57-0\" class=\"reference\"><a href=\"#cite_note-obf-main-57\">&#91;57&#93;</a></sup> The combination of a continued need for new <a href=\"/wiki/Algorithm\" title=\"Algorithm\">algorithms</a> for the analysis of emerging types of biological readouts, the potential for innovative <i><a href=\"/wiki/In_silico\" title=\"In silico\">in silico</a></i> experiments, and freely available <a href=\"/wiki/Open_code\" class=\"mw-redirect\" title=\"Open code\">open code</a> bases have created opportunities for research groups to contribute to both bioinformatics regardless of <a href=\"/wiki/Funding_of_science\" title=\"Funding of science\">funding</a>. The open source tools often act as incubators of ideas, or community-supported <a href=\"/wiki/Plug-in_(computing)\" title=\"Plug-in (computing)\">plug-ins</a> in commercial applications. They may also provide <i><a href=\"/wiki/De_facto\" title=\"De facto\">de facto</a></i> standards and shared object models for assisting with the challenge of bioinformation integration.\\n</p><p>Open-source bioinformatics software includes <a href=\"/wiki/Bioconductor\" title=\"Bioconductor\">Bioconductor</a>, <a href=\"/wiki/BioPerl\" title=\"BioPerl\">BioPerl</a>, <a href=\"/wiki/Biopython\" title=\"Biopython\">Biopython</a>, <a href=\"/wiki/BioJava\" title=\"BioJava\">BioJava</a>, <a href=\"/wiki/BioJS\" title=\"BioJS\">BioJS</a>, <a href=\"/wiki/BioRuby\" title=\"BioRuby\">BioRuby</a>, <a href=\"/wiki/Bioclipse\" title=\"Bioclipse\">Bioclipse</a>, <a href=\"/wiki/EMBOSS\" title=\"EMBOSS\">EMBOSS</a>, <a href=\"/wiki/.NET_Bio\" title=\".NET Bio\">.NET Bio</a>, <a href=\"/wiki/Orange_(software)\" title=\"Orange (software)\">Orange</a> with its bioinformatics add-on, <a href=\"/wiki/Apache_Taverna\" title=\"Apache Taverna\">Apache Taverna</a>, <a href=\"/wiki/UGENE\" title=\"UGENE\">UGENE</a> and <a href=\"/wiki/GenoCAD\" title=\"GenoCAD\">GenoCAD</a>. \\n</p><p>The non-profit <a href=\"/wiki/Open_Bioinformatics_Foundation\" title=\"Open Bioinformatics Foundation\">Open Bioinformatics Foundation</a><sup id=\"cite_ref-obf-main_57-1\" class=\"reference\"><a href=\"#cite_note-obf-main-57\">&#91;57&#93;</a></sup> and the annual <a href=\"/wiki/Bioinformatics_Open_Source_Conference\" title=\"Bioinformatics Open Source Conference\">Bioinformatics Open Source Conference</a> promote open-source bioinformatics software.<sup id=\"cite_ref-obf-bosc_58-0\" class=\"reference\"><a href=\"#cite_note-obf-bosc-58\">&#91;58&#93;</a></sup>\\n</p>\\n<h3><span class=\"mw-headline\" id=\"Web_services_in_bioinformatics\">Web services in bioinformatics</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=36\" title=\"Edit section: Web services in bioinformatics\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p><a href=\"/wiki/SOAP\" title=\"SOAP\">SOAP</a>- and <a href=\"/wiki/REST\" class=\"mw-redirect\" title=\"REST\">REST</a>-based interfaces have been developed to allow client computers to use algorithms, data and computing resources from servers in other parts of the world. The main advantage are that end users do not have to deal with software and database maintenance overheads.\\n</p><p>Basic bioinformatics services are classified by the <a href=\"/wiki/European_Bioinformatics_Institute\" title=\"European Bioinformatics Institute\">EBI</a> into three categories: <a href=\"/wiki/Sequence_alignment_software\" class=\"mw-redirect\" title=\"Sequence alignment software\">SSS</a> (Sequence Search Services), <a href=\"/wiki/Multiple_sequence_alignment\" title=\"Multiple sequence alignment\">MSA</a> (Multiple Sequence Alignment), and <a href=\"#Sequence_analysis\">BSA</a> (Biological Sequence Analysis).<sup id=\"cite_ref-59\" class=\"reference\"><a href=\"#cite_note-59\">&#91;59&#93;</a></sup> The availability of these <a href=\"/wiki/Service-orientation\" title=\"Service-orientation\">service-oriented</a> bioinformatics resources demonstrate the applicability of web-based bioinformatics solutions, and range from a collection of standalone tools with a common data format under a single web-based interface, to integrative, distributed and extensible <a href=\"/wiki/Bioinformatics_workflow_management_systems\" class=\"mw-redirect\" title=\"Bioinformatics workflow management systems\">bioinformatics workflow management systems</a>.\\n</p>\\n<h4><span class=\"mw-headline\" id=\"Bioinformatics_workflow_management_systems\">Bioinformatics workflow management systems</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=37\" title=\"Edit section: Bioinformatics workflow management systems\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h4>\\n<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1033289096\"><div role=\"note\" class=\"hatnote navigation-not-searchable\">Main article: <a href=\"/wiki/Bioinformatics_workflow_management_systems\" class=\"mw-redirect\" title=\"Bioinformatics workflow management systems\">Bioinformatics workflow management systems</a></div>\\n<p>A <a href=\"/wiki/Bioinformatics_workflow_management_systems\" class=\"mw-redirect\" title=\"Bioinformatics workflow management systems\">bioinformatics workflow management system</a> is a specialized form of a <a href=\"/wiki/Workflow_management_system\" title=\"Workflow management system\">workflow management system</a> designed specifically to compose and execute a series of computational or data manipulation steps, or a workflow, in a Bioinformatics application. Such systems are designed to\\n</p>\\n<ul><li>provide an easy-to-use environment for individual application scientists themselves to create their own workflows,</li>\\n<li>provide interactive tools for the scientists enabling them to execute their workflows and view their results in real-time,</li>\\n<li>simplify the process of sharing and reusing workflows between the scientists, and</li>\\n<li>enable scientists to track the <a href=\"/wiki/Provenance\" title=\"Provenance\">provenance</a> of the workflow execution results and the workflow creation steps.</li></ul>\\n<p>Some of the platforms giving this service: <a href=\"/wiki/Galaxy_(computational_biology)\" title=\"Galaxy (computational biology)\">Galaxy</a>, <a href=\"/wiki/Kepler_scientific_workflow_system\" title=\"Kepler scientific workflow system\">Kepler</a>, <a href=\"/wiki/Apache_Taverna\" title=\"Apache Taverna\">Taverna</a>, <a href=\"/wiki/UGENE\" title=\"UGENE\">UGENE</a>, <a href=\"/wiki/Anduril_(workflow_engine)\" title=\"Anduril (workflow engine)\">Anduril</a>, <a href=\"/wiki/High-performance_Integrated_Virtual_Environment\" title=\"High-performance Integrated Virtual Environment\">HIVE</a>.\\n</p>\\n<h3><span class=\"mw-headline\" id=\"BioCompute_and_BioCompute_Objects\">BioCompute and BioCompute Objects</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=38\" title=\"Edit section: BioCompute and BioCompute Objects\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h3>\\n<p>In 2014, the <a href=\"/wiki/Food_and_Drug_Administration\" title=\"Food and Drug Administration\">US Food and Drug Administration</a> sponsored a conference held at the <a href=\"/wiki/National_Institutes_of_Health\" title=\"National Institutes of Health\">National Institutes of Health</a> Bethesda Campus to discuss reproducibility in bioinformatics.<sup id=\"cite_ref-60\" class=\"reference\"><a href=\"#cite_note-60\">&#91;60&#93;</a></sup> Over the next three years, a consortium of stakeholders met regularly to discuss what would become BioCompute paradigm.<sup id=\"cite_ref-61\" class=\"reference\"><a href=\"#cite_note-61\">&#91;61&#93;</a></sup> These stakeholders included representatives from government, industry, and academic entities. Session leaders represented numerous branches of the FDA and NIH Institutes and Centers, non-profit entities including the <a href=\"/wiki/Human_Variome_Project\" title=\"Human Variome Project\">Human Variome Project</a> and the <a href=\"/wiki/European_Federation_for_Medical_Informatics\" title=\"European Federation for Medical Informatics\">European Federation for Medical Informatics</a>, and research institutions including <a href=\"/wiki/Stanford_University\" title=\"Stanford University\">Stanford</a>, the <a href=\"/wiki/New_York_Genome_Center\" title=\"New York Genome Center\">New York Genome Center</a>, and the <a href=\"/wiki/George_Washington_University\" title=\"George Washington University\">George Washington University</a>.\\n</p><p>It was decided that the BioCompute paradigm would be in the form of digital \\'lab notebooks\\' which allow for the reproducibility, replication, review, and reuse, of bioinformatics protocols. This was proposed to enable greater continuity within a research group over the course of normal personnel flux while furthering the exchange of ideas between groups. The US FDA funded this work so that information on pipelines would be more transparent and accessible to their regulatory staff.<sup id=\"cite_ref-62\" class=\"reference\"><a href=\"#cite_note-62\">&#91;62&#93;</a></sup>\\n</p><p>In 2016, the group reconvened at the NIH in Bethesda and discussed the potential for a <a href=\"/wiki/BioCompute_Object\" title=\"BioCompute Object\">BioCompute Object</a>, an instance of the BioCompute paradigm. This work was copied as both a \"standard trial use\" document and a preprint paper uploaded to bioRxiv. The BioCompute object allows for the JSON-ized record to be shared among employees, collaborators, and regulators.<sup id=\"cite_ref-63\" class=\"reference\"><a href=\"#cite_note-63\">&#91;63&#93;</a></sup><sup id=\"cite_ref-64\" class=\"reference\"><a href=\"#cite_note-64\">&#91;64&#93;</a></sup>\\n</p>\\n<h2><span class=\"mw-headline\" id=\"Education_platforms\">Education platforms</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=39\" title=\"Edit section: Education platforms\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<p>Bioinformatics is not only taught as in-person <a href=\"/wiki/Master%27s_degree\" title=\"Master&#39;s degree\">masters degree</a> at many universities. The computational nature of bioinformatics lends it to <a href=\"/wiki/Educational_technology\" title=\"Educational technology\">computer-aided and online learning</a>.<sup id=\"cite_ref-65\" class=\"reference\"><a href=\"#cite_note-65\">&#91;65&#93;</a></sup><sup id=\"cite_ref-66\" class=\"reference\"><a href=\"#cite_note-66\">&#91;66&#93;</a></sup> Software platforms designed to teach bioinformatics concepts and methods include <a href=\"/wiki/Rosalind_(education_platform)\" title=\"Rosalind (education platform)\">Rosalind</a> and online courses offered through the <a href=\"/wiki/Swiss_Institute_of_Bioinformatics\" title=\"Swiss Institute of Bioinformatics\">Swiss Institute of Bioinformatics</a> Training Portal. The <a href=\"/wiki/Canadian_Bioinformatics_Workshops\" title=\"Canadian Bioinformatics Workshops\">Canadian Bioinformatics Workshops</a> provides videos and slides from training workshops on their website under a <a href=\"/wiki/Creative_Commons\" title=\"Creative Commons\">Creative Commons</a> license. The 4273\\xcf\\x80 project or 4273pi project<sup id=\"cite_ref-67\" class=\"reference\"><a href=\"#cite_note-67\">&#91;67&#93;</a></sup> also offers open source educational materials for free. The course runs on low cost <a href=\"/wiki/Raspberry_Pi\" title=\"Raspberry Pi\">Raspberry Pi</a> computers and has been used to teach adults and school pupils.<sup id=\"cite_ref-68\" class=\"reference\"><a href=\"#cite_note-68\">&#91;68&#93;</a></sup><sup id=\"cite_ref-69\" class=\"reference\"><a href=\"#cite_note-69\">&#91;69&#93;</a></sup> 4283 is actively developed by a consortium of academics and research staff who have run research level bioinformatics using Raspberry Pi computers and the 4283\\xcf\\x80 operating system.<sup id=\"cite_ref-70\" class=\"reference\"><a href=\"#cite_note-70\">&#91;70&#93;</a></sup><sup id=\"cite_ref-71\" class=\"reference\"><a href=\"#cite_note-71\">&#91;71&#93;</a></sup>\\n</p><p><a href=\"/wiki/Massive_open_online_course\" title=\"Massive open online course\">MOOC</a> platforms also provide online certifications in bioinformatics and related disciplines, including <a href=\"/wiki/Coursera\" title=\"Coursera\">Coursera</a>\\'s Bioinformatics Specialization (<a href=\"/wiki/University_of_California,_San_Diego\" title=\"University of California, San Diego\">UC San Diego</a>) and Genomic Data Science Specialization (<a href=\"/wiki/Johns_Hopkins_University\" title=\"Johns Hopkins University\">Johns Hopkins</a>) as well as <a href=\"/wiki/EdX\" title=\"EdX\">EdX</a>\\'s Data Analysis for Life Sciences XSeries (<a href=\"/wiki/Harvard_University\" title=\"Harvard University\">Harvard</a>).\\n</p>\\n<h2><span class=\"mw-headline\" id=\"Conferences\">Conferences</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=40\" title=\"Edit section: Conferences\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<p>There are several large conferences that are concerned with bioinformatics. Some of the most notable examples are <a href=\"/wiki/Intelligent_Systems_for_Molecular_Biology\" title=\"Intelligent Systems for Molecular Biology\">Intelligent Systems for Molecular Biology</a> (ISMB), <a href=\"/wiki/European_Conference_on_Computational_Biology\" title=\"European Conference on Computational Biology\">European Conference on Computational Biology</a> (ECCB), and <a href=\"/wiki/Research_in_Computational_Molecular_Biology\" title=\"Research in Computational Molecular Biology\">Research in Computational Molecular Biology</a> (RECOMB).\\n</p>\\n<h2><span class=\"mw-headline\" id=\"See_also\">See also</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=41\" title=\"Edit section: See also\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<style data-mw-deduplicate=\"TemplateStyles:r1147244281\">.mw-parser-output .div-col{margin-top:0.3em;column-width:30em}.mw-parser-output .div-col-small{font-size:90%}.mw-parser-output .div-col-rules{column-rule:1px solid #aaa}.mw-parser-output .div-col dl,.mw-parser-output .div-col ol,.mw-parser-output .div-col ul{margin-top:0}.mw-parser-output .div-col li,.mw-parser-output .div-col dd{page-break-inside:avoid;break-inside:avoid-column}.mw-parser-output .plainlist ol,.mw-parser-output .plainlist ul{line-height:inherit;list-style:none;margin:0}.mw-parser-output .plainlist ol li,.mw-parser-output .plainlist ul li{margin-bottom:0}</style><div class=\"div-col\" style=\"column-width: 30em;\">\\n<ul><li><a href=\"/wiki/Biodiversity_informatics\" title=\"Biodiversity informatics\">Biodiversity informatics</a></li>\\n<li><a href=\"/wiki/Bioinformatics_companies\" class=\"mw-redirect\" title=\"Bioinformatics companies\">Bioinformatics companies</a></li>\\n<li><a href=\"/wiki/Computational_biology\" title=\"Computational biology\">Computational biology</a></li>\\n<li><a href=\"/wiki/Computational_biomodeling\" class=\"mw-redirect\" title=\"Computational biomodeling\">Computational biomodeling</a></li>\\n<li><a href=\"/wiki/Computational_genomics\" title=\"Computational genomics\">Computational genomics</a></li>\\n<li><a href=\"/wiki/Cyberbiosecurity\" title=\"Cyberbiosecurity\">Cyberbiosecurity</a></li>\\n<li><a href=\"/wiki/Functional_genomics\" title=\"Functional genomics\">Functional genomics</a></li>\\n<li><a href=\"/wiki/Health_informatics\" title=\"Health informatics\">Health informatics</a></li>\\n<li><a href=\"/wiki/International_Society_for_Computational_Biology\" title=\"International Society for Computational Biology\">International Society for Computational Biology</a></li>\\n<li><a href=\"/wiki/Jumping_library\" title=\"Jumping library\">Jumping library</a></li>\\n<li><a href=\"/wiki/List_of_bioinformatics_institutions\" title=\"List of bioinformatics institutions\">List of bioinformatics institutions</a></li>\\n<li><a href=\"/wiki/List_of_open-source_bioinformatics_software\" title=\"List of open-source bioinformatics software\">List of open-source bioinformatics software</a></li>\\n<li><a href=\"/wiki/List_of_bioinformatics_journals\" title=\"List of bioinformatics journals\">List of bioinformatics journals</a></li>\\n<li><a href=\"/wiki/Metabolomics\" title=\"Metabolomics\">Metabolomics</a></li>\\n<li><a href=\"/wiki/Nucleic_acid_sequence\" title=\"Nucleic acid sequence\">Nucleic acid sequence</a></li>\\n<li><a href=\"/wiki/Phylogenetics\" title=\"Phylogenetics\">Phylogenetics</a></li>\\n<li><a href=\"/wiki/Proteomics\" title=\"Proteomics\">Proteomics</a></li>\\n<li><a href=\"/wiki/Gene_Disease_Database\" title=\"Gene Disease Database\">Gene Disease Database</a></li></ul></div>\\n<h2><span class=\"mw-headline\" id=\"References\">References</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=42\" title=\"Edit section: References\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<style data-mw-deduplicate=\"TemplateStyles:r1011085734\">.mw-parser-output .reflist{font-size:90%;margin-bottom:0.5em;list-style-type:decimal}.mw-parser-output .reflist .references{font-size:100%;margin-bottom:0;list-style-type:inherit}.mw-parser-output .reflist-columns-2{column-width:30em}.mw-parser-output .reflist-columns-3{column-width:25em}.mw-parser-output .reflist-columns{margin-top:0.3em}.mw-parser-output .reflist-columns ol{margin-top:0}.mw-parser-output .reflist-columns li{page-break-inside:avoid;break-inside:avoid-column}.mw-parser-output .reflist-upper-alpha{list-style-type:upper-alpha}.mw-parser-output .reflist-upper-roman{list-style-type:upper-roman}.mw-parser-output .reflist-lower-alpha{list-style-type:lower-alpha}.mw-parser-output .reflist-lower-greek{list-style-type:lower-greek}.mw-parser-output .reflist-lower-roman{list-style-type:lower-roman}</style><div class=\"reflist\">\\n<div class=\"mw-references-wrap mw-references-columns\"><ol class=\"references\">\\n<li id=\"cite_note-1\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-1\">^</a></b></span> <span class=\"reference-text\"><style data-mw-deduplicate=\"TemplateStyles:r1133582631\">.mw-parser-output cite.citation{font-style:inherit;word-wrap:break-word}.mw-parser-output .citation q{quotes:\"\\\\\"\"\"\\\\\"\"\"\\'\"\"\\'\"}.mw-parser-output .citation:target{background-color:rgba(0,127,255,0.133)}.mw-parser-output .id-lock-free a,.mw-parser-output .citation .cs1-lock-free a{background:url(\"//upload.wikimedia.org/wikipedia/commons/6/65/Lock-green.svg\")right 0.1em center/9px no-repeat}.mw-parser-output .id-lock-limited a,.mw-parser-output .id-lock-registration a,.mw-parser-output .citation .cs1-lock-limited a,.mw-parser-output .citation .cs1-lock-registration a{background:url(\"//upload.wikimedia.org/wikipedia/commons/d/d6/Lock-gray-alt-2.svg\")right 0.1em center/9px no-repeat}.mw-parser-output .id-lock-subscription a,.mw-parser-output .citation .cs1-lock-subscription a{background:url(\"//upload.wikimedia.org/wikipedia/commons/a/aa/Lock-red-alt-2.svg\")right 0.1em center/9px no-repeat}.mw-parser-output .cs1-ws-icon a{background:url(\"//upload.wikimedia.org/wikipedia/commons/4/4c/Wikisource-logo.svg\")right 0.1em center/12px no-repeat}.mw-parser-output .cs1-code{color:inherit;background:inherit;border:none;padding:inherit}.mw-parser-output .cs1-hidden-error{display:none;color:#d33}.mw-parser-output .cs1-visible-error{color:#d33}.mw-parser-output .cs1-maint{display:none;color:#3a3;margin-left:0.3em}.mw-parser-output .cs1-format{font-size:95%}.mw-parser-output .cs1-kern-left{padding-left:0.2em}.mw-parser-output .cs1-kern-right{padding-right:0.2em}.mw-parser-output .citation .mw-selflink{font-weight:inherit}</style><cite id=\"CITEREFLesk2013\" class=\"citation web cs1\">Lesk AM (26 July 2013). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.britannica.com/science/bioinformatics\">\"Bioinformatics\"</a>. <i>Encyclopaedia Britannica</i>. <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20210414103621/https://www.britannica.com/science/bioinformatics\">Archived</a> from the original on 14 April 2021<span class=\"reference-accessdate\">. Retrieved <span class=\"nowrap\">17 April</span> 2017</span>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=Encyclopaedia+Britannica&amp;rft.atitle=Bioinformatics&amp;rft.date=2013-07-26&amp;rft.aulast=Lesk&amp;rft.aufirst=AM&amp;rft_id=https%3A%2F%2Fwww.britannica.com%2Fscience%2Fbioinformatics&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-:0-2\"><span class=\"mw-cite-backlink\">^ <a href=\"#cite_ref-:0_2-0\"><sup><i><b>a</b></i></sup></a> <a href=\"#cite_ref-:0_2-1\"><sup><i><b>b</b></i></sup></a></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFSimMinaryLevitt2012\" class=\"citation journal cs1\">Sim AY, Minary P, Levitt M (June 2012). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4028509\">\"Modeling nucleic acids\"</a>. <i>Current Opinion in Structural Biology</i>. <b>22</b> (3): 273\\xe2\\x80\\x938. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1016%2Fj.sbi.2012.03.012\">10.1016/j.sbi.2012.03.012</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4028509\">4028509</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/22538125\">22538125</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Current+Opinion+in+Structural+Biology&amp;rft.atitle=Modeling+nucleic+acids&amp;rft.volume=22&amp;rft.issue=3&amp;rft.pages=273-8&amp;rft.date=2012-06&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4028509%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F22538125&amp;rft_id=info%3Adoi%2F10.1016%2Fj.sbi.2012.03.012&amp;rft.aulast=Sim&amp;rft.aufirst=AY&amp;rft.au=Minary%2C+P&amp;rft.au=Levitt%2C+M&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4028509&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-3\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-3\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFDawsonMaciejczykJankowskaBujnicki2016\" class=\"citation journal cs1\">Dawson WK, Maciejczyk M, Jankowska EJ, Bujnicki JM (July 2016). <a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1016%2Fj.ymeth.2016.04.026\">\"Coarse-grained modeling of RNA 3D structure\"</a>. <i>Methods</i>. <b>103</b>: 138\\xe2\\x80\\x9356. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1016%2Fj.ymeth.2016.04.026\">10.1016/j.ymeth.2016.04.026</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/27125734\">27125734</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Methods&amp;rft.atitle=Coarse-grained+modeling+of+RNA+3D+structure&amp;rft.volume=103&amp;rft.pages=138-56&amp;rft.date=2016-07&amp;rft_id=info%3Adoi%2F10.1016%2Fj.ymeth.2016.04.026&amp;rft_id=info%3Apmid%2F27125734&amp;rft.aulast=Dawson&amp;rft.aufirst=WK&amp;rft.au=Maciejczyk%2C+M&amp;rft.au=Jankowska%2C+EJ&amp;rft.au=Bujnicki%2C+JM&amp;rft_id=https%3A%2F%2Fdoi.org%2F10.1016%252Fj.ymeth.2016.04.026&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-4\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-4\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFKmiecikGrontKolinskiWieteska2016\" class=\"citation journal cs1\">Kmiecik S, Gront D, Kolinski M, Wieteska L, Dawid AE, Kolinski A (July 2016). <a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1021%2Facs.chemrev.6b00163\">\"Coarse-Grained Protein Models and Their Applications\"</a>. <i>Chemical Reviews</i>. <b>116</b> (14): 7898\\xe2\\x80\\x93936. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1021%2Facs.chemrev.6b00163\">10.1021/acs.chemrev.6b00163</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/27333362\">27333362</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Chemical+Reviews&amp;rft.atitle=Coarse-Grained+Protein+Models+and+Their+Applications&amp;rft.volume=116&amp;rft.issue=14&amp;rft.pages=7898-936&amp;rft.date=2016-07&amp;rft_id=info%3Adoi%2F10.1021%2Facs.chemrev.6b00163&amp;rft_id=info%3Apmid%2F27333362&amp;rft.aulast=Kmiecik&amp;rft.aufirst=S&amp;rft.au=Gront%2C+D&amp;rft.au=Kolinski%2C+M&amp;rft.au=Wieteska%2C+L&amp;rft.au=Dawid%2C+AE&amp;rft.au=Kolinski%2C+A&amp;rft_id=https%3A%2F%2Fdoi.org%2F10.1021%252Facs.chemrev.6b00163&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-5\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-5\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFWong2016\" class=\"citation book cs1\">Wong KC (2016). <i>Computational Biology and Bioinformatics: Gene Regulation</i>. CRC Press/Taylor &amp; Francis Group. <a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/9781498724975\" title=\"Special:BookSources/9781498724975\"><bdi>9781498724975</bdi></a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Computational+Biology+and+Bioinformatics%3A+Gene+Regulation&amp;rft.pub=CRC+Press%2FTaylor+%26+Francis+Group&amp;rft.date=2016&amp;rft.isbn=9781498724975&amp;rft.aulast=Wong&amp;rft.aufirst=KC&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-6\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-6\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFJoyceZhangBradleyHavranek2015\" class=\"citation journal cs1\">Joyce AP, Zhang C, Bradley P, Havranek JJ (January 2015). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4366589\">\"Structure-based modeling of protein: DNA specificity\"</a>. <i>Briefings in Functional Genomics</i>. <b>14</b> (1): 39\\xe2\\x80\\x9349. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1093%2Fbfgp%2Felu044\">10.1093/bfgp/elu044</a></span>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4366589\">4366589</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/25414269\">25414269</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Briefings+in+Functional+Genomics&amp;rft.atitle=Structure-based+modeling+of+protein%3A+DNA+specificity&amp;rft.volume=14&amp;rft.issue=1&amp;rft.pages=39-49&amp;rft.date=2015-01&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4366589%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F25414269&amp;rft_id=info%3Adoi%2F10.1093%2Fbfgp%2Felu044&amp;rft.aulast=Joyce&amp;rft.aufirst=AP&amp;rft.au=Zhang%2C+C&amp;rft.au=Bradley%2C+P&amp;rft.au=Havranek%2C+JJ&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4366589&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-7\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-7\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFSpigaDegiacomiDal_Peraro2014\" class=\"citation book cs1\">Spiga E, Degiacomi MT, Dal Peraro M (2014). \"New Strategies for Integrative Dynamic Modeling of Macromolecular Assembly\".  In Karabencheva-Christova T (ed.). <i>Biomolecular Modelling and Simulations</i>. Advances in Protein Chemistry and Structural Biology. Vol.&#160;96. Academic Press. pp.&#160;77\\xe2\\x80\\x93111. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1016%2Fbs.apcsb.2014.06.008\">10.1016/bs.apcsb.2014.06.008</a>. <a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/9780128000137\" title=\"Special:BookSources/9780128000137\"><bdi>9780128000137</bdi></a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/25443955\">25443955</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=bookitem&amp;rft.atitle=New+Strategies+for+Integrative+Dynamic+Modeling+of+Macromolecular+Assembly&amp;rft.btitle=Biomolecular+Modelling+and+Simulations&amp;rft.series=Advances+in+Protein+Chemistry+and+Structural+Biology&amp;rft.pages=77-111&amp;rft.pub=Academic+Press&amp;rft.date=2014&amp;rft_id=info%3Apmid%2F25443955&amp;rft_id=info%3Adoi%2F10.1016%2Fbs.apcsb.2014.06.008&amp;rft.isbn=9780128000137&amp;rft.aulast=Spiga&amp;rft.aufirst=E&amp;rft.au=Degiacomi%2C+MT&amp;rft.au=Dal+Peraro%2C+M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-8\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-8\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFCiemnyKurcinskiKamelKolinski2018\" class=\"citation journal cs1\">Ciemny M, Kurcinski M, Kamel K, Kolinski A, Alam N, Schueler-Furman O, Kmiecik S (August 2018). <a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1016%2Fj.drudis.2018.05.006\">\"Protein-peptide docking: opportunities and challenges\"</a>. <i>Drug Discovery Today</i>. <b>23</b> (8): 1530\\xe2\\x80\\x931537. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1016%2Fj.drudis.2018.05.006\">10.1016/j.drudis.2018.05.006</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/29733895\">29733895</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Drug+Discovery+Today&amp;rft.atitle=Protein-peptide+docking%3A+opportunities+and+challenges&amp;rft.volume=23&amp;rft.issue=8&amp;rft.pages=1530-1537&amp;rft.date=2018-08&amp;rft_id=info%3Adoi%2F10.1016%2Fj.drudis.2018.05.006&amp;rft_id=info%3Apmid%2F29733895&amp;rft.aulast=Ciemny&amp;rft.aufirst=M&amp;rft.au=Kurcinski%2C+M&amp;rft.au=Kamel%2C+K&amp;rft.au=Kolinski%2C+A&amp;rft.au=Alam%2C+N&amp;rft.au=Schueler-Furman%2C+O&amp;rft.au=Kmiecik%2C+S&amp;rft_id=https%3A%2F%2Fdoi.org%2F10.1016%252Fj.drudis.2018.05.006&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-9\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-9\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFOuzounisValencia2003\" class=\"citation journal cs1\">Ouzounis, C. A.; Valencia, A. (2003). <a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1093%2Fbioinformatics%2Fbtg309\">\"Early bioinformatics: the birth of a discipline\\xe2\\x80\\x94a personal view\"</a>. <i>Bioinformatics</i>. <b>19</b> (17): 2176\\xe2\\x80\\x932190. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1093%2Fbioinformatics%2Fbtg309\">10.1093/bioinformatics/btg309</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/14630646\">14630646</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.atitle=Early+bioinformatics%3A+the+birth+of+a+discipline%E2%80%94a+personal+view&amp;rft.volume=19&amp;rft.issue=17&amp;rft.pages=2176-2190&amp;rft.date=2003&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtg309&amp;rft_id=info%3Apmid%2F14630646&amp;rft.aulast=Ouzounis&amp;rft.aufirst=C.+A.&amp;rft.au=Valencia%2C+A.&amp;rft_id=https%3A%2F%2Fdoi.org%2F10.1093%252Fbioinformatics%252Fbtg309&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-Hogeweg2011-10\"><span class=\"mw-cite-backlink\">^ <a href=\"#cite_ref-Hogeweg2011_10-0\"><sup><i><b>a</b></i></sup></a> <a href=\"#cite_ref-Hogeweg2011_10-1\"><sup><i><b>b</b></i></sup></a></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFHogeweg2011\" class=\"citation journal cs1\">Hogeweg P (March 2011).  Searls DB (ed.). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3068925\">\"The roots of bioinformatics in theoretical biology\"</a>. <i>PLOS Computational Biology</i>. <b>7</b> (3): e1002021. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/2011PLSCB...7E2021H\">2011PLSCB...7E2021H</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1371%2Fjournal.pcbi.1002021\">10.1371/journal.pcbi.1002021</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3068925\">3068925</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/21483479\">21483479</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+Computational+Biology&amp;rft.atitle=The+roots+of+bioinformatics+in+theoretical+biology&amp;rft.volume=7&amp;rft.issue=3&amp;rft.pages=e1002021&amp;rft.date=2011-03&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3068925%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F21483479&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002021&amp;rft_id=info%3Abibcode%2F2011PLSCB...7E2021H&amp;rft.aulast=Hogeweg&amp;rft.aufirst=P&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3068925&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-11\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-11\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFHesperHogeweg1970\" class=\"citation journal cs1\">Hesper B, Hogeweg P (1970). \"Bio-informatica: een werkconcept\". <i>Kameleon</i>. <b>1</b> (6): 28\\xe2\\x80\\x9329.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Kameleon&amp;rft.atitle=Bio-informatica%3A+een+werkconcept&amp;rft.volume=1&amp;rft.issue=6&amp;rft.pages=28-29&amp;rft.date=1970&amp;rft.aulast=Hesper&amp;rft.aufirst=B&amp;rft.au=Hogeweg%2C+P&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-12\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-12\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFHesperHogeweg2021\" class=\"citation arxiv cs1\">Hesper B, Hogeweg P (2021). \"Bio-informatics: a working concept. A translation of \"Bio-informatica: een werkconcept\" by B. Hesper and P. Hogeweg\". <a href=\"/wiki/ArXiv_(identifier)\" class=\"mw-redirect\" title=\"ArXiv (identifier)\">arXiv</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://arxiv.org/abs/2111.11832v1\">2111.11832v1</a></span> [<a rel=\"nofollow\" class=\"external text\" href=\"//arxiv.org/archive/q-bio.OT\">q-bio.OT</a>].</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=preprint&amp;rft.jtitle=arXiv&amp;rft.atitle=Bio-informatics%3A+a+working+concept.+A+translation+of+%22Bio-informatica%3A+een+werkconcept%22+by+B.+Hesper+and+P.+Hogeweg&amp;rft.date=2021&amp;rft_id=info%3Aarxiv%2F2111.11832v1&amp;rft.aulast=Hesper&amp;rft.aufirst=B&amp;rft.au=Hogeweg%2C+P&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-13\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-13\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFHogeweg1978\" class=\"citation journal cs1\">Hogeweg P (1978). \"Simulating the growth of cellular forms\". <i>Simulation</i>. <b>31</b> (3): 90\\xe2\\x80\\x9396. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1177%2F003754977803100305\">10.1177/003754977803100305</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:61206099\">61206099</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Simulation&amp;rft.atitle=Simulating+the+growth+of+cellular+forms&amp;rft.volume=31&amp;rft.issue=3&amp;rft.pages=90-96&amp;rft.date=1978&amp;rft_id=info%3Adoi%2F10.1177%2F003754977803100305&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A61206099%23id-name%3DS2CID&amp;rft.aulast=Hogeweg&amp;rft.aufirst=P&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-14\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-14\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFColby2022\" class=\"citation web cs1\">Colby B (2022). <a rel=\"nofollow\" class=\"external text\" href=\"https://sequencing.com/education-center/whole-genome-sequencing/whole-genome-sequencing-cost\">\"Whole Genome Sequencing Cost\"</a>. <i>Sequencing.com</i>. <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20220315025036/https://sequencing.com/education-center/whole-genome-sequencing/whole-genome-sequencing-cost\">Archived</a> from the original on 15 March 2022<span class=\"reference-accessdate\">. Retrieved <span class=\"nowrap\">8 April</span> 2022</span>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=Sequencing.com&amp;rft.atitle=Whole+Genome+Sequencing+Cost&amp;rft.date=2022&amp;rft.aulast=Colby&amp;rft.aufirst=B&amp;rft_id=https%3A%2F%2Fsequencing.com%2Feducation-center%2Fwhole-genome-sequencing%2Fwhole-genome-sequencing-cost&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-15\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-15\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFMoody2004\" class=\"citation book cs1\">Moody G (2004). <span class=\"cs1-lock-registration\" title=\"Free registration required\"><a rel=\"nofollow\" class=\"external text\" href=\"https://archive.org/details/digitalcodeoflif0000mood\"><i>Digital Code of Life: How Bioinformatics is Revolutionizing Science, Medicine, and Business</i></a></span>. <a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/978-0-471-32788-2\" title=\"Special:BookSources/978-0-471-32788-2\"><bdi>978-0-471-32788-2</bdi></a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Digital+Code+of+Life%3A+How+Bioinformatics+is+Revolutionizing+Science%2C+Medicine%2C+and+Business&amp;rft.date=2004&amp;rft.isbn=978-0-471-32788-2&amp;rft.aulast=Moody&amp;rft.aufirst=G&amp;rft_id=https%3A%2F%2Farchive.org%2Fdetails%2Fdigitalcodeoflif0000mood&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-16\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-16\">^</a></b></span> <span class=\"reference-text\">Dayhoff, M.O. (1966) Atlas of protein sequence and structure. National Biomedical Research Foundation, 215 pp.</span>\\n</li>\\n<li id=\"cite_note-pmid17775169-17\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-pmid17775169_17-0\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFEckDayhoff1966\" class=\"citation journal cs1\">Eck RV, Dayhoff MO (April 1966). \"Evolution of the structure of ferredoxin based on living relics of primitive amino Acid sequences\". <i>Science</i>. <b>152</b> (3720): 363\\xe2\\x80\\x936. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/1966Sci...152..363E\">1966Sci...152..363E</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1126%2Fscience.152.3720.363\">10.1126/science.152.3720.363</a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/17775169\">17775169</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:23208558\">23208558</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=Evolution+of+the+structure+of+ferredoxin+based+on+living+relics+of+primitive+amino+Acid+sequences&amp;rft.volume=152&amp;rft.issue=3720&amp;rft.pages=363-6&amp;rft.date=1966-04&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.152.3720.363&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A23208558%23id-name%3DS2CID&amp;rft_id=info%3Apmid%2F17775169&amp;rft_id=info%3Abibcode%2F1966Sci...152..363E&amp;rft.aulast=Eck&amp;rft.aufirst=RV&amp;rft.au=Dayhoff%2C+MO&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-18\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-18\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFJohnsonWu2000\" class=\"citation journal cs1\">Johnson G, Wu TT (January 2000). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC102431\">\"Kabat database and its applications: 30 years after the first variability plot\"</a>. <i>Nucleic Acids Research</i>. <b>28</b> (1): 214\\xe2\\x80\\x938. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1093%2Fnar%2F28.1.214\">10.1093/nar/28.1.214</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC102431\">102431</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/10592229\">10592229</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=Kabat+database+and+its+applications%3A+30+years+after+the+first+variability+plot&amp;rft.volume=28&amp;rft.issue=1&amp;rft.pages=214-8&amp;rft.date=2000-01&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC102431%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F10592229&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F28.1.214&amp;rft.aulast=Johnson&amp;rft.aufirst=G&amp;rft.au=Wu%2C+TT&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC102431&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-19\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-19\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFEricksonAltman1979\" class=\"citation journal cs1\">Erickson JW, Altman GG (1979). \"A Search for Patterns in the Nucleotide Sequence of the MS2 Genome\". <i>Journal of Mathematical Biology</i>. <b>7</b> (3): 219\\xe2\\x80\\x93230. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1007%2FBF00275725\">10.1007/BF00275725</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:85199492\">85199492</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Mathematical+Biology&amp;rft.atitle=A+Search+for+Patterns+in+the+Nucleotide+Sequence+of+the+MS2+Genome&amp;rft.volume=7&amp;rft.issue=3&amp;rft.pages=219-230&amp;rft.date=1979&amp;rft_id=info%3Adoi%2F10.1007%2FBF00275725&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A85199492%23id-name%3DS2CID&amp;rft.aulast=Erickson&amp;rft.aufirst=JW&amp;rft.au=Altman%2C+GG&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-20\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-20\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFShulmanSteinbergWestmoreland1981\" class=\"citation journal cs1\">Shulman MJ, Steinberg CM, Westmoreland N (February 1981). \"The coding function of nucleotide sequences can be discerned by statistical analysis\". <i>Journal of Theoretical Biology</i>. <b>88</b> (3): 409\\xe2\\x80\\x9320. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/1981JThBi..88..409S\">1981JThBi..88..409S</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1016%2F0022-5193%2881%2990274-5\">10.1016/0022-5193(81)90274-5</a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/6456380\">6456380</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Theoretical+Biology&amp;rft.atitle=The+coding+function+of+nucleotide+sequences+can+be+discerned+by+statistical+analysis&amp;rft.volume=88&amp;rft.issue=3&amp;rft.pages=409-20&amp;rft.date=1981-02&amp;rft_id=info%3Apmid%2F6456380&amp;rft_id=info%3Adoi%2F10.1016%2F0022-5193%2881%2990274-5&amp;rft_id=info%3Abibcode%2F1981JThBi..88..409S&amp;rft.aulast=Shulman&amp;rft.aufirst=MJ&amp;rft.au=Steinberg%2C+CM&amp;rft.au=Westmoreland%2C+N&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-21\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-21\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFXiong2006\" class=\"citation book cs1\">Xiong J (2006). <span class=\"cs1-lock-limited\" title=\"Free access subject to limited trial, subscription normally required\"><a rel=\"nofollow\" class=\"external text\" href=\"https://archive.org/details/essentialbioinfo00xion\"><i>Essential Bioinformatics</i></a></span>. Cambridge, United Kingdom: Cambridge University Press. pp.&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://archive.org/details/essentialbioinfo00xion/page/n13\">4</a>. <a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/978-0-511-16815-4\" title=\"Special:BookSources/978-0-511-16815-4\"><bdi>978-0-511-16815-4</bdi></a> &#8211; via Internet Archive.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Essential+Bioinformatics&amp;rft.place=Cambridge%2C+United+Kingdom&amp;rft.pages=4&amp;rft.pub=Cambridge+University+Press&amp;rft.date=2006&amp;rft.isbn=978-0-511-16815-4&amp;rft.aulast=Xiong&amp;rft.aufirst=J&amp;rft_id=https%3A%2F%2Farchive.org%2Fdetails%2Fessentialbioinfo00xion&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-pmid870828-22\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-pmid870828_22-0\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFSangerAirBarrellBrown1977\" class=\"citation journal cs1\">Sanger F, Air GM, Barrell BG, Brown NL, Coulson AR, Fiddes CA,  et&#160;al. (February 1977). \"Nucleotide sequence of bacteriophage phi X174 DNA\". <i>Nature</i>. <b>265</b> (5596): 687\\xe2\\x80\\x9395. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/1977Natur.265..687S\">1977Natur.265..687S</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2F265687a0\">10.1038/265687a0</a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/870828\">870828</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:4206886\">4206886</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Nucleotide+sequence+of+bacteriophage+phi+X174+DNA&amp;rft.volume=265&amp;rft.issue=5596&amp;rft.pages=687-95&amp;rft.date=1977-02&amp;rft_id=info%3Adoi%2F10.1038%2F265687a0&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A4206886%23id-name%3DS2CID&amp;rft_id=info%3Apmid%2F870828&amp;rft_id=info%3Abibcode%2F1977Natur.265..687S&amp;rft.aulast=Sanger&amp;rft.aufirst=F&amp;rft.au=Air%2C+GM&amp;rft.au=Barrell%2C+BG&amp;rft.au=Brown%2C+NL&amp;rft.au=Coulson%2C+AR&amp;rft.au=Fiddes%2C+CA&amp;rft.au=Hutchison%2C+CA&amp;rft.au=Slocombe%2C+PM&amp;rft.au=Smith%2C+M&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-pmid18073190-23\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-pmid18073190_23-0\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFBensonKarsch-MizrachiLipmanOstell2008\" class=\"citation journal cs1\">Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (January 2008). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2238942\">\"GenBank\"</a>. <i>Nucleic Acids Research</i>. <b>36</b> (Database issue): D25-30. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1093%2Fnar%2Fgkm929\">10.1093/nar/gkm929</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2238942\">2238942</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/18073190\">18073190</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=GenBank&amp;rft.volume=36&amp;rft.issue=Database+issue&amp;rft.pages=D25-30&amp;rft.date=2008-01&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2238942%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F18073190&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkm929&amp;rft.aulast=Benson&amp;rft.aufirst=DA&amp;rft.au=Karsch-Mizrachi%2C+I&amp;rft.au=Lipman%2C+DJ&amp;rft.au=Ostell%2C+J&amp;rft.au=Wheeler%2C+DL&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2238942&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-pmid7542800-24\"><span class=\"mw-cite-backlink\">^ <a href=\"#cite_ref-pmid7542800_24-0\"><sup><i><b>a</b></i></sup></a> <a href=\"#cite_ref-pmid7542800_24-1\"><sup><i><b>b</b></i></sup></a> <a href=\"#cite_ref-pmid7542800_24-2\"><sup><i><b>c</b></i></sup></a></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFFleischmannAdamsWhiteClayton1995\" class=\"citation journal cs1\">Fleischmann RD, Adams MD, White O, Clayton RA, Kirkness EF, Kerlavage AR,  et&#160;al. (July 1995). \"Whole-genome random sequencing and assembly of Haemophilus influenzae Rd\". <i>Science</i>. <b>269</b> (5223): 496\\xe2\\x80\\x93512. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/1995Sci...269..496F\">1995Sci...269..496F</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1126%2Fscience.7542800\">10.1126/science.7542800</a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/7542800\">7542800</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=Whole-genome+random+sequencing+and+assembly+of+Haemophilus+influenzae+Rd&amp;rft.volume=269&amp;rft.issue=5223&amp;rft.pages=496-512&amp;rft.date=1995-07&amp;rft_id=info%3Apmid%2F7542800&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.7542800&amp;rft_id=info%3Abibcode%2F1995Sci...269..496F&amp;rft.aulast=Fleischmann&amp;rft.aufirst=RD&amp;rft.au=Adams%2C+MD&amp;rft.au=White%2C+O&amp;rft.au=Clayton%2C+RA&amp;rft.au=Kirkness%2C+EF&amp;rft.au=Kerlavage%2C+AR&amp;rft.au=Bult%2C+CJ&amp;rft.au=Tomb%2C+JF&amp;rft.au=Dougherty%2C+BA&amp;rft.au=Merrick%2C+JM&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-25\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-25\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFStein2001\" class=\"citation journal cs1\">Stein, Lincoln (2001). \"Genome annotation: from sequence to biology\". <i>Nature</i>. <b>2</b> (7): 493\\xe2\\x80\\x93503. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2F35080529\">10.1038/35080529</a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/11433356\">11433356</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:12044602\">12044602</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Genome+annotation%3A+from+sequence+to+biology&amp;rft.volume=2&amp;rft.issue=7&amp;rft.pages=493-503&amp;rft.date=2001&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A12044602%23id-name%3DS2CID&amp;rft_id=info%3Apmid%2F11433356&amp;rft_id=info%3Adoi%2F10.1038%2F35080529&amp;rft.aulast=Stein&amp;rft.aufirst=Lincoln&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-26\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-26\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFErdinLisewskiLichtarge2011\" class=\"citation journal cs1\">Erdin S, Lisewski AM, Lichtarge O (April 2011). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3120633\">\"Protein function prediction: towards integration of similarity metrics\"</a>. <i>Current Opinion in Structural Biology</i>. <b>21</b> (2): 180\\xe2\\x80\\x938. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1016%2Fj.sbi.2011.02.001\">10.1016/j.sbi.2011.02.001</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3120633\">3120633</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/21353529\">21353529</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Current+Opinion+in+Structural+Biology&amp;rft.atitle=Protein+function+prediction%3A+towards+integration+of+similarity+metrics&amp;rft.volume=21&amp;rft.issue=2&amp;rft.pages=180-8&amp;rft.date=2011-04&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3120633%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F21353529&amp;rft_id=info%3Adoi%2F10.1016%2Fj.sbi.2011.02.001&amp;rft.aulast=Erdin&amp;rft.aufirst=S&amp;rft.au=Lisewski%2C+AM&amp;rft.au=Lichtarge%2C+O&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3120633&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-27\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-27\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFCarvajal-Rodr\\xc3\\xadguez2010\" class=\"citation journal cs1\">Carvajal-Rodr\\xc3\\xadguez A (March 2010). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2851118\">\"Simulation of genes and genomes forward in time\"</a>. <i>Current Genomics</i>. <b>11</b> (1): 58\\xe2\\x80\\x9361. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.2174%2F138920210790218007\">10.2174/138920210790218007</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2851118\">2851118</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/20808525\">20808525</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Current+Genomics&amp;rft.atitle=Simulation+of+genes+and+genomes+forward+in+time&amp;rft.volume=11&amp;rft.issue=1&amp;rft.pages=58-61&amp;rft.date=2010-03&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2851118%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F20808525&amp;rft_id=info%3Adoi%2F10.2174%2F138920210790218007&amp;rft.aulast=Carvajal-Rodr%C3%ADguez&amp;rft.aufirst=A&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2851118&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-28\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-28\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFBrown2002\" class=\"citation book cs1\">Brown TA (2002). \"Mutation, Repair and Recombination\". <i>Genomes</i> (2nd&#160;ed.). Manchester (UK): Oxford.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=bookitem&amp;rft.atitle=Mutation%2C+Repair+and+Recombination&amp;rft.btitle=Genomes&amp;rft.place=Manchester+%28UK%29&amp;rft.edition=2nd&amp;rft.pub=Oxford&amp;rft.date=2002&amp;rft.aulast=Brown&amp;rft.aufirst=TA&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-29\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-29\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFCarterFieglerPiper2002\" class=\"citation journal cs1\">Carter NP, Fiegler H, Piper J (October 2002). \"Comparative analysis of comparative genomic hybridization microarray technologies: report of a workshop sponsored by the Wellcome Trust\". <i>Cytometry</i>. <b>49</b> (2): 43\\xe2\\x80\\x938. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1002%2Fcyto.10153\">10.1002/cyto.10153</a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/12357458\">12357458</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Cytometry&amp;rft.atitle=Comparative+analysis+of+comparative+genomic+hybridization+microarray+technologies%3A+report+of+a+workshop+sponsored+by+the+Wellcome+Trust&amp;rft.volume=49&amp;rft.issue=2&amp;rft.pages=43-8&amp;rft.date=2002-10&amp;rft_id=info%3Adoi%2F10.1002%2Fcyto.10153&amp;rft_id=info%3Apmid%2F12357458&amp;rft.aulast=Carter&amp;rft.aufirst=NP&amp;rft.au=Fiegler%2C+H&amp;rft.au=Piper%2C+J&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-30\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-30\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFChaudhariGuptaDutta2016\" class=\"citation journal cs1\">Chaudhari NM, Gupta VK, Dutta C (April 2016). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4829868\">\"BPGA- an ultra-fast pan-genome analysis pipeline\"</a>. <i>Scientific Reports</i>. <b>6</b>: 24373. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/2016NatSR...624373C\">2016NatSR...624373C</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2Fsrep24373\">10.1038/srep24373</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4829868\">4829868</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/27071527\">27071527</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Scientific+Reports&amp;rft.atitle=BPGA-+an+ultra-fast+pan-genome+analysis+pipeline&amp;rft.volume=6&amp;rft.pages=24373&amp;rft.date=2016-04&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4829868%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F27071527&amp;rft_id=info%3Adoi%2F10.1038%2Fsrep24373&amp;rft_id=info%3Abibcode%2F2016NatSR...624373C&amp;rft.aulast=Chaudhari&amp;rft.aufirst=NM&amp;rft.au=Gupta%2C+VK&amp;rft.au=Dutta%2C+C&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4829868&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-Demerec1945-31\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-Demerec1945_31-0\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFAston2014\" class=\"citation journal cs1\">Aston KI (May 2014). <a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1111%2Fj.2047-2927.2014.00188.x\">\"Genetic susceptibility to male infertility: news from genome-wide association studies\"</a>. <i>Andrology</i>. <b>2</b> (3): 315\\xe2\\x80\\x9321. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1111%2Fj.2047-2927.2014.00188.x\">10.1111/j.2047-2927.2014.00188.x</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/24574159\">24574159</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:206007180\">206007180</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Andrology&amp;rft.atitle=Genetic+susceptibility+to+male+infertility%3A+news+from+genome-wide+association+studies&amp;rft.volume=2&amp;rft.issue=3&amp;rft.pages=315-21&amp;rft.date=2014-05&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A206007180%23id-name%3DS2CID&amp;rft_id=info%3Apmid%2F24574159&amp;rft_id=info%3Adoi%2F10.1111%2Fj.2047-2927.2014.00188.x&amp;rft.aulast=Aston&amp;rft.aufirst=KI&amp;rft_id=https%3A%2F%2Fdoi.org%2F10.1111%252Fj.2047-2927.2014.00188.x&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-V\\xc3\\xa9ron2013-32\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-V\\xc3\\xa9ron2013_32-0\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFV\\xc3\\xa9ronBleinCox2014\" class=\"citation journal cs1\">V\\xc3\\xa9ron A, Blein S, Cox DG (2014). \"Genome-wide association studies and the clinic: a focus on breast cancer\". <i>Biomarkers in Medicine</i>. <b>8</b> (2): 287\\xe2\\x80\\x9396. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.2217%2Fbmm.13.121\">10.2217/bmm.13.121</a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/24521025\">24521025</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biomarkers+in+Medicine&amp;rft.atitle=Genome-wide+association+studies+and+the+clinic%3A+a+focus+on+breast+cancer&amp;rft.volume=8&amp;rft.issue=2&amp;rft.pages=287-96&amp;rft.date=2014&amp;rft_id=info%3Adoi%2F10.2217%2Fbmm.13.121&amp;rft_id=info%3Apmid%2F24521025&amp;rft.aulast=V%C3%A9ron&amp;rft.aufirst=A&amp;rft.au=Blein%2C+S&amp;rft.au=Cox%2C+DG&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-Tosto2013-33\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-Tosto2013_33-0\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFTostoReitz2013\" class=\"citation journal cs1\">Tosto G, Reitz C (October 2013). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3809844\">\"Genome-wide association studies in Alzheimer\\'s disease: a review\"</a>. <i>Current Neurology and Neuroscience Reports</i>. <b>13</b> (10): 381. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1007%2Fs11910-013-0381-0\">10.1007/s11910-013-0381-0</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3809844\">3809844</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/23954969\">23954969</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Current+Neurology+and+Neuroscience+Reports&amp;rft.atitle=Genome-wide+association+studies+in+Alzheimer%27s+disease%3A+a+review&amp;rft.volume=13&amp;rft.issue=10&amp;rft.pages=381&amp;rft.date=2013-10&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3809844%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F23954969&amp;rft_id=info%3Adoi%2F10.1007%2Fs11910-013-0381-0&amp;rft.aulast=Tosto&amp;rft.aufirst=G&amp;rft.au=Reitz%2C+C&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3809844&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-Londin2013-34\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-Londin2013_34-0\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFLondinYadavSurreyKricka2013\" class=\"citation book cs1\">Londin E, Yadav P, Surrey S, Kricka LJ, Fortina P (2013). \"Use of linkage analysis, genome-wide association studies, and next-generation sequencing in the identification of disease-causing mutations\". <i>Pharmacogenomics</i>. Methods in Molecular Biology. Vol.&#160;1015. pp.&#160;127\\xe2\\x80\\x9346. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1007%2F978-1-62703-435-7_8\">10.1007/978-1-62703-435-7_8</a>. <a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/978-1-62703-434-0\" title=\"Special:BookSources/978-1-62703-434-0\"><bdi>978-1-62703-434-0</bdi></a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/23824853\">23824853</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=bookitem&amp;rft.atitle=Use+of+linkage+analysis%2C+genome-wide+association+studies%2C+and+next-generation+sequencing+in+the+identification+of+disease-causing+mutations&amp;rft.btitle=Pharmacogenomics&amp;rft.series=Methods+in+Molecular+Biology&amp;rft.pages=127-46&amp;rft.date=2013&amp;rft_id=info%3Apmid%2F23824853&amp;rft_id=info%3Adoi%2F10.1007%2F978-1-62703-435-7_8&amp;rft.isbn=978-1-62703-434-0&amp;rft.aulast=Londin&amp;rft.aufirst=E&amp;rft.au=Yadav%2C+P&amp;rft.au=Surrey%2C+S&amp;rft.au=Kricka%2C+LJ&amp;rft.au=Fortina%2C+P&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-35\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-35\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFHindorffSethupathyJunkinsRamos2009\" class=\"citation journal cs1\">Hindorff LA, Sethupathy P, Junkins HA, Ramos EM, Mehta JP, Collins FS, Manolio TA (June 2009). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687147\">\"Potential etiologic and functional implications of genome-wide association loci for human diseases and traits\"</a>. <i>Proceedings of the National Academy of Sciences of the United States of America</i>. <b>106</b> (23): 9362\\xe2\\x80\\x937. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/2009PNAS..106.9362H\">2009PNAS..106.9362H</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1073%2Fpnas.0903103106\">10.1073/pnas.0903103106</a></span>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2687147\">2687147</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/19474294\">19474294</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;rft.atitle=Potential+etiologic+and+functional+implications+of+genome-wide+association+loci+for+human+diseases+and+traits&amp;rft.volume=106&amp;rft.issue=23&amp;rft.pages=9362-7&amp;rft.date=2009-06&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687147%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F19474294&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0903103106&amp;rft_id=info%3Abibcode%2F2009PNAS..106.9362H&amp;rft.aulast=Hindorff&amp;rft.aufirst=LA&amp;rft.au=Sethupathy%2C+P&amp;rft.au=Junkins%2C+HA&amp;rft.au=Ramos%2C+EM&amp;rft.au=Mehta%2C+JP&amp;rft.au=Collins%2C+FS&amp;rft.au=Manolio%2C+TA&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2687147&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-36\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-36\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFHall2010\" class=\"citation book cs1\">Hall LO (2010). \"Finding the right genes for disease and prognosis prediction\". <i>2010 International Conference on System Science and Engineering</i>. <i>System Science and Engineering (ICSSE),2010 International Conference</i>. pp.&#160;1\\xe2\\x80\\x932. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1109%2FICSSE.2010.5551766\">10.1109/ICSSE.2010.5551766</a>. <a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/978-1-4244-6472-2\" title=\"Special:BookSources/978-1-4244-6472-2\"><bdi>978-1-4244-6472-2</bdi></a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:21622726\">21622726</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=bookitem&amp;rft.atitle=Finding+the+right+genes+for+disease+and+prognosis+prediction&amp;rft.btitle=2010+International+Conference+on+System+Science+and+Engineering&amp;rft.pages=1-2&amp;rft.date=2010&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A21622726%23id-name%3DS2CID&amp;rft_id=info%3Adoi%2F10.1109%2FICSSE.2010.5551766&amp;rft.isbn=978-1-4244-6472-2&amp;rft.aulast=Hall&amp;rft.aufirst=LO&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-37\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-37\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFManolioCollinsCoxGoldstein2009\" class=\"citation journal cs1\">Manolio, Teri A.; Collins, Francis S.; Cox, Nancy J.;  et&#160;al. (October 2009). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2831613\">\"Finding the missing heritability of complex diseases\"</a>. <i>Nature</i>. <b>461</b> (7265): 747\\xe2\\x80\\x93753. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/2009Natur.461..747M\">2009Natur.461..747M</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2Fnature08494\">10.1038/nature08494</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2831613\">2831613</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/19812666\">19812666</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Finding+the+missing+heritability+of+complex+diseases&amp;rft.volume=461&amp;rft.issue=7265&amp;rft.pages=747-753&amp;rft.date=2009-10&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2831613%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F19812666&amp;rft_id=info%3Adoi%2F10.1038%2Fnature08494&amp;rft_id=info%3Abibcode%2F2009Natur.461..747M&amp;rft.aulast=Manolio&amp;rft.aufirst=Teri+A.&amp;rft.au=Collins%2C+Francis+S.&amp;rft.au=Cox%2C+Nancy+J.&amp;rft.au=Goldstein%2C+David+B.&amp;rft.au=Hindorff%2C+Lucia+A.&amp;rft.au=Hunter%2C+David+J.&amp;rft.au=McCarthy%2C+Mark+I.&amp;rft.au=Ramos%2C+Erin+M.&amp;rft.au=Cardon%2C+Lon+R.&amp;rft.au=Chakravarti%2C+Aravinda&amp;rft.au=Cho%2C+Judy+H.&amp;rft.au=Guttmacher%2C+Alan+E.&amp;rft.au=Kong%2C+Augustine&amp;rft.au=Kruglyak%2C+Leonid&amp;rft.au=Mardis%2C+Elaine&amp;rft.au=Rotimi%2C+Charles+N.&amp;rft.au=Slatkin%2C+Montgomery&amp;rft.au=Valle%2C+David&amp;rft.au=Whittemore%2C+Alice+S.&amp;rft.au=Boehnke%2C+Michael&amp;rft.au=Clark%2C+Andrew+G.&amp;rft.au=Eichler%2C+Evan+E.&amp;rft.au=Gibson%2C+Greg&amp;rft.au=Haines%2C+Jonathan+L.&amp;rft.au=Mackay%2C+Trudy+F.+C.&amp;rft.au=McCarroll%2C+Steven+A.&amp;rft.au=Visscher%2C+Peter+M.&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2831613&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-38\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-38\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFWainschteinJainZhengAslibekyan2022\" class=\"citation journal cs1\">Wainschtein, Pierrick; Jain, Deepti; Zheng, Zhili;  et&#160;al. (March 2022). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9119698\">\"Assessing the contribution of rare variants to complex trait heritability from whole-genome sequence data\"</a>. <i>Nature Genetics</i>. <b>54</b> (3): 263\\xe2\\x80\\x93273. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2Fs41588-021-00997-7\">10.1038/s41588-021-00997-7</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9119698\">9119698</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/35256806\">35256806</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Genetics&amp;rft.atitle=Assessing+the+contribution+of+rare+variants+to+complex+trait+heritability+from+whole-genome+sequence+data&amp;rft.volume=54&amp;rft.issue=3&amp;rft.pages=263-273&amp;rft.date=2022-03&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9119698%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F35256806&amp;rft_id=info%3Adoi%2F10.1038%2Fs41588-021-00997-7&amp;rft.aulast=Wainschtein&amp;rft.aufirst=Pierrick&amp;rft.au=Jain%2C+Deepti&amp;rft.au=Zheng%2C+Zhili&amp;rft.au=Aslibekyan%2C+Stella&amp;rft.au=Becker%2C+Diane&amp;rft.au=Bi%2C+Wenjian&amp;rft.au=Brody%2C+Jennifer&amp;rft.au=Carlson%2C+Jenna+C.&amp;rft.au=Correa%2C+Adolfo&amp;rft.au=Du%2C+Margaret+Mengmeng&amp;rft.au=Fernandez-Rhodes%2C+Lindsay&amp;rft.au=Ferrier%2C+Kendra+R.&amp;rft.au=Graff%2C+Misa&amp;rft.au=Guo%2C+Xiuqing&amp;rft.au=He%2C+Jiang&amp;rft.au=Heard-Costa%2C+Nancy+L.&amp;rft.au=Highland%2C+Heather+M.&amp;rft.au=Hirschhorn%2C+Joel+N.&amp;rft.au=Howard-Claudio%2C+Candace+M.&amp;rft.au=Isasi%2C+Carmen+R.&amp;rft.au=Jackson%2C+Rebecca&amp;rft.au=Jiang%2C+Jicai&amp;rft.au=Joehanes%2C+Roby&amp;rft.au=Justice%2C+Anne+E.&amp;rft.au=Kalyani%2C+Rita+R.&amp;rft.au=Kardia%2C+Sharon&amp;rft.au=Lange%2C+Ethan&amp;rft.au=LeBoff%2C+Meryl&amp;rft.au=Lee%2C+Seunggeun&amp;rft.au=Li%2C+Xihao&amp;rft.au=Li%2C+Zilin&amp;rft.au=Lim%2C+Elise&amp;rft.au=Lin%2C+Danyu&amp;rft.au=Lin%2C+Xihong&amp;rft.au=Liu%2C+Simin&amp;rft.au=Lu%2C+Yingchang&amp;rft.au=Manson%2C+JoAnn&amp;rft.au=Martin%2C+Lisa&amp;rft.au=McHugh%2C+Caitlin&amp;rft.au=Mikulla%2C+Julie&amp;rft.au=Musani%2C+Solomon+K.&amp;rft.au=Ng%2C+Maggie&amp;rft.au=Nickerson%2C+Deborah&amp;rft.au=Palmer%2C+Nicholette&amp;rft.au=Perry%2C+James&amp;rft.au=Peters%2C+Ulrike&amp;rft.au=Preuss%2C+Michael&amp;rft.au=Qi%2C+Qibin&amp;rft.au=Raffield%2C+Laura&amp;rft.au=Rasmussen-Torvik%2C+Laura&amp;rft.au=Reiner%2C+Alex&amp;rft.au=Russell%2C+Emily+M.&amp;rft.au=Sitlani%2C+Colleen&amp;rft.au=Smith%2C+Jennifer&amp;rft.au=Spracklen%2C+Cassandra+N.&amp;rft.au=Wang%2C+Tao&amp;rft.au=Wang%2C+Zhe&amp;rft.au=Wessel%2C+Jennifer&amp;rft.au=Xu%2C+Hanfei&amp;rft.au=Yaser%2C+Mohammad&amp;rft.au=Yoneyama%2C+Sachiko&amp;rft.au=Young%2C+Kendra+A.&amp;rft.au=Zhang%2C+Jingwen&amp;rft.au=Zhang%2C+Xinruo&amp;rft.au=Zhou%2C+Hufeng&amp;rft.au=Zhu%2C+Xiaofeng&amp;rft.au=Zoellner%2C+Sebastian&amp;rft.au=Abe%2C+Namiko&amp;rft.au=Abecasis%2C+Gon%C3%A7alo&amp;rft.au=Aguet%2C+Francois&amp;rft.au=Almasy%2C+Laura&amp;rft.au=Alonso%2C+Alvaro&amp;rft.au=Ament%2C+Seth&amp;rft.au=Anderson%2C+Peter&amp;rft.au=Anugu%2C+Pramod&amp;rft.au=Applebaum-Bowden%2C+Deborah&amp;rft.au=Ardlie%2C+Kristin&amp;rft.au=Arking%2C+Dan&amp;rft.au=Ashley-Koch%2C+Allison&amp;rft.au=Assimes%2C+Tim&amp;rft.au=Auer%2C+Paul&amp;rft.au=Avramopoulos%2C+Dimitrios&amp;rft.au=Ayas%2C+Najib&amp;rft.au=Balasubramanian%2C+Adithya&amp;rft.au=Barnard%2C+John&amp;rft.au=Barnes%2C+Kathleen&amp;rft.au=Barr%2C+R.+Graham&amp;rft.au=Barron-Casella%2C+Emily&amp;rft.au=Barwick%2C+Lucas&amp;rft.au=Beaty%2C+Terri&amp;rft.au=Beck%2C+Gerald&amp;rft.au=Becker%2C+Lewis&amp;rft.au=Beer%2C+Rebecca&amp;rft.au=Beitelshees%2C+Amber&amp;rft.au=Benjamin%2C+Emelia&amp;rft.au=Benos%2C+Takis&amp;rft.au=Bezerra%2C+Marcos&amp;rft.au=Bielak%2C+Larry&amp;rft.au=Bis%2C+Joshua&amp;rft.au=Blackwell%2C+Thomas&amp;rft.au=Blangero%2C+John&amp;rft.au=Bowden%2C+Donald+W.&amp;rft.au=Bowler%2C+Russell&amp;rft.au=Broeckel%2C+Ulrich&amp;rft.au=Broome%2C+Jai&amp;rft.au=Brown%2C+Deborah&amp;rft.au=Bunting%2C+Karen&amp;rft.au=Burchard%2C+Esteban&amp;rft.au=Bustamante%2C+Carlos&amp;rft.au=Buth%2C+Erin&amp;rft.au=Cade%2C+Brian&amp;rft.au=Cardwell%2C+Jonathan&amp;rft.au=Carey%2C+Vincent&amp;rft.au=Carrier%2C+Julie&amp;rft.au=Carson%2C+April&amp;rft.au=Carty%2C+Cara&amp;rft.au=Casaburi%2C+Richard&amp;rft.au=Romero%2C+Juan+P.+Casas&amp;rft.au=Casella%2C+James&amp;rft.au=Castaldi%2C+Peter&amp;rft.au=Chaffin%2C+Mark&amp;rft.au=Chang%2C+Christy&amp;rft.au=Chang%2C+Yi-Cheng&amp;rft.au=Chavan%2C+Sameer&amp;rft.au=Chen%2C+Bo-Juen&amp;rft.au=Chen%2C+Wei-Min&amp;rft.au=Cho%2C+Michael&amp;rft.au=Choi%2C+Seung+Hoan&amp;rft.au=Chuang%2C+Lee-Ming&amp;rft.au=Chung%2C+Ren-Hua&amp;rft.au=Clish%2C+Clary&amp;rft.au=Comhair%2C+Suzy&amp;rft.au=Conomos%2C+Matthew&amp;rft.au=Cornell%2C+Elaine&amp;rft.au=Crandall%2C+Carolyn&amp;rft.au=Crapo%2C+James&amp;rft.au=Curran%2C+Joanne&amp;rft.au=Curtis%2C+Jeffrey&amp;rft.au=Custer%2C+Brian&amp;rft.au=Damcott%2C+Coleen&amp;rft.au=Darbar%2C+Dawood&amp;rft.au=David%2C+Sean&amp;rft.au=Davis%2C+Colleen&amp;rft.au=Daya%2C+Michelle&amp;rft.au=de+las+Fuentes%2C+Lisa&amp;rft.au=de+Vries%2C+Paul&amp;rft.au=DeBaun%2C+Michael&amp;rft.au=Deka%2C+Ranjan&amp;rft.au=DeMeo%2C+Dawn&amp;rft.au=Devine%2C+Scott&amp;rft.au=Dinh%2C+Huyen&amp;rft.au=Doddapaneni%2C+Harsha&amp;rft.au=Duan%2C+Qing&amp;rft.au=Dugan-Perez%2C+Shannon&amp;rft.au=Duggirala%2C+Ravi&amp;rft.au=Durda%2C+Jon+Peter&amp;rft.au=Dutcher%2C+Susan+K.&amp;rft.au=Eaton%2C+Charles&amp;rft.au=Ekunwe%2C+Lynette&amp;rft.au=El+Boueiz%2C+Adel&amp;rft.au=Emery%2C+Leslie&amp;rft.au=Erzurum%2C+Serpil&amp;rft.au=Farber%2C+Charles&amp;rft.au=Farek%2C+Jesse&amp;rft.au=Fingerlin%2C+Tasha&amp;rft.au=Flickinger%2C+Matthew&amp;rft.au=Franceschini%2C+Nora&amp;rft.au=Frazar%2C+Chris&amp;rft.au=Fu%2C+Mao&amp;rft.au=Fullerton%2C+Stephanie+M.&amp;rft.au=Fulton%2C+Lucinda&amp;rft.au=Gabriel%2C+Stacey&amp;rft.au=Gan%2C+Weiniu&amp;rft.au=Gao%2C+Shanshan&amp;rft.au=Gao%2C+Yan&amp;rft.au=Gass%2C+Margery&amp;rft.au=Geiger%2C+Heather&amp;rft.au=Gelb%2C+Bruce&amp;rft.au=Geraci%2C+Mark&amp;rft.au=Germer%2C+Soren&amp;rft.au=Gerszten%2C+Robert&amp;rft.au=Ghosh%2C+Auyon&amp;rft.au=Gibbs%2C+Richard&amp;rft.au=Gignoux%2C+Chris&amp;rft.au=Gladwin%2C+Mark&amp;rft.au=Glahn%2C+David&amp;rft.au=Gogarten%2C+Stephanie&amp;rft.au=Gong%2C+Da-Wei&amp;rft.au=Goring%2C+Harald&amp;rft.au=Graw%2C+Sharon&amp;rft.au=Gray%2C+Kathryn+J.&amp;rft.au=Grine%2C+Daniel&amp;rft.au=Gross%2C+Colin&amp;rft.au=Gu%2C+C.+Charles&amp;rft.au=Guan%2C+Yue&amp;rft.au=Gupta%2C+Namrata&amp;rft.au=Haas%2C+David+M.&amp;rft.au=Haessler%2C+Jeff&amp;rft.au=Hall%2C+Michael&amp;rft.au=Han%2C+Yi&amp;rft.au=Hanly%2C+Patrick&amp;rft.au=Harris%2C+Daniel&amp;rft.au=Hawley%2C+Nicola+L.&amp;rft.au=Heavner%2C+Ben&amp;rft.au=Herrington%2C+David&amp;rft.au=Hersh%2C+Craig&amp;rft.au=Hidalgo%2C+Bertha&amp;rft.au=Hixson%2C+James&amp;rft.au=Hobbs%2C+Brian&amp;rft.au=Hokanson%2C+John&amp;rft.au=Hong%2C+Elliott&amp;rft.au=Hoth%2C+Karin&amp;rft.au=Hsiung%2C+Chao+Agnes&amp;rft.au=Hu%2C+Jianhong&amp;rft.au=Hung%2C+Yi-Jen&amp;rft.au=Huston%2C+Haley&amp;rft.au=Hwu%2C+Chii+Min&amp;rft.au=Irvin%2C+Marguerite+Ryan&amp;rft.au=Jaquish%2C+Cashell&amp;rft.au=Johnsen%2C+Jill&amp;rft.au=Johnson%2C+Andrew&amp;rft.au=Johnson%2C+Craig&amp;rft.au=Johnston%2C+Rich&amp;rft.au=Jones%2C+Kimberly&amp;rft.au=Kang%2C+Hyun+Min&amp;rft.au=Kaplan%2C+Robert&amp;rft.au=Kelly%2C+Shannon&amp;rft.au=Kenny%2C+Eimear&amp;rft.au=Kessler%2C+Michael&amp;rft.au=Khan%2C+Alyna&amp;rft.au=Khan%2C+Ziad&amp;rft.au=Kim%2C+Wonji&amp;rft.au=Kimoff%2C+John&amp;rft.au=Kinney%2C+Greg&amp;rft.au=Konkle%2C+Barbara&amp;rft.au=Kramer%2C+Holly&amp;rft.au=Lange%2C+Christoph&amp;rft.au=Lee%2C+Jiwon&amp;rft.au=Lee%2C+Sandra&amp;rft.au=Lee%2C+Wen-Jane&amp;rft.au=LeFaive%2C+Jonathon&amp;rft.au=Levine%2C+David&amp;rft.au=Levy%2C+Dan&amp;rft.au=Lewis%2C+Joshua&amp;rft.au=Li%2C+Xiaohui&amp;rft.au=Li%2C+Yun&amp;rft.au=Lin%2C+Henry&amp;rft.au=Lin%2C+Honghuang&amp;rft.au=Liu%2C+Yongmei&amp;rft.au=Liu%2C+Yu&amp;rft.au=Lunetta%2C+Kathryn&amp;rft.au=Luo%2C+James&amp;rft.au=Magalang%2C+Ulysses&amp;rft.au=Mahaney%2C+Michael&amp;rft.au=Make%2C+Barry&amp;rft.au=Manichaikul%2C+Ani&amp;rft.au=Manning%2C+Alisa&amp;rft.au=Marton%2C+Melissa&amp;rft.au=Mathai%2C+Susan&amp;rft.au=May%2C+Susanne&amp;rft.au=McArdle%2C+Patrick&amp;rft.au=McFarland%2C+Sean&amp;rft.au=McGoldrick%2C+Daniel&amp;rft.au=McNeil%2C+Becky&amp;rft.au=Mei%2C+Hao&amp;rft.au=Meigs%2C+James&amp;rft.au=Menon%2C+Vipin&amp;rft.au=Mestroni%2C+Luisa&amp;rft.au=Metcalf%2C+Ginger&amp;rft.au=Meyers%2C+Deborah+A.&amp;rft.au=Mignot%2C+Emmanuel&amp;rft.au=Mikulla%2C+Julie&amp;rft.au=Min%2C+Nancy&amp;rft.au=Minear%2C+Mollie&amp;rft.au=Minster%2C+Ryan+L.&amp;rft.au=Moll%2C+Matt&amp;rft.au=Momin%2C+Zeineen&amp;rft.au=Montasser%2C+May+E.&amp;rft.au=Montgomery%2C+Courtney&amp;rft.au=Muzny%2C+Donna&amp;rft.au=Mychaleckyj%2C+Josyf+C.&amp;rft.au=Nadkarni%2C+Girish&amp;rft.au=Naik%2C+Rakhi&amp;rft.au=Naseri%2C+Take&amp;rft.au=Natarajan%2C+Pradeep&amp;rft.au=Nekhai%2C+Sergei&amp;rft.au=Nelson%2C+Sarah+C.&amp;rft.au=Neltner%2C+Bonnie&amp;rft.au=Nessner%2C+Caitlin&amp;rft.au=Nkechinyere%2C+Osuji&amp;rft.au=O%E2%80%99Connor%2C+Tim&amp;rft.au=Ochs-Balcom%2C+Heather&amp;rft.au=Okwuonu%2C+Geoffrey&amp;rft.au=Pack%2C+Allan&amp;rft.au=Paik%2C+David+T.&amp;rft.au=Palmer%2C+Nicholette&amp;rft.au=Pankow%2C+James&amp;rft.au=Papanicolaou%2C+George&amp;rft.au=Parker%2C+Cora&amp;rft.au=Peloso%2C+Gina&amp;rft.au=Peralta%2C+Juan+Manuel&amp;rft.au=Perez%2C+Marco&amp;rft.au=Peyser%2C+Patricia&amp;rft.au=Phillips%2C+Lawrence+S.&amp;rft.au=Pleiness%2C+Jacob&amp;rft.au=Pollin%2C+Toni&amp;rft.au=Post%2C+Wendy&amp;rft.au=Becker%2C+Julia+Powers&amp;rft.au=Boorgula%2C+Meher+Preethi&amp;rft.au=Qasba%2C+Pankaj&amp;rft.au=Qiao%2C+Dandi&amp;rft.au=Qin%2C+Zhaohui&amp;rft.au=Rafaels%2C+Nicholas&amp;rft.au=Rajendran%2C+Mahitha&amp;rft.au=Rao%2C+D.+C.&amp;rft.au=Ratan%2C+Aakrosh&amp;rft.au=Reed%2C+Robert&amp;rft.au=Reeves%2C+Catherine&amp;rft.au=Reupena%2C+Muagututi%E2%80%98a+Sefuiva&amp;rft.au=Rice%2C+Ken&amp;rft.au=Robillard%2C+Rebecca&amp;rft.au=Robine%2C+Nicolas&amp;rft.au=Roselli%2C+Carolina&amp;rft.au=Ruczinski%2C+Ingo&amp;rft.au=Runnels%2C+Alexi&amp;rft.au=Russell%2C+Pamela&amp;rft.au=Ruuska%2C+Sarah&amp;rft.au=Ryan%2C+Kathleen&amp;rft.au=Sabino%2C+Ester+Cerdeira&amp;rft.au=Saleheen%2C+Danish&amp;rft.au=Salimi%2C+Shabnam&amp;rft.au=Salvi%2C+Sejal&amp;rft.au=Salzberg%2C+Steven&amp;rft.au=Sandow%2C+Kevin&amp;rft.au=Sankaran%2C+Vijay+G.&amp;rft.au=Santibanez%2C+Jireh&amp;rft.au=Schwander%2C+Karen&amp;rft.au=Schwartz%2C+David&amp;rft.au=Sciurba%2C+Frank&amp;rft.au=Seidman%2C+Christine&amp;rft.au=Seidman%2C+Jonathan&amp;rft.au=Sheehan%2C+Vivien&amp;rft.au=Sherman%2C+Stephanie+L.&amp;rft.au=Shetty%2C+Amol&amp;rft.au=Shetty%2C+Aniket&amp;rft.au=Sheu%2C+Wayne+Hui-Heng&amp;rft.au=Silver%2C+Brian&amp;rft.au=Silverman%2C+Edwin&amp;rft.au=Skomro%2C+Robert&amp;rft.au=Smith%2C+Albert+Vernon&amp;rft.au=Smith%2C+Josh&amp;rft.au=Smith%2C+Tanja&amp;rft.au=Smoller%2C+Sylvia&amp;rft.au=Snively%2C+Beverly&amp;rft.au=Snyder%2C+Michael&amp;rft.au=Sofer%2C+Tamar&amp;rft.au=Sotoodehnia%2C+Nona&amp;rft.au=Stilp%2C+Adrienne+M.&amp;rft.au=Storm%2C+Garrett&amp;rft.au=Streeten%2C+Elizabeth&amp;rft.au=Su%2C+Jessica+Lasky&amp;rft.au=Sung%2C+Yun+Ju&amp;rft.au=Sylvia%2C+Jody&amp;rft.au=Szpiro%2C+Adam&amp;rft.au=Taliun%2C+Daniel&amp;rft.au=Tang%2C+Hua&amp;rft.au=Taub%2C+Margaret&amp;rft.au=Taylor%2C+Kent+D.&amp;rft.au=Taylor%2C+Matthew&amp;rft.au=Taylor%2C+Simeon&amp;rft.au=Telen%2C+Marilyn&amp;rft.au=Thornton%2C+Timothy+A.&amp;rft.au=Threlkeld%2C+Machiko&amp;rft.au=Tinker%2C+Lesley&amp;rft.au=Tirschwell%2C+David&amp;rft.au=Tishkoff%2C+Sarah&amp;rft.au=Tiwari%2C+Hemant&amp;rft.au=Tong%2C+Catherine&amp;rft.au=Tracy%2C+Russell&amp;rft.au=Tsai%2C+Michael&amp;rft.au=Vaidya%2C+Dhananjay&amp;rft.au=Van+Den+Berg%2C+David&amp;rft.au=VandeHaar%2C+Peter&amp;rft.au=Vrieze%2C+Scott&amp;rft.au=Walker%2C+Tarik&amp;rft.au=Wallace%2C+Robert&amp;rft.au=Walts%2C+Avram&amp;rft.au=Wang%2C+Fei+Fei&amp;rft.au=Wang%2C+Heming&amp;rft.au=Wang%2C+Jiongming&amp;rft.au=Watson%2C+Karol&amp;rft.au=Watt%2C+Jennifer&amp;rft.au=Weeks%2C+Daniel+E.&amp;rft.au=Weinstock%2C+Joshua&amp;rft.au=Weiss%2C+Scott+T.&amp;rft.au=Weng%2C+Lu-Chen&amp;rft.au=Willer%2C+Cristen&amp;rft.au=Williams%2C+Kayleen&amp;rft.au=Williams%2C+L.+Keoki&amp;rft.au=Wilson%2C+Carla&amp;rft.au=Wilson%2C+James&amp;rft.au=Winterkorn%2C+Lara&amp;rft.au=Wong%2C+Quenna&amp;rft.au=Wu%2C+Joseph&amp;rft.au=Xu%2C+Huichun&amp;rft.au=Yang%2C+Ivana&amp;rft.au=Yu%2C+Ketian&amp;rft.au=Zekavat%2C+Seyedeh+Maryam&amp;rft.au=Zhang%2C+Yingze&amp;rft.au=Zhao%2C+Snow+Xueyan&amp;rft.au=Zhao%2C+Wei&amp;rft.au=Zody%2C+Michael&amp;rft.au=Cupples%2C+L.+Adrienne&amp;rft.au=Shadyab%2C+Aladdin+H.&amp;rft.au=McKnight%2C+Barbara&amp;rft.au=Shoemaker%2C+Benjamin+M.&amp;rft.au=Mitchell%2C+Braxton+D.&amp;rft.au=Psaty%2C+Bruce+M.&amp;rft.au=Kooperberg%2C+Charles&amp;rft.au=Liu%2C+Ching-Ti&amp;rft.au=Albert%2C+Christine+M.&amp;rft.au=Roden%2C+Dan&amp;rft.au=Chasman%2C+Daniel+I.&amp;rft.au=Darbar%2C+Dawood&amp;rft.au=Lloyd-Jones%2C+Donald+M.&amp;rft.au=Arnett%2C+Donna+K.&amp;rft.au=Regan%2C+Elizabeth+A.&amp;rft.au=Boerwinkle%2C+Eric&amp;rft.au=Rotter%2C+Jerome+I.&amp;rft.au=O%E2%80%99Connell%2C+Jeffrey+R.&amp;rft.au=Yanek%2C+Lisa+R.&amp;rft.au=de+Andrade%2C+Mariza&amp;rft.au=Allison%2C+Matthew+A.&amp;rft.au=McDonald%2C+Merry-Lynn+N.&amp;rft.au=Chung%2C+Mina+K.&amp;rft.au=Fornage%2C+Myriam&amp;rft.au=Chami%2C+Nathalie&amp;rft.au=Smith%2C+Nicholas+L.&amp;rft.au=Ellinor%2C+Patrick+T.&amp;rft.au=Vasan%2C+Ramachandran+S.&amp;rft.au=Mathias%2C+Rasika+A.&amp;rft.au=Loos%2C+Ruth+J.+F.&amp;rft.au=Rich%2C+Stephen+S.&amp;rft.au=Lubitz%2C+Steven+A.&amp;rft.au=Heckbert%2C+Susan+R.&amp;rft.au=Redline%2C+Susan&amp;rft.au=Guo%2C+Xiuqing&amp;rft.au=Chen%2C+Y.+-D+Ida&amp;rft.au=Laurie%2C+Cecelia+A.&amp;rft.au=Hernandez%2C+Ryan+D.&amp;rft.au=McGarvey%2C+Stephen+T.&amp;rft.au=Goddard%2C+Michael+E.&amp;rft.au=Laurie%2C+Cathy+C.&amp;rft.au=North%2C+Kari+E.&amp;rft.au=Lange%2C+Leslie+A.&amp;rft.au=Weir%2C+Bruce+S.&amp;rft.au=Yengo%2C+Loic&amp;rft.au=Yang%2C+Jian&amp;rft.au=Visscher%2C+Peter+M.&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC9119698&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-39\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-39\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFTaliunHarrisKesslerCarlson2021\" class=\"citation journal cs1\">Taliun, Daniel; Harris, Daniel N.; Kessler, Michael D.;  et&#160;al. (February 2021). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7875770\">\"Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program\"</a>. <i>Nature</i>. <b>590</b> (7845): 290\\xe2\\x80\\x93299. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/2021Natur.590..290T\">2021Natur.590..290T</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2Fs41586-021-03205-y\">10.1038/s41586-021-03205-y</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7875770\">7875770</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/33568819\">33568819</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Sequencing+of+53%2C831+diverse+genomes+from+the+NHLBI+TOPMed+Program&amp;rft.volume=590&amp;rft.issue=7845&amp;rft.pages=290-299&amp;rft.date=2021-02&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7875770%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F33568819&amp;rft_id=info%3Adoi%2F10.1038%2Fs41586-021-03205-y&amp;rft_id=info%3Abibcode%2F2021Natur.590..290T&amp;rft.aulast=Taliun&amp;rft.aufirst=Daniel&amp;rft.au=Harris%2C+Daniel+N.&amp;rft.au=Kessler%2C+Michael+D.&amp;rft.au=Carlson%2C+Jedidiah&amp;rft.au=Szpiech%2C+Zachary+A.&amp;rft.au=Torres%2C+Raul&amp;rft.au=Taliun%2C+Sarah+A.+Gagliano&amp;rft.au=Corvelo%2C+Andr%C3%A9&amp;rft.au=Gogarten%2C+Stephanie+M.&amp;rft.au=Kang%2C+Hyun+Min&amp;rft.au=Pitsillides%2C+Achilleas+N.&amp;rft.au=LeFaive%2C+Jonathon&amp;rft.au=Lee%2C+Seung-been&amp;rft.au=Tian%2C+Xiaowen&amp;rft.au=Browning%2C+Brian+L.&amp;rft.au=Das%2C+Sayantan&amp;rft.au=Emde%2C+Anne-Katrin&amp;rft.au=Clarke%2C+Wayne+E.&amp;rft.au=Loesch%2C+Douglas+P.&amp;rft.au=Shetty%2C+Amol+C.&amp;rft.au=Blackwell%2C+Thomas+W.&amp;rft.au=Smith%2C+Albert+V.&amp;rft.au=Wong%2C+Quenna&amp;rft.au=Liu%2C+Xiaoming&amp;rft.au=Conomos%2C+Matthew+P.&amp;rft.au=Bobo%2C+Dean+M.&amp;rft.au=Aguet%2C+Fran%C3%A7ois&amp;rft.au=Albert%2C+Christine&amp;rft.au=Alonso%2C+Alvaro&amp;rft.au=Ardlie%2C+Kristin+G.&amp;rft.au=Arking%2C+Dan+E.&amp;rft.au=Aslibekyan%2C+Stella&amp;rft.au=Auer%2C+Paul+L.&amp;rft.au=Barnard%2C+John&amp;rft.au=Barr%2C+R.+Graham&amp;rft.au=Barwick%2C+Lucas&amp;rft.au=Becker%2C+Lewis+C.&amp;rft.au=Beer%2C+Rebecca+L.&amp;rft.au=Benjamin%2C+Emelia+J.&amp;rft.au=Bielak%2C+Lawrence+F.&amp;rft.au=Blangero%2C+John&amp;rft.au=Boehnke%2C+Michael&amp;rft.au=Bowden%2C+Donald+W.&amp;rft.au=Brody%2C+Jennifer+A.&amp;rft.au=Burchard%2C+Esteban+G.&amp;rft.au=Cade%2C+Brian+E.&amp;rft.au=Casella%2C+James+F.&amp;rft.au=Chalazan%2C+Brandon&amp;rft.au=Chasman%2C+Daniel+I.&amp;rft.au=Chen%2C+Yii-Der+Ida&amp;rft.au=Cho%2C+Michael+H.&amp;rft.au=Choi%2C+Seung+Hoan&amp;rft.au=Chung%2C+Mina+K.&amp;rft.au=Clish%2C+Clary+B.&amp;rft.au=Correa%2C+Adolfo&amp;rft.au=Curran%2C+Joanne+E.&amp;rft.au=Custer%2C+Brian&amp;rft.au=Darbar%2C+Dawood&amp;rft.au=Daya%2C+Michelle&amp;rft.au=de+Andrade%2C+Mariza&amp;rft.au=DeMeo%2C+Dawn+L.&amp;rft.au=Dutcher%2C+Susan+K.&amp;rft.au=Ellinor%2C+Patrick+T.&amp;rft.au=Emery%2C+Leslie+S.&amp;rft.au=Eng%2C+Celeste&amp;rft.au=Fatkin%2C+Diane&amp;rft.au=Fingerlin%2C+Tasha&amp;rft.au=Forer%2C+Lukas&amp;rft.au=Fornage%2C+Myriam&amp;rft.au=Franceschini%2C+Nora&amp;rft.au=Fuchsberger%2C+Christian&amp;rft.au=Fullerton%2C+Stephanie+M.&amp;rft.au=Germer%2C+Soren&amp;rft.au=Gladwin%2C+Mark+T.&amp;rft.au=Gottlieb%2C+Daniel+J.&amp;rft.au=Guo%2C+Xiuqing&amp;rft.au=Hall%2C+Michael+E.&amp;rft.au=He%2C+Jiang&amp;rft.au=Heard-Costa%2C+Nancy+L.&amp;rft.au=Heckbert%2C+Susan+R.&amp;rft.au=Irvin%2C+Marguerite+R.&amp;rft.au=Johnsen%2C+Jill+M.&amp;rft.au=Johnson%2C+Andrew+D.&amp;rft.au=Kaplan%2C+Robert&amp;rft.au=Kardia%2C+Sharon+L.+R.&amp;rft.au=Kelly%2C+Tanika&amp;rft.au=Kelly%2C+Shannon&amp;rft.au=Kenny%2C+Eimear+E.&amp;rft.au=Kiel%2C+Douglas+P.&amp;rft.au=Klemmer%2C+Robert&amp;rft.au=Konkle%2C+Barbara+A.&amp;rft.au=Kooperberg%2C+Charles&amp;rft.au=K%C3%B6ttgen%2C+Anna&amp;rft.au=Lange%2C+Leslie+A.&amp;rft.au=Lasky-Su%2C+Jessica&amp;rft.au=Levy%2C+Daniel&amp;rft.au=Lin%2C+Xihong&amp;rft.au=Lin%2C+Keng-Han&amp;rft.au=Liu%2C+Chunyu&amp;rft.au=Loos%2C+Ruth+J.+F.&amp;rft.au=Garman%2C+Lori&amp;rft.au=Gerszten%2C+Robert&amp;rft.au=Lubitz%2C+Steven+A.&amp;rft.au=Lunetta%2C+Kathryn+L.&amp;rft.au=Mak%2C+Angel+C.+Y.&amp;rft.au=Manichaikul%2C+Ani&amp;rft.au=Manning%2C+Alisa+K.&amp;rft.au=Mathias%2C+Rasika+A.&amp;rft.au=McManus%2C+David+D.&amp;rft.au=McGarvey%2C+Stephen+T.&amp;rft.au=Meigs%2C+James+B.&amp;rft.au=Meyers%2C+Deborah+A.&amp;rft.au=Mikulla%2C+Julie+L.&amp;rft.au=Minear%2C+Mollie+A.&amp;rft.au=Mitchell%2C+Braxton+D.&amp;rft.au=Mohanty%2C+Sanghamitra&amp;rft.au=Montasser%2C+May+E.&amp;rft.au=Montgomery%2C+Courtney&amp;rft.au=Morrison%2C+Alanna+C.&amp;rft.au=Murabito%2C+Joanne+M.&amp;rft.au=Natale%2C+Andrea&amp;rft.au=Natarajan%2C+Pradeep&amp;rft.au=Nelson%2C+Sarah+C.&amp;rft.au=North%2C+Kari+E.&amp;rft.au=O%E2%80%99Connell%2C+Jeffrey+R.&amp;rft.au=Palmer%2C+Nicholette+D.&amp;rft.au=Pankratz%2C+Nathan&amp;rft.au=Peloso%2C+Gina+M.&amp;rft.au=Peyser%2C+Patricia+A.&amp;rft.au=Pleiness%2C+Jacob&amp;rft.au=Post%2C+Wendy+S.&amp;rft.au=Psaty%2C+Bruce+M.&amp;rft.au=Rao%2C+D.+C.&amp;rft.au=Redline%2C+Susan&amp;rft.au=Reiner%2C+Alexander+P.&amp;rft.au=Roden%2C+Dan&amp;rft.au=Rotter%2C+Jerome+I.&amp;rft.au=Ruczinski%2C+Ingo&amp;rft.au=Sarnowski%2C+Chlo%C3%A9&amp;rft.au=Schoenherr%2C+Sebastian&amp;rft.au=Schwartz%2C+David+A.&amp;rft.au=Seo%2C+Jeong-Sun&amp;rft.au=Seshadri%2C+Sudha&amp;rft.au=Sheehan%2C+Vivien+A.&amp;rft.au=Sheu%2C+Wayne+H.&amp;rft.au=Shoemaker%2C+M.+Benjamin&amp;rft.au=Smith%2C+Nicholas+L.&amp;rft.au=Smith%2C+Jennifer+A.&amp;rft.au=Sotoodehnia%2C+Nona&amp;rft.au=Stilp%2C+Adrienne+M.&amp;rft.au=Tang%2C+Weihong&amp;rft.au=Taylor%2C+Kent+D.&amp;rft.au=Telen%2C+Marilyn&amp;rft.au=Thornton%2C+Timothy+A.&amp;rft.au=Tracy%2C+Russell+P.&amp;rft.au=Van+Den+Berg%2C+David+J.&amp;rft.au=Vasan%2C+Ramachandran+S.&amp;rft.au=Viaud-Martinez%2C+Karine+A.&amp;rft.au=Vrieze%2C+Scott&amp;rft.au=Weeks%2C+Daniel+E.&amp;rft.au=Weir%2C+Bruce+S.&amp;rft.au=Weiss%2C+Scott+T.&amp;rft.au=Weng%2C+Lu-Chen&amp;rft.au=Willer%2C+Cristen+J.&amp;rft.au=Zhang%2C+Yingze&amp;rft.au=Zhao%2C+Xutong&amp;rft.au=Arnett%2C+Donna+K.&amp;rft.au=Ashley-Koch%2C+Allison+E.&amp;rft.au=Barnes%2C+Kathleen+C.&amp;rft.au=Boerwinkle%2C+Eric&amp;rft.au=Gabriel%2C+Stacey&amp;rft.au=Gibbs%2C+Richard&amp;rft.au=Rice%2C+Kenneth+M.&amp;rft.au=Rich%2C+Stephen+S.&amp;rft.au=Silverman%2C+Edwin+K.&amp;rft.au=Qasba%2C+Pankaj&amp;rft.au=Gan%2C+Weiniu&amp;rft.au=Papanicolaou%2C+George+J.&amp;rft.au=Nickerson%2C+Deborah+A.&amp;rft.au=Browning%2C+Sharon+R.&amp;rft.au=Zody%2C+Michael+C.&amp;rft.au=Z%C3%B6llner%2C+Sebastian&amp;rft.au=Wilson%2C+James+G.&amp;rft.au=Cupples%2C+L.+Adrienne&amp;rft.au=Laurie%2C+Cathy+C.&amp;rft.au=Jaquish%2C+Cashell+E.&amp;rft.au=Hernandez%2C+Ryan+D.&amp;rft.au=O%E2%80%99Connor%2C+Timothy+D.&amp;rft.au=Abecasis%2C+Gon%C3%A7alo+R.&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7875770&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-40\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-40\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFLiLiZhouGaynor2020\" class=\"citation journal cs1\">Li, Xihao; Li, Zilin; Zhou, Hufeng;  et&#160;al. (September 2020). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7483769\">\"Dynamic incorporation of multiple in silico functional annotations empowers rare variant association analysis of large whole-genome sequencing studies at scale\"</a>. <i>Nature Genetics</i>. <b>52</b> (9): 969\\xe2\\x80\\x93983. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2Fs41588-020-0676-4\">10.1038/s41588-020-0676-4</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7483769\">7483769</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/32839606\">32839606</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Genetics&amp;rft.atitle=Dynamic+incorporation+of+multiple+in+silico+functional+annotations+empowers+rare+variant+association+analysis+of+large+whole-genome+sequencing+studies+at+scale&amp;rft.volume=52&amp;rft.issue=9&amp;rft.pages=969-983&amp;rft.date=2020-09&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7483769%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F32839606&amp;rft_id=info%3Adoi%2F10.1038%2Fs41588-020-0676-4&amp;rft.aulast=Li&amp;rft.aufirst=Xihao&amp;rft.au=Li%2C+Zilin&amp;rft.au=Zhou%2C+Hufeng&amp;rft.au=Gaynor%2C+Sheila+M.&amp;rft.au=Liu%2C+Yaowu&amp;rft.au=Chen%2C+Han&amp;rft.au=Sun%2C+Ryan&amp;rft.au=Dey%2C+Rounak&amp;rft.au=Arnett%2C+Donna+K.&amp;rft.au=Aslibekyan%2C+Stella&amp;rft.au=Ballantyne%2C+Christie+M.&amp;rft.au=Bielak%2C+Lawrence+F.&amp;rft.au=Blangero%2C+John&amp;rft.au=Boerwinkle%2C+Eric&amp;rft.au=Bowden%2C+Donald+W.&amp;rft.au=Broome%2C+Jai+G.&amp;rft.au=Conomos%2C+Matthew+P.&amp;rft.au=Correa%2C+Adolfo&amp;rft.au=Cupples%2C+L.+Adrienne&amp;rft.au=Curran%2C+Joanne+E.&amp;rft.au=Freedman%2C+Barry+I.&amp;rft.au=Guo%2C+Xiuqing&amp;rft.au=Hindy%2C+George&amp;rft.au=Irvin%2C+Marguerite+R.&amp;rft.au=Kardia%2C+Sharon+L.+R.&amp;rft.au=Kathiresan%2C+Sekar&amp;rft.au=Khan%2C+Alyna+T.&amp;rft.au=Kooperberg%2C+Charles+L.&amp;rft.au=Laurie%2C+Cathy+C.&amp;rft.au=Liu%2C+X.+Shirley&amp;rft.au=Mahaney%2C+Michael+C.&amp;rft.au=Manichaikul%2C+Ani+W.&amp;rft.au=Martin%2C+Lisa+W.&amp;rft.au=Mathias%2C+Rasika+A.&amp;rft.au=McGarvey%2C+Stephen+T.&amp;rft.au=Mitchell%2C+Braxton+D.&amp;rft.au=Montasser%2C+May+E.&amp;rft.au=Moore%2C+Jill+E.&amp;rft.au=Morrison%2C+Alanna+C.&amp;rft.au=O%E2%80%99Connell%2C+Jeffrey+R.&amp;rft.au=Palmer%2C+Nicholette+D.&amp;rft.au=Pampana%2C+Akhil&amp;rft.au=Peralta%2C+Juan+M.&amp;rft.au=Peyser%2C+Patricia+A.&amp;rft.au=Psaty%2C+Bruce+M.&amp;rft.au=Redline%2C+Susan&amp;rft.au=Rice%2C+Kenneth+M.&amp;rft.au=Rich%2C+Stephen+S.&amp;rft.au=Smith%2C+Jennifer+A.&amp;rft.au=Tiwari%2C+Hemant+K.&amp;rft.au=Tsai%2C+Michael+Y.&amp;rft.au=Vasan%2C+Ramachandran+S.&amp;rft.au=Wang%2C+Fei+Fei&amp;rft.au=Weeks%2C+Daniel+E.&amp;rft.au=Weng%2C+Zhiping&amp;rft.au=Wilson%2C+James+G.&amp;rft.au=Yanek%2C+Lisa+R.&amp;rft.au=Abe%2C+Namiko&amp;rft.au=Abecasis%2C+Gon%C3%A7alo+R.&amp;rft.au=Aguet%2C+Francois&amp;rft.au=Albert%2C+Christine&amp;rft.au=Almasy%2C+Laura&amp;rft.au=Alonso%2C+Alvaro&amp;rft.au=Ament%2C+Seth&amp;rft.au=Anderson%2C+Peter&amp;rft.au=Anugu%2C+Pramod&amp;rft.au=Applebaum-Bowden%2C+Deborah&amp;rft.au=Ardlie%2C+Kristin&amp;rft.au=Arking%2C+Dan&amp;rft.au=Arnett%2C+Donna+K.&amp;rft.au=Ashley-Koch%2C+Allison&amp;rft.au=Aslibekyan%2C+Stella&amp;rft.au=Assimes%2C+Tim&amp;rft.au=Auer%2C+Paul&amp;rft.au=Avramopoulos%2C+Dimitrios&amp;rft.au=Barnard%2C+John&amp;rft.au=Barnes%2C+Kathleen&amp;rft.au=Barr%2C+R.+Graham&amp;rft.au=Barron-Casella%2C+Emily&amp;rft.au=Barwick%2C+Lucas&amp;rft.au=Beaty%2C+Terri&amp;rft.au=Beck%2C+Gerald&amp;rft.au=Becker%2C+Diane&amp;rft.au=Becker%2C+Lewis&amp;rft.au=Beer%2C+Rebecca&amp;rft.au=Beitelshees%2C+Amber&amp;rft.au=Benjamin%2C+Emelia&amp;rft.au=Benos%2C+Takis&amp;rft.au=Bezerra%2C+Marcos&amp;rft.au=Bielak%2C+Lawrence+F.&amp;rft.au=Bis%2C+Joshua&amp;rft.au=Blackwell%2C+Thomas&amp;rft.au=Blangero%2C+John&amp;rft.au=Boerwinkle%2C+Eric&amp;rft.au=Bowden%2C+Donald+W.&amp;rft.au=Bowler%2C+Russell&amp;rft.au=Brody%2C+Jennifer&amp;rft.au=Broeckel%2C+Ulrich&amp;rft.au=Broome%2C+Jai+G.&amp;rft.au=Bunting%2C+Karen&amp;rft.au=Burchard%2C+Esteban&amp;rft.au=Bustamante%2C+Carlos&amp;rft.au=Buth%2C+Erin&amp;rft.au=Cade%2C+Brian&amp;rft.au=Cardwell%2C+Jonathan&amp;rft.au=Carey%2C+Vincent&amp;rft.au=Carty%2C+Cara&amp;rft.au=Casaburi%2C+Richard&amp;rft.au=Casella%2C+James&amp;rft.au=Castaldi%2C+Peter&amp;rft.au=Chaffin%2C+Mark&amp;rft.au=Chang%2C+Christy&amp;rft.au=Chang%2C+Yi-Cheng&amp;rft.au=Chasman%2C+Daniel&amp;rft.au=Chavan%2C+Sameer&amp;rft.au=Chen%2C+Bo-Juen&amp;rft.au=Chen%2C+Wei-Min&amp;rft.au=Chen%2C+Yii-Der+Ida&amp;rft.au=Cho%2C+Michael&amp;rft.au=Choi%2C+Seung+Hoan&amp;rft.au=Chuang%2C+Lee-Ming&amp;rft.au=Chung%2C+Mina&amp;rft.au=Chung%2C+Ren-Hua&amp;rft.au=Clish%2C+Clary&amp;rft.au=Comhair%2C+Suzy&amp;rft.au=Conomos%2C+Matthew+P.&amp;rft.au=Cornell%2C+Elaine&amp;rft.au=Correa%2C+Adolfo&amp;rft.au=Crandall%2C+Carolyn&amp;rft.au=Crapo%2C+James&amp;rft.au=Cupples%2C+L.+Adrienne&amp;rft.au=Curran%2C+Joanne+E.&amp;rft.au=Curtis%2C+Jeffrey&amp;rft.au=Custer%2C+Brian&amp;rft.au=Damcott%2C+Coleen&amp;rft.au=Darbar%2C+Dawood&amp;rft.au=Das%2C+Sayantan&amp;rft.au=David%2C+Sean&amp;rft.au=Davis%2C+Colleen&amp;rft.au=Daya%2C+Michelle&amp;rft.au=de+Andrade%2C+Mariza&amp;rft.au=Fuentes%2C+Lisa+de+las&amp;rft.au=DeBaun%2C+Michael&amp;rft.au=Deka%2C+Ranjan&amp;rft.au=DeMeo%2C+Dawn&amp;rft.au=Devine%2C+Scott&amp;rft.au=Duan%2C+Qing&amp;rft.au=Duggirala%2C+Ravi&amp;rft.au=Durda%2C+Jon+Peter&amp;rft.au=Dutcher%2C+Susan&amp;rft.au=Eaton%2C+Charles&amp;rft.au=Ekunwe%2C+Lynette&amp;rft.au=El+Boueiz%2C+Adel&amp;rft.au=Ellinor%2C+Patrick&amp;rft.au=Emery%2C+Leslie&amp;rft.au=Erzurum%2C+Serpil&amp;rft.au=Farber%2C+Charles&amp;rft.au=Fingerlin%2C+Tasha&amp;rft.au=Flickinger%2C+Matthew&amp;rft.au=Fornage%2C+Myriam&amp;rft.au=Franceschini%2C+Nora&amp;rft.au=Frazar%2C+Chris&amp;rft.au=Fu%2C+Mao&amp;rft.au=Fullerton%2C+Stephanie+M.&amp;rft.au=Fulton%2C+Lucinda&amp;rft.au=Gabriel%2C+Stacey&amp;rft.au=Gan%2C+Weiniu&amp;rft.au=Gao%2C+Shanshan&amp;rft.au=Gao%2C+Yan&amp;rft.au=Gass%2C+Margery&amp;rft.au=Gelb%2C+Bruce&amp;rft.au=Geng%2C+Xiaoqi&amp;rft.au=Geraci%2C+Mark&amp;rft.au=Germer%2C+Soren&amp;rft.au=Gerszten%2C+Robert&amp;rft.au=Ghosh%2C+Auyon&amp;rft.au=Gibbs%2C+Richard&amp;rft.au=Gignoux%2C+Chris&amp;rft.au=Gladwin%2C+Mark&amp;rft.au=Glahn%2C+David&amp;rft.au=Gogarten%2C+Stephanie&amp;rft.au=Gong%2C+Da-Wei&amp;rft.au=Goring%2C+Harald&amp;rft.au=Graw%2C+Sharon&amp;rft.au=Grine%2C+Daniel&amp;rft.au=Gu%2C+C.+Charles&amp;rft.au=Guan%2C+Yue&amp;rft.au=Guo%2C+Xiuqing&amp;rft.au=Gupta%2C+Namrata&amp;rft.au=Haessler%2C+Jeff&amp;rft.au=Hall%2C+Michael&amp;rft.au=Harris%2C+Daniel&amp;rft.au=Hawley%2C+Nicola+L.&amp;rft.au=He%2C+Jiang&amp;rft.au=Heckbert%2C+Susan&amp;rft.au=Hernandez%2C+Ryan&amp;rft.au=Herrington%2C+David&amp;rft.au=Hersh%2C+Craig&amp;rft.au=Hidalgo%2C+Bertha&amp;rft.au=Hixson%2C+James&amp;rft.au=Hobbs%2C+Brian&amp;rft.au=Hokanson%2C+John&amp;rft.au=Hong%2C+Elliott&amp;rft.au=Hoth%2C+Karin&amp;rft.au=Hsiung%2C+Chao&amp;rft.au=Hung%2C+Yi-Jen&amp;rft.au=Huston%2C+Haley&amp;rft.au=Hwu%2C+Chii+Min&amp;rft.au=Irvin%2C+Marguerite+R.&amp;rft.au=Jackson%2C+Rebecca&amp;rft.au=Jain%2C+Deepti&amp;rft.au=Jaquish%2C+Cashell&amp;rft.au=Jhun%2C+Min+A.&amp;rft.au=Johnsen%2C+Jill&amp;rft.au=Johnson%2C+Andrew&amp;rft.au=Johnson%2C+Craig&amp;rft.au=Johnston%2C+Rich&amp;rft.au=Jones%2C+Kimberly&amp;rft.au=Kang%2C+Hyun+Min&amp;rft.au=Kaplan%2C+Robert&amp;rft.au=Kardia%2C+Sharon+L.+R.&amp;rft.au=Kathiresan%2C+Sekar&amp;rft.au=Kelly%2C+Shannon&amp;rft.au=Kenny%2C+Eimear&amp;rft.au=Kessler%2C+Michael&amp;rft.au=Khan%2C+Alyna+T.&amp;rft.au=Kim%2C+Wonji&amp;rft.au=Kinney%2C+Greg&amp;rft.au=Konkle%2C+Barbara&amp;rft.au=Kooperberg%2C+Charles+L.&amp;rft.au=Kramer%2C+Holly&amp;rft.au=Lange%2C+Christoph&amp;rft.au=Lange%2C+Ethan&amp;rft.au=Lange%2C+Leslie&amp;rft.au=Laurie%2C+Cathy+C.&amp;rft.au=Laurie%2C+Cecelia&amp;rft.au=LeBoff%2C+Meryl&amp;rft.au=Lee%2C+Jiwon&amp;rft.au=Lee%2C+Seunggeun+Shawn&amp;rft.au=Lee%2C+Wen-Jane&amp;rft.au=LeFaive%2C+Jonathon&amp;rft.au=Levine%2C+David&amp;rft.au=Levy%2C+Dan&amp;rft.au=Lewis%2C+Joshua&amp;rft.au=Li%2C+Xiaohui&amp;rft.au=Li%2C+Yun&amp;rft.au=Lin%2C+Henry&amp;rft.au=Lin%2C+Honghuang&amp;rft.au=Lin%2C+Keng+Han&amp;rft.au=Lin%2C+Xihong&amp;rft.au=Liu%2C+Simin&amp;rft.au=Liu%2C+Yongmei&amp;rft.au=Liu%2C+Yu&amp;rft.au=Loos%2C+Ruth+J.+F.&amp;rft.au=Lubitz%2C+Steven&amp;rft.au=Lunetta%2C+Kathryn&amp;rft.au=Luo%2C+James&amp;rft.au=Mahaney%2C+Michael+C.&amp;rft.au=Make%2C+Barry&amp;rft.au=Manichaikul%2C+Ani+W.&amp;rft.au=Manson%2C+JoAnn&amp;rft.au=Margolin%2C+Lauren&amp;rft.au=Martin%2C+Lisa+W.&amp;rft.au=Mathai%2C+Susan&amp;rft.au=Mathias%2C+Rasika+A.&amp;rft.au=May%2C+Susanne&amp;rft.au=McArdle%2C+Patrick&amp;rft.au=McDonald%2C+Merry-Lynn&amp;rft.au=McFarland%2C+Sean&amp;rft.au=McGarvey%2C+Stephen+T.&amp;rft.au=McGoldrick%2C+Daniel&amp;rft.au=McHugh%2C+Caitlin&amp;rft.au=Mei%2C+Hao&amp;rft.au=Mestroni%2C+Luisa&amp;rft.au=Meyers%2C+Deborah+A.&amp;rft.au=Mikulla%2C+Julie&amp;rft.au=Min%2C+Nancy&amp;rft.au=Minear%2C+Mollie&amp;rft.au=Minster%2C+Ryan+L.&amp;rft.au=Mitchell%2C+Braxton+D.&amp;rft.au=Moll%2C+Matt&amp;rft.au=Montasser%2C+May+E.&amp;rft.au=Montgomery%2C+Courtney&amp;rft.au=Moscati%2C+Arden&amp;rft.au=Musani%2C+Solomon&amp;rft.au=Mwasongwe%2C+Stanford&amp;rft.au=Mychaleckyj%2C+Josyf+C.&amp;rft.au=Nadkarni%2C+Girish&amp;rft.au=Naik%2C+Rakhi&amp;rft.au=Naseri%2C+Take&amp;rft.au=Natarajan%2C+Pradeep&amp;rft.au=Nekhai%2C+Sergei&amp;rft.au=Nelson%2C+Sarah+C.&amp;rft.au=Neltner%2C+Bonnie&amp;rft.au=Nickerson%2C+Deborah&amp;rft.au=North%2C+Kari&amp;rft.au=O%E2%80%99Connell%2C+Jeffrey+R.&amp;rft.au=O%E2%80%99Connor%2C+Tim&amp;rft.au=Ochs-Balcom%2C+Heather&amp;rft.au=Paik%2C+David&amp;rft.au=Palmer%2C+Nicholette+D.&amp;rft.au=Pankow%2C+James&amp;rft.au=Papanicolaou%2C+George&amp;rft.au=Parsa%2C+Afshin&amp;rft.au=Peralta%2C+Juan+M.&amp;rft.au=Perez%2C+Marco&amp;rft.au=Perry%2C+James&amp;rft.au=Peters%2C+Ulrike&amp;rft.au=Peyser%2C+Patricia+A.&amp;rft.au=Phillips%2C+Lawrence+S.&amp;rft.au=Pollin%2C+Toni&amp;rft.au=Post%2C+Wendy&amp;rft.au=Becker%2C+Julia+Powers&amp;rft.au=Boorgula%2C+Meher+Preethi&amp;rft.au=Preuss%2C+Michael&amp;rft.au=Psaty%2C+Bruce+M.&amp;rft.au=Qasba%2C+Pankaj&amp;rft.au=Qiao%2C+Dandi&amp;rft.au=Qin%2C+Zhaohui&amp;rft.au=Rafaels%2C+Nicholas&amp;rft.au=Raffield%2C+Laura&amp;rft.au=Vasan%2C+Ramachandran+S.&amp;rft.au=Rao%2C+D.+C.&amp;rft.au=Rasmussen-Torvik%2C+Laura&amp;rft.au=Ratan%2C+Aakrosh&amp;rft.au=Redline%2C+Susan&amp;rft.au=Reed%2C+Robert&amp;rft.au=Regan%2C+Elizabeth&amp;rft.au=Reiner%2C+Alex&amp;rft.au=Reupena%2C+Muagututi%E2%80%98a+Sefuiva&amp;rft.au=Rice%2C+Kenneth+M.&amp;rft.au=Rich%2C+Stephen+S.&amp;rft.au=Roden%2C+Dan&amp;rft.au=Roselli%2C+Carolina&amp;rft.au=Rotter%2C+Jerome+I.&amp;rft.au=Ruczinski%2C+Ingo&amp;rft.au=Russell%2C+Pamela&amp;rft.au=Ruuska%2C+Sarah&amp;rft.au=Ryan%2C+Kathleen&amp;rft.au=Sabino%2C+Ester+Cerdeira&amp;rft.au=Saleheen%2C+Danish&amp;rft.au=Salimi%2C+Shabnam&amp;rft.au=Salzberg%2C+Steven&amp;rft.au=Sandow%2C+Kevin&amp;rft.au=Sankaran%2C+Vijay+G.&amp;rft.au=Scheller%2C+Christopher&amp;rft.au=Schmidt%2C+Ellen&amp;rft.au=Schwander%2C+Karen&amp;rft.au=Schwartz%2C+David&amp;rft.au=Sciurba%2C+Frank&amp;rft.au=Seidman%2C+Christine&amp;rft.au=Seidman%2C+Jonathan&amp;rft.au=Sheehan%2C+Vivien&amp;rft.au=Sherman%2C+Stephanie+L.&amp;rft.au=Shetty%2C+Amol&amp;rft.au=Shetty%2C+Aniket&amp;rft.au=Sheu%2C+Wayne+Hui-Heng&amp;rft.au=Shoemaker%2C+M.+Benjamin&amp;rft.au=Silver%2C+Brian&amp;rft.au=Silverman%2C+Edwin&amp;rft.au=Smith%2C+Jennifer+A.&amp;rft.au=Smith%2C+Josh&amp;rft.au=Smith%2C+Nicholas&amp;rft.au=Smith%2C+Tanja&amp;rft.au=Smoller%2C+Sylvia&amp;rft.au=Snively%2C+Beverly&amp;rft.au=Snyder%2C+Michael&amp;rft.au=Sofer%2C+Tamar&amp;rft.au=Sotoodehnia%2C+Nona&amp;rft.au=Stilp%2C+Adrienne+M.&amp;rft.au=Storm%2C+Garrett&amp;rft.au=Streeten%2C+Elizabeth&amp;rft.au=Su%2C+Jessica+Lasky&amp;rft.au=Sung%2C+Yun+Ju&amp;rft.au=Sylvia%2C+Jody&amp;rft.au=Szpiro%2C+Adam&amp;rft.au=Sztalryd%2C+Carole&amp;rft.au=Taliun%2C+Daniel&amp;rft.au=Tang%2C+Hua&amp;rft.au=Taub%2C+Margaret&amp;rft.au=Taylor%2C+Kent+D.&amp;rft.au=Taylor%2C+Matthew&amp;rft.au=Taylor%2C+Simeon&amp;rft.au=Telen%2C+Marilyn&amp;rft.au=Thornton%2C+Timothy+A.&amp;rft.au=Threlkeld%2C+Machiko&amp;rft.au=Tinker%2C+Lesley&amp;rft.au=Tirschwell%2C+David&amp;rft.au=Tishkoff%2C+Sarah&amp;rft.au=Tiwari%2C+Hemant+K.&amp;rft.au=Tong%2C+Catherine&amp;rft.au=Tracy%2C+Russell&amp;rft.au=Tsai%2C+Michael+Y.&amp;rft.au=Vaidya%2C+Dhananjay&amp;rft.au=Van+Den+Berg%2C+David&amp;rft.au=VandeHaar%2C+Peter&amp;rft.au=Vrieze%2C+Scott&amp;rft.au=Walker%2C+Tarik&amp;rft.au=Wallace%2C+Robert&amp;rft.au=Walts%2C+Avram&amp;rft.au=Wang%2C+Fei+Fei&amp;rft.au=Wang%2C+Heming&amp;rft.au=Watson%2C+Karol&amp;rft.au=Weeks%2C+Daniel+E.&amp;rft.au=Weir%2C+Bruce&amp;rft.au=Weiss%2C+Scott&amp;rft.au=Weng%2C+Lu-Chen&amp;rft.au=Wessel%2C+Jennifer&amp;rft.au=Willer%2C+Cristen+J.&amp;rft.au=Williams%2C+Kayleen&amp;rft.au=Williams%2C+L.+Keoki&amp;rft.au=Wilson%2C+Carla&amp;rft.au=Wilson%2C+James+G.&amp;rft.au=Wong%2C+Quenna&amp;rft.au=Wu%2C+Joseph&amp;rft.au=Xu%2C+Huichun&amp;rft.au=Yanek%2C+Lisa+R.&amp;rft.au=Yang%2C+Ivana&amp;rft.au=Yang%2C+Rongze&amp;rft.au=Zaghloul%2C+Norann&amp;rft.au=Zekavat%2C+Maryam&amp;rft.au=Zhang%2C+Yingze&amp;rft.au=Zhao%2C+Snow+Xueyan&amp;rft.au=Zhao%2C+Wei&amp;rft.au=Zhi%2C+Degui&amp;rft.au=Zhou%2C+Xiang&amp;rft.au=Zhu%2C+Xiaofeng&amp;rft.au=Zody%2C+Michael&amp;rft.au=Zoellner%2C+Sebastian&amp;rft.au=Abdalla%2C+Moustafa&amp;rft.au=Abecasis%2C+Gon%C3%A7alo+R.&amp;rft.au=Arnett%2C+Donna+K.&amp;rft.au=Aslibekyan%2C+Stella&amp;rft.au=Assimes%2C+Tim&amp;rft.au=Atkinson%2C+Elizabeth&amp;rft.au=Ballantyne%2C+Christie+M.&amp;rft.au=Beitelshees%2C+Amber&amp;rft.au=Bielak%2C+Lawrence+F.&amp;rft.au=Bis%2C+Joshua&amp;rft.au=Bodea%2C+Corneliu&amp;rft.au=Boerwinkle%2C+Eric&amp;rft.au=Bowden%2C+Donald+W.&amp;rft.au=Brody%2C+Jennifer&amp;rft.au=Cade%2C+Brian&amp;rft.au=Carlson%2C+Jenna&amp;rft.au=Chang%2C+I-Shou&amp;rft.au=Chen%2C+Yii-Der+Ida&amp;rft.au=Chun%2C+Sung&amp;rft.au=Chung%2C+Ren-Hua&amp;rft.au=Conomos%2C+Matthew+P.&amp;rft.au=Correa%2C+Adolfo&amp;rft.au=Cupples%2C+L.+Adrienne&amp;rft.au=Damcott%2C+Coleen&amp;rft.au=de+Vries%2C+Paul&amp;rft.au=Do%2C+Ron&amp;rft.au=Elliott%2C+Amanda&amp;rft.au=Fu%2C+Mao&amp;rft.au=Ganna%2C+Andrea&amp;rft.au=Gong%2C+Da-Wei&amp;rft.au=Graham%2C+Sarah&amp;rft.au=Haas%2C+Mary&amp;rft.au=Haring%2C+Bernhard&amp;rft.au=He%2C+Jiang&amp;rft.au=Heckbert%2C+Susan&amp;rft.au=Himes%2C+Blanca&amp;rft.au=Hixson%2C+James&amp;rft.au=Irvin%2C+Marguerite+R.&amp;rft.au=Jain%2C+Deepti&amp;rft.au=Jarvik%2C+Gail&amp;rft.au=Jhun%2C+Min+A.&amp;rft.au=Jiang%2C+Jicai&amp;rft.au=Jun%2C+Goo&amp;rft.au=Kalyani%2C+Rita&amp;rft.au=Kardia%2C+Sharon+L.+R.&amp;rft.au=Kathiresan%2C+Sekar&amp;rft.au=Khera%2C+Amit&amp;rft.au=Klarin%2C+Derek&amp;rft.au=Kooperberg%2C+Charles+L.&amp;rft.au=Kral%2C+Brian&amp;rft.au=Lange%2C+Leslie&amp;rft.au=Laurie%2C+Cathy+C.&amp;rft.au=Laurie%2C+Cecelia&amp;rft.au=Lemaitre%2C+Rozenn&amp;rft.au=Li%2C+Zilin&amp;rft.au=Li%2C+Xihao&amp;rft.au=Lin%2C+Xihong&amp;rft.au=Mahaney%2C+Michael+C.&amp;rft.au=Manichaikul%2C+Ani+W.&amp;rft.au=Martin%2C+Lisa+W.&amp;rft.au=Mathias%2C+Rasika+A.&amp;rft.au=Mathur%2C+Ravi&amp;rft.au=McGarvey%2C+Stephen+T.&amp;rft.au=McHugh%2C+Caitlin&amp;rft.au=McLenithan%2C+John&amp;rft.au=Mikulla%2C+Julie&amp;rft.au=Mitchell%2C+Braxton+D.&amp;rft.au=Montasser%2C+May+E.&amp;rft.au=Moran%2C+Andrew&amp;rft.au=Morrison%2C+Alanna+C.&amp;rft.au=Nakao%2C+Tetsushi&amp;rft.au=Natarajan%2C+Pradeep&amp;rft.au=Nickerson%2C+Deborah&amp;rft.au=North%2C+Kari&amp;rft.au=O%E2%80%99Connell%2C+Jeffrey+R.&amp;rft.au=O%E2%80%99Donnell%2C+Christopher&amp;rft.au=Palmer%2C+Nicholette+D.&amp;rft.au=Pampana%2C+Akhil&amp;rft.au=Patel%2C+Aniruddh&amp;rft.au=Peloso%2C+Gina+M.&amp;rft.au=Perry%2C+James&amp;rft.au=Peters%2C+Ulrike&amp;rft.au=Peyser%2C+Patricia+A.&amp;rft.au=Pirruccello%2C+James&amp;rft.au=Pollin%2C+Toni&amp;rft.au=Preuss%2C+Michael&amp;rft.au=Psaty%2C+Bruce+M.&amp;rft.au=Rao%2C+D.+C.&amp;rft.au=Redline%2C+Susan&amp;rft.au=Reed%2C+Robert&amp;rft.au=Reiner%2C+Alex&amp;rft.au=Rich%2C+Stephen+S.&amp;rft.au=Rosenthal%2C+Samantha&amp;rft.au=Rotter%2C+Jerome+I.&amp;rft.au=Schoenberg%2C+Jenny&amp;rft.au=Selvaraj%2C+Margaret+Sunitha&amp;rft.au=Sheu%2C+Wayne+Hui-Heng&amp;rft.au=Smith%2C+Jennifer+A.&amp;rft.au=Sofer%2C+Tamar&amp;rft.au=Stilp%2C+Adrienne+M.&amp;rft.au=Sunyaev%2C+Shamil+R.&amp;rft.au=Surakka%2C+Ida&amp;rft.au=Sztalryd%2C+Carole&amp;rft.au=Tang%2C+Hua&amp;rft.au=Taylor%2C+Kent+D.&amp;rft.au=Tsai%2C+Michael+Y.&amp;rft.au=Uddin%2C+Md+Mesbah&amp;rft.au=Urbut%2C+Sarah&amp;rft.au=Verbanck%2C+Marie&amp;rft.au=Von+Holle%2C+Ann&amp;rft.au=Wang%2C+Heming&amp;rft.au=Wang%2C+Fei+Fei&amp;rft.au=Wiggins%2C+Kerri&amp;rft.au=Willer%2C+Cristen+J.&amp;rft.au=Wilson%2C+James+G.&amp;rft.au=Wolford%2C+Brooke&amp;rft.au=Xu%2C+Huichun&amp;rft.au=Yanek%2C+Lisa+R.&amp;rft.au=Zaghloul%2C+Norann&amp;rft.au=Zekavat%2C+Maryam&amp;rft.au=Zhang%2C+Jingwen&amp;rft.au=Neale%2C+Benjamin+M.&amp;rft.au=Sunyaev%2C+Shamil+R.&amp;rft.au=Abecasis%2C+Gon%C3%A7alo+R.&amp;rft.au=Rotter%2C+Jerome+I.&amp;rft.au=Willer%2C+Cristen+J.&amp;rft.au=Peloso%2C+Gina+M.&amp;rft.au=Natarajan%2C+Pradeep&amp;rft.au=Lin%2C+Xihong&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC7483769&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-41\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-41\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFLiLiZhouGaynor2022\" class=\"citation journal cs1\">Li, Zilin; Li, Xihao; Zhou, Hufeng;  et&#160;al. (December 2022). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10008172\">\"A framework for detecting noncoding rare-variant associations of large-scale whole-genome sequencing studies\"</a>. <i>Nature Methods</i>. <b>19</b> (12): 1599\\xe2\\x80\\x931611. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2Fs41592-022-01640-x\">10.1038/s41592-022-01640-x</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10008172\">10008172</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/36303018\">36303018</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:243873361\">243873361</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Methods&amp;rft.atitle=A+framework+for+detecting+noncoding+rare-variant+associations+of+large-scale+whole-genome+sequencing+studies&amp;rft.volume=19&amp;rft.issue=12&amp;rft.pages=1599-1611&amp;rft.date=2022-12&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10008172%23id-name%3DPMC&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A243873361%23id-name%3DS2CID&amp;rft_id=info%3Apmid%2F36303018&amp;rft_id=info%3Adoi%2F10.1038%2Fs41592-022-01640-x&amp;rft.aulast=Li&amp;rft.aufirst=Zilin&amp;rft.au=Li%2C+Xihao&amp;rft.au=Zhou%2C+Hufeng&amp;rft.au=Gaynor%2C+Sheila+M.&amp;rft.au=Selvaraj%2C+Margaret+Sunitha&amp;rft.au=Arapoglou%2C+Theodore&amp;rft.au=Quick%2C+Corbin&amp;rft.au=Liu%2C+Yaowu&amp;rft.au=Chen%2C+Han&amp;rft.au=Sun%2C+Ryan&amp;rft.au=Dey%2C+Rounak&amp;rft.au=Arnett%2C+Donna+K.&amp;rft.au=Auer%2C+Paul+L.&amp;rft.au=Bielak%2C+Lawrence+F.&amp;rft.au=Bis%2C+Joshua+C.&amp;rft.au=Blackwell%2C+Thomas+W.&amp;rft.au=Blangero%2C+John&amp;rft.au=Boerwinkle%2C+Eric&amp;rft.au=Bowden%2C+Donald+W.&amp;rft.au=Brody%2C+Jennifer+A.&amp;rft.au=Cade%2C+Brian+E.&amp;rft.au=Conomos%2C+Matthew+P.&amp;rft.au=Correa%2C+Adolfo&amp;rft.au=Cupples%2C+L.+Adrienne&amp;rft.au=Curran%2C+Joanne+E.&amp;rft.au=de+Vries%2C+Paul+S.&amp;rft.au=Duggirala%2C+Ravindranath&amp;rft.au=Franceschini%2C+Nora&amp;rft.au=Freedman%2C+Barry+I.&amp;rft.au=G%C3%B6ring%2C+Harald+H.+H.&amp;rft.au=Guo%2C+Xiuqing&amp;rft.au=Kalyani%2C+Rita+R.&amp;rft.au=Kooperberg%2C+Charles&amp;rft.au=Kral%2C+Brian+G.&amp;rft.au=Lange%2C+Leslie+A.&amp;rft.au=Lin%2C+Bridget+M.&amp;rft.au=Manichaikul%2C+Ani&amp;rft.au=Manning%2C+Alisa+K.&amp;rft.au=Martin%2C+Lisa+W.&amp;rft.au=Mathias%2C+Rasika+A.&amp;rft.au=Meigs%2C+James+B.&amp;rft.au=Mitchell%2C+Braxton+D.&amp;rft.au=Montasser%2C+May+E.&amp;rft.au=Morrison%2C+Alanna+C.&amp;rft.au=Naseri%2C+Take&amp;rft.au=O%E2%80%99Connell%2C+Jeffrey+R.&amp;rft.au=Palmer%2C+Nicholette+D.&amp;rft.au=Peyser%2C+Patricia+A.&amp;rft.au=Psaty%2C+Bruce+M.&amp;rft.au=Raffield%2C+Laura+M.&amp;rft.au=Redline%2C+Susan&amp;rft.au=Reiner%2C+Alexander+P.&amp;rft.au=Reupena%2C+Muagututi%E2%80%99a+Sefuiva&amp;rft.au=Rice%2C+Kenneth+M.&amp;rft.au=Rich%2C+Stephen+S.&amp;rft.au=Smith%2C+Jennifer+A.&amp;rft.au=Taylor%2C+Kent+D.&amp;rft.au=Taub%2C+Margaret+A.&amp;rft.au=Vasan%2C+Ramachandran+S.&amp;rft.au=Weeks%2C+Daniel+E.&amp;rft.au=Wilson%2C+James+G.&amp;rft.au=Yanek%2C+Lisa+R.&amp;rft.au=Zhao%2C+Wei&amp;rft.au=Abe%2C+Namiko&amp;rft.au=Abecasis%2C+Gon%C3%A7alo&amp;rft.au=Aguet%2C+Francois&amp;rft.au=Albert%2C+Christine&amp;rft.au=Almasy%2C+Laura&amp;rft.au=Alonso%2C+Alvaro&amp;rft.au=Ament%2C+Seth&amp;rft.au=Anderson%2C+Peter&amp;rft.au=Anugu%2C+Pramod&amp;rft.au=Applebaum-Bowden%2C+Deborah&amp;rft.au=Ardlie%2C+Kristin&amp;rft.au=Arking%2C+Dan&amp;rft.au=Ashley-Koch%2C+Allison&amp;rft.au=Aslibekyan%2C+Stella&amp;rft.au=Assimes%2C+Tim&amp;rft.au=Avramopoulos%2C+Dimitrios&amp;rft.au=Ayas%2C+Najib&amp;rft.au=Balasubramanian%2C+Adithya&amp;rft.au=Barnard%2C+John&amp;rft.au=Barnes%2C+Kathleen&amp;rft.au=Barr%2C+R.+Graham&amp;rft.au=Barron-Casella%2C+Emily&amp;rft.au=Barwick%2C+Lucas&amp;rft.au=Beaty%2C+Terri&amp;rft.au=Beck%2C+Gerald&amp;rft.au=Becker%2C+Diane&amp;rft.au=Becker%2C+Lewis&amp;rft.au=Beer%2C+Rebecca&amp;rft.au=Beitelshees%2C+Amber&amp;rft.au=Benjamin%2C+Emelia&amp;rft.au=Benos%2C+Takis&amp;rft.au=Bezerra%2C+Marcos&amp;rft.au=Blue%2C+Nathan&amp;rft.au=Bowler%2C+Russell&amp;rft.au=Broeckel%2C+Ulrich&amp;rft.au=Broome%2C+Jai&amp;rft.au=Brown%2C+Deborah&amp;rft.au=Bunting%2C+Karen&amp;rft.au=Burchard%2C+Esteban&amp;rft.au=Bustamante%2C+Carlos&amp;rft.au=Buth%2C+Erin&amp;rft.au=Cardwell%2C+Jonathan&amp;rft.au=Carey%2C+Vincent&amp;rft.au=Carrier%2C+Julie&amp;rft.au=Carson%2C+April&amp;rft.au=Carty%2C+Cara&amp;rft.au=Casaburi%2C+Richard&amp;rft.au=Casas+Romero%2C+Juan+P.&amp;rft.au=Casella%2C+James&amp;rft.au=Castaldi%2C+Peter&amp;rft.au=Chaffin%2C+Mark&amp;rft.au=Chang%2C+Christy&amp;rft.au=Chang%2C+Yi-Cheng&amp;rft.au=Chasman%2C+Daniel&amp;rft.au=Chavan%2C+Sameer&amp;rft.au=Chen%2C+Bo-Juen&amp;rft.au=Chen%2C+Wei-Min&amp;rft.au=Chen%2C+Yii-Der+Ida&amp;rft.au=Cho%2C+Michael&amp;rft.au=Choi%2C+Seung+Hoan&amp;rft.au=Chuang%2C+Lee-Ming&amp;rft.au=Chung%2C+Mina&amp;rft.au=Chung%2C+Ren-Hua&amp;rft.au=Clish%2C+Clary&amp;rft.au=Comhair%2C+Suzy&amp;rft.au=Cornell%2C+Elaine&amp;rft.au=Crandall%2C+Carolyn&amp;rft.au=Crapo%2C+James&amp;rft.au=Curtis%2C+Jeffrey&amp;rft.au=Custer%2C+Brian&amp;rft.au=Damcott%2C+Coleen&amp;rft.au=Darbar%2C+Dawood&amp;rft.au=David%2C+Sean&amp;rft.au=Davis%2C+Colleen&amp;rft.au=Daya%2C+Michelle&amp;rft.au=de+Andrade%2C+Mariza&amp;rft.au=Fuentes%2C+Lisa+de+las&amp;rft.au=DeBaun%2C+Michael&amp;rft.au=Deka%2C+Ranjan&amp;rft.au=DeMeo%2C+Dawn&amp;rft.au=Devine%2C+Scott&amp;rft.au=Dinh%2C+Huyen&amp;rft.au=Doddapaneni%2C+Harsha&amp;rft.au=Duan%2C+Qing&amp;rft.au=Dugan-Perez%2C+Shannon&amp;rft.au=Durda%2C+Jon+Peter&amp;rft.au=Dutcher%2C+Susan+K.&amp;rft.au=Eaton%2C+Charles&amp;rft.au=Ekunwe%2C+Lynette&amp;rft.au=El+Boueiz%2C+Adel&amp;rft.au=Ellinor%2C+Patrick&amp;rft.au=Emery%2C+Leslie&amp;rft.au=Erzurum%2C+Serpil&amp;rft.au=Farber%2C+Charles&amp;rft.au=Farek%2C+Jesse&amp;rft.au=Fingerlin%2C+Tasha&amp;rft.au=Flickinger%2C+Matthew&amp;rft.au=Fornage%2C+Myriam&amp;rft.au=Frazar%2C+Chris&amp;rft.au=Fu%2C+Mao&amp;rft.au=Fullerton%2C+Stephanie+M.&amp;rft.au=Fulton%2C+Lucinda&amp;rft.au=Gabriel%2C+Stacey&amp;rft.au=Gan%2C+Weiniu&amp;rft.au=Gao%2C+Shanshan&amp;rft.au=Gao%2C+Yan&amp;rft.au=Gass%2C+Margery&amp;rft.au=Geiger%2C+Heather&amp;rft.au=Gelb%2C+Bruce&amp;rft.au=Geraci%2C+Mark&amp;rft.au=Germer%2C+Soren&amp;rft.au=Gerszten%2C+Robert&amp;rft.au=Ghosh%2C+Auyon&amp;rft.au=Gibbs%2C+Richard&amp;rft.au=Gignoux%2C+Chris&amp;rft.au=Gladwin%2C+Mark&amp;rft.au=Glahn%2C+David&amp;rft.au=Gogarten%2C+Stephanie&amp;rft.au=Gong%2C+Da-Wei&amp;rft.au=Graw%2C+Sharon&amp;rft.au=Gray%2C+Kathryn+J.&amp;rft.au=Grine%2C+Daniel&amp;rft.au=Gross%2C+Colin&amp;rft.au=Gu%2C+C.+Charles&amp;rft.au=Guan%2C+Yue&amp;rft.au=Gupta%2C+Namrata&amp;rft.au=Hall%2C+Michael&amp;rft.au=Han%2C+Yi&amp;rft.au=Hanly%2C+Patrick&amp;rft.au=Harris%2C+Daniel&amp;rft.au=Hawley%2C+Nicola+L.&amp;rft.au=He%2C+Jiang&amp;rft.au=Heavner%2C+Ben&amp;rft.au=Heckbert%2C+Susan&amp;rft.au=Hernandez%2C+Ryan&amp;rft.au=Herrington%2C+David&amp;rft.au=Hersh%2C+Craig&amp;rft.au=Hidalgo%2C+Bertha&amp;rft.au=Hixson%2C+James&amp;rft.au=Hobbs%2C+Brian&amp;rft.au=Hokanson%2C+John&amp;rft.au=Hong%2C+Elliott&amp;rft.au=Hoth%2C+Karin&amp;rft.au=Hsiung%2C+Chao&amp;rft.au=Hu%2C+Jianhong&amp;rft.au=Hung%2C+Yi-Jen&amp;rft.au=Huston%2C+Haley&amp;rft.au=Hwu%2C+Chii+Min&amp;rft.au=Irvin%2C+Marguerite+Ryan&amp;rft.au=Jackson%2C+Rebecca&amp;rft.au=Jain%2C+Deepti&amp;rft.au=Jaquish%2C+Cashell&amp;rft.au=Johnsen%2C+Jill&amp;rft.au=Johnson%2C+Andrew&amp;rft.au=Johnson%2C+Craig&amp;rft.au=Johnston%2C+Rich&amp;rft.au=Jones%2C+Kimberly&amp;rft.au=Kang%2C+Hyun+Min&amp;rft.au=Kaplan%2C+Robert&amp;rft.au=Kardia%2C+Sharon&amp;rft.au=Kelly%2C+Shannon&amp;rft.au=Kenny%2C+Eimear&amp;rft.au=Kessler%2C+Michael&amp;rft.au=Khan%2C+Alyna&amp;rft.au=Khan%2C+Ziad&amp;rft.au=Kim%2C+Wonji&amp;rft.au=Kimoff%2C+John&amp;rft.au=Kinney%2C+Greg&amp;rft.au=Konkle%2C+Barbara&amp;rft.au=Kramer%2C+Holly&amp;rft.au=Lange%2C+Christoph&amp;rft.au=Lange%2C+Ethan&amp;rft.au=Laurie%2C+Cathy&amp;rft.au=Laurie%2C+Cecelia&amp;rft.au=LeBoff%2C+Meryl&amp;rft.au=Lee%2C+Jiwon&amp;rft.au=Lee%2C+Sandra&amp;rft.au=Lee%2C+Wen-Jane&amp;rft.au=LeFaive%2C+Jonathon&amp;rft.au=Levine%2C+David&amp;rft.au=Lewis%2C+Joshua&amp;rft.au=Li%2C+Xiaohui&amp;rft.au=Li%2C+Yun&amp;rft.au=Lin%2C+Henry&amp;rft.au=Lin%2C+Honghuang&amp;rft.au=Liu%2C+Simin&amp;rft.au=Liu%2C+Yongmei&amp;rft.au=Liu%2C+Yu&amp;rft.au=Loos%2C+Ruth+J.+F.&amp;rft.au=Lubitz%2C+Steven&amp;rft.au=Lunetta%2C+Kathryn&amp;rft.au=Luo%2C+James&amp;rft.au=Magalang%2C+Ulysses&amp;rft.au=Mahaney%2C+Michael&amp;rft.au=Make%2C+Barry&amp;rft.au=Manson%2C+JoAnn&amp;rft.au=Marton%2C+Melissa&amp;rft.au=Mathai%2C+Susan&amp;rft.au=May%2C+Susanne&amp;rft.au=McArdle%2C+Patrick&amp;rft.au=McDonald%2C+Merry-Lynn&amp;rft.au=McFarland%2C+Sean&amp;rft.au=McGoldrick%2C+Daniel&amp;rft.au=McHugh%2C+Caitlin&amp;rft.au=McNeil%2C+Becky&amp;rft.au=Mei%2C+Hao&amp;rft.au=Menon%2C+Vipin&amp;rft.au=Mestroni%2C+Luisa&amp;rft.au=Metcalf%2C+Ginger&amp;rft.au=Meyers%2C+Deborah+A.&amp;rft.au=Mignot%2C+Emmanuel&amp;rft.au=Mikulla%2C+Julie&amp;rft.au=Min%2C+Nancy&amp;rft.au=Minear%2C+Mollie&amp;rft.au=Minster%2C+Ryan+L.&amp;rft.au=Moll%2C+Matt&amp;rft.au=Momin%2C+Zeineen&amp;rft.au=Montgomery%2C+Courtney&amp;rft.au=Muzny%2C+Donna&amp;rft.au=Mychaleckyj%2C+Josyf+C.&amp;rft.au=Nadkarni%2C+Girish&amp;rft.au=Naik%2C+Rakhi&amp;rft.au=Nekhai%2C+Sergei&amp;rft.au=Nelson%2C+Sarah+C.&amp;rft.au=Neltner%2C+Bonnie&amp;rft.au=Nessner%2C+Caitlin&amp;rft.au=Nickerson%2C+Deborah&amp;rft.au=Nkechinyere%2C+Osuji&amp;rft.au=North%2C+Kari&amp;rft.au=O%E2%80%99Connor%2C+Tim&amp;rft.au=Ochs-Balcom%2C+Heather&amp;rft.au=Okwuonu%2C+Geoffrey&amp;rft.au=Pack%2C+Allan&amp;rft.au=Paik%2C+David+T.&amp;rft.au=Pankow%2C+James&amp;rft.au=Papanicolaou%2C+George&amp;rft.au=Parker%2C+Cora&amp;rft.au=Peralta%2C+Juan+Manuel&amp;rft.au=Perez%2C+Marco&amp;rft.au=Perry%2C+James&amp;rft.au=Peters%2C+Ulrike&amp;rft.au=Phillips%2C+Lawrence+S.&amp;rft.au=Pleiness%2C+Jacob&amp;rft.au=Pollin%2C+Toni&amp;rft.au=Post%2C+Wendy&amp;rft.au=Becker%2C+Julia+Powers&amp;rft.au=Boorgula%2C+Meher+Preethi&amp;rft.au=Preuss%2C+Michael&amp;rft.au=Qasba%2C+Pankaj&amp;rft.au=Qiao%2C+Dandi&amp;rft.au=Qin%2C+Zhaohui&amp;rft.au=Rafaels%2C+Nicholas&amp;rft.au=Rajendran%2C+Mahitha&amp;rft.au=Rao%2C+D.+C.&amp;rft.au=Rasmussen-Torvik%2C+Laura&amp;rft.au=Ratan%2C+Aakrosh&amp;rft.au=Reed%2C+Robert&amp;rft.au=Reeves%2C+Catherine&amp;rft.au=Regan%2C+Elizabeth&amp;rft.au=Robillard%2C+Rebecca&amp;rft.au=Robine%2C+Nicolas&amp;rft.au=Roden%2C+Dan&amp;rft.au=Roselli%2C+Carolina&amp;rft.au=Ruczinski%2C+Ingo&amp;rft.au=Runnels%2C+Alexi&amp;rft.au=Russell%2C+Pamela&amp;rft.au=Ruuska%2C+Sarah&amp;rft.au=Ryan%2C+Kathleen&amp;rft.au=Sabino%2C+Ester+Cerdeira&amp;rft.au=Saleheen%2C+Danish&amp;rft.au=Salimi%2C+Shabnam&amp;rft.au=Salvi%2C+Sejal&amp;rft.au=Salzberg%2C+Steven&amp;rft.au=Sandow%2C+Kevin&amp;rft.au=Sankaran%2C+Vijay+G.&amp;rft.au=Santibanez%2C+Jireh&amp;rft.au=Schwander%2C+Karen&amp;rft.au=Schwartz%2C+David&amp;rft.au=Sciurba%2C+Frank&amp;rft.au=Seidman%2C+Christine&amp;rft.au=Seidman%2C+Jonathan&amp;rft.au=S%C3%A9ri%C3%A8s%2C+Fr%C3%A9d%C3%A9ric&amp;rft.au=Sheehan%2C+Vivien&amp;rft.au=Sherman%2C+Stephanie+L.&amp;rft.au=Shetty%2C+Amol&amp;rft.au=Shetty%2C+Aniket&amp;rft.au=Sheu%2C+Wayne+Hui-Heng&amp;rft.au=Shoemaker%2C+M.+Benjamin&amp;rft.au=Silver%2C+Brian&amp;rft.au=Silverman%2C+Edwin&amp;rft.au=Skomro%2C+Robert&amp;rft.au=Smith%2C+Albert+Vernon&amp;rft.au=Smith%2C+Josh&amp;rft.au=Smith%2C+Nicholas&amp;rft.au=Smith%2C+Tanja&amp;rft.au=Smoller%2C+Sylvia&amp;rft.au=Snively%2C+Beverly&amp;rft.au=Snyder%2C+Michael&amp;rft.au=Sofer%2C+Tamar&amp;rft.au=Sotoodehnia%2C+Nona&amp;rft.au=Stilp%2C+Adrienne+M.&amp;rft.au=Storm%2C+Garrett&amp;rft.au=Streeten%2C+Elizabeth&amp;rft.au=Su%2C+Jessica+Lasky&amp;rft.au=Sung%2C+Yun+Ju&amp;rft.au=Sylvia%2C+Jody&amp;rft.au=Szpiro%2C+Adam&amp;rft.au=Taliun%2C+Daniel&amp;rft.au=Tang%2C+Hua&amp;rft.au=Taub%2C+Margaret&amp;rft.au=Taylor%2C+Matthew&amp;rft.au=Taylor%2C+Simeon&amp;rft.au=Telen%2C+Marilyn&amp;rft.au=Thornton%2C+Timothy+A.&amp;rft.au=Threlkeld%2C+Machiko&amp;rft.au=Tinker%2C+Lesley&amp;rft.au=Tirschwell%2C+David&amp;rft.au=Tishkoff%2C+Sarah&amp;rft.au=Tiwari%2C+Hemant&amp;rft.au=Tong%2C+Catherine&amp;rft.au=Tracy%2C+Russell&amp;rft.au=Tsai%2C+Michael&amp;rft.au=Vaidya%2C+Dhananjay&amp;rft.au=Van+Den+Berg%2C+David&amp;rft.au=VandeHaar%2C+Peter&amp;rft.au=Vrieze%2C+Scott&amp;rft.au=Walker%2C+Tarik&amp;rft.au=Wallace%2C+Robert&amp;rft.au=Walts%2C+Avram&amp;rft.au=Wang%2C+Fei+Fei&amp;rft.au=Wang%2C+Heming&amp;rft.au=Wang%2C+Jiongming&amp;rft.au=Watson%2C+Karol&amp;rft.au=Watt%2C+Jennifer&amp;rft.au=Weinstock%2C+Joshua&amp;rft.au=Weir%2C+Bruce&amp;rft.au=Weiss%2C+Scott+T.&amp;rft.au=Weng%2C+Lu-Chen&amp;rft.au=Wessel%2C+Jennifer&amp;rft.au=Williams%2C+Kayleen&amp;rft.au=Williams%2C+L.+Keoki&amp;rft.au=Wilson%2C+Carla&amp;rft.au=Winterkorn%2C+Lara&amp;rft.au=Wong%2C+Quenna&amp;rft.au=Wu%2C+Joseph&amp;rft.au=Xu%2C+Huichun&amp;rft.au=Yang%2C+Ivana&amp;rft.au=Yu%2C+Ketian&amp;rft.au=Zekavat%2C+Seyedeh+Maryam&amp;rft.au=Zhang%2C+Yingze&amp;rft.au=Zhao%2C+Snow+Xueyan&amp;rft.au=Zhu%2C+Xiaofeng&amp;rft.au=Ziv%2C+Elad&amp;rft.au=Zody%2C+Michael&amp;rft.au=Zoellner%2C+Sebastian&amp;rft.au=Atkinson%2C+Elizabeth&amp;rft.au=Ballantyne%2C+Christie&amp;rft.au=Bao%2C+Wei&amp;rft.au=Bhattacharya%2C+Romit&amp;rft.au=Bielak%2C+Larry&amp;rft.au=Bis%2C+Joshua&amp;rft.au=Bodea%2C+Corneliu&amp;rft.au=Brody%2C+Jennifer&amp;rft.au=Cade%2C+Brian&amp;rft.au=Calvo%2C+Sarah&amp;rft.au=Carlson%2C+Jenna&amp;rft.au=Chang%2C+I-Shou&amp;rft.au=Cho%2C+So+Mi&amp;rft.au=de+Vries%2C+Paul&amp;rft.au=Diallo%2C+Ana+F.&amp;rft.au=Do%2C+Ron&amp;rft.au=Dron%2C+Jacqueline&amp;rft.au=Elliott%2C+Amanda&amp;rft.au=Finucane%2C+Hilary&amp;rft.au=Floyd%2C+Caitlin&amp;rft.au=Ganna%2C+Andrea&amp;rft.au=Gong%2C+Dawei&amp;rft.au=Graham%2C+Sarah&amp;rft.au=Haas%2C+Mary&amp;rft.au=Haring%2C+Bernhard&amp;rft.au=Heemann%2C+Scott&amp;rft.au=Himes%2C+Blanca&amp;rft.au=Jarvik%2C+Gail&amp;rft.au=Jiang%2C+Jicai&amp;rft.au=Joehanes%2C+Roby&amp;rft.au=Joseph%2C+Paule+Valery&amp;rft.au=Jun%2C+Goo&amp;rft.au=Kalyani%2C+Rita&amp;rft.au=Kanai%2C+Masahiro&amp;rft.au=Kathiresan%2C+Sekar&amp;rft.au=Khera%2C+Amit&amp;rft.au=Khetarpal%2C+Sumeet&amp;rft.au=Klarin%2C+Derek&amp;rft.au=Koyama%2C+Satoshi&amp;rft.au=Kral%2C+Brian&amp;rft.au=Lange%2C+Leslie&amp;rft.au=Lemaitre%2C+Rozenn&amp;rft.au=Li%2C+Changwei&amp;rft.au=Lu%2C+Yingchang&amp;rft.au=Martin%2C+Lisa&amp;rft.au=Mathias%2C+Rasika&amp;rft.au=Mathur%2C+Ravi&amp;rft.au=McGarvey%2C+Stephen&amp;rft.au=McLenithan%2C+John&amp;rft.au=Miller%2C+Amy&amp;rft.au=Mootha%2C+Vamsi&amp;rft.au=Moran%2C+Andrew&amp;rft.au=Nakao%2C+Tetsushi&amp;rft.au=O%E2%80%99Connell%2C+Jeff&amp;rft.au=O%E2%80%99Donnell%2C+Christopher&amp;rft.au=Palmer%2C+Nicholette&amp;rft.au=Paruchuri%2C+Kaavya&amp;rft.au=Patel%2C+Aniruddh&amp;rft.au=Peloso%2C+Gina&amp;rft.au=Pettinger%2C+Mary&amp;rft.au=Peyser%2C+Patricia&amp;rft.au=Pirruccello%2C+James&amp;rft.au=Psaty%2C+Bruce&amp;rft.au=Reiner%2C+Alex&amp;rft.au=Rich%2C+Stephen&amp;rft.au=Rosenthal%2C+Samantha&amp;rft.au=Rotter%2C+Jerome&amp;rft.au=Smith%2C+Jennifer&amp;rft.au=Sunyaev%2C+Shamil+R.&amp;rft.au=Surakka%2C+Ida&amp;rft.au=Sztalryd%2C+Carole&amp;rft.au=Trinder%2C+Mark&amp;rft.au=Uddin%2C+Md+Mesbah&amp;rft.au=Urbut%2C+Sarah&amp;rft.au=Van+Buren%2C+Eric&amp;rft.au=Verbanck%2C+Marie&amp;rft.au=Von+Holle%2C+Ann&amp;rft.au=Wang%2C+Yuxuan&amp;rft.au=Wiggins%2C+Kerri&amp;rft.au=Wilkins%2C+John&amp;rft.au=Willer%2C+Cristen&amp;rft.au=Wilson%2C+James&amp;rft.au=Wolford%2C+Brooke&amp;rft.au=Yanek%2C+Lisa&amp;rft.au=Yu%2C+Zhi&amp;rft.au=Zaghloul%2C+Norann&amp;rft.au=Zhang%2C+Jingwen&amp;rft.au=Zhou%2C+Ying&amp;rft.au=Rotter%2C+Jerome+I.&amp;rft.au=Willer%2C+Cristen+J.&amp;rft.au=Natarajan%2C+Pradeep&amp;rft.au=Peloso%2C+Gina+M.&amp;rft.au=Lin%2C+Xihong&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10008172&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-42\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-42\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite class=\"citation journal cs1\">\"STAARpipeline: an all-in-one rare-variant tool for biobank-scale whole-genome sequencing data\". <i>Nature Methods</i>. <b>19</b> (12): 1532\\xe2\\x80\\x931533. December 2022. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2Fs41592-022-01641-w\">10.1038/s41592-022-01641-w</a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/36316564\">36316564</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:253246835\">253246835</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Methods&amp;rft.atitle=STAARpipeline%3A+an+all-in-one+rare-variant+tool+for+biobank-scale+whole-genome+sequencing+data&amp;rft.volume=19&amp;rft.issue=12&amp;rft.pages=1532-1533&amp;rft.date=2022-12&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A253246835%23id-name%3DS2CID&amp;rft_id=info%3Apmid%2F36316564&amp;rft_id=info%3Adoi%2F10.1038%2Fs41592-022-01641-w&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-43\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-43\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFLiQuickZhouGaynor2023\" class=\"citation journal cs1\">Li, Xihao; Quick, Corbin; Zhou, Hufeng; Gaynor, Sheila M.; Liu, Yaowu; Chen, Han; Selvaraj, Margaret Sunitha; Sun, Ryan; Dey, Rounak; Arnett, Donna K.; Bielak, Lawrence F.; Bis, Joshua C.; Blangero, John; Boerwinkle, Eric; Bowden, Donald W.; Brody, Jennifer A.; Cade, Brian E.; Correa, Adolfo; Cupples, L. Adrienne; Curran, Joanne E.; de Vries, Paul S.; Duggirala, Ravindranath; Freedman, Barry I.; G\\xc3\\xb6ring, Harald H. H.; Guo, Xiuqing; Haessler, Jeffrey; Kalyani, Rita R.; Kooperberg, Charles; Kral, Brian G.; Lange, Leslie A.; Manichaikul, Ani; Martin, Lisa W.; McGarvey, Stephen T.; Mitchell, Braxton D.; Montasser, May E.; Morrison, Alanna C.; Naseri, Take; O\\xe2\\x80\\x99Connell, Jeffrey R.; Palmer, Nicholette D.; Peyser, Patricia A.; Psaty, Bruce M.; Raffield, Laura M.; Redline, Susan; Reiner, Alexander P.; Reupena, Muagututi\\xe2\\x80\\x99a Sefuiva; Rice, Kenneth M.; Rich, Stephen S.; Sitlani, Colleen M.; Smith, Jennifer A.; Taylor, Kent D.; Vasan, Ramachandran S.; Willer, Cristen J.; Wilson, James G.; Yanek, Lisa R.; Zhao, Wei; NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium; TOPMed Lipids Working Group; Rotter, Jerome I.; Natarajan, Pradeep; Peloso, Gina M.; Li, Zilin; Lin, Xihong (January 2023). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10084891\">\"Powerful, scalable and resource-efficient meta-analysis of rare variant associations in large whole genome sequencing studies\"</a>. <i>Nature Genetics</i>. <b>55</b> (1): 154\\xe2\\x80\\x93164. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2Fs41588-022-01225-6\">10.1038/s41588-022-01225-6</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10084891\">10084891</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/36564505\">36564505</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:255084231\">255084231</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature+Genetics&amp;rft.atitle=Powerful%2C+scalable+and+resource-efficient+meta-analysis+of+rare+variant+associations+in+large+whole+genome+sequencing+studies&amp;rft.volume=55&amp;rft.issue=1&amp;rft.pages=154-164&amp;rft.date=2023-01&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10084891%23id-name%3DPMC&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A255084231%23id-name%3DS2CID&amp;rft_id=info%3Apmid%2F36564505&amp;rft_id=info%3Adoi%2F10.1038%2Fs41588-022-01225-6&amp;rft.aulast=Li&amp;rft.aufirst=Xihao&amp;rft.au=Quick%2C+Corbin&amp;rft.au=Zhou%2C+Hufeng&amp;rft.au=Gaynor%2C+Sheila+M.&amp;rft.au=Liu%2C+Yaowu&amp;rft.au=Chen%2C+Han&amp;rft.au=Selvaraj%2C+Margaret+Sunitha&amp;rft.au=Sun%2C+Ryan&amp;rft.au=Dey%2C+Rounak&amp;rft.au=Arnett%2C+Donna+K.&amp;rft.au=Bielak%2C+Lawrence+F.&amp;rft.au=Bis%2C+Joshua+C.&amp;rft.au=Blangero%2C+John&amp;rft.au=Boerwinkle%2C+Eric&amp;rft.au=Bowden%2C+Donald+W.&amp;rft.au=Brody%2C+Jennifer+A.&amp;rft.au=Cade%2C+Brian+E.&amp;rft.au=Correa%2C+Adolfo&amp;rft.au=Cupples%2C+L.+Adrienne&amp;rft.au=Curran%2C+Joanne+E.&amp;rft.au=de+Vries%2C+Paul+S.&amp;rft.au=Duggirala%2C+Ravindranath&amp;rft.au=Freedman%2C+Barry+I.&amp;rft.au=G%C3%B6ring%2C+Harald+H.+H.&amp;rft.au=Guo%2C+Xiuqing&amp;rft.au=Haessler%2C+Jeffrey&amp;rft.au=Kalyani%2C+Rita+R.&amp;rft.au=Kooperberg%2C+Charles&amp;rft.au=Kral%2C+Brian+G.&amp;rft.au=Lange%2C+Leslie+A.&amp;rft.au=Manichaikul%2C+Ani&amp;rft.au=Martin%2C+Lisa+W.&amp;rft.au=McGarvey%2C+Stephen+T.&amp;rft.au=Mitchell%2C+Braxton+D.&amp;rft.au=Montasser%2C+May+E.&amp;rft.au=Morrison%2C+Alanna+C.&amp;rft.au=Naseri%2C+Take&amp;rft.au=O%E2%80%99Connell%2C+Jeffrey+R.&amp;rft.au=Palmer%2C+Nicholette+D.&amp;rft.au=Peyser%2C+Patricia+A.&amp;rft.au=Psaty%2C+Bruce+M.&amp;rft.au=Raffield%2C+Laura+M.&amp;rft.au=Redline%2C+Susan&amp;rft.au=Reiner%2C+Alexander+P.&amp;rft.au=Reupena%2C+Muagututi%E2%80%99a+Sefuiva&amp;rft.au=Rice%2C+Kenneth+M.&amp;rft.au=Rich%2C+Stephen+S.&amp;rft.au=Sitlani%2C+Colleen+M.&amp;rft.au=Smith%2C+Jennifer+A.&amp;rft.au=Taylor%2C+Kent+D.&amp;rft.au=Vasan%2C+Ramachandran+S.&amp;rft.au=Willer%2C+Cristen+J.&amp;rft.au=Wilson%2C+James+G.&amp;rft.au=Yanek%2C+Lisa+R.&amp;rft.au=Zhao%2C+Wei&amp;rft.au=NHLBI+Trans-Omics+for+Precision+Medicine+%28TOPMed%29+Consortium&amp;rft.au=TOPMed+Lipids+Working+Group&amp;rft.au=Rotter%2C+Jerome+I.&amp;rft.au=Natarajan%2C+Pradeep&amp;rft.au=Peloso%2C+Gina+M.&amp;rft.au=Li%2C+Zilin&amp;rft.au=Lin%2C+Xihong&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10084891&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-44\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-44\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFVazquezde_la_TorreValencia2012\" class=\"citation journal cs1\">Vazquez M, de la Torre V, Valencia A (27 December 2012). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531315\">\"Chapter 14: Cancer genome analysis\"</a>. <i>PLOS Computational Biology</i>. <b>8</b> (12): e1002824. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/2012PLSCB...8E2824V\">2012PLSCB...8E2824V</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1371%2Fjournal.pcbi.1002824\">10.1371/journal.pcbi.1002824</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531315\">3531315</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/23300415\">23300415</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+Computational+Biology&amp;rft.atitle=Chapter+14%3A+Cancer+genome+analysis&amp;rft.volume=8&amp;rft.issue=12&amp;rft.pages=e1002824&amp;rft.date=2012-12-27&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3531315%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F23300415&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.1002824&amp;rft_id=info%3Abibcode%2F2012PLSCB...8E2824V&amp;rft.aulast=Vazquez&amp;rft.aufirst=M&amp;rft.au=de+la+Torre%2C+V&amp;rft.au=Valencia%2C+A&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3531315&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-45\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-45\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFHye-JungJaswinderMartinSamuel2014\" class=\"citation book cs1\">Hye-Jung EC, Jaswinder K, Martin K, Samuel AA, Marco AM (2014). \"Second-Generation Sequencing for Cancer Genome Analysis\".  In Dellaire G, Berman JN, Arceci RJ (eds.). <i>Cancer Genomics</i>. Boston (US): Academic Press. pp.&#160;13\\xe2\\x80\\x9330. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1016%2FB978-0-12-396967-5.00002-5\">10.1016/B978-0-12-396967-5.00002-5</a>. <a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/9780123969675\" title=\"Special:BookSources/9780123969675\"><bdi>9780123969675</bdi></a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=bookitem&amp;rft.atitle=Second-Generation+Sequencing+for+Cancer+Genome+Analysis&amp;rft.btitle=Cancer+Genomics&amp;rft.place=Boston+%28US%29&amp;rft.pages=13-30&amp;rft.pub=Academic+Press&amp;rft.date=2014&amp;rft_id=info%3Adoi%2F10.1016%2FB978-0-12-396967-5.00002-5&amp;rft.isbn=9780123969675&amp;rft.aulast=Hye-Jung&amp;rft.aufirst=EC&amp;rft.au=Jaswinder%2C+K&amp;rft.au=Martin%2C+K&amp;rft.au=Samuel%2C+AA&amp;rft.au=Marco%2C+AM&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-46\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-46\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFGrauBen-GalPoschGrosse2006\" class=\"citation journal cs1\">Grau J, Ben-Gal I, Posch S, Grosse I (July 2006). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1538886\">\"VOMBAT: prediction of transcription factor binding sites using variable order Bayesian trees\"</a>. <i>Nucleic Acids Research</i>. <b>34</b> (Web Server issue): W529-33. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1093%2Fnar%2Fgkl212\">10.1093/nar/gkl212</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1538886\">1538886</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/16845064\">16845064</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.atitle=VOMBAT%3A+prediction+of+transcription+factor+binding+sites+using+variable+order+Bayesian+trees&amp;rft.volume=34&amp;rft.issue=Web+Server+issue&amp;rft.pages=W529-33&amp;rft.date=2006-07&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1538886%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F16845064&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkl212&amp;rft.aulast=Grau&amp;rft.aufirst=J&amp;rft.au=Ben-Gal%2C+I&amp;rft.au=Posch%2C+S&amp;rft.au=Grosse%2C+I&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1538886&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-47\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-47\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite class=\"citation web cs1\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.proteinatlas.org/\">\"The Human Protein Atlas\"</a>. <i>www.proteinatlas.org</i>. <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20200304041657/http://www.proteinatlas.org/\">Archived</a> from the original on 4 March 2020<span class=\"reference-accessdate\">. Retrieved <span class=\"nowrap\">2 October</span> 2017</span>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=www.proteinatlas.org&amp;rft.atitle=The+Human+Protein+Atlas&amp;rft_id=https%3A%2F%2Fwww.proteinatlas.org%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-48\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-48\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite class=\"citation web cs1\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.proteinatlas.org/humancell\">\"The human cell\"</a>. <i>www.proteinatlas.org</i>. <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20171002215345/https://www.proteinatlas.org/humancell\">Archived</a> from the original on 2 October 2017<span class=\"reference-accessdate\">. Retrieved <span class=\"nowrap\">2 October</span> 2017</span>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=www.proteinatlas.org&amp;rft.atitle=The+human+cell&amp;rft_id=https%3A%2F%2Fwww.proteinatlas.org%2Fhumancell&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-49\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-49\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFThul\\xc3\\x85kessonWikingMahdessian2017\" class=\"citation journal cs1\">Thul PJ, \\xc3\\x85kesson L, Wiking M, Mahdessian D, Geladaki A, Ait Blal H,  et&#160;al. (May 2017). \"A subcellular map of the human proteome\". <i>Science</i>. <b>356</b> (6340): eaal3321. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1126%2Fscience.aal3321\">10.1126/science.aal3321</a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/28495876\">28495876</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:10744558\">10744558</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Science&amp;rft.atitle=A+subcellular+map+of+the+human+proteome&amp;rft.volume=356&amp;rft.issue=6340&amp;rft.pages=eaal3321&amp;rft.date=2017-05&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A10744558%23id-name%3DS2CID&amp;rft_id=info%3Apmid%2F28495876&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.aal3321&amp;rft.aulast=Thul&amp;rft.aufirst=PJ&amp;rft.au=%C3%85kesson%2C+L&amp;rft.au=Wiking%2C+M&amp;rft.au=Mahdessian%2C+D&amp;rft.au=Geladaki%2C+A&amp;rft.au=Ait+Blal%2C+H&amp;rft.au=Alm%2C+T&amp;rft.au=Asplund%2C+A&amp;rft.au=Bj%C3%B6rk%2C+L&amp;rft.au=Breckels%2C+LM&amp;rft.au=B%C3%A4ckstr%C3%B6m%2C+A&amp;rft.au=Danielsson%2C+F&amp;rft.au=Fagerberg%2C+L&amp;rft.au=Fall%2C+J&amp;rft.au=Gatto%2C+L&amp;rft.au=Gnann%2C+C&amp;rft.au=Hober%2C+S&amp;rft.au=Hjelmare%2C+M&amp;rft.au=Johansson%2C+F&amp;rft.au=Lee%2C+S&amp;rft.au=Lindskog%2C+C&amp;rft.au=Mulder%2C+J&amp;rft.au=Mulvey%2C+CM&amp;rft.au=Nilsson%2C+P&amp;rft.au=Oksvold%2C+P&amp;rft.au=Rockberg%2C+J&amp;rft.au=Schutten%2C+R&amp;rft.au=Schwenk%2C+JM&amp;rft.au=Sivertsson%2C+%C3%85&amp;rft.au=Sj%C3%B6stedt%2C+E&amp;rft.au=Skogs%2C+M&amp;rft.au=Stadler%2C+C&amp;rft.au=Sullivan%2C+DP&amp;rft.au=Tegel%2C+H&amp;rft.au=Winsnes%2C+C&amp;rft.au=Zhang%2C+C&amp;rft.au=Zwahlen%2C+M&amp;rft.au=Mardinoglu%2C+A&amp;rft.au=Pont%C3%A9n%2C+F&amp;rft.au=von+Feilitzen%2C+K&amp;rft.au=Lilley%2C+KS&amp;rft.au=Uhl%C3%A9n%2C+M&amp;rft.au=Lundberg%2C+E&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-50\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-50\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFAyNoble2015\" class=\"citation journal cs1\">Ay F, Noble WS (September 2015). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4556012\">\"Analysis methods for studying the 3D architecture of the genome\"</a>. <i>Genome Biology</i>. <b>16</b> (1): 183. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1186%2Fs13059-015-0745-7\">10.1186/s13059-015-0745-7</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4556012\">4556012</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/26328929\">26328929</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Genome+Biology&amp;rft.atitle=Analysis+methods+for+studying+the+3D+architecture+of+the+genome&amp;rft.volume=16&amp;rft.issue=1&amp;rft.pages=183&amp;rft.date=2015-09&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4556012%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F26328929&amp;rft_id=info%3Adoi%2F10.1186%2Fs13059-015-0745-7&amp;rft.aulast=Ay&amp;rft.aufirst=F&amp;rft.au=Noble%2C+WS&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4556012&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-51\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-51\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFKryshtafovychSchwedeTopfFidelis2019\" class=\"citation journal cs1\">Kryshtafovych, A.; Schwede, T.; Topf, M.; Fidelis, K.; Moult, J. (2019). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6927249\">\"Critical Assessment of Methods of Protein Structure Prediction (CASP) \\xe2\\x80\\x93 Round XIII\"</a>. <i>Proteins</i>. <b>87</b> (12): 1011\\xe2\\x80\\x931020. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1002%2Fprot.25823\">10.1002/prot.25823</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6927249\">6927249</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/31589781\">31589781</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Proteins&amp;rft.atitle=Critical+Assessment+of+Methods+of+Protein+Structure+Prediction+%28CASP%29+%E2%80%93+Round+XIII&amp;rft.volume=87&amp;rft.issue=12&amp;rft.pages=1011-1020&amp;rft.date=2019&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6927249%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F31589781&amp;rft_id=info%3Adoi%2F10.1002%2Fprot.25823&amp;rft.aulast=Kryshtafovych&amp;rft.aufirst=A.&amp;rft.au=Schwede%2C+T.&amp;rft.au=Topf%2C+M.&amp;rft.au=Fidelis%2C+K.&amp;rft.au=Moult%2C+J.&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6927249&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-52\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-52\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite class=\"citation web cs1\"><a rel=\"nofollow\" class=\"external text\" href=\"https://predictioncenter.org/casp14/\">\"Home - CASP14\"</a>. <i>predictioncenter.org</i>. <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20230130200222/https://predictioncenter.org/casp14/\">Archived</a> from the original on 30 January 2023<span class=\"reference-accessdate\">. Retrieved <span class=\"nowrap\">12 June</span> 2023</span>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=predictioncenter.org&amp;rft.atitle=Home+-+CASP14&amp;rft_id=https%3A%2F%2Fpredictioncenter.org%2Fcasp14%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-53\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-53\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFHoyRobinsonTrentKakar2007\" class=\"citation journal cs1\">Hoy JA, Robinson H, Trent JT, Kakar S, Smagghe BJ, Hargrove MS (August 2007). \"Plant hemoglobins: a molecular fossil record for the evolution of oxygen transport\". <i>Journal of Molecular Biology</i>. <b>371</b> (1): 168\\xe2\\x80\\x9379. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1016%2Fj.jmb.2007.05.029\">10.1016/j.jmb.2007.05.029</a>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/17560601\">17560601</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Journal+of+Molecular+Biology&amp;rft.atitle=Plant+hemoglobins%3A+a+molecular+fossil+record+for+the+evolution+of+oxygen+transport&amp;rft.volume=371&amp;rft.issue=1&amp;rft.pages=168-79&amp;rft.date=2007-08&amp;rft_id=info%3Adoi%2F10.1016%2Fj.jmb.2007.05.029&amp;rft_id=info%3Apmid%2F17560601&amp;rft.aulast=Hoy&amp;rft.aufirst=JA&amp;rft.au=Robinson%2C+H&amp;rft.au=Trent%2C+JT&amp;rft.au=Kakar%2C+S&amp;rft.au=Smagghe%2C+BJ&amp;rft.au=Hargrove%2C+MS&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-54\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-54\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFJumperEvansPritzelGreen2021\" class=\"citation journal cs1\">Jumper, John; Evans, Richard; Pritzel, Alexander; Green, Tim; Figurnov, Michael; Ronneberger, Olaf; Tunyasuvunakool, Kathryn; Bates, Russ; \\xc5\\xbd\\xc3\\xaddek, Augustin; Potapenko, Anna; Bridgland, Alex; Meyer, Clemens; Kohl, Simon A. A.; Ballard, Andrew J.; Cowie, Andrew (August 2021). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8371605\">\"Highly accurate protein structure prediction with AlphaFold\"</a>. <i>Nature</i>. <b>596</b> (7873): 583\\xe2\\x80\\x93589. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/2021Natur.596..583J\">2021Natur.596..583J</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1038%2Fs41586-021-03819-2\">10.1038/s41586-021-03819-2</a>. <a href=\"/wiki/ISSN_(identifier)\" class=\"mw-redirect\" title=\"ISSN (identifier)\">ISSN</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://www.worldcat.org/issn/1476-4687\">1476-4687</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8371605\">8371605</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/34265844\">34265844</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Nature&amp;rft.atitle=Highly+accurate+protein+structure+prediction+with+AlphaFold&amp;rft.volume=596&amp;rft.issue=7873&amp;rft.pages=583-589&amp;rft.date=2021-08&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8371605%23id-name%3DPMC&amp;rft_id=info%3Abibcode%2F2021Natur.596..583J&amp;rft_id=info%3Apmid%2F34265844&amp;rft_id=info%3Adoi%2F10.1038%2Fs41586-021-03819-2&amp;rft.issn=1476-4687&amp;rft.aulast=Jumper&amp;rft.aufirst=John&amp;rft.au=Evans%2C+Richard&amp;rft.au=Pritzel%2C+Alexander&amp;rft.au=Green%2C+Tim&amp;rft.au=Figurnov%2C+Michael&amp;rft.au=Ronneberger%2C+Olaf&amp;rft.au=Tunyasuvunakool%2C+Kathryn&amp;rft.au=Bates%2C+Russ&amp;rft.au=%C5%BD%C3%ADdek%2C+Augustin&amp;rft.au=Potapenko%2C+Anna&amp;rft.au=Bridgland%2C+Alex&amp;rft.au=Meyer%2C+Clemens&amp;rft.au=Kohl%2C+Simon+A.+A.&amp;rft.au=Ballard%2C+Andrew+J.&amp;rft.au=Cowie%2C+Andrew&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8371605&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-55\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-55\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite class=\"citation web cs1\"><a rel=\"nofollow\" class=\"external text\" href=\"https://alphafold.ebi.ac.uk/\">\"AlphaFold Protein Structure Database\"</a>. <i>alphafold.ebi.ac.uk</i>. <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20210724013505/https://alphafold.ebi.ac.uk/\">Archived</a> from the original on 24 July 2021<span class=\"reference-accessdate\">. Retrieved <span class=\"nowrap\">10 October</span> 2022</span>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=alphafold.ebi.ac.uk&amp;rft.atitle=AlphaFold+Protein+Structure+Database&amp;rft_id=https%3A%2F%2Falphafold.ebi.ac.uk%2F&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-56\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-56\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFTitzRajagopalaGollH\\xc3\\xa4user2008\" class=\"citation journal cs1\">Titz B, Rajagopala SV, Goll J, H\\xc3\\xa4user R, McKevitt MT, Palzkill T, Uetz P (May 2008).  Hall N (ed.). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2386257\">\"The binary protein interactome of Treponema pallidum--the syphilis spirochete\"</a>. <i>PLOS ONE</i>. <b>3</b> (5): e2292. <a href=\"/wiki/Bibcode_(identifier)\" class=\"mw-redirect\" title=\"Bibcode (identifier)\">Bibcode</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://ui.adsabs.harvard.edu/abs/2008PLoSO...3.2292T\">2008PLoSO...3.2292T</a>. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1371%2Fjournal.pone.0002292\">10.1371/journal.pone.0002292</a></span>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2386257\">2386257</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/18509523\">18509523</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+ONE&amp;rft.atitle=The+binary+protein+interactome+of+Treponema+pallidum--the+syphilis+spirochete&amp;rft.volume=3&amp;rft.issue=5&amp;rft.pages=e2292&amp;rft.date=2008-05&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2386257%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F18509523&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pone.0002292&amp;rft_id=info%3Abibcode%2F2008PLoSO...3.2292T&amp;rft.aulast=Titz&amp;rft.aufirst=B&amp;rft.au=Rajagopala%2C+SV&amp;rft.au=Goll%2C+J&amp;rft.au=H%C3%A4user%2C+R&amp;rft.au=McKevitt%2C+MT&amp;rft.au=Palzkill%2C+T&amp;rft.au=Uetz%2C+P&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2386257&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-obf-main-57\"><span class=\"mw-cite-backlink\">^ <a href=\"#cite_ref-obf-main_57-0\"><sup><i><b>a</b></i></sup></a> <a href=\"#cite_ref-obf-main_57-1\"><sup><i><b>b</b></i></sup></a></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite class=\"citation web cs1\"><a rel=\"nofollow\" class=\"external text\" href=\"http://www.open-bio.org/wiki/Main_Page\">\"Open Bioinformatics Foundation: About us\"</a>. <i>Official website</i>. <a href=\"/wiki/Open_Bioinformatics_Foundation\" title=\"Open Bioinformatics Foundation\">Open Bioinformatics Foundation</a>. <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20110512022059/http://open-bio.org/wiki/Main_Page\">Archived</a> from the original on 12 May 2011<span class=\"reference-accessdate\">. Retrieved <span class=\"nowrap\">10 May</span> 2011</span>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=Official+website&amp;rft.atitle=Open+Bioinformatics+Foundation%3A+About+us&amp;rft_id=http%3A%2F%2Fwww.open-bio.org%2Fwiki%2FMain_Page&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-obf-bosc-58\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-obf-bosc_58-0\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite class=\"citation web cs1\"><a rel=\"nofollow\" class=\"external text\" href=\"http://www.open-bio.org/wiki/BOSC\">\"Open Bioinformatics Foundation: BOSC\"</a>. <i>Official website</i>. <a href=\"/wiki/Open_Bioinformatics_Foundation\" title=\"Open Bioinformatics Foundation\">Open Bioinformatics Foundation</a>. <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20110718175922/http://www.open-bio.org/wiki/BOSC\">Archived</a> from the original on 18 July 2011<span class=\"reference-accessdate\">. Retrieved <span class=\"nowrap\">10 May</span> 2011</span>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=Official+website&amp;rft.atitle=Open+Bioinformatics+Foundation%3A+BOSC&amp;rft_id=http%3A%2F%2Fwww.open-bio.org%2Fwiki%2FBOSC&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-59\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-59\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFNisbetElder_IVMiner2009\" class=\"citation book cs1\">Nisbet R, Elder IV J, Miner G (2009). <a rel=\"nofollow\" class=\"external text\" href=\"https://books.google.com/books?id=U5np34a5fmQC&amp;q=bioinformatics%20service%20categories%20EBI&amp;pg=PA328\">\"Bioinformatics\"</a>. <i>Handbook of Statistical Analysis and Data Mining Applications</i>. Academic Press. p.&#160;328. <a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/978-0080912035\" title=\"Special:BookSources/978-0080912035\"><bdi>978-0080912035</bdi></a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=bookitem&amp;rft.atitle=Bioinformatics&amp;rft.btitle=Handbook+of+Statistical+Analysis+and+Data+Mining+Applications&amp;rft.pages=328&amp;rft.pub=Academic+Press&amp;rft.date=2009&amp;rft.isbn=978-0080912035&amp;rft.aulast=Nisbet&amp;rft.aufirst=R&amp;rft.au=Elder+IV%2C+J&amp;rft.au=Miner%2C+G&amp;rft_id=https%3A%2F%2Fbooks.google.com%2Fbooks%3Fid%3DU5np34a5fmQC%26q%3Dbioinformatics%2520service%2520categories%2520EBI%26pg%3DPA328&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-60\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-60\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFOffice_of_the_Commissioner\" class=\"citation web cs1\">Office of the Commissioner. <a rel=\"nofollow\" class=\"external text\" href=\"https://www.fda.gov/ScienceResearch/SpecialTopics/RegulatoryScience/ucm389561.htm\">\"Advancing Regulatory Science \\xe2\\x80\\x93 Sept. 24\\xe2\\x80\\x9325, 2014 Public Workshop: Next Generation Sequencing Standards\"</a>. <i>www.fda.gov</i>. <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20171114200347/https://www.fda.gov/ScienceResearch/SpecialTopics/RegulatoryScience/ucm389561.htm\">Archived</a> from the original on 14 November 2017<span class=\"reference-accessdate\">. Retrieved <span class=\"nowrap\">30 November</span> 2017</span>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=www.fda.gov&amp;rft.atitle=Advancing+Regulatory+Science+%E2%80%93+Sept.+24%E2%80%9325%2C+2014+Public+Workshop%3A+Next+Generation+Sequencing+Standards&amp;rft.au=Office+of+the+Commissioner&amp;rft_id=https%3A%2F%2Fwww.fda.gov%2FScienceResearch%2FSpecialTopics%2FRegulatoryScience%2Fucm389561.htm&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-61\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-61\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFSimonyanGoecksMazumder2017\" class=\"citation journal cs1\">Simonyan V, Goecks J, Mazumder R (2017). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5510742\">\"Biocompute Objects-A Step towards Evaluation and Validation of Biomedical Scientific Computations\"</a>. <i>PDA Journal of Pharmaceutical Science and Technology</i>. <b>71</b> (2): 136\\xe2\\x80\\x93146. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.5731%2Fpdajpst.2016.006734\">10.5731/pdajpst.2016.006734</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5510742\">5510742</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/27974626\">27974626</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PDA+Journal+of+Pharmaceutical+Science+and+Technology&amp;rft.atitle=Biocompute+Objects-A+Step+towards+Evaluation+and+Validation+of+Biomedical+Scientific+Computations&amp;rft.volume=71&amp;rft.issue=2&amp;rft.pages=136-146&amp;rft.date=2017&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5510742%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F27974626&amp;rft_id=info%3Adoi%2F10.5731%2Fpdajpst.2016.006734&amp;rft.aulast=Simonyan&amp;rft.aufirst=V&amp;rft.au=Goecks%2C+J&amp;rft.au=Mazumder%2C+R&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5510742&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-62\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-62\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFOffice_of_the_Commissioner\" class=\"citation web cs1\">Office of the Commissioner. <a rel=\"nofollow\" class=\"external text\" href=\"https://www.fda.gov/ScienceResearch/SpecialTopics/RegulatoryScience/ucm491893.htm\">\"Advancing Regulatory Science \\xe2\\x80\\x93 Community-based development of HTS standards for validating data and computation and encouraging interoperability\"</a>. <i>www.fda.gov</i>. <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20180126133504/https://www.fda.gov/ScienceResearch/SpecialTopics/RegulatoryScience/ucm491893.htm\">Archived</a> from the original on 26 January 2018<span class=\"reference-accessdate\">. Retrieved <span class=\"nowrap\">30 November</span> 2017</span>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=unknown&amp;rft.jtitle=www.fda.gov&amp;rft.atitle=Advancing+Regulatory+Science+%E2%80%93+Community-based+development+of+HTS+standards+for+validating+data+and+computation+and+encouraging+interoperability&amp;rft.au=Office+of+the+Commissioner&amp;rft_id=https%3A%2F%2Fwww.fda.gov%2FScienceResearch%2FSpecialTopics%2FRegulatoryScience%2Fucm491893.htm&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-63\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-63\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFAlterovitzDeanGobleCrusoe2018\" class=\"citation journal cs1\">Alterovitz G, Dean D, Goble C, Crusoe MR, Soiland-Reyes S, Bell A,  et&#160;al. (December 2018). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6338479\">\"Enabling precision medicine via standard communication of HTS provenance, analysis, and results\"</a>. <i>PLOS Biology</i>. <b>16</b> (12): e3000099. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1371%2Fjournal.pbio.3000099\">10.1371/journal.pbio.3000099</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6338479\">6338479</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/30596645\">30596645</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=PLOS+Biology&amp;rft.atitle=Enabling+precision+medicine+via+standard+communication+of+HTS+provenance%2C+analysis%2C+and+results&amp;rft.volume=16&amp;rft.issue=12&amp;rft.pages=e3000099&amp;rft.date=2018-12&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6338479%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F30596645&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pbio.3000099&amp;rft.aulast=Alterovitz&amp;rft.aufirst=G&amp;rft.au=Dean%2C+D&amp;rft.au=Goble%2C+C&amp;rft.au=Crusoe%2C+MR&amp;rft.au=Soiland-Reyes%2C+S&amp;rft.au=Bell%2C+A&amp;rft.au=Hayes%2C+A&amp;rft.au=Suresh%2C+A&amp;rft.au=Purkayastha%2C+A&amp;rft.au=King%2C+CH&amp;rft.au=Taylor%2C+D&amp;rft.au=Johanson%2C+E&amp;rft.au=Thompson%2C+EE&amp;rft.au=Donaldson%2C+E&amp;rft.au=Morizono%2C+H&amp;rft.au=Tsang%2C+H&amp;rft.au=Vora%2C+JK&amp;rft.au=Goecks%2C+J&amp;rft.au=Yao%2C+J&amp;rft.au=Almeida%2C+JS&amp;rft.au=Keeney%2C+J&amp;rft.au=Addepalli%2C+K&amp;rft.au=Krampis%2C+K&amp;rft.au=Smith%2C+KM&amp;rft.au=Guo%2C+L&amp;rft.au=Walderhaug%2C+M&amp;rft.au=Schito%2C+M&amp;rft.au=Ezewudo%2C+M&amp;rft.au=Guimera%2C+N&amp;rft.au=Walsh%2C+P&amp;rft.au=Kahsay%2C+R&amp;rft.au=Gottipati%2C+S&amp;rft.au=Rodwell%2C+TC&amp;rft.au=Bloom%2C+T&amp;rft.au=Lai%2C+Y&amp;rft.au=Simonyan%2C+V&amp;rft.au=Mazumder%2C+R&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC6338479&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-64\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-64\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite class=\"citation cs2\"><a rel=\"nofollow\" class=\"external text\" href=\"https://github.com/biocompute-objects/HTS-CSRS\"><i>BioCompute Object (BCO) project is a collaborative and community-driven framework to standardize HTS computational data. 1. BCO Specification Document: user manual for understanding and creating B.</i></a>, biocompute-objects, 3 September 2017, <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20180627081221/https://github.com/biocompute-objects/HTS-CSRS\">archived</a> from the original on 27 June 2018<span class=\"reference-accessdate\">, retrieved <span class=\"nowrap\">30 November</span> 2017</span></cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=BioCompute+Object+%28BCO%29+project+is+a+collaborative+and+community-driven+framework+to+standardize+HTS+computational+data.+1.+BCO+Specification+Document%3A+user+manual+for+understanding+and+creating+B.&amp;rft.pub=biocompute-objects&amp;rft.date=2017-09-03&amp;rft_id=https%3A%2F%2Fgithub.com%2Fbiocompute-objects%2FHTS-CSRS&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-65\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-65\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFCampbell2003\" class=\"citation journal cs1\">Campbell, A. Malcolm (1 June 2003). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC162192\">\"Public Access for Teaching Genomics, Proteomics, and Bioinformatics\"</a>. <i>Cell Biology Education</i>. <b>2</b> (2): 98\\xe2\\x80\\x93111. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1187%2Fcbe.03-02-0007\">10.1187/cbe.03-02-0007</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC162192\">162192</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/12888845\">12888845</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Cell+Biology+Education&amp;rft.atitle=Public+Access+for+Teaching+Genomics%2C+Proteomics%2C+and+Bioinformatics&amp;rft.volume=2&amp;rft.issue=2&amp;rft.pages=98-111&amp;rft.date=2003-06-01&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC162192%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F12888845&amp;rft_id=info%3Adoi%2F10.1187%2Fcbe.03-02-0007&amp;rft.aulast=Campbell&amp;rft.aufirst=A.+Malcolm&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC162192&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-66\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-66\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFArenas2021\" class=\"citation journal cs1\">Arenas, Miguel (September 2021). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8426940\">\"General considerations for online teaching practices in bioinformatics in the time of COVID \\xe2\\x80\\x9019\"</a>. <i>Biochemistry and Molecular Biology Education</i>. <b>49</b> (5): 683\\xe2\\x80\\x93684. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1002%2Fbmb.21558\">10.1002/bmb.21558</a>. <a href=\"/wiki/ISSN_(identifier)\" class=\"mw-redirect\" title=\"ISSN (identifier)\">ISSN</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://www.worldcat.org/issn/1470-8175\">1470-8175</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8426940\">8426940</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/34231941\">34231941</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Biochemistry+and+Molecular+Biology+Education&amp;rft.atitle=General+considerations+for+online+teaching+practices+in+bioinformatics+in+the+time+of+COVID+%E2%80%9019&amp;rft.volume=49&amp;rft.issue=5&amp;rft.pages=683-684&amp;rft.date=2021-09&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8426940%23id-name%3DPMC&amp;rft.issn=1470-8175&amp;rft_id=info%3Apmid%2F34231941&amp;rft_id=info%3Adoi%2F10.1002%2Fbmb.21558&amp;rft.aulast=Arenas&amp;rft.aufirst=Miguel&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8426940&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-67\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-67\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFBarkerFerrierHollandMitchell2013\" class=\"citation journal cs1\">Barker D, Ferrier DE, Holland PW, Mitchell JB, Plaisier H, Ritchie MG, Smart SD (August 2013). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3751261\">\"4273\\xcf\\x80: bioinformatics education on low cost ARM hardware\"</a>. <i>BMC Bioinformatics</i>. <b>13</b>: 522. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1186%2F1471-2105-14-243\">10.1186/1471-2105-14-243</a></span>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3751261\">3751261</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/23937194\">23937194</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.atitle=4273%CF%80%3A+bioinformatics+education+on+low+cost+ARM+hardware&amp;rft.volume=13&amp;rft.pages=522&amp;rft.date=2013-08&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3751261%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F23937194&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-14-243&amp;rft.aulast=Barker&amp;rft.aufirst=D&amp;rft.au=Ferrier%2C+DE&amp;rft.au=Holland%2C+PW&amp;rft.au=Mitchell%2C+JB&amp;rft.au=Plaisier%2C+H&amp;rft.au=Ritchie%2C+MG&amp;rft.au=Smart%2C+SD&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3751261&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-68\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-68\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFBarkerAldersonMcDonaghPlaisier2015\" class=\"citation journal cs1\">Barker D, Alderson RG, McDonagh JL, Plaisier H, Comrie MM, Duncan L,  et&#160;al. (2015). \"University-level practical activities in bioinformatics benefit voluntary groups of pupils in the last 2 years of school\". <i>International Journal of STEM Education</i>. <b>2</b> (17). <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1186%2Fs40594-015-0030-z\">10.1186/s40594-015-0030-z</a>. <a href=\"/wiki/S2CID_(identifier)\" class=\"mw-redirect\" title=\"S2CID (identifier)\">S2CID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://api.semanticscholar.org/CorpusID:256396656\">256396656</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=International+Journal+of+STEM+Education&amp;rft.atitle=University-level+practical+activities+in+bioinformatics+benefit+voluntary+groups+of+pupils+in+the+last+2+years+of+school&amp;rft.volume=2&amp;rft.issue=17&amp;rft.date=2015&amp;rft_id=info%3Adoi%2F10.1186%2Fs40594-015-0030-z&amp;rft_id=https%3A%2F%2Fapi.semanticscholar.org%2FCorpusID%3A256396656%23id-name%3DS2CID&amp;rft.aulast=Barker&amp;rft.aufirst=D&amp;rft.au=Alderson%2C+RG&amp;rft.au=McDonagh%2C+JL&amp;rft.au=Plaisier%2C+H&amp;rft.au=Comrie%2C+MM&amp;rft.au=Duncan%2C+L&amp;rft.au=Muirhead%2C+GT&amp;rft.au=Sweeney%2C+SD&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-69\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-69\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFMcDonaghBarkerAlderson2016\" class=\"citation journal cs1\">McDonagh JL, Barker D, Alderson RG (2016). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4775721\">\"Bringing computational science to the public\"</a>. <i>SpringerPlus</i>. <b>5</b> (259): 259. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1186%2Fs40064-016-1856-7\">10.1186/s40064-016-1856-7</a>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4775721\">4775721</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/27006868\">27006868</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=SpringerPlus&amp;rft.atitle=Bringing+computational+science+to+the+public&amp;rft.volume=5&amp;rft.issue=259&amp;rft.pages=259&amp;rft.date=2016&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4775721%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F27006868&amp;rft_id=info%3Adoi%2F10.1186%2Fs40064-016-1856-7&amp;rft.aulast=McDonagh&amp;rft.aufirst=JL&amp;rft.au=Barker%2C+D&amp;rft.au=Alderson%2C+RG&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4775721&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-70\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-70\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFRobsonBarker2015\" class=\"citation journal cs1\">Robson JF, Barker D (October 2015). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4604092\">\"Comparison of the protein-coding gene content of Chlamydia trachomatis and Protochlamydia amoebophila using a Raspberry Pi computer\"</a>. <i>BMC Research Notes</i>. <b>8</b> (561): 561. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1186%2Fs13104-015-1476-2\">10.1186/s13104-015-1476-2</a></span>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4604092\">4604092</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/26462790\">26462790</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Research+Notes&amp;rft.atitle=Comparison+of+the+protein-coding+gene+content+of+Chlamydia+trachomatis+and+Protochlamydia+amoebophila+using+a+Raspberry+Pi+computer&amp;rft.volume=8&amp;rft.issue=561&amp;rft.pages=561&amp;rft.date=2015-10&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4604092%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F26462790&amp;rft_id=info%3Adoi%2F10.1186%2Fs13104-015-1476-2&amp;rft.aulast=Robson&amp;rft.aufirst=JF&amp;rft.au=Barker%2C+D&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4604092&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n<li id=\"cite_note-71\"><span class=\"mw-cite-backlink\"><b><a href=\"#cite_ref-71\">^</a></b></span> <span class=\"reference-text\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFWreggelsworthBarker2015\" class=\"citation journal cs1\">Wreggelsworth KM, Barker D (October 2015). <a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4606965\">\"A comparison of the protein-coding genomes of two green sulphur bacteria, Chlorobium tepidum TLS and Pelodictyon phaeoclathratiforme BU-1\"</a>. <i>BMC Research Notes</i>. <b>8</b> (565): 565. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1186%2Fs13104-015-1535-8\">10.1186/s13104-015-1535-8</a></span>. <a href=\"/wiki/PMC_(identifier)\" class=\"mw-redirect\" title=\"PMC (identifier)\">PMC</a>&#160;<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4606965\">4606965</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/26467441\">26467441</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=BMC+Research+Notes&amp;rft.atitle=A+comparison+of+the+protein-coding+genomes+of+two+green+sulphur+bacteria%2C+Chlorobium+tepidum+TLS+and+Pelodictyon+phaeoclathratiforme+BU-1&amp;rft.volume=8&amp;rft.issue=565&amp;rft.pages=565&amp;rft.date=2015-10&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4606965%23id-name%3DPMC&amp;rft_id=info%3Apmid%2F26467441&amp;rft_id=info%3Adoi%2F10.1186%2Fs13104-015-1535-8&amp;rft.aulast=Wreggelsworth&amp;rft.aufirst=KM&amp;rft.au=Barker%2C+D&amp;rft_id=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4606965&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></span>\\n</li>\\n</ol></div></div>\\n<h2><span class=\"mw-headline\" id=\"Further_reading\">Further reading</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=43\" title=\"Edit section: Further reading\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<style data-mw-deduplicate=\"TemplateStyles:r1054258005\">.mw-parser-output .refbegin{font-size:90%;margin-bottom:0.5em}.mw-parser-output .refbegin-hanging-indents>ul{margin-left:0}.mw-parser-output .refbegin-hanging-indents>ul>li{margin-left:0;padding-left:3.2em;text-indent:-3.2em}.mw-parser-output .refbegin-hanging-indents ul,.mw-parser-output .refbegin-hanging-indents ul li{list-style:none}@media(max-width:720px){.mw-parser-output .refbegin-hanging-indents>ul>li{padding-left:1.6em;text-indent:-1.6em}}.mw-parser-output .refbegin-columns{margin-top:0.3em}.mw-parser-output .refbegin-columns ul{margin-top:0}.mw-parser-output .refbegin-columns li{page-break-inside:avoid;break-inside:avoid-column}</style><div class=\"refbegin refbegin-columns references-column-width\" style=\"column-width: 35em\">\\n<ul><li>Sehgal et al.&#160;: Structural, phylogenetic and docking studies of D-amino acid oxidase activator(DAOA ), a candidate schizophrenia gene. Theoretical Biology and Medical Modelling 2013 10 :3.</li>\\n<li>Achuthsankar S Nair <a rel=\"nofollow\" class=\"external text\" href=\"http://print.achuth.googlepages.com/BINFTutorialV5.0CSI07.pdf\">Computational Biology &amp; Bioinformatics \\xe2\\x80\\x93 A gentle Overview</a> <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20081216212125/http://print.achuth.googlepages.com/BINFTutorialV5.0CSI07.pdf\">Archived</a> 16 December 2008 at the <a href=\"/wiki/Wayback_Machine\" title=\"Wayback Machine\">Wayback Machine</a>, Communications of Computer Society of India, January 2007</li>\\n<li><a href=\"/wiki/Srinivas_Aluru\" title=\"Srinivas Aluru\">Aluru, Srinivas</a>, ed. <i>Handbook of Computational Molecular Biology</i>. Chapman &amp; Hall/Crc, 2006. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/1-58488-406-1\" title=\"Special:BookSources/1-58488-406-1\">1-58488-406-1</a> (Chapman &amp; Hall/Crc Computer and Information Science Series)</li>\\n<li>Baldi, P and Brunak, S, <i>Bioinformatics: The Machine Learning Approach</i>, 2nd edition. MIT Press, 2001. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-262-02506-X\" title=\"Special:BookSources/0-262-02506-X\">0-262-02506-X</a></li>\\n<li>Barnes, M.R. and Gray, I.C., eds., <i>Bioinformatics for Geneticists</i>, first edition. Wiley, 2003. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-470-84394-2\" title=\"Special:BookSources/0-470-84394-2\">0-470-84394-2</a></li>\\n<li>Baxevanis, A.D. and Ouellette, B.F.F., eds., <i>Bioinformatics: A Practical Guide to the Analysis of Genes and Proteins</i>, third edition. Wiley, 2005. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-471-47878-4\" title=\"Special:BookSources/0-471-47878-4\">0-471-47878-4</a></li>\\n<li>Baxevanis, A.D., Petsko, G.A., Stein, L.D., and Stormo, G.D., eds., <i><a href=\"/wiki/Current_Protocols\" title=\"Current Protocols\">Current Protocols</a> in Bioinformatics</i>. Wiley, 2007. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-471-25093-7\" title=\"Special:BookSources/0-471-25093-7\">0-471-25093-7</a></li>\\n<li>Cristianini, N. and Hahn, M. <a rel=\"nofollow\" class=\"external text\" href=\"http://www.computational-genomics.net/\"><i>Introduction to Computational Genomics</i></a> <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20090104042023/http://www.computational-genomics.net/\">Archived</a> 4 January 2009 at the <a href=\"/wiki/Wayback_Machine\" title=\"Wayback Machine\">Wayback Machine</a>, Cambridge University Press, 2006. (<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/9780521671910\" title=\"Special:BookSources/9780521671910\">9780521671910</a> |<link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-521-67191-4\" title=\"Special:BookSources/0-521-67191-4\">0-521-67191-4</a>)</li>\\n<li>Durbin, R., S. Eddy, A. Krogh and G. Mitchison, <i>Biological sequence analysis</i>. Cambridge University Press, 1998. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-521-62971-3\" title=\"Special:BookSources/0-521-62971-3\">0-521-62971-3</a></li>\\n<li><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><cite id=\"CITEREFGilbert2004\" class=\"citation journal cs1\">Gilbert D (September 2004). <a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1093%2Fbib%2F5.3.300\">\"Bioinformatics software resources\"</a>. <i>Briefings in Bioinformatics</i>. <b>5</b> (3): 300\\xe2\\x80\\x934. <a href=\"/wiki/Doi_(identifier)\" class=\"mw-redirect\" title=\"Doi (identifier)\">doi</a>:<span class=\"cs1-lock-free\" title=\"Freely accessible\"><a rel=\"nofollow\" class=\"external text\" href=\"https://doi.org/10.1093%2Fbib%2F5.3.300\">10.1093/bib/5.3.300</a></span>. <a href=\"/wiki/PMID_(identifier)\" class=\"mw-redirect\" title=\"PMID (identifier)\">PMID</a>&#160;<a rel=\"nofollow\" class=\"external text\" href=\"https://pubmed.ncbi.nlm.nih.gov/15383216\">15383216</a>.</cite><span title=\"ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.genre=article&amp;rft.jtitle=Briefings+in+Bioinformatics&amp;rft.atitle=Bioinformatics+software+resources&amp;rft.volume=5&amp;rft.issue=3&amp;rft.pages=300-4&amp;rft.date=2004-09&amp;rft_id=info%3Adoi%2F10.1093%2Fbib%2F5.3.300&amp;rft_id=info%3Apmid%2F15383216&amp;rft.aulast=Gilbert&amp;rft.aufirst=D&amp;rft_id=https%3A%2F%2Fdoi.org%2F10.1093%252Fbib%252F5.3.300&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3ABioinformatics\" class=\"Z3988\"></span></li>\\n<li>Keedwell, E., <i>Intelligent Bioinformatics: The Application of Artificial Intelligence Techniques to Bioinformatics Problems</i>. Wiley, 2005. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-470-02175-6\" title=\"Special:BookSources/0-470-02175-6\">0-470-02175-6</a></li>\\n<li>Kohane, et al. <i>Microarrays for an Integrative Genomics.</i> The MIT Press, 2002. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-262-11271-X\" title=\"Special:BookSources/0-262-11271-X\">0-262-11271-X</a></li>\\n<li>Lund, O. et al. <i>Immunological Bioinformatics.</i> The MIT Press, 2005. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-262-12280-4\" title=\"Special:BookSources/0-262-12280-4\">0-262-12280-4</a></li>\\n<li><a href=\"/wiki/Lior_Pachter\" title=\"Lior Pachter\">Pachter, Lior</a> and <a href=\"/wiki/Bernd_Sturmfels\" title=\"Bernd Sturmfels\">Sturmfels, Bernd</a>. \"Algebraic Statistics for Computational Biology\" Cambridge University Press, 2005. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-521-85700-7\" title=\"Special:BookSources/0-521-85700-7\">0-521-85700-7</a></li>\\n<li>Pevzner, Pavel A. <i>Computational Molecular Biology: An Algorithmic Approach</i> The MIT Press, 2000. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-262-16197-4\" title=\"Special:BookSources/0-262-16197-4\">0-262-16197-4</a></li>\\n<li>Soinov, L. <a rel=\"nofollow\" class=\"external text\" href=\"http://jprr.org/index.php/jprr/article/view/8/5\">Bioinformatics and Pattern Recognition Come Together</a> <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20130510213503/http://jprr.org/index.php/jprr/article/view/8/5\">Archived</a> 10 May 2013 at the <a href=\"/wiki/Wayback_Machine\" title=\"Wayback Machine\">Wayback Machine</a> Journal of Pattern Recognition Research (<a rel=\"nofollow\" class=\"external text\" href=\"http://www.jprr.org\">JPRR</a> <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20080908110041/http://www.jprr.org/\">Archived</a> 8 September 2008 at the <a href=\"/wiki/Wayback_Machine\" title=\"Wayback Machine\">Wayback Machine</a>), Vol 1 (1) 2006 p.&#160;37\\xe2\\x80\\x9341</li>\\n<li>Stevens, Hallam, <i>Life Out of Sequence: A Data-Driven History of Bioinformatics</i>, Chicago: The University of Chicago Press, 2013, <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/9780226080208\" title=\"Special:BookSources/9780226080208\">9780226080208</a></li>\\n<li>Tisdall, James. \"Beginning Perl for Bioinformatics\" O\\'Reilly, 2001. <link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1133582631\"><a href=\"/wiki/ISBN_(identifier)\" class=\"mw-redirect\" title=\"ISBN (identifier)\">ISBN</a>&#160;<a href=\"/wiki/Special:BookSources/0-596-00080-4\" title=\"Special:BookSources/0-596-00080-4\">0-596-00080-4</a></li>\\n<li><a rel=\"nofollow\" class=\"external text\" href=\"http://www.nap.edu/catalog/11480.html\">Catalyzing Inquiry at the Interface of Computing and Biology (2005) CSTB report</a> <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20070128222920/http://www.nap.edu/catalog/11480.html\">Archived</a> 28 January 2007 at the <a href=\"/wiki/Wayback_Machine\" title=\"Wayback Machine\">Wayback Machine</a></li>\\n<li><a rel=\"nofollow\" class=\"external text\" href=\"http://www.nap.edu/catalog/2121.html\">Calculating the Secrets of Life: Contributions of the Mathematical Sciences and computing to Molecular Biology (1995)</a> <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20080706035211/http://www.nap.edu/catalog/2121.html\">Archived</a> 6 July 2008 at the <a href=\"/wiki/Wayback_Machine\" title=\"Wayback Machine\">Wayback Machine</a></li>\\n<li><a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20071222091912/http://ocw.mit.edu/OcwWeb/Biology/7-91JSpring2004/LectureNotes/index.htm\">Foundations of Computational and Systems Biology MIT Course</a></li>\\n<li><a rel=\"nofollow\" class=\"external text\" href=\"http://compbio.mit.edu/6.047/\">Computational Biology: Genomes, Networks, Evolution Free MIT Course</a> <a rel=\"nofollow\" class=\"external text\" href=\"https://web.archive.org/web/20130408034631/http://compbio.mit.edu/6.047/\">Archived</a> 8 April 2013 at the <a href=\"/wiki/Wayback_Machine\" title=\"Wayback Machine\">Wayback Machine</a></li></ul>\\n</div>\\n<h2><span class=\"mw-headline\" id=\"External_links\">External links</span><span class=\"mw-editsection\"><span class=\"mw-editsection-bracket\">[</span><a href=\"/w/index.php?title=Bioinformatics&amp;action=edit&amp;section=44\" title=\"Edit section: External links\">edit</a><span class=\"mw-editsection-bracket\">]</span></span></h2>\\n<style data-mw-deduplicate=\"TemplateStyles:r1126788409\">.mw-parser-output .plainlist ol,.mw-parser-output .plainlist ul{line-height:inherit;list-style:none;margin:0;padding:0}.mw-parser-output .plainlist ol li,.mw-parser-output .plainlist ul li{margin-bottom:0}</style><style data-mw-deduplicate=\"TemplateStyles:r1045330069\">.mw-parser-output .sidebar{width:22em;float:right;clear:right;margin:0.5em 0 1em 1em;background:#f8f9fa;border:1px solid #aaa;padding:0.2em;text-align:center;line-height:1.4em;font-size:88%;border-collapse:collapse;display:table}body.skin-minerva .mw-parser-output .sidebar{display:table!important;float:right!important;margin:0.5em 0 1em 1em!important}.mw-parser-output .sidebar-subgroup{width:100%;margin:0;border-spacing:0}.mw-parser-output .sidebar-left{float:left;clear:left;margin:0.5em 1em 1em 0}.mw-parser-output .sidebar-none{float:none;clear:both;margin:0.5em 1em 1em 0}.mw-parser-output .sidebar-outer-title{padding:0 0.4em 0.2em;font-size:125%;line-height:1.2em;font-weight:bold}.mw-parser-output .sidebar-top-image{padding:0.4em}.mw-parser-output .sidebar-top-caption,.mw-parser-output .sidebar-pretitle-with-top-image,.mw-parser-output .sidebar-caption{padding:0.2em 0.4em 0;line-height:1.2em}.mw-parser-output .sidebar-pretitle{padding:0.4em 0.4em 0;line-height:1.2em}.mw-parser-output .sidebar-title,.mw-parser-output .sidebar-title-with-pretitle{padding:0.2em 0.8em;font-size:145%;line-height:1.2em}.mw-parser-output .sidebar-title-with-pretitle{padding:0.1em 0.4em}.mw-parser-output .sidebar-image{padding:0.2em 0.4em 0.4em}.mw-parser-output .sidebar-heading{padding:0.1em 0.4em}.mw-parser-output .sidebar-content{padding:0 0.5em 0.4em}.mw-parser-output .sidebar-content-with-subgroup{padding:0.1em 0.4em 0.2em}.mw-parser-output .sidebar-above,.mw-parser-output .sidebar-below{padding:0.3em 0.8em;font-weight:bold}.mw-parser-output .sidebar-collapse .sidebar-above,.mw-parser-output .sidebar-collapse .sidebar-below{border-top:1px solid #aaa;border-bottom:1px solid #aaa}.mw-parser-output .sidebar-navbar{text-align:right;font-size:115%;padding:0 0.4em 0.4em}.mw-parser-output .sidebar-list-title{padding:0 0.4em;text-align:left;font-weight:bold;line-height:1.6em;font-size:105%}.mw-parser-output .sidebar-list-title-c{padding:0 0.4em;text-align:center;margin:0 3.3em}@media(max-width:720px){body.mediawiki .mw-parser-output .sidebar{width:100%!important;clear:both;float:none!important;margin-left:0!important;margin-right:0!important}}</style><table class=\"sidebar nomobile nowraplinks\" style=\"width:auto;text-align:left;\"><tbody><tr><td class=\"sidebar-pretitle\"><a href=\"/wiki/Wikipedia:The_Wikipedia_Library\" title=\"Wikipedia:The Wikipedia Library\">Library resources</a> about <br /> <b>Bioinformatics</b> <hr /></td></tr><tr><td class=\"sidebar-content plainlist\">\\n<ul><li><a class=\"external text\" href=\"https://ftl.toolforge.org/cgi-bin/ftl?st=wp&amp;su=Bioinformatics\">Resources in your library</a></li>\\n<li><a class=\"external text\" href=\"https://ftl.toolforge.org/cgi-bin/ftl?st=wp&amp;su=Bioinformatics&amp;library=0CHOOSE0\">Resources in other libraries</a></li></ul></td>\\n</tr></tbody></table><style data-mw-deduplicate=\"TemplateStyles:r1040133893\">.mw-parser-output .spoken-wikipedia{border:1px solid #a2a9b1;background-color:#f8f9fa;margin:0.5em 0;padding:0.2em;line-height:1.5em;font-size:90%}.mw-parser-output .spoken-wikipedia-header{text-align:center}.mw-parser-output .spoken-wikipedia-listen-to{font-weight:bold}.mw-parser-output .spoken-wikipedia-files{text-align:center;margin-top:10px;margin-bottom:0.4em}.mw-parser-output .spoken-wikipedia-icon{float:left;margin-left:5px;margin-top:10px}.mw-parser-output .spoken-wikipedia-disclaimer{margin-left:60px;margin-top:10px;font-size:95%;line-height:1.4em}.mw-parser-output .spoken-wikipedia-footer{margin-top:10px;text-align:center}@media(min-width:720px){.mw-parser-output .spoken-wikipedia{width:20em;float:right;clear:right;margin-left:1em}}</style><div class=\"spoken-wikipedia sisterproject noprint haudio\"><div class=\"spoken-wikipedia-header\"><span class=\"spoken-wikipedia-listen-to\">Listen to this article</span> (<span class=\"duration\"><span class=\"min\">37</span> minutes</span>)</div><div class=\"spoken-wikipedia-files\"><div class=\"center\"><div class=\"floatnone\"><audio id=\"mwe_player_0\" controls=\"\" preload=\"none\" width=\"200\" style=\"width:200px;\" data-durationhint=\"2221\" data-mwtitle=\"En-Bioinformatics.ogg\" data-mwprovider=\"wikimediacommons\"><source src=\"//upload.wikimedia.org/wikipedia/commons/4/43/En-Bioinformatics.ogg\" type=\"audio/ogg; codecs=&quot;vorbis&quot;\" data-title=\"Original Ogg file (68 kbps)\" data-shorttitle=\"Ogg source\" data-width=\"0\" data-height=\"0\" /><source src=\"//upload.wikimedia.org/wikipedia/commons/transcoded/4/43/En-Bioinformatics.ogg/En-Bioinformatics.ogg.mp3\" type=\"audio/mpeg\" data-title=\"MP3\" data-shorttitle=\"MP3\" data-transcodekey=\"mp3\" data-width=\"0\" data-height=\"0\" /></audio></div></div>\\n</div><div class=\"spoken-wikipedia-icon\"><img alt=\"Spoken Wikipedia icon\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/4/47/Sound-icon.svg/45px-Sound-icon.svg.png\" decoding=\"async\" title=\"Spoken Wikipedia\" width=\"45\" height=\"34\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/4/47/Sound-icon.svg/68px-Sound-icon.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/4/47/Sound-icon.svg/90px-Sound-icon.svg.png 2x\" data-file-width=\"128\" data-file-height=\"96\" /></div><div class=\"spoken-wikipedia-disclaimer\"><a href=\"/wiki/File:En-Bioinformatics.ogg\" title=\"File:En-Bioinformatics.ogg\">This audio file</a> was created from a revision of this article dated 20&#160;September&#160;2013<span style=\"display:none\">&#160;(<span class=\"bday dtstart published updated\">2013-09-20</span>)</span>, and does not reflect subsequent edits.</div><div class=\"spoken-wikipedia-footer\">(<a href=\"/wiki/Wikipedia:Media_help\" class=\"mw-redirect\" title=\"Wikipedia:Media help\">Audio help</a>&#160;\\xc2\\xb7 <a href=\"/wiki/Wikipedia:Spoken_articles\" title=\"Wikipedia:Spoken articles\">More spoken articles</a>)</div></div>\\n<ul><li><a href=\"/wiki/File:Wiktionary-logo-en-v2.svg\" class=\"image\"><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/9/99/Wiktionary-logo-en-v2.svg/16px-Wiktionary-logo-en-v2.svg.png\" decoding=\"async\" width=\"16\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/9/99/Wiktionary-logo-en-v2.svg/24px-Wiktionary-logo-en-v2.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/99/Wiktionary-logo-en-v2.svg/32px-Wiktionary-logo-en-v2.svg.png 2x\" data-file-width=\"512\" data-file-height=\"512\" /></a> The dictionary definition of <a href=\"https://en.wiktionary.org/wiki/bioinformatics\" class=\"extiw\" title=\"wiktionary:bioinformatics\"><i>bioinformatics</i></a> at Wiktionary</li>\\n<li><a href=\"/wiki/File:Wikiversity_logo_2017.svg\" class=\"image\"><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/0/0b/Wikiversity_logo_2017.svg/16px-Wikiversity_logo_2017.svg.png\" decoding=\"async\" width=\"16\" height=\"13\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/0/0b/Wikiversity_logo_2017.svg/24px-Wikiversity_logo_2017.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/0/0b/Wikiversity_logo_2017.svg/32px-Wikiversity_logo_2017.svg.png 2x\" data-file-width=\"626\" data-file-height=\"512\" /></a> Learning materials related to <a href=\"https://en.wikiversity.org/wiki/Special:Search/Bioinformatics\" class=\"extiw\" title=\"v:Special:Search/Bioinformatics\">Bioinformatics</a> at Wikiversity</li>\\n<li><a href=\"/wiki/File:Commons-logo.svg\" class=\"image\"><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/12px-Commons-logo.svg.png\" decoding=\"async\" width=\"12\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/18px-Commons-logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/24px-Commons-logo.svg.png 2x\" data-file-width=\"1024\" data-file-height=\"1376\" /></a> Media related to <a href=\"https://commons.wikimedia.org/wiki/Category:Bioinformatics\" class=\"extiw\" title=\"commons:Category:Bioinformatics\">Bioinformatics</a> at Wikimedia Commons</li>\\n<li><a rel=\"nofollow\" class=\"external text\" href=\"http://expasy.org\">Bioinformatics Resource Portal (SIB)</a></li></ul>\\n<div class=\"navbox-styles\"><style data-mw-deduplicate=\"TemplateStyles:r1129693374\">.mw-parser-output .hlist dl,.mw-parser-output .hlist ol,.mw-parser-output .hlist ul{margin:0;padding:0}.mw-parser-output .hlist dd,.mw-parser-output .hlist dt,.mw-parser-output .hlist li{margin:0;display:inline}.mw-parser-output .hlist.inline,.mw-parser-output .hlist.inline dl,.mw-parser-output .hlist.inline ol,.mw-parser-output .hlist.inline ul,.mw-parser-output .hlist dl dl,.mw-parser-output .hlist dl ol,.mw-parser-output .hlist dl ul,.mw-parser-output .hlist ol dl,.mw-parser-output .hlist ol ol,.mw-parser-output .hlist ol ul,.mw-parser-output .hlist ul dl,.mw-parser-output .hlist ul ol,.mw-parser-output .hlist ul ul{display:inline}.mw-parser-output .hlist .mw-empty-li{display:none}.mw-parser-output .hlist dt::after{content:\": \"}.mw-parser-output .hlist dd::after,.mw-parser-output .hlist li::after{content:\" \\xc2\\xb7 \";font-weight:bold}.mw-parser-output .hlist dd:last-child::after,.mw-parser-output .hlist dt:last-child::after,.mw-parser-output .hlist li:last-child::after{content:none}.mw-parser-output .hlist dd dd:first-child::before,.mw-parser-output .hlist dd dt:first-child::before,.mw-parser-output .hlist dd li:first-child::before,.mw-parser-output .hlist dt dd:first-child::before,.mw-parser-output .hlist dt dt:first-child::before,.mw-parser-output .hlist dt li:first-child::before,.mw-parser-output .hlist li dd:first-child::before,.mw-parser-output .hlist li dt:first-child::before,.mw-parser-output .hlist li li:first-child::before{content:\" (\";font-weight:normal}.mw-parser-output .hlist dd dd:last-child::after,.mw-parser-output .hlist dd dt:last-child::after,.mw-parser-output .hlist dd li:last-child::after,.mw-parser-output .hlist dt dd:last-child::after,.mw-parser-output .hlist dt dt:last-child::after,.mw-parser-output .hlist dt li:last-child::after,.mw-parser-output .hlist li dd:last-child::after,.mw-parser-output .hlist li dt:last-child::after,.mw-parser-output .hlist li li:last-child::after{content:\")\";font-weight:normal}.mw-parser-output .hlist ol{counter-reset:listitem}.mw-parser-output .hlist ol>li{counter-increment:listitem}.mw-parser-output .hlist ol>li::before{content:\" \"counter(listitem)\"\\\\a0 \"}.mw-parser-output .hlist dd ol>li:first-child::before,.mw-parser-output .hlist dt ol>li:first-child::before,.mw-parser-output .hlist li ol>li:first-child::before{content:\" (\"counter(listitem)\"\\\\a0 \"}</style><style data-mw-deduplicate=\"TemplateStyles:r1061467846\">.mw-parser-output .navbox{box-sizing:border-box;border:1px solid #a2a9b1;width:100%;clear:both;font-size:88%;text-align:center;padding:1px;margin:1em auto 0}.mw-parser-output .navbox .navbox{margin-top:0}.mw-parser-output .navbox+.navbox,.mw-parser-output .navbox+.navbox-styles+.navbox{margin-top:-1px}.mw-parser-output .navbox-inner,.mw-parser-output .navbox-subgroup{width:100%}.mw-parser-output .navbox-group,.mw-parser-output .navbox-title,.mw-parser-output .navbox-abovebelow{padding:0.25em 1em;line-height:1.5em;text-align:center}.mw-parser-output .navbox-group{white-space:nowrap;text-align:right}.mw-parser-output .navbox,.mw-parser-output .navbox-subgroup{background-color:#fdfdfd}.mw-parser-output .navbox-list{line-height:1.5em;border-color:#fdfdfd}.mw-parser-output .navbox-list-with-group{text-align:left;border-left-width:2px;border-left-style:solid}.mw-parser-output tr+tr>.navbox-abovebelow,.mw-parser-output tr+tr>.navbox-group,.mw-parser-output tr+tr>.navbox-image,.mw-parser-output tr+tr>.navbox-list{border-top:2px solid #fdfdfd}.mw-parser-output .navbox-title{background-color:#ccf}.mw-parser-output .navbox-abovebelow,.mw-parser-output .navbox-group,.mw-parser-output .navbox-subgroup .navbox-title{background-color:#ddf}.mw-parser-output .navbox-subgroup .navbox-group,.mw-parser-output .navbox-subgroup .navbox-abovebelow{background-color:#e6e6ff}.mw-parser-output .navbox-even{background-color:#f7f7f7}.mw-parser-output .navbox-odd{background-color:transparent}.mw-parser-output .navbox .hlist td dl,.mw-parser-output .navbox .hlist td ol,.mw-parser-output .navbox .hlist td ul,.mw-parser-output .navbox td.hlist dl,.mw-parser-output .navbox td.hlist ol,.mw-parser-output .navbox td.hlist ul{padding:0.125em 0}.mw-parser-output .navbox .navbar{display:block;font-size:100%}.mw-parser-output .navbox-title .navbar{float:left;text-align:left;margin-right:0.5em}</style></div><div role=\"navigation\" class=\"navbox\" aria-labelledby=\"Bioinformatics\" style=\"padding:3px\"><table class=\"nowraplinks mw-collapsible autocollapse navbox-inner\" style=\"border-spacing:0;background:transparent;color:inherit\"><tbody><tr><th scope=\"col\" class=\"navbox-title\" colspan=\"2\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><style data-mw-deduplicate=\"TemplateStyles:r1063604349\">.mw-parser-output .navbar{display:inline;font-size:88%;font-weight:normal}.mw-parser-output .navbar-collapse{float:left;text-align:left}.mw-parser-output .navbar-boxtext{word-spacing:0}.mw-parser-output .navbar ul{display:inline-block;white-space:nowrap;line-height:inherit}.mw-parser-output .navbar-brackets::before{margin-right:-0.125em;content:\"[ \"}.mw-parser-output .navbar-brackets::after{margin-left:-0.125em;content:\" ]\"}.mw-parser-output .navbar li{word-spacing:-0.125em}.mw-parser-output .navbar a>span,.mw-parser-output .navbar a>abbr{text-decoration:inherit}.mw-parser-output .navbar-mini abbr{font-variant:small-caps;border-bottom:none;text-decoration:none;cursor:inherit}.mw-parser-output .navbar-ct-full{font-size:114%;margin:0 7em}.mw-parser-output .navbar-ct-mini{font-size:114%;margin:0 4em}</style><div class=\"navbar plainlinks hlist navbar-mini\"><ul><li class=\"nv-view\"><a href=\"/wiki/Template:Bioinformatics\" title=\"Template:Bioinformatics\"><abbr title=\"View this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">v</abbr></a></li><li class=\"nv-talk\"><a href=\"/wiki/Template_talk:Bioinformatics\" title=\"Template talk:Bioinformatics\"><abbr title=\"Discuss this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">t</abbr></a></li><li class=\"nv-edit\"><a class=\"external text\" href=\"https://en.wikipedia.org/w/index.php?title=Template:Bioinformatics&amp;action=edit\"><abbr title=\"Edit this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">e</abbr></a></li></ul></div><div id=\"Bioinformatics\" style=\"font-size:114%;margin:0 4em\"><a class=\"mw-selflink selflink\">Bioinformatics</a></div></th></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Databases</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li>Sequence databases: <a href=\"/wiki/GenBank\" title=\"GenBank\">GenBank</a>, <a href=\"/wiki/European_Nucleotide_Archive\" title=\"European Nucleotide Archive\">European Nucleotide Archive</a>, <a href=\"/wiki/DNA_Data_Bank_of_Japan\" title=\"DNA Data Bank of Japan\">DNA Data Bank of Japan</a> and <a href=\"/wiki/China_National_GeneBank\" title=\"China National GeneBank\">China National GeneBank</a></li>\\n<li>Secondary databases: <a href=\"/wiki/UniProt\" title=\"UniProt\">UniProt</a>, database of protein sequences grouping together <a href=\"/wiki/UniProt#UniProtKB.2FSwiss-Prot\" title=\"UniProt\">Swiss-Prot</a>, <a href=\"/wiki/UniProt#UniProtKB.2FTrEMBL\" title=\"UniProt\">TrEMBL</a> and <a href=\"/wiki/Protein_Information_Resource\" title=\"Protein Information Resource\">Protein Information Resource</a></li>\\n<li>Other databases: <a href=\"/wiki/BioNumbers\" title=\"BioNumbers\">BioNumbers</a>, <a href=\"/wiki/Protein_Data_Bank\" title=\"Protein Data Bank\">Protein Data Bank</a>, <a href=\"/wiki/Ensembl_genome_database_project\" title=\"Ensembl genome database project\">Ensembl</a> and <a href=\"/wiki/InterPro\" title=\"InterPro\">InterPro</a></li>\\n<li>Specialised genomic databases: <a href=\"/wiki/Barcode_of_Life_Data_System\" title=\"Barcode of Life Data System\">BOLD</a>, <a href=\"/wiki/Saccharomyces_Genome_Database\" title=\"Saccharomyces Genome Database\">Saccharomyces Genome Database</a>, <a href=\"/wiki/FlyBase\" title=\"FlyBase\">FlyBase</a>, <a href=\"/wiki/VectorBase\" title=\"VectorBase\">VectorBase</a>, <a href=\"/wiki/WormBase\" title=\"WormBase\">WormBase</a>, <a href=\"/wiki/Rat_Genome_Database\" title=\"Rat Genome Database\">Rat Genome Database</a>, <a href=\"/wiki/PHI-base\" title=\"PHI-base\">PHI-base</a>, <a href=\"/wiki/The_Arabidopsis_Information_Resource\" title=\"The Arabidopsis Information Resource\">Arabidopsis Information Resource</a>, <a href=\"/wiki/GISAID\" title=\"GISAID\">GISAID</a> and <a href=\"/wiki/Zebrafish_Information_Network\" title=\"Zebrafish Information Network\">Zebrafish Information Network</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Software</th><td class=\"navbox-list-with-group navbox-list navbox-even hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/BLAST\" class=\"mw-redirect\" title=\"BLAST\">BLAST</a></li>\\n<li><a href=\"/wiki/Bowtie_(sequence_analysis)\" title=\"Bowtie (sequence analysis)\">Bowtie</a></li>\\n<li><a href=\"/wiki/Clustal\" title=\"Clustal\">Clustal</a></li>\\n<li><a href=\"/wiki/EMBOSS\" title=\"EMBOSS\">EMBOSS</a></li>\\n<li><a href=\"/wiki/HMMER\" title=\"HMMER\">HMMER</a></li>\\n<li><a href=\"/wiki/MUSCLE_(alignment_software)\" title=\"MUSCLE (alignment software)\">MUSCLE</a></li>\\n<li><a href=\"/wiki/Phylogenetic_Assignment_of_Named_Global_Outbreak_Lineages\" title=\"Phylogenetic Assignment of Named Global Outbreak Lineages\">PANGOLIN</a></li>\\n<li><a href=\"/wiki/SAMtools\" title=\"SAMtools\">SAMtools</a></li>\\n<li><a href=\"/wiki/Short_Oligonucleotide_Analysis_Package\" title=\"Short Oligonucleotide Analysis Package\">SOAP suite</a></li>\\n<li><a href=\"/wiki/TopHat_(bioinformatics)\" title=\"TopHat (bioinformatics)\">TopHat</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Other</th><td class=\"navbox-list-with-group navbox-list navbox-odd hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li>Server: <a href=\"/wiki/ExPASy\" class=\"mw-redirect\" title=\"ExPASy\">ExPASy</a></li>\\n<li>Ontology: <a href=\"/wiki/Gene_ontology\" class=\"mw-redirect\" title=\"Gene ontology\">Gene Ontology</a></li>\\n<li><a href=\"/wiki/Rosalind_(education_platform)\" title=\"Rosalind (education platform)\">Rosalind (education platform)</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Institutions</th><td class=\"navbox-list-with-group navbox-list navbox-even hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Broad_Institute\" title=\"Broad Institute\">Broad Institute</a></li>\\n<li><a href=\"/wiki/Computational_Biology_Department\" class=\"mw-redirect\" title=\"Computational Biology Department\">Computational Biology Department</a> (CBD)</li>\\n<li><a href=\"/wiki/COSBI\" title=\"COSBI\">Microsoft Research - University of Trento Centre for Computational and Systems Biology</a> (COSBI)</li>\\n<li><a href=\"/wiki/Database_Center_for_Life_Science\" title=\"Database Center for Life Science\">Database Center for Life Science</a> (DBCLS)</li>\\n<li><a href=\"/wiki/DNA_Data_Bank_of_Japan\" title=\"DNA Data Bank of Japan\">DNA Data Bank of Japan</a> (DDBJ)</li>\\n<li><a href=\"/wiki/European_Bioinformatics_Institute\" title=\"European Bioinformatics Institute\">European Bioinformatics Institute</a> (EMBL-EBI)</li>\\n<li><a href=\"/wiki/European_Molecular_Biology_Laboratory\" title=\"European Molecular Biology Laboratory\">European Molecular Biology Laboratory</a> (EMBL)</li>\\n<li><a href=\"/wiki/Flatiron_Institute\" title=\"Flatiron Institute\">Flatiron Institute</a></li>\\n<li><a href=\"/wiki/J._Craig_Venter_Institute\" title=\"J. Craig Venter Institute\">J. Craig Venter Institute</a> (JCVI)</li>\\n<li><a href=\"/wiki/Max_Planck_Institute_of_Molecular_Cell_Biology_and_Genetics\" title=\"Max Planck Institute of Molecular Cell Biology and Genetics\">Max Planck Institute of Molecular Cell Biology and Genetics</a> (MPI-CBG)</li>\\n<li><a href=\"/wiki/National_Center_for_Biotechnology_Information\" title=\"National Center for Biotechnology Information\">US National Center for Biotechnology Information</a> (NCBI)</li>\\n<li><a href=\"/wiki/National_Institute_of_Genetics\" title=\"National Institute of Genetics\">Japanese Institute of Genetics</a></li>\\n<li><a href=\"/wiki/Netherlands_Bioinformatics_Centre\" title=\"Netherlands Bioinformatics Centre\">Netherlands Bioinformatics Centre</a> (NBIC)</li>\\n<li><a href=\"/wiki/Philippine_Genome_Center\" title=\"Philippine Genome Center\">Philippine Genome Center</a> (PGC)</li>\\n<li><a href=\"/wiki/Scripps_Research\" title=\"Scripps Research\">Scripps Research</a></li>\\n<li><a href=\"/wiki/Swiss_Institute_of_Bioinformatics\" title=\"Swiss Institute of Bioinformatics\">Swiss Institute of Bioinformatics</a> (SIB)</li>\\n<li><a href=\"/wiki/Wellcome_Sanger_Institute\" title=\"Wellcome Sanger Institute\">Wellcome Sanger Institute</a></li>\\n<li><a href=\"/wiki/Whitehead_Institute\" title=\"Whitehead Institute\">Whitehead Institute</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Organizations</th><td class=\"navbox-list-with-group navbox-list navbox-odd hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/African_Society_for_Bioinformatics_and_Computational_Biology\" title=\"African Society for Bioinformatics and Computational Biology\">African Society for Bioinformatics and Computational Biology</a> (ASBCB)</li>\\n<li><a href=\"/wiki/Australia_Bioinformatics_Resource\" title=\"Australia Bioinformatics Resource\">Australia Bioinformatics Resource</a> (EMBL-AR)</li>\\n<li><a href=\"/wiki/EMBnet\" title=\"EMBnet\">European Molecular Biology network</a> (EMBnet)</li>\\n<li><a href=\"/wiki/International_Nucleotide_Sequence_Database_Collaboration\" title=\"International Nucleotide Sequence Database Collaboration\">International Nucleotide Sequence Database Collaboration</a> (INSDC)</li>\\n<li><a href=\"/wiki/International_Society_for_Biocuration\" title=\"International Society for Biocuration\">International Society for Biocuration</a> (ISB)</li>\\n<li><a href=\"/wiki/International_Society_for_Computational_Biology\" title=\"International Society for Computational Biology\">International Society for Computational Biology</a> (ISCB)\\n<ul><li><a href=\"/wiki/International_Society_for_Computational_Biology_Student_Council\" title=\"International Society for Computational Biology Student Council\">Student Council</a> (ISCB-SC)</li></ul></li>\\n<li><a href=\"/wiki/Institute_of_Genomics_and_Integrative_Biology\" title=\"Institute of Genomics and Integrative Biology\">Institute of Genomics and Integrative Biology</a> (CSIR-IGIB)</li>\\n<li><a href=\"/wiki/Japanese_Society_for_Bioinformatics\" title=\"Japanese Society for Bioinformatics\">Japanese Society for Bioinformatics</a> (JSBi)</li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Meetings</th><td class=\"navbox-list-with-group navbox-list navbox-even hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Basel_Computational_Biology_Conference\" title=\"Basel Computational Biology Conference\">Basel Computational Biology Conference\\xe2\\x80\\x8e</a> ([BC<sup>2</sup>])</li>\\n<li><a href=\"/wiki/European_Conference_on_Computational_Biology\" title=\"European Conference on Computational Biology\">European Conference on Computational Biology</a> (ECCB)</li>\\n<li><a href=\"/wiki/Intelligent_Systems_for_Molecular_Biology\" title=\"Intelligent Systems for Molecular Biology\">Intelligent Systems for Molecular Biology</a> (ISMB)</li>\\n<li><a href=\"/wiki/International_Conference_on_Bioinformatics\" title=\"International Conference on Bioinformatics\">International Conference on Bioinformatics</a> (InCoB)</li>\\n<li><a href=\"/wiki/International_Conference_on_Computational_Intelligence_Methods_for_Bioinformatics_and_Biostatistics\" title=\"International Conference on Computational Intelligence Methods for Bioinformatics and Biostatistics\">International Conference on Computational Intelligence Methods for Bioinformatics and Biostatistics</a> (CIBB)</li>\\n<li><a href=\"/wiki/ISCB_Africa_ASBCB_Conference_on_Bioinformatics\" title=\"ISCB Africa ASBCB Conference on Bioinformatics\">ISCB Africa ASBCB Conference on Bioinformatics</a></li>\\n<li><a href=\"/wiki/Pacific_Symposium_on_Biocomputing\" title=\"Pacific Symposium on Biocomputing\">Pacific Symposium on Biocomputing</a> (PSB)</li>\\n<li><a href=\"/wiki/Research_in_Computational_Molecular_Biology\" title=\"Research in Computational Molecular Biology\">Research in Computational Molecular Biology</a> (RECOMB)</li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">File formats</th><td class=\"navbox-list-with-group navbox-list navbox-odd hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/CRAM_(file_format)\" title=\"CRAM (file format)\">CRAM format</a></li>\\n<li><a href=\"/wiki/FASTA_format\" title=\"FASTA format\">FASTA format</a></li>\\n<li><a href=\"/wiki/FASTQ_format\" title=\"FASTQ format\">FASTQ format</a></li>\\n<li><a href=\"/wiki/NeXML_format\" title=\"NeXML format\">NeXML format</a></li>\\n<li><a href=\"/wiki/Nexus_file\" title=\"Nexus file\">Nexus format</a></li>\\n<li><a href=\"/wiki/Pileup_format\" title=\"Pileup format\">Pileup format</a></li>\\n<li><a href=\"/wiki/SAM_(file_format)\" title=\"SAM (file format)\">SAM format</a></li>\\n<li><a href=\"/wiki/Stockholm_format\" title=\"Stockholm format\">Stockholm format</a></li>\\n<li><a href=\"/wiki/Variant_Call_Format\" title=\"Variant Call Format\">VCF format</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Related topics</th><td class=\"navbox-list-with-group navbox-list navbox-even hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Computational_biology\" title=\"Computational biology\">Computational biology</a></li>\\n<li><a href=\"/wiki/List_of_biobanks\" title=\"List of biobanks\">List of biobanks</a></li>\\n<li><a href=\"/wiki/List_of_biological_databases\" title=\"List of biological databases\">List of biological databases</a></li>\\n<li><a href=\"/wiki/Molecular_phylogenetics\" title=\"Molecular phylogenetics\">Molecular phylogenetics</a></li>\\n<li><a href=\"/wiki/Sequencing\" title=\"Sequencing\">Sequencing</a></li>\\n<li><a href=\"/wiki/Sequence_database\" title=\"Sequence database\">Sequence database</a></li>\\n<li><a href=\"/wiki/Sequence_alignment\" title=\"Sequence alignment\">Sequence alignment</a></li></ul>\\n</div></td></tr><tr><td class=\"navbox-abovebelow hlist\" colspan=\"2\"><div>\\n<ul><li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/16px-Symbol_category_class.svg.png\" decoding=\"async\" title=\"Category\" width=\"16\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/23px-Symbol_category_class.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/31px-Symbol_category_class.svg.png 2x\" data-file-width=\"180\" data-file-height=\"185\" /> <b><a href=\"/wiki/Category:Bioinformatics\" title=\"Category:Bioinformatics\">Category</a></b></li>\\n<li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/12px-Commons-logo.svg.png\" decoding=\"async\" title=\"Commons page\" width=\"12\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/18px-Commons-logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/24px-Commons-logo.svg.png 2x\" data-file-width=\"1024\" data-file-height=\"1376\" /> <b><a href=\"https://commons.wikimedia.org/wiki/Category:Bioinformatics\" class=\"extiw\" title=\"commons:Category:Bioinformatics\">Commons</a></b></li></ul>\\n</div></td></tr></tbody></table></div>\\n<div class=\"navbox-styles\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1061467846\"></div><div role=\"navigation\" class=\"navbox\" aria-labelledby=\"Information_processing\" style=\"padding:3px\"><table class=\"nowraplinks mw-collapsible autocollapse navbox-inner\" style=\"border-spacing:0;background:transparent;color:inherit\"><tbody><tr><th scope=\"col\" class=\"navbox-title\" colspan=\"2\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1063604349\"><div class=\"navbar plainlinks hlist navbar-mini\"><ul><li class=\"nv-view\"><a href=\"/wiki/Template:Informatics\" title=\"Template:Informatics\"><abbr title=\"View this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">v</abbr></a></li><li class=\"nv-talk\"><a href=\"/wiki/Template_talk:Informatics\" title=\"Template talk:Informatics\"><abbr title=\"Discuss this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">t</abbr></a></li><li class=\"nv-edit\"><a class=\"external text\" href=\"https://en.wikipedia.org/w/index.php?title=Template:Informatics&amp;action=edit\"><abbr title=\"Edit this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">e</abbr></a></li></ul></div><div id=\"Information_processing\" style=\"font-size:114%;margin:0 4em\"><a href=\"/wiki/Information_processing\" class=\"mw-redirect\" title=\"Information processing\">Information processing</a></div></th></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Information processes</th><td class=\"navbox-list-with-group navbox-list navbox-odd hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\"></div><table class=\"nowraplinks navbox-subgroup\" style=\"border-spacing:0\"><tbody><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">information processes by function</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Perception\" title=\"Perception\">perception</a></li>\\n<li><a href=\"/wiki/Attention\" title=\"Attention\">attention</a></li>\\n<li><a href=\"/wiki/Social_influence\" title=\"Social influence\">influence</a></li>\\n<li><a href=\"/wiki/Remote_control\" title=\"Remote control\">operating</a></li>\\n<li><a href=\"/wiki/Communication\" title=\"Communication\">communication</a></li>\\n<li><a href=\"/wiki/Reason\" title=\"Reason\">reasoning</a></li>\\n<li><a href=\"/wiki/Learning\" title=\"Learning\">learning</a></li>\\n<li><a href=\"/wiki/Information_storage\" class=\"mw-redirect\" title=\"Information storage\">storing</a></li>\\n<li><a href=\"/wiki/Decision-making\" title=\"Decision-making\">decision-making</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">information processing abstractions</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Event_processing\" class=\"mw-redirect\" title=\"Event processing\">event processing</a></li>\\n<li><a href=\"/wiki/Semiotics\" title=\"Semiotics\">sign processesing</a></li>\\n<li><a href=\"/wiki/Signal\" title=\"Signal\">signal processing</a></li>\\n<li><a href=\"/wiki/Data_processing\" title=\"Data processing\">data processing</a></li>\\n<li><a href=\"/wiki/Stream_processing\" title=\"Stream processing\">stream processing</a></li>\\n<li><a href=\"/wiki/Multi-agent_system\" title=\"Multi-agent system\">agent processing</a></li>\\n<li><a href=\"/wiki/State_(computer_science)\" title=\"State (computer science)\">state processing</a></li></ul>\\n</div></td></tr></tbody></table><div></div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Information processors</th><td class=\"navbox-list-with-group navbox-list navbox-odd hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\"></div><table class=\"nowraplinks navbox-subgroup\" style=\"border-spacing:0\"><tbody><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">natural</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Natural_computing#Nature_as_information_processing\" title=\"Natural computing\">nature as information processing</a></li>\\n<li><a href=\"/wiki/Information_processing_theory#Humans_as_Information_Processing_Systems\" title=\"Information processing theory\">humans as information processing systems</a></li>\\n<li><a href=\"/wiki/Social_information_processing\" title=\"Social information processing\">society as information processing system</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">mixed</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Mixed_reality\" title=\"Mixed reality\">mixed reality</a></li>\\n<li><a href=\"/wiki/Brain%E2%80%93computer_interface\" title=\"Brain\\xe2\\x80\\x93computer interface\">brain\\xe2\\x80\\x93computer interface</a></li>\\n<li><a href=\"/wiki/Physical_computing\" title=\"Physical computing\">physical computing</a></li>\\n<li><a href=\"/wiki/Human%E2%80%93computer_interaction\" title=\"Human\\xe2\\x80\\x93computer interaction\">human\\xe2\\x80\\x93computer interaction</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">artificial</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Processor_(computing)\" title=\"Processor (computing)\">processors</a> and <a href=\"/wiki/Process_(computing)\" title=\"Process (computing)\">processes</a></li>\\n<li><a href=\"/wiki/Bio-inspired_computing\" title=\"Bio-inspired computing\">bio-inspired computing</a></li>\\n<li><a href=\"/wiki/Ubiquitous_computing\" title=\"Ubiquitous computing\">ubiquitous computing</a></li>\\n<li><a href=\"/wiki/Artificial_brain\" title=\"Artificial brain\">artificial brain</a> and <a href=\"/wiki/Mind_uploading\" title=\"Mind uploading\">mind uploading</a></li>\\n<li><a href=\"/wiki/Virtual_reality\" title=\"Virtual reality\">virtual reality</a></li>\\n<li><a href=\"/wiki/Virtual_world\" title=\"Virtual world\">virtual world</a></li></ul>\\n</div></td></tr></tbody></table><div></div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Information processing <br />theories and concepts</th><td class=\"navbox-list-with-group navbox-list navbox-odd hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\"></div><table class=\"nowraplinks navbox-subgroup\" style=\"border-spacing:0\"><tbody><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">in biology</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Computational_biology\" title=\"Computational biology\">computational</a> and <a href=\"/wiki/Systems_biology\" title=\"Systems biology\">systems biology</a></li>\\n<li><a href=\"/wiki/Genome_informatics\" title=\"Genome informatics\">genetic informatics</a> and <a href=\"/wiki/Cellular_computing\" class=\"mw-redirect\" title=\"Cellular computing\">cellular computing</a></li>\\n<li><a href=\"/wiki/Computational_neuroscience\" title=\"Computational neuroscience\">computational neuroscience</a> and <a href=\"/wiki/Neurocomputing\" class=\"mw-redirect\" title=\"Neurocomputing\">neurocomputing</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">in cognitive psychology</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Information_processing_theory\" title=\"Information processing theory\">information processing theory</a></li>\\n<li><a href=\"/wiki/Mind\" title=\"Mind\">mind</a> and <a href=\"/wiki/Intelligence\" title=\"Intelligence\">intelligence</a></li>\\n<li><a href=\"/wiki/Cognitive_informatics\" class=\"mw-redirect\" title=\"Cognitive informatics\">cognitive informatics</a> and <a href=\"/wiki/Neuroinformatics\" title=\"Neuroinformatics\">neuroinformatics</a></li>\\n<li><a href=\"/wiki/Behavior_informatics\" title=\"Behavior informatics\">behavior informatics</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">in computer science</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Neural_computation\" title=\"Neural computation\">neural computation</a></li>\\n<li><a href=\"/wiki/Theory_of_computation\" title=\"Theory of computation\">computation theory</a></li>\\n<li><a href=\"/wiki/Algorithms\" class=\"mw-redirect\" title=\"Algorithms\">algorithms</a> and <a href=\"/wiki/Information_structure\" title=\"Information structure\">information structures</a></li>\\n<li><a href=\"/wiki/Circuit_(computer_science)\" title=\"Circuit (computer science)\">computational circuits</a></li>\\n<li><a href=\"/wiki/Artificial_intelligence\" title=\"Artificial intelligence\">artificial intelligence</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">in philosophy</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Computational_theory_of_mind\" title=\"Computational theory of mind\">computational theory of mind</a></li>\\n<li><a href=\"/wiki/Philosophy_of_information\" title=\"Philosophy of information\">philosophy of information</a></li>\\n<li><a href=\"/wiki/Philosophy_of_artificial_intelligence\" title=\"Philosophy of artificial intelligence\">philosophy of artificial intelligence</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">interdisciplinary</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Information_theory\" title=\"Information theory\">information theory</a></li>\\n<li><a href=\"/wiki/Decision_theory\" title=\"Decision theory\">decision theory</a></li>\\n<li><a href=\"/wiki/Systems_theory\" title=\"Systems theory\">systems theory</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%;font-weight:normal;\">other</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Infosphere\" title=\"Infosphere\">infosphere</a></li>\\n<li><a href=\"/wiki/Inforg\" title=\"Inforg\">inforg</a></li>\\n<li><i><a href=\"/wiki/Decoding_the_Universe\" title=\"Decoding the Universe\">Decoding the Universe</a></i></li>\\n<li><a href=\"/wiki/Information_overload\" title=\"Information overload\">information overload</a></li></ul>\\n</div></td></tr></tbody></table><div></div></td></tr></tbody></table></div>\\n<div class=\"navbox-styles\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1061467846\"></div><div role=\"navigation\" class=\"navbox\" aria-labelledby=\"Omics\" style=\"padding:3px\"><table class=\"nowraplinks hlist mw-collapsible autocollapse navbox-inner\" style=\"border-spacing:0;background:transparent;color:inherit\"><tbody><tr><th scope=\"col\" class=\"navbox-title\" colspan=\"2\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1063604349\"><div class=\"navbar plainlinks hlist navbar-mini\"><ul><li class=\"nv-view\"><a href=\"/wiki/Template:Genomics\" title=\"Template:Genomics\"><abbr title=\"View this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">v</abbr></a></li><li class=\"nv-talk\"><a href=\"/wiki/Template_talk:Genomics\" title=\"Template talk:Genomics\"><abbr title=\"Discuss this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">t</abbr></a></li><li class=\"nv-edit\"><a class=\"external text\" href=\"https://en.wikipedia.org/w/index.php?title=Template:Genomics&amp;action=edit\"><abbr title=\"Edit this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">e</abbr></a></li></ul></div><div id=\"Omics\" style=\"font-size:114%;margin:0 4em\"><a href=\"/wiki/Omics\" title=\"Omics\">Omics</a></div></th></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Genomics\" title=\"Genomics\">Genomics</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Cognitive_genomics\" title=\"Cognitive genomics\">Cognitive genomics</a></li>\\n<li><a href=\"/wiki/Computational_genomics\" title=\"Computational genomics\">Computational genomics</a></li>\\n<li><a href=\"/wiki/Comparative_genomics\" title=\"Comparative genomics\">Comparative genomics</a></li>\\n<li><a href=\"/wiki/Functional_genomics\" title=\"Functional genomics\">Functional genomics</a></li>\\n<li><a href=\"/wiki/Genome_project\" title=\"Genome project\">Genome project</a>\\n<ul><li><a href=\"/wiki/Human_Genome_Project\" title=\"Human Genome Project\">Human Genome Project</a></li></ul></li>\\n<li><a href=\"/wiki/Metagenomics\" title=\"Metagenomics\">Metagenomics</a>\\n<ul><li><a href=\"/wiki/Human_Microbiome_Project\" title=\"Human Microbiome Project\">Human Microbiome Project</a></li></ul></li>\\n<li><a href=\"/wiki/Pangenomics\" class=\"mw-redirect\" title=\"Pangenomics\">Pangenomics</a></li>\\n<li><a href=\"/wiki/Personal_genomics\" title=\"Personal genomics\">Personal genomics</a></li>\\n<li><a href=\"/wiki/Population_genomics\" title=\"Population genomics\">Population genomics</a></li>\\n<li><a href=\"/wiki/Social_genomics\" class=\"mw-redirect\" title=\"Social genomics\">Social genomics</a></li>\\n<li><a href=\"/wiki/Structural_genomics\" title=\"Structural genomics\">Structural genomics</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a class=\"mw-selflink selflink\">Bioinformatics</a></th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Biochip\" title=\"Biochip\">Biochip</a></li>\\n<li><a href=\"/wiki/Cheminformatics\" title=\"Cheminformatics\">Cheminformatics</a></li>\\n<li><a href=\"/wiki/Chemogenomics\" title=\"Chemogenomics\">Chemogenomics</a></li>\\n<li><a href=\"/wiki/Connectomics\" title=\"Connectomics\">Connectomics</a>\\n<ul><li><a href=\"/wiki/Human_Connectome_Project\" title=\"Human Connectome Project\">Human Connectome Project</a></li></ul></li>\\n<li><a href=\"/wiki/Epigenomics\" title=\"Epigenomics\">Epigenomics</a>\\n<ul><li><a href=\"/wiki/Human_Epigenome_Project\" title=\"Human Epigenome Project\">Human Epigenome Project</a></li></ul></li>\\n<li><a href=\"/wiki/Glycomics\" title=\"Glycomics\">Glycomics</a></li>\\n<li><a href=\"/wiki/Immunomics\" title=\"Immunomics\">Immunomics</a></li>\\n<li><a href=\"/wiki/Lipidomics\" title=\"Lipidomics\">Lipidomics</a></li>\\n<li><a href=\"/wiki/Metabolomics\" title=\"Metabolomics\">Metabolomics</a></li>\\n<li><a href=\"/wiki/Microbiome\" title=\"Microbiome\">Microbiomics</a></li>\\n<li><a href=\"/wiki/Nutrigenomics\" class=\"mw-redirect\" title=\"Nutrigenomics\">Nutrigenomics</a></li>\\n<li><a href=\"/wiki/Paleopolyploidy\" title=\"Paleopolyploidy\">Paleopolyploidy</a></li>\\n<li><a href=\"/wiki/Pharmacogenetics\" class=\"mw-redirect\" title=\"Pharmacogenetics\">Pharmacogenetics</a></li>\\n<li><a href=\"/wiki/Pharmacogenomics\" title=\"Pharmacogenomics\">Pharmacogenomics</a></li>\\n<li><a href=\"/wiki/Systems_biology\" title=\"Systems biology\">Systems biology</a></li>\\n<li><a href=\"/wiki/Toxicogenomics\" title=\"Toxicogenomics\">Toxicogenomics</a></li>\\n<li><a href=\"/wiki/Transcriptomics\" class=\"mw-redirect\" title=\"Transcriptomics\">Transcriptomics</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Structural_biology\" title=\"Structural biology\">Structural biology</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Proteomics\" title=\"Proteomics\">Proteomics</a>\\n<ul><li><a href=\"/wiki/Human_proteome_project\" class=\"mw-redirect\" title=\"Human proteome project\">Human proteome project</a></li></ul></li>\\n<li><a href=\"/wiki/Call-map_proteomics\" class=\"mw-redirect\" title=\"Call-map proteomics\">Call-map proteomics</a></li>\\n<li><a href=\"/wiki/Structure-based_drug_design\" class=\"mw-redirect\" title=\"Structure-based drug design\">Structure-based drug design</a></li>\\n<li><a href=\"/wiki/Expression_proteomics\" class=\"mw-redirect\" title=\"Expression proteomics\">Expression proteomics</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Research tools</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/2-D_electrophoresis\" class=\"mw-redirect\" title=\"2-D electrophoresis\">2-D electrophoresis</a></li>\\n<li><a href=\"/wiki/Mass_spectrometer\" class=\"mw-redirect\" title=\"Mass spectrometer\">Mass spectrometer</a></li>\\n<li><a href=\"/wiki/Electrospray_ionization\" title=\"Electrospray ionization\">Electrospray ionization</a></li>\\n<li><a href=\"/wiki/Matrix-assisted_laser_desorption_ionization\" class=\"mw-redirect\" title=\"Matrix-assisted laser desorption ionization\">Matrix-assisted laser desorption ionization</a></li>\\n<li><a href=\"/wiki/Matrix-assisted_laser_desorption_ionization-time_of_flight_mass_spectrometer\" class=\"mw-redirect\" title=\"Matrix-assisted laser desorption ionization-time of flight mass spectrometer\">Matrix-assisted laser desorption ionization-time of flight mass spectrometer</a></li>\\n<li><a href=\"/wiki/Microfluidic-based_tools\" class=\"mw-redirect\" title=\"Microfluidic-based tools\">Microfluidic-based tools</a></li>\\n<li><a href=\"/wiki/Isotope_affinity_tags\" class=\"mw-redirect\" title=\"Isotope affinity tags\">Isotope affinity tags</a></li>\\n<li><a href=\"/wiki/Chromosome_conformation_capture\" title=\"Chromosome conformation capture\">Chromosome conformation capture</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Organizations</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/DNA_Data_Bank_of_Japan\" title=\"DNA Data Bank of Japan\">DNA Data Bank of Japan</a> (JP)</li>\\n<li><a href=\"/wiki/European_Molecular_Biology_Laboratory\" title=\"European Molecular Biology Laboratory\">European Molecular Biology Laboratory</a> (EU)</li>\\n<li><a href=\"/wiki/National_Institutes_of_Health\" title=\"National Institutes of Health\">National Institutes of Health</a> (USA)</li>\\n<li><a href=\"/wiki/Wellcome_Sanger_Institute\" title=\"Wellcome Sanger Institute\">Wellcome Sanger Institute</a> (UK)</li></ul>\\n</div></td></tr><tr><td class=\"navbox-abovebelow\" colspan=\"2\"><div>\\n<ul><li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/en/thumb/d/db/Symbol_list_class.svg/16px-Symbol_list_class.svg.png\" decoding=\"async\" title=\"List-Class article\" width=\"16\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/en/thumb/d/db/Symbol_list_class.svg/23px-Symbol_list_class.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/d/db/Symbol_list_class.svg/31px-Symbol_list_class.svg.png 2x\" data-file-width=\"180\" data-file-height=\"185\" />  <b><a href=\"/wiki/List_of_omics_topics_in_biology\" title=\"List of omics topics in biology\">List</a> </b></li>\\n<li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/16px-Symbol_category_class.svg.png\" decoding=\"async\" title=\"Category\" width=\"16\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/23px-Symbol_category_class.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/31px-Symbol_category_class.svg.png 2x\" data-file-width=\"180\" data-file-height=\"185\" /> <b><a href=\"/wiki/Category:Omics\" title=\"Category:Omics\">Category</a></b></li></ul>\\n</div></td></tr></tbody></table></div>\\n<div class=\"navbox-styles\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1061467846\"></div><div role=\"navigation\" class=\"navbox\" aria-labelledby=\"Biology\" style=\"padding:3px\"><table class=\"nowraplinks hlist mw-collapsible autocollapse navbox-inner\" style=\"border-spacing:0;background:transparent;color:inherit\"><tbody><tr><th scope=\"col\" class=\"navbox-title\" colspan=\"3\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1063604349\"><div class=\"navbar plainlinks hlist navbar-mini\"><ul><li class=\"nv-view\"><a href=\"/wiki/Template:Biology_nav\" title=\"Template:Biology nav\"><abbr title=\"View this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">v</abbr></a></li><li class=\"nv-talk\"><a href=\"/wiki/Template_talk:Biology_nav\" title=\"Template talk:Biology nav\"><abbr title=\"Discuss this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">t</abbr></a></li><li class=\"nv-edit\"><a class=\"external text\" href=\"https://en.wikipedia.org/w/index.php?title=Template:Biology_nav&amp;action=edit\"><abbr title=\"Edit this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">e</abbr></a></li></ul></div><div id=\"Biology\" style=\"font-size:114%;margin:0 4em\"><a href=\"/wiki/Biology\" title=\"Biology\">Biology</a></div></th></tr><tr><td class=\"navbox-abovebelow\" colspan=\"3\"><div>\\n<ul><li>Introduction (<a href=\"/wiki/Introduction_to_genetics\" title=\"Introduction to genetics\">Genetics</a>, <a href=\"/wiki/Introduction_to_evolution\" title=\"Introduction to evolution\">Evolution</a>)</li>\\n<li><a href=\"/wiki/Outline_of_biology\" title=\"Outline of biology\">Outline</a></li>\\n<li><a href=\"/wiki/History_of_biology\" title=\"History of biology\">History</a>\\n<ul><li><a href=\"/wiki/Timeline_of_biology_and_organic_chemistry\" title=\"Timeline of biology and organic chemistry\">Timeline</a></li></ul></li>\\n<li><a href=\"/wiki/Index_of_biology_articles\" title=\"Index of biology articles\">Index</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Biology\" title=\"Biology\">Biology</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\"></div><table class=\"nowraplinks navbox-subgroup\" style=\"border-spacing:0\"><tbody><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Overview</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Science\" title=\"Science\">Science</a></li>\\n<li><a href=\"/wiki/Life\" title=\"Life\">Life</a></li>\\n<li>Properties (<a href=\"/wiki/Adaptation\" title=\"Adaptation\">Adaptation</a>,&#160;<a href=\"/wiki/Metabolism\" title=\"Metabolism\"> Energy processing</a>,&#160;<a href=\"/wiki/Developmental_biology\" title=\"Developmental biology\">Growth</a>,&#160;<a href=\"/wiki/Structure#Biological\" title=\"Structure\">Order</a>,&#160;<a href=\"/wiki/Homeostasis\" title=\"Homeostasis\">Regulation</a>,&#160;<a href=\"/wiki/Reproduction\" title=\"Reproduction\">Reproduction</a>,&#160;<a href=\"/wiki/Stimulus_(physiology)\" title=\"Stimulus (physiology)\">Response to environment</a>)</li>\\n<li><a href=\"/wiki/Biological_organisation\" title=\"Biological organisation\">Hierarchy of life</a> (<a href=\"/wiki/Atom\" title=\"Atom\">Atom</a> &gt;&#160;<a href=\"/wiki/Molecule\" title=\"Molecule\">Molecule</a> &gt;&#160;<a href=\"/wiki/Organelle\" title=\"Organelle\">Organelle</a> &gt;&#160;<a href=\"/wiki/Cell_(biology)\" title=\"Cell (biology)\">Cell</a> &gt;&#160;<a href=\"/wiki/Tissue_(biology)\" title=\"Tissue (biology)\">Tissue</a> &gt;&#160;<a href=\"/wiki/Organ_(biology)\" title=\"Organ (biology)\">Organ</a> &gt;&#160;<a href=\"/wiki/Biological_system\" title=\"Biological system\">Organ system</a> &gt;&#160;<a href=\"/wiki/Organism\" title=\"Organism\">Organism</a> &gt;&#160;<a href=\"/wiki/Population\" title=\"Population\">Population</a> &gt;&#160;<a href=\"/wiki/Biocoenosis\" title=\"Biocoenosis\">Community</a> &gt;&#160;<a href=\"/wiki/Ecosystem\" title=\"Ecosystem\">Ecosystem</a> &gt;&#160;<a href=\"/wiki/Biosphere\" title=\"Biosphere\">Biosphere</a>)</li>\\n<li><a href=\"/wiki/Reductionism#In_science\" title=\"Reductionism\">Reductionistic</a></li>\\n<li><a href=\"/wiki/Emergence\" title=\"Emergence\"> Emergent property</a></li>\\n<li><a href=\"/wiki/Mechanical_philosophy\" class=\"mw-redirect\" title=\"Mechanical philosophy\">Mechanistic</a></li>\\n<li><a href=\"/wiki/Scientific_method\" title=\"Scientific method\">Scientific method</a></li>\\n<li><a href=\"/wiki/Scientific_theory\" title=\"Scientific theory\"> Theory</a></li>\\n<li><a href=\"/wiki/Scientific_law\" title=\"Scientific law\"> Law</a></li>\\n<li><a href=\"/wiki/Peer_review\" title=\"Peer review\">Peer review</a></li>\\n<li><a href=\"/wiki/List_of_biology_journals\" title=\"List of biology journals\">Biology journals</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Chemical basis</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Atom\" title=\"Atom\">Atoms</a></li>\\n<li><a href=\"/wiki/Amino_acid\" title=\"Amino acid\">Amino acids</a></li>\\n<li><a href=\"/wiki/Carbohydrate\" title=\"Carbohydrate\">Carbohydrates</a></li>\\n<li><a href=\"/wiki/Chemical_bond\" title=\"Chemical bond\">Chemical bond</a></li>\\n<li><a href=\"/wiki/Chemical_element\" title=\"Chemical element\">Chemical element</a></li>\\n<li><a href=\"/wiki/Lipid\" title=\"Lipid\">Lipids</a></li>\\n<li><a href=\"/wiki/Matter\" title=\"Matter\">Matter</a>\\n<ul><li><a href=\"/wiki/Quantum_biology\" title=\"Quantum biology\">Quantum</a></li></ul></li>\\n<li><a href=\"/wiki/Molecule\" title=\"Molecule\">Molecules</a></li>\\n<li><a href=\"/wiki/Monomer\" title=\"Monomer\">Monomer</a></li>\\n<li><a href=\"/wiki/Nucleic_acid\" title=\"Nucleic acid\">Nucleic acids</a></li>\\n<li><a href=\"/wiki/Organic_compound\" title=\"Organic compound\">Organic compounds</a></li>\\n<li><a href=\"/wiki/PH\" title=\"PH\">pH</a></li>\\n<li><a href=\"/wiki/Polymer\" title=\"Polymer\">Polymer</a></li>\\n<li><a href=\"/wiki/Protein\" title=\"Protein\">Proteins</a></li>\\n<li><a href=\"/wiki/Water\" title=\"Water\">Water</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Cell_(biology)\" title=\"Cell (biology)\">Cells</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Adenosine_triphosphate\" title=\"Adenosine triphosphate\">ATP</a></li>\\n<li><a href=\"/wiki/Cell_cycle\" title=\"Cell cycle\">Cell cycle</a></li>\\n<li><a href=\"/wiki/Cell_theory\" title=\"Cell theory\">Cell theory</a></li>\\n<li><a href=\"/wiki/Cell_signaling\" title=\"Cell signaling\">Cell signaling</a></li>\\n<li><a href=\"/wiki/Cellular_respiration\" title=\"Cellular respiration\">Cellular respiration</a></li>\\n<li><a href=\"/wiki/Energy\" title=\"Energy\">Energy transformation</a></li>\\n<li><a href=\"/wiki/Enzyme\" title=\"Enzyme\">Enzyme</a></li>\\n<li><a href=\"/wiki/Eukaryote\" title=\"Eukaryote\">Eukaryote</a></li>\\n<li><a href=\"/wiki/Fermentation\" title=\"Fermentation\">Fermentation</a></li>\\n<li><a href=\"/wiki/Metabolism\" title=\"Metabolism\">Metabolism</a></li>\\n<li><a href=\"/wiki/Meiosis\" title=\"Meiosis\">Meiosis</a></li>\\n<li><a href=\"/wiki/Mitosis\" title=\"Mitosis\">Mitosis</a></li>\\n<li><a href=\"/wiki/Photosynthesis\" title=\"Photosynthesis\">Photosynthesis</a></li>\\n<li><a href=\"/wiki/Prokaryote\" title=\"Prokaryote\">Prokaryote</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Genetics\" title=\"Genetics\">Genetics</a></th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/DNA\" title=\"DNA\">DNA</a></li>\\n<li><a href=\"/wiki/Epigenetics\" title=\"Epigenetics\">Epigenetics</a></li>\\n<li><a href=\"/wiki/Evolutionary_developmental_biology\" title=\"Evolutionary developmental biology\">Evolutionary developmental biology</a></li>\\n<li><a href=\"/wiki/Gene_expression\" title=\"Gene expression\">Gene expression</a></li>\\n<li><a href=\"/wiki/Regulation_of_gene_expression\" title=\"Regulation of gene expression\">Gene regulation</a></li>\\n<li><a href=\"/wiki/Genome\" title=\"Genome\">Genomes</a></li>\\n<li><a href=\"/wiki/Mendelian_inheritance\" title=\"Mendelian inheritance\">Mendelian inheritance</a></li>\\n<li><a href=\"/wiki/Post-transcriptional_modification\" title=\"Post-transcriptional modification\">Post-transcriptional modification</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Evolution\" title=\"Evolution\">Evolution</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Adaptation\" title=\"Adaptation\">Adaptation</a></li>\\n<li><a href=\"/wiki/Earliest_known_life_forms\" title=\"Earliest known life forms\">Earliest known life forms</a></li>\\n<li><a href=\"/wiki/Function_(biology)\" title=\"Function (biology)\">Function</a></li>\\n<li><a href=\"/wiki/Genetic_drift\" title=\"Genetic drift\">Genetic drift</a></li>\\n<li><a href=\"/wiki/Gene_flow\" title=\"Gene flow\">Gene flow</a></li>\\n<li><a href=\"/wiki/History_of_life\" title=\"History of life\">History of life</a></li>\\n<li><a href=\"/wiki/Macroevolution\" title=\"Macroevolution\">Macroevolution</a></li>\\n<li><a href=\"/wiki/Microevolution\" title=\"Microevolution\">Microevolution</a></li>\\n<li><a href=\"/wiki/Mutation\" title=\"Mutation\">Mutation</a></li>\\n<li><a href=\"/wiki/Natural_selection\" title=\"Natural selection\">Natural selection</a></li>\\n<li><a href=\"/wiki/Phylogenetics\" title=\"Phylogenetics\">Phylogenetics</a></li>\\n<li><a href=\"/wiki/Speciation\" title=\"Speciation\">Speciation</a></li>\\n<li><a href=\"/wiki/Taxonomy_(biology)\" title=\"Taxonomy (biology)\">Taxonomy</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Biodiversity\" title=\"Biodiversity\">Diversity</a></th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Archaea\" title=\"Archaea\">Archaea</a></li>\\n<li><a href=\"/wiki/Animal\" title=\"Animal\">Animals</a></li>\\n<li><a href=\"/wiki/Bacteria\" title=\"Bacteria\">Bacteria</a></li>\\n<li><a href=\"/wiki/Biodiversity\" title=\"Biodiversity\">Biodiversity</a></li>\\n<li><a href=\"/wiki/Fungus\" title=\"Fungus\">Fungus</a></li>\\n<li><a href=\"/wiki/Invertebrate\" title=\"Invertebrate\">Invertebrates</a></li>\\n<li><a href=\"/wiki/Plant\" title=\"Plant\">Plants</a></li>\\n<li><a href=\"/wiki/Protist\" title=\"Protist\">Protists</a></li>\\n<li><a href=\"/wiki/Vertebrate\" title=\"Vertebrate\">Vertebrates</a></li>\\n<li><a href=\"/wiki/Virus\" title=\"Virus\">Viruses</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Plant form <br /> and function</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Epidermis_(botany)\" title=\"Epidermis (botany)\">Epidermis (botany)</a></li>\\n<li><a href=\"/wiki/Flower\" title=\"Flower\">Flower</a></li>\\n<li><a href=\"/wiki/Ground_tissue\" title=\"Ground tissue\">Ground tissue</a></li>\\n<li><a href=\"/wiki/Leaf\" title=\"Leaf\">Leaf</a></li>\\n<li><a href=\"/wiki/Phloem\" title=\"Phloem\">Phloem</a></li>\\n<li><a href=\"/wiki/Plant_stem\" title=\"Plant stem\">Plant stem</a></li>\\n<li><a href=\"/wiki/Root\" title=\"Root\">Root</a></li>\\n<li><a href=\"/wiki/Shoot\" title=\"Shoot\">Shoot</a></li>\\n<li><a href=\"/wiki/Vascular_plant\" title=\"Vascular plant\">Vascular plant</a></li>\\n<li><a href=\"/wiki/Vascular_tissue\" title=\"Vascular tissue\">Vascular tissue</a></li>\\n<li><a href=\"/wiki/Xylem\" title=\"Xylem\">Xylem</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Animal form <br />and function</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Breathing\" title=\"Breathing\">Breathing</a></li>\\n<li><a href=\"/wiki/Circulatory_system\" title=\"Circulatory system\">Circulatory system</a></li>\\n<li><a href=\"/wiki/Endocrine_system\" title=\"Endocrine system\">Endocrine system</a></li>\\n<li><a href=\"/wiki/Human_digestive_system\" title=\"Human digestive system\">Digestive system</a></li>\\n<li><a href=\"/wiki/Homeostasis\" title=\"Homeostasis\">Homeostasis</a></li>\\n<li><a href=\"/wiki/Immune_system\" title=\"Immune system\">Immune system</a></li>\\n<li><a href=\"/wiki/Internal_environment\" title=\"Internal environment\">Internal environment</a></li>\\n<li><a href=\"/wiki/Muscular_system\" title=\"Muscular system\">Muscular system</a></li>\\n<li><a href=\"/wiki/Nervous_system\" title=\"Nervous system\">Nervous system</a></li>\\n<li><a href=\"/wiki/Reproductive_system\" title=\"Reproductive system\">Reproductive system</a></li>\\n<li><a href=\"/wiki/Respiratory_system\" title=\"Respiratory system\">Respiratory system</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Ecology\" title=\"Ecology\">Ecology</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Biogeochemical_cycle\" title=\"Biogeochemical cycle\">Biogeochemical cycle</a></li>\\n<li><a href=\"/wiki/Biological_interaction\" title=\"Biological interaction\">Biological interaction</a></li>\\n<li><a href=\"/wiki/Biomass\" title=\"Biomass\">Biomass</a></li>\\n<li><a href=\"/wiki/Biome\" title=\"Biome\">Biomes</a></li>\\n<li><a href=\"/wiki/Biosphere\" title=\"Biosphere\">Biosphere</a></li>\\n<li><a href=\"/wiki/Climate\" title=\"Climate\">Climate</a></li>\\n<li><a href=\"/wiki/Climate_change\" title=\"Climate change\">Climate change</a></li>\\n<li><a href=\"/wiki/Community_(ecology)\" title=\"Community (ecology)\">Community</a></li>\\n<li><a href=\"/wiki/Conservation_biology\" title=\"Conservation biology\">Conservation</a></li>\\n<li><a href=\"/wiki/Ecosystem_ecology\" title=\"Ecosystem ecology\">Ecosystem</a></li>\\n<li><a href=\"/wiki/Habitat\" title=\"Habitat\">Habitat</a>\\n<ul><li><a href=\"/wiki/Ecological_niche\" title=\"Ecological niche\">niche</a></li></ul></li>\\n<li><a href=\"/wiki/Microbiome\" title=\"Microbiome\">Microbiome</a></li>\\n<li><a href=\"/wiki/Population_ecology\" title=\"Population ecology\">Population dynamics</a></li>\\n<li><a href=\"/wiki/Resource_(biology)\" title=\"Resource (biology)\">Resources</a></li></ul>\\n</div></td></tr></tbody></table><div></div></td><td class=\"noviewer navbox-image\" rowspan=\"4\" style=\"width:1px;padding:0 0 0 2px\"><div><a href=\"/wiki/File:Tree_of_life_by_Haeckel.jpg\" class=\"image\"><img alt=\"Tree of life by Haeckel.jpg\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/d/de/Tree_of_life_by_Haeckel.jpg/140px-Tree_of_life_by_Haeckel.jpg\" decoding=\"async\" width=\"140\" height=\"222\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/d/de/Tree_of_life_by_Haeckel.jpg/210px-Tree_of_life_by_Haeckel.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/d/de/Tree_of_life_by_Haeckel.jpg/280px-Tree_of_life_by_Haeckel.jpg 2x\" data-file-width=\"1804\" data-file-height=\"2856\" /></a></div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/List_of_research_methods_in_biology\" title=\"List of research methods in biology\">Research <br /> methods</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\"></div><table class=\"nowraplinks navbox-subgroup\" style=\"border-spacing:0\"><tbody><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Laboratory <br /> techniques</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Genetic_engineering\" title=\"Genetic engineering\">Genetic engineering</a></li>\\n<li><a href=\"/wiki/Transformation_(genetics)\" title=\"Transformation (genetics)\">Transformation</a></li>\\n<li><a href=\"/wiki/Gel_electrophoresis\" title=\"Gel electrophoresis\">Gel electrophoresis</a></li>\\n<li><a href=\"/wiki/Chromatography\" title=\"Chromatography\">Chromatography</a></li>\\n<li><a href=\"/wiki/Centrifugation\" title=\"Centrifugation\">Centrifugation</a></li>\\n<li><a href=\"/wiki/Cell_culture\" title=\"Cell culture\">Cell culture</a></li>\\n<li><a href=\"/wiki/DNA_sequencing\" title=\"DNA sequencing\">DNA sequencing</a></li>\\n<li><a href=\"/wiki/DNA_microarray\" title=\"DNA microarray\">DNA microarray</a></li>\\n<li><a href=\"/wiki/Green_fluorescent_protein\" title=\"Green fluorescent protein\">Green fluorescent protein</a></li>\\n<li><a href=\"/wiki/Vector_(molecular_biology)\" title=\"Vector (molecular biology)\">vector</a></li>\\n<li><a href=\"/wiki/Enzyme_assay\" title=\"Enzyme assay\">Enzyme assay</a></li>\\n<li><a href=\"/wiki/Protein_purification\" title=\"Protein purification\">Protein purification</a></li>\\n<li><a href=\"/wiki/Western_blot\" title=\"Western blot\">Western blot</a></li>\\n<li><a href=\"/wiki/Northern_blot\" title=\"Northern blot\">Northern blot</a></li>\\n<li><a href=\"/wiki/Southern_blot\" title=\"Southern blot\">Southern blot</a></li>\\n<li><a href=\"/wiki/Restriction_enzyme\" title=\"Restriction enzyme\">Restriction enzyme</a></li>\\n<li><a href=\"/wiki/Polymerase_chain_reaction\" title=\"Polymerase chain reaction\">Polymerase chain reaction</a></li>\\n<li><a href=\"/wiki/Two-hybrid_screening\" title=\"Two-hybrid screening\">Two-hybrid screening</a></li>\\n<li><i><a href=\"/wiki/In_vivo\" title=\"In vivo\">in vivo</a></i></li>\\n<li><i><a href=\"/wiki/In_vitro\" title=\"In vitro\">in vitro</a></i></li>\\n<li><i><a href=\"/wiki/In_silico\" title=\"In silico\">in silico</a></i></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Field techniques</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Belt_transect\" title=\"Belt transect\">Belt transect</a></li>\\n<li><a href=\"/wiki/Mark_and_recapture\" title=\"Mark and recapture\">mark and recapture</a></li>\\n<li><a href=\"/wiki/Species_discovery_curve\" title=\"Species discovery curve\">species discovery curve</a></li></ul>\\n</div></td></tr></tbody></table><div></div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Branches</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Agricultural_science\" title=\"Agricultural science\">Agricultural science</a></li>\\n<li><a href=\"/wiki/Anatomy\" title=\"Anatomy\">Anatomy</a></li>\\n<li><a href=\"/wiki/Astrobiology\" title=\"Astrobiology\">Astrobiology</a></li>\\n<li><a href=\"/wiki/Biochemistry\" title=\"Biochemistry\">Biochemistry</a></li>\\n<li><a href=\"/wiki/Biogeography\" title=\"Biogeography\">Biogeography</a></li>\\n<li><a href=\"/wiki/Biomechanics\" title=\"Biomechanics\">Biomechanics</a></li>\\n<li><a href=\"/wiki/Biophysics\" title=\"Biophysics\">Biophysics</a></li>\\n<li><a class=\"mw-selflink selflink\">Bioinformatics</a></li>\\n<li><a href=\"/wiki/Biosemiotics\" title=\"Biosemiotics\">Biosemiotics</a></li>\\n<li><a href=\"/wiki/Biostatistics\" title=\"Biostatistics\">Biostatistics</a></li>\\n<li><a href=\"/wiki/Biotechnology\" title=\"Biotechnology\">Biotechnology</a></li>\\n<li><a href=\"/wiki/Botany\" title=\"Botany\">Botany</a></li>\\n<li><a href=\"/wiki/Cell_biology\" title=\"Cell biology\">Cell biology</a></li>\\n<li><a href=\"/wiki/Cellular_microbiology\" title=\"Cellular microbiology\">Cellular microbiology</a></li>\\n<li><a href=\"/wiki/Chronobiology\" title=\"Chronobiology\">Chronobiology</a></li>\\n<li><a href=\"/wiki/Cognitive_biology\" title=\"Cognitive biology\">Cognitive biology</a></li>\\n<li><a href=\"/wiki/Computational_biology\" title=\"Computational biology\">Computational biology</a></li>\\n<li><a href=\"/wiki/Conservation_biology\" title=\"Conservation biology\">Conservation biology</a></li>\\n<li><a href=\"/wiki/Developmental_biology\" title=\"Developmental biology\">Developmental biology</a></li>\\n<li><a href=\"/wiki/Ecology\" title=\"Ecology\">Ecology</a></li>\\n<li><a href=\"/wiki/Epidemiology\" title=\"Epidemiology\">Epidemiology</a></li>\\n<li><a href=\"/wiki/Epigenetics\" title=\"Epigenetics\">Epigenetics</a></li>\\n<li><a href=\"/wiki/Evolutionary_biology\" title=\"Evolutionary biology\">Evolutionary biology</a></li>\\n<li><a href=\"/wiki/Evolutionary_developmental_biology\" title=\"Evolutionary developmental biology\">Evolutionary developmental biology</a></li>\\n<li><a href=\"/wiki/Genetics\" title=\"Genetics\">Genetics</a></li>\\n<li><a href=\"/wiki/Genomics\" title=\"Genomics\">Genomics</a></li>\\n<li><a href=\"/wiki/Histology\" title=\"Histology\">Histology</a></li>\\n<li><a href=\"/wiki/Human_biology\" title=\"Human biology\">Human biology</a></li>\\n<li><a href=\"/wiki/Immunology\" title=\"Immunology\">Immunology</a></li>\\n<li><a href=\"/wiki/Lipidology\" title=\"Lipidology\">Lipidology</a></li>\\n<li><a href=\"/wiki/Marine_biology\" title=\"Marine biology\">Marine biology</a></li>\\n<li><a href=\"/wiki/Mathematical_and_theoretical_biology\" title=\"Mathematical and theoretical biology\">Mathematical biology</a></li>\\n<li><a href=\"/wiki/Microbiology\" title=\"Microbiology\">Microbiology</a></li>\\n<li><a href=\"/wiki/Molecular_biology\" title=\"Molecular biology\">Molecular biology</a></li>\\n<li><a href=\"/wiki/Mycology\" title=\"Mycology\">Mycology</a></li>\\n<li><a href=\"/wiki/Neuroscience\" title=\"Neuroscience\">Neuroscience</a></li>\\n<li><a href=\"/wiki/Paleontology\" title=\"Paleontology\">Paleontology</a></li>\\n<li><a href=\"/wiki/Parasitology\" title=\"Parasitology\">Parasitology</a></li>\\n<li><a href=\"/wiki/Pathology\" title=\"Pathology\">Pathology</a></li>\\n<li><a href=\"/wiki/Pharmacology\" title=\"Pharmacology\">Pharmacology</a></li>\\n<li><a href=\"/wiki/Phycology\" title=\"Phycology\">Phycology</a></li>\\n<li><a href=\"/wiki/Physiology\" title=\"Physiology\">Physiology</a></li>\\n<li><a href=\"/wiki/Reproduction\" title=\"Reproduction\">Reproductive biology</a></li>\\n<li><a href=\"/wiki/Sociobiology\" title=\"Sociobiology\">Sociobiology</a></li>\\n<li><a href=\"/wiki/Structural_biology\" title=\"Structural biology\">Structural biology</a></li>\\n<li><a href=\"/wiki/Synthetic_biology\" title=\"Synthetic biology\">Synthetic biology</a></li>\\n<li><a href=\"/wiki/Systematics\" title=\"Systematics\">Systematics</a></li>\\n<li><a href=\"/wiki/Systems_biology\" title=\"Systems biology\">Systems biology</a></li>\\n<li><a href=\"/wiki/Virology\" title=\"Virology\">Virology</a></li>\\n<li><a href=\"/wiki/Zoology\" title=\"Zoology\">Zoology</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Glossaries</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Glossary_of_biology\" title=\"Glossary of biology\">Biology</a></li>\\n<li><a href=\"/wiki/Glossary_of_botanical_terms\" title=\"Glossary of botanical terms\">Botanical terms</a></li>\\n<li><a href=\"/wiki/Glossary_of_ecology\" title=\"Glossary of ecology\">Ecological terms</a></li>\\n<li><a href=\"/wiki/Glossary_of_plant_morphology\" title=\"Glossary of plant morphology\">Plant morphology terms</a></li></ul>\\n</div></td></tr><tr><td class=\"navbox-abovebelow\" colspan=\"3\"><div>\\n<ul><li><span class=\"nowrap\"><a href=\"/wiki/File:Issoria_lathonia.jpg\" class=\"image\"><img alt=\"icon\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/16px-Issoria_lathonia.jpg\" decoding=\"async\" width=\"16\" height=\"11\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/24px-Issoria_lathonia.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/32px-Issoria_lathonia.jpg 2x\" data-file-width=\"629\" data-file-height=\"445\" /></a>&#160;<a href=\"/wiki/Portal:Biology\" title=\"Portal:Biology\">Biology&#32;portal</a></span></li>\\n<li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/16px-Symbol_category_class.svg.png\" decoding=\"async\" title=\"Category\" width=\"16\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/23px-Symbol_category_class.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/31px-Symbol_category_class.svg.png 2x\" data-file-width=\"180\" data-file-height=\"185\" /> <a href=\"/wiki/Category:Biology\" title=\"Category:Biology\">Category</a></li>\\n<li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/12px-Commons-logo.svg.png\" decoding=\"async\" title=\"Commons page\" width=\"12\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/18px-Commons-logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/24px-Commons-logo.svg.png 2x\" data-file-width=\"1024\" data-file-height=\"1376\" /> <a href=\"https://commons.wikimedia.org/wiki/Category:Biology\" class=\"extiw\" title=\"commons:Category:Biology\">Commons</a></li>\\n<li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/3/37/People_icon.svg/16px-People_icon.svg.png\" decoding=\"async\" title=\"WikiProject\" width=\"16\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/3/37/People_icon.svg/24px-People_icon.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/3/37/People_icon.svg/32px-People_icon.svg.png 2x\" data-file-width=\"100\" data-file-height=\"100\" /> <a href=\"/wiki/Wikipedia:WikiProject_Biology\" title=\"Wikipedia:WikiProject Biology\">WikiProject</a></li></ul>\\n</div></td></tr></tbody></table></div>\\n<div class=\"navbox-styles\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1061467846\"></div><div role=\"navigation\" class=\"navbox\" aria-labelledby=\"Branches_of_biology\" style=\"padding:3px\"><table class=\"nowraplinks mw-collapsible autocollapse navbox-inner\" style=\"border-spacing:0;background:transparent;color:inherit\"><tbody><tr><th scope=\"col\" class=\"navbox-title\" colspan=\"2\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1063604349\"><div class=\"navbar plainlinks hlist navbar-mini\"><ul><li class=\"nv-view\"><a href=\"/wiki/Template:Branches_of_biology\" title=\"Template:Branches of biology\"><abbr title=\"View this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">v</abbr></a></li><li class=\"nv-talk\"><a href=\"/wiki/Template_talk:Branches_of_biology\" title=\"Template talk:Branches of biology\"><abbr title=\"Discuss this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">t</abbr></a></li><li class=\"nv-edit\"><a class=\"external text\" href=\"https://en.wikipedia.org/w/index.php?title=Template:Branches_of_biology&amp;action=edit\"><abbr title=\"Edit this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">e</abbr></a></li></ul></div><div id=\"Branches_of_biology\" style=\"font-size:114%;margin:0 4em\"><a href=\"/wiki/Outline_of_biology#Branches\" title=\"Outline of biology\">Branches of biology</a></div></th></tr><tr><td colspan=\"2\" class=\"navbox-list navbox-odd hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Abiogenesis\" title=\"Abiogenesis\">Abiogenesis</a></li>\\n<li><a href=\"/wiki/Aerobiology\" title=\"Aerobiology\">Aerobiology</a></li>\\n<li><a href=\"/wiki/Agrostology\" title=\"Agrostology\">Agrostology</a></li>\\n<li><a href=\"/wiki/Anatomy\" title=\"Anatomy\">Anatomy</a></li>\\n<li><a href=\"/wiki/Astrobiology\" title=\"Astrobiology\">Astrobiology</a></li>\\n<li><a href=\"/wiki/Bacteriology\" title=\"Bacteriology\">Bacteriology</a></li>\\n<li><a href=\"/wiki/Biochemistry\" title=\"Biochemistry\">Biochemistry</a></li>\\n<li><a href=\"/wiki/Biogeography\" title=\"Biogeography\">Biogeography</a></li>\\n<li><a href=\"/wiki/Biogeology\" title=\"Biogeology\">Biogeology</a></li>\\n<li><a class=\"mw-selflink selflink\">Bioinformatics</a></li>\\n<li><a href=\"/wiki/Biological_engineering\" title=\"Biological engineering\">Biological engineering</a></li>\\n<li><a href=\"/wiki/Biomechanics\" title=\"Biomechanics\">Biomechanics</a></li>\\n<li><a href=\"/wiki/Biophysics\" title=\"Biophysics\">Biophysics</a></li>\\n<li><a href=\"/wiki/Biosemiotics\" title=\"Biosemiotics\">Biosemiotics</a></li>\\n<li><a href=\"/wiki/Biostatistics\" title=\"Biostatistics\">Biostatistics</a></li>\\n<li><a href=\"/wiki/Biotechnology\" title=\"Biotechnology\">Biotechnology</a></li>\\n<li><a href=\"/wiki/Botany\" title=\"Botany\">Botany</a></li>\\n<li><a href=\"/wiki/Cell_biology\" title=\"Cell biology\">Cell biology</a></li>\\n<li><a href=\"/wiki/Cellular_microbiology\" title=\"Cellular microbiology\">Cellular microbiology</a></li>\\n<li><a href=\"/wiki/Chemical_biology\" title=\"Chemical biology\">Chemical biology</a></li>\\n<li><a href=\"/wiki/Chronobiology\" title=\"Chronobiology\">Chronobiology</a></li>\\n<li><a href=\"/wiki/Cognitive_biology\" title=\"Cognitive biology\">Cognitive biology</a></li>\\n<li><a href=\"/wiki/Computational_biology\" title=\"Computational biology\">Computational biology</a></li>\\n<li><a href=\"/wiki/Conservation_biology\" title=\"Conservation biology\">Conservation biology</a></li>\\n<li><a href=\"/wiki/Cryobiology\" title=\"Cryobiology\">Cryobiology</a></li>\\n<li><a href=\"/wiki/Cytogenetics\" title=\"Cytogenetics\">Cytogenetics</a></li>\\n<li><a href=\"/wiki/Dendrology\" title=\"Dendrology\">Dendrology</a></li>\\n<li><a href=\"/wiki/Developmental_biology\" title=\"Developmental biology\">Developmental biology</a></li>\\n<li><a href=\"/wiki/Ecological_genetics\" title=\"Ecological genetics\">Ecological genetics</a></li>\\n<li><a href=\"/wiki/Ecology\" title=\"Ecology\">Ecology</a></li>\\n<li><a href=\"/wiki/Embryology\" title=\"Embryology\">Embryology</a></li>\\n<li><a href=\"/wiki/Epidemiology\" title=\"Epidemiology\">Epidemiology</a></li>\\n<li><a href=\"/wiki/Epigenetics\" title=\"Epigenetics\">Epigenetics</a></li>\\n<li><a href=\"/wiki/Evolutionary_biology\" title=\"Evolutionary biology\">Evolutionary biology</a></li>\\n<li><a href=\"/wiki/Freshwater_biology\" title=\"Freshwater biology\">Freshwater biology</a></li>\\n<li><a href=\"/wiki/Genetics\" title=\"Genetics\">Genetics</a></li>\\n<li><a href=\"/wiki/Genomics\" title=\"Genomics\">Genomics</a></li>\\n<li><a href=\"/wiki/Geobiology\" title=\"Geobiology\">Geobiology</a></li>\\n<li><a href=\"/wiki/Gerontology\" title=\"Gerontology\">Gerontology</a></li>\\n<li><a href=\"/wiki/Herpetology\" title=\"Herpetology\">Herpetology</a></li>\\n<li><a href=\"/wiki/Histology\" title=\"Histology\">Histology</a></li>\\n<li><a href=\"/wiki/Human_biology\" title=\"Human biology\">Human biology</a></li>\\n<li><a href=\"/wiki/Ichthyology\" title=\"Ichthyology\">Ichthyology</a></li>\\n<li><a href=\"/wiki/Immunology\" title=\"Immunology\">Immunology</a></li>\\n<li><a href=\"/wiki/Lipidology\" title=\"Lipidology\">Lipidology</a></li>\\n<li><a href=\"/wiki/Mammalogy\" title=\"Mammalogy\">Mammalogy</a></li>\\n<li><a href=\"/wiki/Marine_biology\" title=\"Marine biology\">Marine biology</a></li>\\n<li><a href=\"/wiki/Mathematical_and_theoretical_biology\" title=\"Mathematical and theoretical biology\">Mathematical biology</a></li>\\n<li><a href=\"/wiki/Microbiology\" title=\"Microbiology\">Microbiology</a></li>\\n<li><a href=\"/wiki/Molecular_biology\" title=\"Molecular biology\">Molecular biology</a></li>\\n<li><a href=\"/wiki/Mycology\" title=\"Mycology\">Mycology</a></li>\\n<li><a href=\"/wiki/Neontology\" title=\"Neontology\">Neontology</a></li>\\n<li><a href=\"/wiki/Neuroscience\" title=\"Neuroscience\">Neuroscience</a></li>\\n<li><a href=\"/wiki/Nutritional_science\" title=\"Nutritional science\">Nutrition</a></li>\\n<li><a href=\"/wiki/Ornithology\" title=\"Ornithology\">Ornithology</a></li>\\n<li><a href=\"/wiki/Osteology\" title=\"Osteology\">Osteology</a></li>\\n<li><a href=\"/wiki/Paleontology\" title=\"Paleontology\">Paleontology</a></li>\\n<li><a href=\"/wiki/Parasitology\" title=\"Parasitology\">Parasitology</a></li>\\n<li><a href=\"/wiki/Pathology\" title=\"Pathology\">Pathology</a></li>\\n<li><a href=\"/wiki/Pharmacology\" title=\"Pharmacology\">Pharmacology</a></li>\\n<li><a href=\"/wiki/Photobiology\" title=\"Photobiology\">Photobiology</a></li>\\n<li><a href=\"/wiki/Phycology\" title=\"Phycology\">Phycology</a></li>\\n<li><a href=\"/wiki/Phylogenetics\" title=\"Phylogenetics\">Phylogenetics</a></li>\\n<li><a href=\"/wiki/Physiology\" title=\"Physiology\">Physiology</a></li>\\n<li><a href=\"/wiki/Pomology\" title=\"Pomology\">Pomology</a></li>\\n<li><a href=\"/wiki/Primatology\" title=\"Primatology\">Primatology</a></li>\\n<li><a href=\"/wiki/Proteomics\" title=\"Proteomics\">Proteomics</a></li>\\n<li><a href=\"/wiki/Protistology\" title=\"Protistology\">Protistology</a></li>\\n<li><a href=\"/wiki/Quantum_biology\" title=\"Quantum biology\">Quantum biology</a></li>\\n<li><a href=\"/wiki/Reproductive_biology\" title=\"Reproductive biology\">Reproductive biology</a></li>\\n<li><a href=\"/wiki/Sociobiology\" title=\"Sociobiology\">Sociobiology</a></li>\\n<li><a href=\"/wiki/Structural_biology\" title=\"Structural biology\">Structural biology</a></li>\\n<li><a href=\"/wiki/Synthetic_biology\" title=\"Synthetic biology\">Synthetic biology</a></li>\\n<li><a href=\"/wiki/Systematics\" title=\"Systematics\">Systematics</a></li>\\n<li><a href=\"/wiki/Systems_biology\" title=\"Systems biology\">Systems biology</a></li>\\n<li><a href=\"/wiki/Taxonomy_(biology)\" title=\"Taxonomy (biology)\">Taxonomy</a></li>\\n<li><a href=\"/wiki/Teratology\" title=\"Teratology\">Teratology</a></li>\\n<li><a href=\"/wiki/Toxicology\" title=\"Toxicology\">Toxicology</a></li>\\n<li><a href=\"/wiki/Virology\" title=\"Virology\">Virology</a></li>\\n<li><a href=\"/wiki/Virophysics\" title=\"Virophysics\">Virophysics</a></li>\\n<li><a href=\"/wiki/Xenobiology\" title=\"Xenobiology\">Xenobiology</a></li>\\n<li><a href=\"/wiki/Zoology\" title=\"Zoology\">Zoology</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">See also</th><td class=\"navbox-list-with-group navbox-list navbox-even hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/History_of_biology\" title=\"History of biology\">History of biology</a></li>\\n<li><a href=\"/wiki/Nobel_Prize_in_Physiology_or_Medicine\" title=\"Nobel Prize in Physiology or Medicine\">Nobel Prize in Physiology or Medicine</a></li>\\n<li><a href=\"/wiki/Timeline_of_biology_and_organic_chemistry\" title=\"Timeline of biology and organic chemistry\">Timeline of biology and organic chemistry</a></li></ul>\\n</div></td></tr></tbody></table></div>\\n<div class=\"navbox-styles\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1061467846\"></div><div role=\"navigation\" class=\"navbox\" aria-labelledby=\"Computer_science\" style=\"padding:3px\"><table class=\"nowraplinks hlist mw-collapsible mw-collapsed navbox-inner\" style=\"border-spacing:0;background:transparent;color:inherit\"><tbody><tr><th scope=\"col\" class=\"navbox-title\" colspan=\"3\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1063604349\"><div class=\"navbar plainlinks hlist navbar-mini\"><ul><li class=\"nv-view\"><a href=\"/wiki/Template:Computer_science\" title=\"Template:Computer science\"><abbr title=\"View this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">v</abbr></a></li><li class=\"nv-talk\"><a href=\"/wiki/Template_talk:Computer_science\" title=\"Template talk:Computer science\"><abbr title=\"Discuss this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">t</abbr></a></li><li class=\"nv-edit\"><a class=\"external text\" href=\"https://en.wikipedia.org/w/index.php?title=Template:Computer_science&amp;action=edit\"><abbr title=\"Edit this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">e</abbr></a></li></ul></div><div id=\"Computer_science\" style=\"font-size:114%;margin:0 4em\"><a href=\"/wiki/Computer_science\" title=\"Computer science\">Computer science</a></div></th></tr><tr><td class=\"navbox-abovebelow\" colspan=\"3\"><div>Note: This template roughly follows the 2012 <a href=\"/wiki/ACM_Computing_Classification_System\" title=\"ACM Computing Classification System\">ACM Computing Classification System</a>.</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Computer_hardware\" title=\"Computer hardware\">Hardware</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Printed_circuit_board\" title=\"Printed circuit board\">Printed circuit board</a></li>\\n<li><a href=\"/wiki/Peripheral\" title=\"Peripheral\">Peripheral</a></li>\\n<li><a href=\"/wiki/Integrated_circuit\" title=\"Integrated circuit\">Integrated circuit</a></li>\\n<li><a href=\"/wiki/Very_Large_Scale_Integration\" title=\"Very Large Scale Integration\">Very Large Scale Integration</a></li>\\n<li><a href=\"/wiki/System_on_a_chip\" title=\"System on a chip\">Systems on Chip (SoCs)</a></li>\\n<li><a href=\"/wiki/Green_computing\" title=\"Green computing\">Energy consumption (Green computing)</a></li>\\n<li><a href=\"/wiki/Electronic_design_automation\" title=\"Electronic design automation\">Electronic design automation</a></li>\\n<li><a href=\"/wiki/Hardware_acceleration\" title=\"Hardware acceleration\">Hardware acceleration</a></li></ul>\\n</div></td><td class=\"noviewer navbox-image\" rowspan=\"17\" style=\"width:1px;padding:0 0 0 2px\"><div><a href=\"/wiki/File:Computer_Retro.svg\" class=\"image\"><img alt=\"Computer Retro.svg\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/5/51/Computer_Retro.svg/50px-Computer_Retro.svg.png\" decoding=\"async\" width=\"50\" height=\"50\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/5/51/Computer_Retro.svg/75px-Computer_Retro.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/5/51/Computer_Retro.svg/100px-Computer_Retro.svg.png 2x\" data-file-width=\"512\" data-file-height=\"512\" /></a></div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Computer systems organization</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Computer_architecture\" title=\"Computer architecture\">Computer architecture</a></li>\\n<li><a href=\"/wiki/Embedded_system\" title=\"Embedded system\">Embedded system</a></li>\\n<li><a href=\"/wiki/Real-time_computing\" title=\"Real-time computing\">Real-time computing</a></li>\\n<li><a href=\"/wiki/Dependability\" title=\"Dependability\">Dependability</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Computer_network\" title=\"Computer network\">Networks</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Network_architecture\" title=\"Network architecture\">Network architecture</a></li>\\n<li><a href=\"/wiki/Network_protocol\" class=\"mw-redirect\" title=\"Network protocol\">Network protocol</a></li>\\n<li><a href=\"/wiki/Networking_hardware\" title=\"Networking hardware\">Network components</a></li>\\n<li><a href=\"/wiki/Network_scheduler\" title=\"Network scheduler\">Network scheduler</a></li>\\n<li><a href=\"/wiki/Network_performance\" title=\"Network performance\">Network performance evaluation</a></li>\\n<li><a href=\"/wiki/Network_service\" title=\"Network service\">Network service</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Software organization</th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Interpreter_(computing)\" title=\"Interpreter (computing)\">Interpreter</a></li>\\n<li><a href=\"/wiki/Middleware\" title=\"Middleware\">Middleware</a></li>\\n<li><a href=\"/wiki/Virtual_machine\" title=\"Virtual machine\">Virtual machine</a></li>\\n<li><a href=\"/wiki/Operating_system\" title=\"Operating system\">Operating system</a></li>\\n<li><a href=\"/wiki/Software_quality\" title=\"Software quality\">Software quality</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Programming_language_theory\" title=\"Programming language theory\">Software notations</a> and <a href=\"/wiki/Programming_tool\" title=\"Programming tool\">tools</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Programming_paradigm\" title=\"Programming paradigm\">Programming paradigm</a></li>\\n<li><a href=\"/wiki/Programming_language\" title=\"Programming language\">Programming language</a></li>\\n<li><a href=\"/wiki/Compiler_construction\" class=\"mw-redirect\" title=\"Compiler construction\">Compiler</a></li>\\n<li><a href=\"/wiki/Domain-specific_language\" title=\"Domain-specific language\">Domain-specific language</a></li>\\n<li><a href=\"/wiki/Modeling_language\" title=\"Modeling language\">Modeling language</a></li>\\n<li><a href=\"/wiki/Software_framework\" title=\"Software framework\">Software framework</a></li>\\n<li><a href=\"/wiki/Integrated_development_environment\" title=\"Integrated development environment\">Integrated development environment</a></li>\\n<li><a href=\"/wiki/Software_configuration_management\" title=\"Software configuration management\">Software configuration management</a></li>\\n<li><a href=\"/wiki/Library_(computing)\" title=\"Library (computing)\">Software library</a></li>\\n<li><a href=\"/wiki/Software_repository\" title=\"Software repository\">Software repository</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Software_development\" title=\"Software development\">Software development</a></th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Control_variable_(programming)\" class=\"mw-redirect\" title=\"Control variable (programming)\">Control variable</a></li>\\n<li><a href=\"/wiki/Software_development_process\" title=\"Software development process\">Software development process</a></li>\\n<li><a href=\"/wiki/Requirements_analysis\" title=\"Requirements analysis\">Requirements analysis</a></li>\\n<li><a href=\"/wiki/Software_design\" title=\"Software design\">Software design</a></li>\\n<li><a href=\"/wiki/Software_construction\" title=\"Software construction\">Software construction</a></li>\\n<li><a href=\"/wiki/Software_deployment\" title=\"Software deployment\">Software deployment</a></li>\\n<li><a href=\"/wiki/Software_engineering\" title=\"Software engineering\">Software engineering</a></li>\\n<li><a href=\"/wiki/Software_maintenance\" title=\"Software maintenance\">Software maintenance</a></li>\\n<li><a href=\"/wiki/Programming_team\" title=\"Programming team\">Programming team</a></li>\\n<li><a href=\"/wiki/Open-source_software\" title=\"Open-source software\">Open-source model</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Theory_of_computation\" title=\"Theory of computation\">Theory of computation</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Model_of_computation\" title=\"Model of computation\">Model of computation</a></li>\\n<li><a href=\"/wiki/Formal_language\" title=\"Formal language\">Formal language</a></li>\\n<li><a href=\"/wiki/Automata_theory\" title=\"Automata theory\">Automata theory</a></li>\\n<li><a href=\"/wiki/Computability_theory\" title=\"Computability theory\">Computability theory</a></li>\\n<li><a href=\"/wiki/Computational_complexity_theory\" title=\"Computational complexity theory\">Computational complexity theory</a></li>\\n<li><a href=\"/wiki/Logic_in_computer_science\" title=\"Logic in computer science\">Logic</a></li>\\n<li><a href=\"/wiki/Semantics_(computer_science)\" title=\"Semantics (computer science)\">Semantics</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Algorithm\" title=\"Algorithm\">Algorithms</a></th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Algorithm_design\" class=\"mw-redirect\" title=\"Algorithm design\">Algorithm design</a></li>\\n<li><a href=\"/wiki/Analysis_of_algorithms\" title=\"Analysis of algorithms\">Analysis of algorithms</a></li>\\n<li><a href=\"/wiki/Algorithmic_efficiency\" title=\"Algorithmic efficiency\">Algorithmic efficiency</a></li>\\n<li><a href=\"/wiki/Randomized_algorithm\" title=\"Randomized algorithm\">Randomized algorithm</a></li>\\n<li><a href=\"/wiki/Computational_geometry\" title=\"Computational geometry\">Computational geometry</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Mathematics of computing</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Discrete_mathematics\" title=\"Discrete mathematics\">Discrete mathematics</a></li>\\n<li><a href=\"/wiki/Probability\" title=\"Probability\">Probability</a></li>\\n<li><a href=\"/wiki/Statistics\" title=\"Statistics\">Statistics</a></li>\\n<li><a href=\"/wiki/Mathematical_software\" title=\"Mathematical software\">Mathematical software</a></li>\\n<li><a href=\"/wiki/Information_theory\" title=\"Information theory\">Information theory</a></li>\\n<li><a href=\"/wiki/Mathematical_analysis\" title=\"Mathematical analysis\">Mathematical analysis</a></li>\\n<li><a href=\"/wiki/Numerical_analysis\" title=\"Numerical analysis\">Numerical analysis</a></li>\\n<li><a href=\"/wiki/Theoretical_computer_science\" title=\"Theoretical computer science\">Theoretical computer science</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Information_system\" title=\"Information system\">Information systems</a></th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Database\" title=\"Database\">Database management system</a></li>\\n<li><a href=\"/wiki/Computer_data_storage\" title=\"Computer data storage\">Information storage systems</a></li>\\n<li><a href=\"/wiki/Enterprise_information_system\" title=\"Enterprise information system\">Enterprise information system</a></li>\\n<li><a href=\"/wiki/Social_software\" title=\"Social software\">Social information systems</a></li>\\n<li><a href=\"/wiki/Geographic_information_system\" title=\"Geographic information system\">Geographic information system</a></li>\\n<li><a href=\"/wiki/Decision_support_system\" title=\"Decision support system\">Decision support system</a></li>\\n<li><a href=\"/wiki/Process_control\" class=\"mw-redirect\" title=\"Process control\">Process control system</a></li>\\n<li><a href=\"/wiki/Multimedia_database\" title=\"Multimedia database\">Multimedia information system</a></li>\\n<li><a href=\"/wiki/Data_mining\" title=\"Data mining\">Data mining</a></li>\\n<li><a href=\"/wiki/Digital_library\" title=\"Digital library\">Digital library</a></li>\\n<li><a href=\"/wiki/Computing_platform\" title=\"Computing platform\">Computing platform</a></li>\\n<li><a href=\"/wiki/Digital_marketing\" title=\"Digital marketing\">Digital marketing</a></li>\\n<li><a href=\"/wiki/World_Wide_Web\" title=\"World Wide Web\">World Wide Web</a></li>\\n<li><a href=\"/wiki/Information_retrieval\" title=\"Information retrieval\">Information retrieval</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Computer_security\" title=\"Computer security\">Security</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Cryptography\" title=\"Cryptography\">Cryptography</a></li>\\n<li><a href=\"/wiki/Formal_methods\" title=\"Formal methods\">Formal methods</a></li>\\n<li><a href=\"/wiki/Security_service_(telecommunication)\" title=\"Security service (telecommunication)\">Security services</a></li>\\n<li><a href=\"/wiki/Intrusion_detection_system\" title=\"Intrusion detection system\">Intrusion detection system</a></li>\\n<li><a href=\"/wiki/Computer_security_compromised_by_hardware_failure\" title=\"Computer security compromised by hardware failure\">Hardware security</a></li>\\n<li><a href=\"/wiki/Network_security\" title=\"Network security\">Network security</a></li>\\n<li><a href=\"/wiki/Information_security\" title=\"Information security\">Information security</a></li>\\n<li><a href=\"/wiki/Application_security\" title=\"Application security\">Application security</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Human%E2%80%93computer_interaction\" title=\"Human\\xe2\\x80\\x93computer interaction\">Human\\xe2\\x80\\x93computer interaction</a></th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Interaction_design\" title=\"Interaction design\">Interaction design</a></li>\\n<li><a href=\"/wiki/Social_computing\" title=\"Social computing\">Social computing</a></li>\\n<li><a href=\"/wiki/Ubiquitous_computing\" title=\"Ubiquitous computing\">Ubiquitous computing</a></li>\\n<li><a href=\"/wiki/Visualization_(graphics)\" title=\"Visualization (graphics)\">Visualization</a></li>\\n<li><a href=\"/wiki/Computer_accessibility\" title=\"Computer accessibility\">Accessibility</a></li>\\n<li><a href=\"/wiki/Synthography\" title=\"Synthography\">Synthography</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Concurrency_(computer_science)\" title=\"Concurrency (computer science)\">Concurrency</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Concurrent_computing\" title=\"Concurrent computing\">Concurrent computing</a></li>\\n<li><a href=\"/wiki/Parallel_computing\" title=\"Parallel computing\">Parallel computing</a></li>\\n<li><a href=\"/wiki/Distributed_computing\" title=\"Distributed computing\">Distributed computing</a></li>\\n<li><a href=\"/wiki/Multithreading_(computer_architecture)\" title=\"Multithreading (computer architecture)\">Multithreading</a></li>\\n<li><a href=\"/wiki/Multiprocessing\" title=\"Multiprocessing\">Multiprocessing</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Artificial_intelligence\" title=\"Artificial intelligence\">Artificial intelligence</a></th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Natural_language_processing\" title=\"Natural language processing\">Natural language processing</a></li>\\n<li><a href=\"/wiki/Knowledge_representation_and_reasoning\" title=\"Knowledge representation and reasoning\">Knowledge representation and reasoning</a></li>\\n<li><a href=\"/wiki/Computer_vision\" title=\"Computer vision\">Computer vision</a></li>\\n<li><a href=\"/wiki/Automated_planning_and_scheduling\" title=\"Automated planning and scheduling\">Automated planning and scheduling</a></li>\\n<li><a href=\"/wiki/Mathematical_optimization\" title=\"Mathematical optimization\">Search methodology</a></li>\\n<li><a href=\"/wiki/Control_theory\" title=\"Control theory\">Control method</a></li>\\n<li><a href=\"/wiki/Philosophy_of_artificial_intelligence\" title=\"Philosophy of artificial intelligence\">Philosophy of artificial intelligence</a></li>\\n<li><a href=\"/wiki/Distributed_artificial_intelligence\" title=\"Distributed artificial intelligence\">Distributed artificial intelligence</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Machine_learning\" title=\"Machine learning\">Machine learning</a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Supervised_learning\" title=\"Supervised learning\">Supervised learning</a></li>\\n<li><a href=\"/wiki/Unsupervised_learning\" title=\"Unsupervised learning\">Unsupervised learning</a></li>\\n<li><a href=\"/wiki/Reinforcement_learning\" title=\"Reinforcement learning\">Reinforcement learning</a></li>\\n<li><a href=\"/wiki/Multi-task_learning\" title=\"Multi-task learning\">Multi-task learning</a></li>\\n<li><a href=\"/wiki/Cross-validation_(statistics)\" title=\"Cross-validation (statistics)\">Cross-validation</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Computer_graphics\" title=\"Computer graphics\">Graphics</a></th><td class=\"navbox-list-with-group navbox-list navbox-even\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Computer_animation\" title=\"Computer animation\">Animation</a></li>\\n<li><a href=\"/wiki/Rendering_(computer_graphics)\" title=\"Rendering (computer graphics)\">Rendering</a></li>\\n<li><a href=\"/wiki/Photograph_manipulation\" title=\"Photograph manipulation\">Photograph manipulation</a></li>\\n<li><a href=\"/wiki/Graphics_processing_unit\" title=\"Graphics processing unit\">Graphics processing unit</a></li>\\n<li><a href=\"/wiki/Mixed_reality\" title=\"Mixed reality\">Mixed reality</a></li>\\n<li><a href=\"/wiki/Virtual_reality\" title=\"Virtual reality\">Virtual reality</a></li>\\n<li><a href=\"/wiki/Image_compression\" title=\"Image compression\">Image compression</a></li>\\n<li><a href=\"/wiki/Solid_modeling\" title=\"Solid modeling\">Solid modeling</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Applied computing</th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/E-commerce\" title=\"E-commerce\">E-commerce</a></li>\\n<li><a href=\"/wiki/Enterprise_software\" title=\"Enterprise software\">Enterprise software</a></li>\\n<li><a href=\"/wiki/Computational_mathematics\" title=\"Computational mathematics\">Computational mathematics</a></li>\\n<li><a href=\"/wiki/Computational_physics\" title=\"Computational physics\">Computational physics</a></li>\\n<li><a href=\"/wiki/Computational_chemistry\" title=\"Computational chemistry\">Computational chemistry</a></li>\\n<li><a href=\"/wiki/Computational_biology\" title=\"Computational biology\">Computational biology</a></li>\\n<li><a href=\"/wiki/Computational_social_science\" title=\"Computational social science\">Computational social science</a></li>\\n<li><a href=\"/wiki/Computational_engineering\" title=\"Computational engineering\">Computational engineering</a></li>\\n<li><a href=\"/wiki/Health_informatics\" title=\"Health informatics\">Computational healthcare</a></li>\\n<li><a href=\"/wiki/Digital_art\" title=\"Digital art\">Digital art</a></li>\\n<li><a href=\"/wiki/Electronic_publishing\" title=\"Electronic publishing\">Electronic publishing</a></li>\\n<li><a href=\"/wiki/Cyberwarfare\" title=\"Cyberwarfare\">Cyberwarfare</a></li>\\n<li><a href=\"/wiki/Electronic_voting\" title=\"Electronic voting\">Electronic voting</a></li>\\n<li><a href=\"/wiki/Video_game\" title=\"Video game\">Video games</a></li>\\n<li><a href=\"/wiki/Word_processor\" title=\"Word processor\">Word processing</a></li>\\n<li><a href=\"/wiki/Operations_research\" title=\"Operations research\">Operations research</a></li>\\n<li><a href=\"/wiki/Educational_technology\" title=\"Educational technology\">Educational technology</a></li>\\n<li><a href=\"/wiki/Document_management_system\" title=\"Document management system\">Document management</a></li></ul>\\n</div></td></tr><tr><td class=\"navbox-abovebelow\" colspan=\"3\"><div>\\n<ul><li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/16px-Symbol_category_class.svg.png\" decoding=\"async\" title=\"Category\" width=\"16\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/23px-Symbol_category_class.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/9/96/Symbol_category_class.svg/31px-Symbol_category_class.svg.png 2x\" data-file-width=\"180\" data-file-height=\"185\" /> <b><a href=\"/wiki/Category:Computer_science\" title=\"Category:Computer science\">Category</a></b></li>\\n<li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/4/41/Global_thinking.svg/10px-Global_thinking.svg.png\" decoding=\"async\" title=\"Outline\" width=\"10\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/4/41/Global_thinking.svg/15px-Global_thinking.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/4/41/Global_thinking.svg/21px-Global_thinking.svg.png 2x\" data-file-width=\"130\" data-file-height=\"200\" /> <b><a href=\"/wiki/Outline_of_computer_science\" title=\"Outline of computer science\">Outline</a></b></li>\\n<li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/3/37/People_icon.svg/16px-People_icon.svg.png\" decoding=\"async\" title=\"WikiProject\" width=\"16\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/3/37/People_icon.svg/24px-People_icon.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/3/37/People_icon.svg/32px-People_icon.svg.png 2x\" data-file-width=\"100\" data-file-height=\"100\" /><b><a href=\"/wiki/Wikipedia:WikiProject_Computer_science\" title=\"Wikipedia:WikiProject Computer science\">WikiProject</a></b></li>\\n<li><img alt=\"\" src=\"//upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/12px-Commons-logo.svg.png\" decoding=\"async\" title=\"Commons page\" width=\"12\" height=\"16\" class=\"noviewer\" srcset=\"//upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/18px-Commons-logo.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/24px-Commons-logo.svg.png 2x\" data-file-width=\"1024\" data-file-height=\"1376\" /> <b><a href=\"https://commons.wikimedia.org/wiki/Category:Computer_science\" class=\"extiw\" title=\"commons:Category:Computer science\">Commons</a></b></li></ul>\\n</div></td></tr></tbody></table></div>\\n<div class=\"navbox-styles\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1061467846\"></div><div role=\"navigation\" class=\"navbox\" aria-labelledby=\"Health_informatics\" style=\"padding:3px\"><table class=\"nowraplinks mw-collapsible autocollapse navbox-inner\" style=\"border-spacing:0;background:transparent;color:inherit\"><tbody><tr><th scope=\"col\" class=\"navbox-title\" colspan=\"2\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1063604349\"><div class=\"navbar plainlinks hlist navbar-mini\"><ul><li class=\"nv-view\"><a href=\"/wiki/Template:Health_informatics\" title=\"Template:Health informatics\"><abbr title=\"View this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">v</abbr></a></li><li class=\"nv-talk\"><a href=\"/wiki/Template_talk:Health_informatics\" title=\"Template talk:Health informatics\"><abbr title=\"Discuss this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">t</abbr></a></li><li class=\"nv-edit\"><a class=\"external text\" href=\"https://en.wikipedia.org/w/index.php?title=Template:Health_informatics&amp;action=edit\"><abbr title=\"Edit this template\" style=\";;background:none transparent;border:none;box-shadow:none;padding:0;\">e</abbr></a></li></ul></div><div id=\"Health_informatics\" style=\"font-size:114%;margin:0 4em\"><a href=\"/wiki/Health_informatics\" title=\"Health informatics\">Health informatics</a></div></th></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Subdisciplines</th><td class=\"navbox-list-with-group navbox-list navbox-odd hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Medical_image_computing\" title=\"Medical image computing\">Medical image computing</a> and <a href=\"/wiki/Imaging_informatics\" title=\"Imaging informatics\">imaging informatics</a></li>\\n<li><a href=\"/wiki/Artificial_intelligence_in_healthcare\" title=\"Artificial intelligence in healthcare\">Artificial intelligence in healthcare</a></li>\\n<li><a href=\"/wiki/Neuroinformatics\" title=\"Neuroinformatics\">Neuroinformatics</a> in healthcare</li>\\n<li><a href=\"/wiki/Behavior_informatics\" title=\"Behavior informatics\">Behavior informatics</a> in healthcare</li>\\n<li><a href=\"/wiki/Computational_biology\" title=\"Computational biology\">Computational biology</a> in healthcare</li>\\n<li><a href=\"/wiki/Translational_bioinformatics\" title=\"Translational bioinformatics\">Translational bioinformatics</a></li>\\n<li><a href=\"/wiki/Translational_medicine\" title=\"Translational medicine\">Translational medicine</a></li>\\n<li><a href=\"/wiki/Health_information_technology\" title=\"Health information technology\">health information technology</a></li>\\n<li><a href=\"/wiki/Telemedicine\" class=\"mw-redirect\" title=\"Telemedicine\">Telemedicine</a></li>\\n<li><a href=\"/wiki/Public_health_informatics\" title=\"Public health informatics\">Public health informatics</a></li>\\n<li><a href=\"/wiki/Health_information_management\" title=\"Health information management\">Health information management</a></li>\\n<li><a href=\"/wiki/Consumer_health_informatics\" title=\"Consumer health informatics\">Consumer health informatics</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Medical_classification\" title=\"Medical classification\">Medical classification</a></th><td class=\"navbox-list-with-group navbox-list navbox-even hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Continuity_of_Care_Record\" title=\"Continuity of Care Record\">Continuity of Care Record</a></li>\\n<li><a href=\"/wiki/HRHIS\" title=\"HRHIS\">HRHIS</a></li>\\n<li><a href=\"/wiki/International_Statistical_Classification_of_Diseases_and_Related_Health_Problems\" class=\"mw-redirect\" title=\"International Statistical Classification of Diseases and Related Health Problems\">ICD</a></li>\\n<li><a href=\"/wiki/ISO_27799\" class=\"mw-redirect\" title=\"ISO 27799\">ISO 27799</a></li>\\n<li><a href=\"/wiki/LOINC\" title=\"LOINC\">LOINC</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Professional organizations</th><td class=\"navbox-list-with-group navbox-list navbox-odd hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/American_Association_for_Medical_Systems_and_Informatics\" title=\"American Association for Medical Systems and Informatics\">American Association for Medical Systems and Informatics</a></li>\\n<li><a href=\"/wiki/American_Medical_Informatics_Association\" title=\"American Medical Informatics Association\">American Medical Informatics Association</a></li>\\n<li><a href=\"/wiki/Health_Informatics_Society_of_Australia\" title=\"Health Informatics Society of Australia\">Australian Society</a> / <a href=\"/wiki/Australasian_College_of_Health_Informatics\" title=\"Australasian College of Health Informatics\">Australasian College</a> (to merge from 2020)</li>\\n<li><a href=\"/wiki/Brazilian_Society_of_Health_Informatics\" title=\"Brazilian Society of Health Informatics\">Brazilian Society</a></li>\\n<li><a href=\"/wiki/European_Federation_for_Medical_Informatics\" title=\"European Federation for Medical Informatics\">European Federation</a></li>\\n<li><a href=\"/wiki/Indian_Association_for_Medical_Informatics\" title=\"Indian Association for Medical Informatics\">Indian Association</a></li>\\n<li><a href=\"/wiki/International_Medical_Informatics_Association\" title=\"International Medical Informatics Association\">International Association</a></li>\\n<li><a href=\"/wiki/American_College_of_Medical_Informatics\" title=\"American College of Medical Informatics\">American College of Medical Informatics</a></li></ul>\\n</div></td></tr><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\">Other concepts</th><td class=\"navbox-list-with-group navbox-list navbox-even hlist\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><a href=\"/wiki/Electronic_health_record\" title=\"Electronic health record\">Electronic health record</a></li>\\n<li><a href=\"/wiki/Health_Level_7\" title=\"Health Level 7\">Health Level 7</a></li>\\n<li><a href=\"/wiki/Remote_manipulator\" title=\"Remote manipulator\">Remote manipulator</a></li>\\n<li><a href=\"/wiki/Personalized_medicine\" title=\"Personalized medicine\">Personalized medicine</a> / <a href=\"/wiki/Precision_medicine\" class=\"mw-redirect\" title=\"Precision medicine\">precision medicine</a></li>\\n<li><a href=\"/wiki/List_of_medical_and_health_informatics_journals\" title=\"List of medical and health informatics journals\">List of medical and health informatics journals</a></li>\\n<li><a href=\"/wiki/OpenEHR\" title=\"OpenEHR\">openEHR</a></li></ul>\\n</div></td></tr></tbody></table></div>\\n<style data-mw-deduplicate=\"TemplateStyles:r1130092004\">.mw-parser-output .portal-bar{font-size:88%;font-weight:bold;display:flex;justify-content:center;align-items:baseline}.mw-parser-output .portal-bar-bordered{padding:0 2em;background-color:#fdfdfd;border:1px solid #a2a9b1;clear:both;margin:1em auto 0}.mw-parser-output .portal-bar-related{font-size:100%;justify-content:flex-start}.mw-parser-output .portal-bar-unbordered{padding:0 1.7em;margin-left:0}.mw-parser-output .portal-bar-header{margin:0 1em 0 0.5em;flex:0 0 auto;min-height:24px}.mw-parser-output .portal-bar-content{display:flex;flex-flow:row wrap;flex:0 1 auto;padding:0.15em 0;column-gap:1em;align-items:baseline;margin:0;list-style:none}.mw-parser-output .portal-bar-content-related{margin:0;list-style:none}.mw-parser-output .portal-bar-item{display:inline-block;margin:0.15em 0.2em;min-height:24px;line-height:24px}@media screen and (max-width:768px){.mw-parser-output .portal-bar{font-size:88%;font-weight:bold;display:flex;flex-flow:column wrap;align-items:baseline}.mw-parser-output .portal-bar-header{text-align:center;flex:0;padding-left:0.5em;margin:0 auto}.mw-parser-output .portal-bar-related{font-size:100%;align-items:flex-start}.mw-parser-output .portal-bar-content{display:flex;flex-flow:row wrap;align-items:center;flex:0;column-gap:1em;border-top:1px solid #a2a9b1;margin:0 auto;list-style:none}.mw-parser-output .portal-bar-content-related{border-top:none;margin:0;list-style:none}}.mw-parser-output .navbox+link+.portal-bar,.mw-parser-output .navbox+style+.portal-bar,.mw-parser-output .navbox+link+.portal-bar-bordered,.mw-parser-output .navbox+style+.portal-bar-bordered,.mw-parser-output .sister-bar+link+.portal-bar,.mw-parser-output .sister-bar+style+.portal-bar,.mw-parser-output .portal-bar+.navbox-styles+.navbox,.mw-parser-output .portal-bar+.navbox-styles+.sister-bar{margin-top:-1px}</style><div class=\"portal-bar noprint metadata noviewer portal-bar-bordered\" role=\"navigation\" aria-label=\"Portals\"><span class=\"portal-bar-header\"><a href=\"/wiki/Wikipedia:Contents/Portals\" title=\"Wikipedia:Contents/Portals\">Portals</a>:</span><ul class=\"portal-bar-content\"><li class=\"portal-bar-item\"><a href=\"/wiki/File:Issoria_lathonia.jpg\" class=\"image\"><img alt=\"icon\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/21px-Issoria_lathonia.jpg\" decoding=\"async\" width=\"21\" height=\"15\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/32px-Issoria_lathonia.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Issoria_lathonia.jpg/42px-Issoria_lathonia.jpg 2x\" data-file-width=\"629\" data-file-height=\"445\" /></a>&#160;<a href=\"/wiki/Portal:Biology\" title=\"Portal:Biology\">Biology</a></li><li class=\"portal-bar-item\"><a href=\"/wiki/File:Tree_of_life.svg\" class=\"image\"><img alt=\"icon\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/0/09/Tree_of_life.svg/21px-Tree_of_life.svg.png\" decoding=\"async\" width=\"21\" height=\"17\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/0/09/Tree_of_life.svg/32px-Tree_of_life.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/0/09/Tree_of_life.svg/42px-Tree_of_life.svg.png 2x\" data-file-width=\"915\" data-file-height=\"760\" /></a>&#160;<a href=\"/wiki/Portal:Evolutionary_biology\" title=\"Portal:Evolutionary biology\">Evolutionary biology</a></li></ul></div>\\n<div class=\"navbox-styles\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1129693374\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1061467846\"><style data-mw-deduplicate=\"TemplateStyles:r1038841319\">.mw-parser-output .tooltip-dotted{border-bottom:1px dotted;cursor:help}</style><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1038841319\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1038841319\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1038841319\"><link rel=\"mw-deduplicated-inline-style\" href=\"mw-data:TemplateStyles:r1038841319\"></div><div role=\"navigation\" class=\"navbox authority-control\" aria-label=\"Navbox\" style=\"padding:3px\"><table class=\"nowraplinks hlist navbox-inner\" style=\"border-spacing:0;background:transparent;color:inherit\"><tbody><tr><th scope=\"row\" class=\"navbox-group\" style=\"width:1%\"><a href=\"/wiki/Help:Authority_control\" title=\"Help:Authority control\">Authority control</a>: National <a href=\"https://www.wikidata.org/wiki/Q128570#identifiers\" title=\"Edit this at Wikidata\"><img alt=\"Edit this at Wikidata\" src=\"//upload.wikimedia.org/wikipedia/en/thumb/8/8a/OOjs_UI_icon_edit-ltr-progressive.svg/10px-OOjs_UI_icon_edit-ltr-progressive.svg.png\" decoding=\"async\" width=\"10\" height=\"10\" style=\"vertical-align: text-top\" class=\"noprint\" srcset=\"//upload.wikimedia.org/wikipedia/en/thumb/8/8a/OOjs_UI_icon_edit-ltr-progressive.svg/15px-OOjs_UI_icon_edit-ltr-progressive.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/8/8a/OOjs_UI_icon_edit-ltr-progressive.svg/20px-OOjs_UI_icon_edit-ltr-progressive.svg.png 2x\" data-file-width=\"20\" data-file-height=\"20\" /></a></th><td class=\"navbox-list-with-group navbox-list navbox-odd\" style=\"width:100%;padding:0\"><div style=\"padding:0 0.25em\">\\n<ul><li><span class=\"uid\"><span class=\"rt-commentedText tooltip tooltip-dotted\" title=\"Bioinformatique\"><a rel=\"nofollow\" class=\"external text\" href=\"https://catalogue.bnf.fr/ark:/12148/cb144116742\">France</a></span></span></li>\\n<li><span class=\"uid\"><span class=\"rt-commentedText tooltip tooltip-dotted\" title=\"Bioinformatique\"><a rel=\"nofollow\" class=\"external text\" href=\"https://data.bnf.fr/ark:/12148/cb144116742\">BnF data</a></span></span></li>\\n<li><span class=\"uid\"><a rel=\"nofollow\" class=\"external text\" href=\"https://d-nb.info/gnd/4611085-9\">Germany</a></span></li>\\n<li><span class=\"uid\"><a rel=\"nofollow\" class=\"external text\" href=\"http://uli.nli.org.il/F/?func=find-b&amp;local_base=NLX10&amp;find_code=UID&amp;request=987007291740605171\">Israel</a></span></li>\\n<li><span class=\"uid\"><span class=\"rt-commentedText tooltip tooltip-dotted\" title=\"Bioinformatics\"><a rel=\"nofollow\" class=\"external text\" href=\"https://id.loc.gov/authorities/sh00003585\">United States</a></span></span></li>\\n<li><span class=\"uid\"><span class=\"rt-commentedText tooltip tooltip-dotted\" title=\"\\xe3\\x83\\x90\\xe3\\x82\\xa4\\xe3\\x82\\xaa\\xe3\\x82\\xa4\\xe3\\x83\\xb3\\xe3\\x83\\x95\\xe3\\x82\\xa9\\xe3\\x83\\x9e\\xe3\\x83\\x86\\xe3\\x82\\xa3\\xe3\\x82\\xaf\\xe3\\x82\\xb9\"><a rel=\"nofollow\" class=\"external text\" href=\"https://id.ndl.go.jp/auth/ndlna/00870420\">Japan</a></span></span></li>\\n<li><span class=\"uid\"><span class=\"rt-commentedText tooltip tooltip-dotted\" title=\"bioinformatika\"><a rel=\"nofollow\" class=\"external text\" href=\"https://aleph.nkp.cz/F/?func=find-c&amp;local_base=aut&amp;ccl_term=ica=ph194800&amp;CON_LNG=ENG\">Czech Republic</a></span></span></li></ul>\\n</div></td></tr></tbody></table></div>\\n<!-- \\nNewPP limit report\\nParsed by mw2331\\nCached time: 20230707215806\\nCache expiry: 1814400\\nReduced expiry: false\\nComplications: [vary\\xe2\\x80\\x90revision\\xe2\\x80\\x90sha1, show\\xe2\\x80\\x90toc]\\nCPU time usage: 1.774 seconds\\nReal time usage: 2.084 seconds\\nPreprocessor visited node count: 18236/1000000\\nPost\\xe2\\x80\\x90expand include size: 489329/2097152 bytes\\nTemplate argument size: 10957/2097152 bytes\\nHighest expansion depth: 16/100\\nExpensive parser function count: 53/500\\nUnstrip recursion depth: 1/20\\nUnstrip post\\xe2\\x80\\x90expand size: 387198/5000000 bytes\\nLua time usage: 1.221/10.000 seconds\\nLua memory usage: 9244762/52428800 bytes\\nLua Profile:\\n    ?                                                                260 ms       22.0%\\n    <mw.lua:694>                                                     100 ms        8.5%\\n    recursiveClone <mwInit.lua:41>                                    80 ms        6.8%\\n    pairsfunc <mw.lua:676>                                            80 ms        6.8%\\n    type                                                              60 ms        5.1%\\n    MediaWiki\\\\Extension\\\\Scribunto\\\\Engines\\\\LuaSandbox\\\\LuaSandboxCallback::callParserFunction       60 ms        5.1%\\n    test <Module:Citation/CS1:1473>                                   40 ms        3.4%\\n    (for generator) <mw.lua:684>                                      40 ms        3.4%\\n    match                                                             40 ms        3.4%\\n    MediaWiki\\\\Extension\\\\Scribunto\\\\Engines\\\\LuaSandbox\\\\LuaSandboxCallback::gsub       40 ms        3.4%\\n    [others]                                                         380 ms       32.2%\\nNumber of Wikibase entities loaded: 1/400\\n-->\\n<!--\\nTransclusion expansion time report (%,ms,calls,template)\\n100.00% 1757.529      1 -total\\n 60.52% 1063.729      1 Template:Reflist\\n 46.38%  815.114     50 Template:Cite_journal\\n  7.21%  126.640     12 Template:Navbox\\n  5.24%   92.095     10 Template:Cite_web\\n  4.48%   78.777      1 Template:Bioinformatics\\n  3.94%   69.184     15 Template:ISBN\\n  2.71%   47.698      1 Template:Short_description\\n  2.59%   45.584      7 Template:Fix\\n  2.42%   42.580      1 Template:Authority_control\\n-->\\n\\n<!-- Saved in parser cache with key enwiki:pcache:idhash:4214-0!canonical and timestamp 20230707215804 and revision id 1163899643. Rendering was triggered because: page-view\\n -->\\n</div><!--esi <esi:include src=\"/esitest-fa8a495983347898/content\" /> --><noscript><img src=\"//en.wikipedia.org/wiki/Special:CentralAutoLogin/start?type=1x1\" alt=\"\" title=\"\" width=\"1\" height=\"1\" style=\"border: none; position: absolute;\" /></noscript>\\n<div class=\"printfooter\" data-nosnippet=\"\">Retrieved from \"<a dir=\"ltr\" href=\"https://en.wikipedia.org/w/index.php?title=Bioinformatics&amp;oldid=1163899643\">https://en.wikipedia.org/w/index.php?title=Bioinformatics&amp;oldid=1163899643</a>\"</div></div>\\n\\t\\t\\t\\t\\t<div id=\"catlinks\" class=\"catlinks\" data-mw=\"interface\"><div id=\"mw-normal-catlinks\" class=\"mw-normal-catlinks\"><a href=\"/wiki/Help:Category\" title=\"Help:Category\">Category</a>: <ul><li><a href=\"/wiki/Category:Bioinformatics\" title=\"Category:Bioinformatics\">Bioinformatics</a></li></ul></div><div id=\"mw-hidden-catlinks\" class=\"mw-hidden-catlinks mw-hidden-cats-hidden\">Hidden categories: <ul><li><a href=\"/wiki/Category:Articles_with_short_description\" title=\"Category:Articles with short description\">Articles with short description</a></li><li><a href=\"/wiki/Category:Short_description_matches_Wikidata\" title=\"Category:Short description matches Wikidata\">Short description matches Wikidata</a></li><li><a href=\"/wiki/Category:Use_dmy_dates_from_September_2020\" title=\"Category:Use dmy dates from September 2020\">Use dmy dates from September 2020</a></li><li><a href=\"/wiki/Category:Articles_with_hAudio_microformats\" title=\"Category:Articles with hAudio microformats\">Articles with hAudio microformats</a></li><li><a href=\"/wiki/Category:Pages_including_recorded_pronunciations\" title=\"Category:Pages including recorded pronunciations\">Pages including recorded pronunciations</a></li><li><a href=\"/wiki/Category:All_articles_with_vague_or_ambiguous_time\" title=\"Category:All articles with vague or ambiguous time\">All articles with vague or ambiguous time</a></li><li><a href=\"/wiki/Category:Vague_or_ambiguous_time_from_June_2023\" title=\"Category:Vague or ambiguous time from June 2023\">Vague or ambiguous time from June 2023</a></li><li><a href=\"/wiki/Category:All_articles_with_specifically_marked_weasel-worded_phrases\" title=\"Category:All articles with specifically marked weasel-worded phrases\">All articles with specifically marked weasel-worded phrases</a></li><li><a href=\"/wiki/Category:Articles_with_specifically_marked_weasel-worded_phrases_from_June_2020\" title=\"Category:Articles with specifically marked weasel-worded phrases from June 2020\">Articles with specifically marked weasel-worded phrases from June 2020</a></li><li><a href=\"/wiki/Category:All_articles_with_unsourced_statements\" title=\"Category:All articles with unsourced statements\">All articles with unsourced statements</a></li><li><a href=\"/wiki/Category:Articles_with_unsourced_statements_from_June_2023\" title=\"Category:Articles with unsourced statements from June 2023\">Articles with unsourced statements from June 2023</a></li><li><a href=\"/wiki/Category:Articles_with_unsourced_statements_from_July_2015\" title=\"Category:Articles with unsourced statements from July 2015\">Articles with unsourced statements from July 2015</a></li><li><a href=\"/wiki/Category:Wikipedia_articles_needing_clarification_from_June_2023\" title=\"Category:Wikipedia articles needing clarification from June 2023\">Wikipedia articles needing clarification from June 2023</a></li><li><a href=\"/wiki/Category:Webarchive_template_wayback_links\" title=\"Category:Webarchive template wayback links\">Webarchive template wayback links</a></li><li><a href=\"/wiki/Category:Spoken_articles\" title=\"Category:Spoken articles\">Spoken articles</a></li><li><a href=\"/wiki/Category:Commons_category_link_from_Wikidata\" title=\"Category:Commons category link from Wikidata\">Commons category link from Wikidata</a></li><li><a href=\"/wiki/Category:Articles_with_BNF_identifiers\" title=\"Category:Articles with BNF identifiers\">Articles with BNF identifiers</a></li><li><a href=\"/wiki/Category:Articles_with_BNFdata_identifiers\" title=\"Category:Articles with BNFdata identifiers\">Articles with BNFdata identifiers</a></li><li><a href=\"/wiki/Category:Articles_with_GND_identifiers\" title=\"Category:Articles with GND identifiers\">Articles with GND identifiers</a></li><li><a href=\"/wiki/Category:Articles_with_J9U_identifiers\" title=\"Category:Articles with J9U identifiers\">Articles with J9U identifiers</a></li><li><a href=\"/wiki/Category:Articles_with_LCCN_identifiers\" title=\"Category:Articles with LCCN identifiers\">Articles with LCCN identifiers</a></li><li><a href=\"/wiki/Category:Articles_with_NDL_identifiers\" title=\"Category:Articles with NDL identifiers\">Articles with NDL identifiers</a></li><li><a href=\"/wiki/Category:Articles_with_NKC_identifiers\" title=\"Category:Articles with NKC identifiers\">Articles with NKC identifiers</a></li></ul></div></div>\\n\\t\\t\\t\\t</div>\\n\\t\\t\\t</main>\\n\\t\\t\\t\\n\\t\\t</div>\\n\\t\\t<div class=\"mw-footer-container\">\\n\\t\\t\\t\\n<footer id=\"footer\" class=\"mw-footer\" role=\"contentinfo\" >\\n\\t<ul id=\"footer-info\">\\n\\t<li id=\"footer-info-lastmod\"> This page was last edited on 7 July 2023, at 01:13<span class=\"anonymous-show\">&#160;(UTC)</span>.</li>\\n\\t<li id=\"footer-info-copyright\">Text is available under the <a rel=\"license\" href=\"https://creativecommons.org/licenses/by-sa/4.0/\">Creative Commons Attribution-ShareAlike License 4.0</a><a rel=\"license\" href=\"//creativecommons.org/licenses/by-sa/4.0/\" style=\"display:none;\"></a>;\\nadditional terms may apply.  By using this site, you agree to the <a href=\"//foundation.wikimedia.org/wiki/Terms_of_Use\">Terms of Use</a> and <a href=\"//foundation.wikimedia.org/wiki/Privacy_policy\">Privacy Policy</a>. Wikipedia\\xc2\\xae is a registered trademark of the <a href=\"//www.wikimediafoundation.org/\">Wikimedia Foundation, Inc.</a>, a non-profit organization.</li>\\n</ul>\\n\\n\\t<ul id=\"footer-places\">\\n\\t<li id=\"footer-places-privacy\"><a href=\"https://foundation.wikimedia.org/wiki/Special:MyLanguage/Policy:Privacy_policy\">Privacy policy</a></li>\\n\\t<li id=\"footer-places-about\"><a href=\"/wiki/Wikipedia:About\">About Wikipedia</a></li>\\n\\t<li id=\"footer-places-disclaimers\"><a href=\"/wiki/Wikipedia:General_disclaimer\">Disclaimers</a></li>\\n\\t<li id=\"footer-places-contact\"><a href=\"//en.wikipedia.org/wiki/Wikipedia:Contact_us\">Contact Wikipedia</a></li>\\n\\t<li id=\"footer-places-wm-codeofconduct\"><a href=\"https://foundation.wikimedia.org/wiki/Special:MyLanguage/Universal_Code_of_Conduct\">Code of Conduct</a></li>\\n\\t<li id=\"footer-places-mobileview\"><a href=\"//en.m.wikipedia.org/w/index.php?title=Bioinformatics&amp;mobileaction=toggle_view_mobile\" class=\"noprint stopMobileRedirectToggle\">Mobile view</a></li>\\n\\t<li id=\"footer-places-developers\"><a href=\"https://developer.wikimedia.org\">Developers</a></li>\\n\\t<li id=\"footer-places-statslink\"><a href=\"https://stats.wikimedia.org/#/en.wikipedia.org\">Statistics</a></li>\\n\\t<li id=\"footer-places-cookiestatement\"><a href=\"https://foundation.wikimedia.org/wiki/Special:MyLanguage/Policy:Cookie_statement\">Cookie statement</a></li>\\n</ul>\\n\\n\\t<ul id=\"footer-icons\" class=\"noprint\">\\n\\t<li id=\"footer-copyrightico\"><a href=\"https://wikimediafoundation.org/\"><img src=\"/static/images/footer/wikimedia-button.png\" srcset=\"/static/images/footer/wikimedia-button-1.5x.png 1.5x, /static/images/footer/wikimedia-button-2x.png 2x\" width=\"88\" height=\"31\" alt=\"Wikimedia Foundation\" loading=\"lazy\" /></a></li>\\n\\t<li id=\"footer-poweredbyico\"><a href=\"https://www.mediawiki.org/\"><img src=\"/static/images/footer/poweredby_mediawiki_88x31.png\" alt=\"Powered by MediaWiki\" srcset=\"/static/images/footer/poweredby_mediawiki_132x47.png 1.5x, /static/images/footer/poweredby_mediawiki_176x62.png 2x\" width=\"88\" height=\"31\" loading=\"lazy\"></a></li>\\n</ul>\\n\\n</footer>\\n\\n\\t\\t</div>\\n\\t</div> \\n</div> \\n<div class=\\'vector-settings\\'>\\n\\t\\n<button\\n\\tid=\"\"\\n\\tclass=\"cdx-button cdx-button--icon-only vector-limited-width-toggle\"\\n\\t><span class=\"vector-icon mw-ui-icon-fullScreen mw-ui-icon-wikimedia-fullScreen\"></span>\\n\\n<span>Toggle limited content width</span>\\n\\n</button>\\n\\n</div>\\n<script>(RLQ=window.RLQ||[]).push(function(){mw.config.set({\"wgHostname\":\"mw2331\",\"wgBackendResponseTime\":2219,\"wgPageParseReport\":{\"limitreport\":{\"cputime\":\"1.774\",\"walltime\":\"2.084\",\"ppvisitednodes\":{\"value\":18236,\"limit\":1000000},\"postexpandincludesize\":{\"value\":489329,\"limit\":2097152},\"templateargumentsize\":{\"value\":10957,\"limit\":2097152},\"expansiondepth\":{\"value\":16,\"limit\":100},\"expensivefunctioncount\":{\"value\":53,\"limit\":500},\"unstrip-depth\":{\"value\":1,\"limit\":20},\"unstrip-size\":{\"value\":387198,\"limit\":5000000},\"entityaccesscount\":{\"value\":1,\"limit\":400},\"timingprofile\":[\"100.00% 1757.529      1 -total\",\" 60.52% 1063.729      1 Template:Reflist\",\" 46.38%  815.114     50 Template:Cite_journal\",\"  7.21%  126.640     12 Template:Navbox\",\"  5.24%   92.095     10 Template:Cite_web\",\"  4.48%   78.777      1 Template:Bioinformatics\",\"  3.94%   69.184     15 Template:ISBN\",\"  2.71%   47.698      1 Template:Short_description\",\"  2.59%   45.584      7 Template:Fix\",\"  2.42%   42.580      1 Template:Authority_control\"]},\"scribunto\":{\"limitreport-timeusage\":{\"value\":\"1.221\",\"limit\":\"10.000\"},\"limitreport-memusage\":{\"value\":9244762,\"limit\":52428800},\"limitreport-logs\":\"table#1 {\\\\n    [\\\\\"size\\\\\"] = \\\\\"tiny\\\\\",\\\\n}\\\\n\",\"limitreport-profile\":[[\"?\",\"260\",\"22.0\"],[\"\\\\u003Cmw.lua:694\\\\u003E\",\"100\",\"8.5\"],[\"recursiveClone \\\\u003CmwInit.lua:41\\\\u003E\",\"80\",\"6.8\"],[\"pairsfunc \\\\u003Cmw.lua:676\\\\u003E\",\"80\",\"6.8\"],[\"type\",\"60\",\"5.1\"],[\"MediaWiki\\\\\\\\Extension\\\\\\\\Scribunto\\\\\\\\Engines\\\\\\\\LuaSandbox\\\\\\\\LuaSandboxCallback::callParserFunction\",\"60\",\"5.1\"],[\"test \\\\u003CModule:Citation/CS1:1473\\\\u003E\",\"40\",\"3.4\"],[\"(for generator) \\\\u003Cmw.lua:684\\\\u003E\",\"40\",\"3.4\"],[\"match\",\"40\",\"3.4\"],[\"MediaWiki\\\\\\\\Extension\\\\\\\\Scribunto\\\\\\\\Engines\\\\\\\\LuaSandbox\\\\\\\\LuaSandboxCallback::gsub\",\"40\",\"3.4\"],[\"[others]\",\"380\",\"32.2\"]]},\"cachereport\":{\"origin\":\"mw2331\",\"timestamp\":\"20230707215806\",\"ttl\":1814400,\"transientcontent\":false}}});});</script>\\n<script type=\"application/ld+json\">{\"@context\":\"https:\\\\/\\\\/schema.org\",\"@type\":\"Article\",\"name\":\"Bioinformatics\",\"url\":\"https:\\\\/\\\\/en.wikipedia.org\\\\/wiki\\\\/Bioinformatics\",\"sameAs\":\"http:\\\\/\\\\/www.wikidata.org\\\\/entity\\\\/Q128570\",\"mainEntity\":\"http:\\\\/\\\\/www.wikidata.org\\\\/entity\\\\/Q128570\",\"author\":{\"@type\":\"Organization\",\"name\":\"Contributors to Wikimedia projects\"},\"publisher\":{\"@type\":\"Organization\",\"name\":\"Wikimedia Foundation, Inc.\",\"logo\":{\"@type\":\"ImageObject\",\"url\":\"https:\\\\/\\\\/www.wikimedia.org\\\\/static\\\\/images\\\\/wmf-hor-googpub.png\"}},\"datePublished\":\"2001-09-20T14:58:39Z\",\"dateModified\":\"2023-07-07T01:13:44Z\",\"image\":\"https:\\\\/\\\\/upload.wikimedia.org\\\\/wikipedia\\\\/commons\\\\/4\\\\/43\\\\/Genome_viewer_screenshot_small.png\",\"headline\":\"computational analysis of large, complex sets of biological data\"}</script><script type=\"application/ld+json\">{\"@context\":\"https:\\\\/\\\\/schema.org\",\"@type\":\"Article\",\"name\":\"Bioinformatics\",\"url\":\"https:\\\\/\\\\/en.wikipedia.org\\\\/wiki\\\\/Bioinformatics\",\"sameAs\":\"http:\\\\/\\\\/www.wikidata.org\\\\/entity\\\\/Q128570\",\"mainEntity\":\"http:\\\\/\\\\/www.wikidata.org\\\\/entity\\\\/Q128570\",\"author\":{\"@type\":\"Organization\",\"name\":\"Contributors to Wikimedia projects\"},\"publisher\":{\"@type\":\"Organization\",\"name\":\"Wikimedia Foundation, Inc.\",\"logo\":{\"@type\":\"ImageObject\",\"url\":\"https:\\\\/\\\\/www.wikimedia.org\\\\/static\\\\/images\\\\/wmf-hor-googpub.png\"}},\"datePublished\":\"2001-09-20T14:58:39Z\",\"dateModified\":\"2023-07-07T01:13:44Z\",\"image\":\"https:\\\\/\\\\/upload.wikimedia.org\\\\/wikipedia\\\\/commons\\\\/4\\\\/43\\\\/Genome_viewer_screenshot_small.png\",\"headline\":\"computational analysis of large, complex sets of biological data\"}</script>\\n</body>\\n</html>'\n"
     ]
    }
   ],
   "source": [
    "print(article)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "import parser"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "parsed_article = bs.BeautifulSoup(article,'html')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "paragraphs = parsed_article.find_all('p')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<p class=\"mw-empty-elt\">\n",
       " </p>,\n",
       " <p><b>Bioinformatics</b> (<span class=\"rt-commentedText nowrap\"><span class=\"IPA nopopups noexcerpt\" lang=\"en-fonipa\"><a href=\"/wiki/Help:IPA/English\" title=\"Help:IPA/English\">/<span style=\"border-bottom:1px dotted\"><span title=\"//: secondary stress follows\"></span><span title=\"'b' in 'buy'\">b</span><span title=\"/a/: 'i' in 'tide'\">a</span><span title=\"/./: syllable break\">.</span><span title=\"/o/: 'o' in 'code'\">o</span><span title=\"//: secondary stress follows\"></span><span title=\"//: 'i' in 'kit'\"></span><span title=\"'n' in 'nigh'\">n</span><span title=\"'f' in 'find'\">f</span><span title=\"/r/: 'er' in 'letter'\">r</span><span title=\"//: primary stress follows\"></span><span title=\"'m' in 'my'\">m</span><span title=\"//: 'a' in 'bad'\"></span><span title=\"'t' in 'tie'\">t</span><span title=\"//: 'i' in 'kit'\"></span><span title=\"'k' in 'kind'\">k</span><span title=\"'s' in 'sigh'\">s</span></span>/</a></span> <span class=\"nowrap\" style=\"font-size:85%\">(<span class=\"unicode haudio\"><span class=\"fn\"><span style=\"white-space:nowrap;margin-right:.25em;\"><a href=\"/wiki/File:En-us-bioinformatics.ogg\" title=\"About this sound\"><img alt=\"\" data-file-height=\"20\" data-file-width=\"20\" decoding=\"async\" height=\"11\" src=\"//upload.wikimedia.org/wikipedia/commons/thumb/8/8a/Loudspeaker.svg/11px-Loudspeaker.svg.png\" srcset=\"//upload.wikimedia.org/wikipedia/commons/thumb/8/8a/Loudspeaker.svg/17px-Loudspeaker.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/8/8a/Loudspeaker.svg/22px-Loudspeaker.svg.png 2x\" width=\"11\"/></a></span><a class=\"internal\" href=\"//upload.wikimedia.org/wikipedia/commons/7/76/En-us-bioinformatics.ogg\" title=\"En-us-bioinformatics.ogg\">listen</a></span></span>)</span></span>) is an <a class=\"mw-redirect\" href=\"/wiki/Interdisciplinary\" title=\"Interdisciplinary\">interdisciplinary</a> field of <a href=\"/wiki/Science\" title=\"Science\">science</a> that develops methods and <a class=\"mw-redirect\" href=\"/wiki/Software_tool\" title=\"Software tool\">software tools</a> for understanding <a href=\"/wiki/Biology\" title=\"Biology\">biological</a> data, especially when the data sets are large and complex. Bioinformatics uses <a href=\"/wiki/Biology\" title=\"Biology\">biology</a>, <a href=\"/wiki/Chemistry\" title=\"Chemistry\">chemistry</a>, <a href=\"/wiki/Physics\" title=\"Physics\">physics</a>, <a href=\"/wiki/Computer_science\" title=\"Computer science\">computer science</a>, <a href=\"/wiki/Computer_programming\" title=\"Computer programming\">computer programming</a>, <a href=\"/wiki/Information_engineering_(field)\" title=\"Information engineering (field)\">information engineering</a>, <a href=\"/wiki/Mathematics\" title=\"Mathematics\">mathematics</a> and <a href=\"/wiki/Statistics\" title=\"Statistics\">statistics</a> to analyze and interpret <a href=\"/wiki/Biological_data\" title=\"Biological data\">biological data</a>. The subsequent process of analyzing and interpreting data is referred to as <a href=\"/wiki/Computational_biology\" title=\"Computational biology\">computational biology</a>. \n",
       " </p>,\n",
       " <p>Computational, statistical, and computer programming techniques have been used  for <a href=\"/wiki/In_silico\" title=\"In silico\">computer simulation</a> analyses of biological queries. They include reused specific analysis \"pipelines\", particularly in the field of <a href=\"/wiki/Genomics\" title=\"Genomics\">genomics</a>, such as by the identification of <a href=\"/wiki/Gene\" title=\"Gene\">genes</a> and single <a href=\"/wiki/Nucleotide\" title=\"Nucleotide\">nucleotide</a> polymorphisms (<a href=\"/wiki/Single-nucleotide_polymorphism\" title=\"Single-nucleotide polymorphism\">SNPs</a>). These pipelines are used to better understand the genetic basis of disease, unique adaptations, desirable properties (esp. in agricultural species), or differences between populations. Bioinformatics also includes <a href=\"/wiki/Proteomics\" title=\"Proteomics\">proteomics</a>, which tries to understand the organizational principles within <a href=\"/wiki/Nucleic_acid\" title=\"Nucleic acid\">nucleic acid</a> and <a href=\"/wiki/Protein\" title=\"Protein\">protein</a> sequences.<sup class=\"reference\" id=\"cite_ref-1\"><a href=\"#cite_note-1\">[1]</a></sup>\n",
       " </p>,\n",
       " <p>Image and <a href=\"/wiki/Signal_processing\" title=\"Signal processing\">signal processing</a> allow extraction of useful results from large amounts of raw data. In the field of genetics, it aids in sequencing and annotating genomes and their observed <a href=\"/wiki/Mutation\" title=\"Mutation\">mutations</a>. Bioinformatics includes <a href=\"/wiki/Text_mining\" title=\"Text mining\">text mining</a> of biological literature and the development of biological and gene <a href=\"/wiki/Ontology_(information_science)\" title=\"Ontology (information science)\">ontologies</a> to organize and query biological data. It also plays a role in the analysis of gene and protein expression and regulation. Bioinformatics tools aid in comparing, analyzing and interpreting genetic and genomic data and more generally in the understanding of evolutionary aspects of molecular biology. At a more integrative level, it helps analyze and catalogue the biological pathways and networks that are an important part of <a href=\"/wiki/Systems_biology\" title=\"Systems biology\">systems biology</a>. In <a href=\"/wiki/Structural_biology\" title=\"Structural biology\">structural biology</a>, it aids in the simulation and modeling of DNA,<sup class=\"reference\" id=\"cite_ref-:0_2-0\"><a href=\"#cite_note-:0-2\">[2]</a></sup> RNA,<sup class=\"reference\" id=\"cite_ref-:0_2-1\"><a href=\"#cite_note-:0-2\">[2]</a></sup><sup class=\"reference\" id=\"cite_ref-3\"><a href=\"#cite_note-3\">[3]</a></sup> proteins<sup class=\"reference\" id=\"cite_ref-4\"><a href=\"#cite_note-4\">[4]</a></sup> as well as biomolecular interactions.<sup class=\"reference\" id=\"cite_ref-5\"><a href=\"#cite_note-5\">[5]</a></sup><sup class=\"reference\" id=\"cite_ref-6\"><a href=\"#cite_note-6\">[6]</a></sup><sup class=\"reference\" id=\"cite_ref-7\"><a href=\"#cite_note-7\">[7]</a></sup><sup class=\"reference\" id=\"cite_ref-8\"><a href=\"#cite_note-8\">[8]</a></sup>\n",
       " </p>,\n",
       " <p>The first definition of the term <i>bioinformatics</i> was coined by <a href=\"/wiki/Paulien_Hogeweg\" title=\"Paulien Hogeweg\">Paulien Hogeweg</a> and <a class=\"new\" href=\"/w/index.php?title=Ben_Hesper&amp;action=edit&amp;redlink=1\" title=\"Ben Hesper (page does not exist)\">Ben Hesper</a> in 1970, to refer to the study of information processes in biotic systems.<sup class=\"reference\" id=\"cite_ref-9\"><a href=\"#cite_note-9\">[9]</a></sup><sup class=\"reference\" id=\"cite_ref-Hogeweg2011_10-0\"><a href=\"#cite_note-Hogeweg2011-10\">[10]</a></sup><sup class=\"reference\" id=\"cite_ref-11\"><a href=\"#cite_note-11\">[11]</a></sup><sup class=\"reference\" id=\"cite_ref-12\"><a href=\"#cite_note-12\">[12]</a></sup><sup class=\"reference\" id=\"cite_ref-13\"><a href=\"#cite_note-13\">[13]</a></sup> This definition placed bioinformatics as a field parallel to <a href=\"/wiki/Biochemistry\" title=\"Biochemistry\">biochemistry</a> (the study of chemical processes in biological systems).<sup class=\"reference\" id=\"cite_ref-Hogeweg2011_10-1\"><a href=\"#cite_note-Hogeweg2011-10\">[10]</a></sup>\n",
       " </p>,\n",
       " <p>Bioinformatics and computational biology involved the analysis of biological data, particularly DNA, RNA, and protein sequences. The field of bioinformatics experienced explosive growth starting in the mid-1990s, driven largely by the <a href=\"/wiki/Human_Genome_Project\" title=\"Human Genome Project\">Human Genome Project</a> and by rapid advances in DNA sequencing technology.\n",
       " </p>,\n",
       " <p>Analyzing biological data to produce meaningful information involves writing and running software programs that use <a href=\"/wiki/Algorithm\" title=\"Algorithm\">algorithms</a> from <a href=\"/wiki/Graph_theory\" title=\"Graph theory\">graph theory</a>, <a href=\"/wiki/Artificial_intelligence\" title=\"Artificial intelligence\">artificial intelligence</a>, <a href=\"/wiki/Soft_computing\" title=\"Soft computing\">soft computing</a>, <a href=\"/wiki/Data_mining\" title=\"Data mining\">data mining</a>, <a class=\"mw-redirect\" href=\"/wiki/Image_processing\" title=\"Image processing\">image processing</a>, and <a href=\"/wiki/Computer_simulation\" title=\"Computer simulation\">computer simulation</a>. The algorithms in turn depend on theoretical foundations such as <a href=\"/wiki/Discrete_mathematics\" title=\"Discrete mathematics\">discrete mathematics</a>, <a href=\"/wiki/Control_theory\" title=\"Control theory\">control theory</a>, <a class=\"mw-redirect\" href=\"/wiki/System_theory\" title=\"System theory\">system theory</a>, <a href=\"/wiki/Information_theory\" title=\"Information theory\">information theory</a>, and <a href=\"/wiki/Statistics\" title=\"Statistics\">statistics</a>.\n",
       " </p>,\n",
       " <p>There has been a tremendous advance in speed and cost reduction since the completion of the Human Genome Project, with some labs able to <a href=\"/wiki/Sequence\" title=\"Sequence\">sequence</a> over 100,000 billion bases each year, and a full genome can be sequenced for $1,000 or less.<sup class=\"reference\" id=\"cite_ref-14\"><a href=\"#cite_note-14\">[14]</a></sup>\n",
       " </p>,\n",
       " <p>Computers became essential in molecular biology when <a class=\"mw-redirect\" href=\"/wiki/Protein_sequences\" title=\"Protein sequences\">protein sequences</a> became available after <a href=\"/wiki/Frederick_Sanger\" title=\"Frederick Sanger\">Frederick Sanger</a> determined the sequence of <a href=\"/wiki/Insulin\" title=\"Insulin\">insulin</a> in the early 1950s. Comparing multiple sequences manually turned out to be impractical. <a href=\"/wiki/Margaret_Oakley_Dayhoff\" title=\"Margaret Oakley Dayhoff\">Margaret Oakley Dayhoff</a>, a pioneer in the field,<sup class=\"reference\" id=\"cite_ref-15\"><a href=\"#cite_note-15\">[15]</a></sup> compiled one of the first protein sequence databases, initially published as books<sup class=\"reference\" id=\"cite_ref-16\"><a href=\"#cite_note-16\">[16]</a></sup> as well as methods of sequence alignment and <a href=\"/wiki/Molecular_evolution\" title=\"Molecular evolution\">molecular evolution</a>.<sup class=\"reference\" id=\"cite_ref-pmid17775169_17-0\"><a href=\"#cite_note-pmid17775169-17\">[17]</a></sup> Another early contributor to bioinformatics was <a href=\"/wiki/Elvin_A._Kabat\" title=\"Elvin A. Kabat\">Elvin A. Kabat</a>, who pioneered biological sequence analysis in 1970 with his comprehensive volumes of antibody sequences released online with Tai Te Wu between 1980 and 1991.<sup class=\"reference\" id=\"cite_ref-18\"><a href=\"#cite_note-18\">[18]</a></sup>\n",
       " </p>,\n",
       " <p>In the 1970s, new techniques for sequencing DNA were applied to bacteriophage MS2 and X174, and the extended nucleotide sequences were then parsed with informational and statistical algorithms.  These studies illustrated that well known features, such as the coding segments and the triplet code, are revealed in straightforward statistical analyses and were the proof of the concept that bioinformatics would be insightful.<sup class=\"reference\" id=\"cite_ref-19\"><a href=\"#cite_note-19\">[19]</a></sup><sup class=\"reference\" id=\"cite_ref-20\"><a href=\"#cite_note-20\">[20]</a></sup>\n",
       " </p>,\n",
       " <p>In order to study how normal cellular activities are altered in different disease states, raw biological data must be combined to form a comprehensive picture of these activities. Therefore<sup class=\"noprint Inline-Template\" style=\"white-space:nowrap;\">[<i><a href=\"/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Chronological_items\" title=\"Wikipedia:Manual of Style/Dates and numbers\"><span title=\"The time period mentioned near this tag is ambiguous. (June 2023)\">when?</span></a></i>]</sup>, the field of bioinformatics has evolved such that the most pressing task now involves the analysis and interpretation of various types of data. This also includes nucleotide and <a class=\"mw-redirect\" href=\"/wiki/Amino_acid_sequence\" title=\"Amino acid sequence\">amino acid sequences</a>, <a href=\"/wiki/Protein_domain\" title=\"Protein domain\">protein domains</a>, and <a href=\"/wiki/Protein_structure\" title=\"Protein structure\">protein structures</a>.<sup class=\"reference\" id=\"cite_ref-21\"><a href=\"#cite_note-21\">[21]</a></sup>\n",
       " </p>,\n",
       " <p>Important sub-disciplines within bioinformatics and <a href=\"/wiki/Computational_biology\" title=\"Computational biology\">computational biology</a> include:\n",
       " </p>,\n",
       " <p>The primary goal of bioinformatics is to increase the understanding of biological processes. What sets it apart from other approaches is its focus on developing and applying computationally intensive techniques to achieve this goal. Examples include: <a href=\"/wiki/Pattern_recognition\" title=\"Pattern recognition\">pattern recognition</a>, <a href=\"/wiki/Data_mining\" title=\"Data mining\">data mining</a>, <a href=\"/wiki/Machine_learning\" title=\"Machine learning\">machine learning</a> algorithms, and <a class=\"mw-redirect\" href=\"/wiki/Biological_Data_Visualization\" title=\"Biological Data Visualization\">visualization</a>. Major research efforts in the field include <a href=\"/wiki/Sequence_alignment\" title=\"Sequence alignment\">sequence alignment</a>, <a class=\"mw-redirect\" href=\"/wiki/Gene_finding\" title=\"Gene finding\">gene finding</a>, <a class=\"mw-redirect\" href=\"/wiki/Genome_assembly\" title=\"Genome assembly\">genome assembly</a>, <a href=\"/wiki/Drug_design\" title=\"Drug design\">drug design</a>, <a href=\"/wiki/Drug_discovery\" title=\"Drug discovery\">drug discovery</a>, <a class=\"mw-redirect\" href=\"/wiki/Protein_structural_alignment\" title=\"Protein structural alignment\">protein structure alignment</a>, <a href=\"/wiki/Protein_structure_prediction\" title=\"Protein structure prediction\">protein structure prediction</a>, prediction of <a href=\"/wiki/Gene_expression\" title=\"Gene expression\">gene expression</a> and <a href=\"/wiki/Protein%E2%80%93protein_interaction\" title=\"Proteinprotein interaction\">proteinprotein interactions</a>, <a class=\"mw-redirect\" href=\"/wiki/Genome-wide_association_studies\" title=\"Genome-wide association studies\">genome-wide association studies</a>, the modeling of <a href=\"/wiki/Evolution\" title=\"Evolution\">evolution</a> and <a href=\"/wiki/Cellular_model\" title=\"Cellular model\">cell division/mitosis.</a>\n",
       " </p>,\n",
       " <p>Bioinformatics entails the creation and advancement of databases, algorithms, computational and statistical techniques, and theory to solve formal and practical problems arising from the management and analysis of biological data.\n",
       " </p>,\n",
       " <p>Over the past few decades, rapid developments in genomic and other molecular research technologies and developments in <a class=\"mw-redirect\" href=\"/wiki/Information_technologies\" title=\"Information technologies\">information technologies</a> have combined to produce a tremendous amount of information related to molecular biology. Bioinformatics is the name given to these mathematical and computing approaches used to glean understanding of biological processes.\n",
       " </p>,\n",
       " <p>Common activities in bioinformatics include mapping and analyzing <a href=\"/wiki/DNA\" title=\"DNA\">DNA</a> and protein sequences, aligning DNA and protein sequences to compare them, and creating and viewing 3-D models of protein structures.\n",
       " </p>,\n",
       " <p>Since the bacteriophage <a href=\"/wiki/Phi_X_174\" title=\"Phi X 174\">Phage -X174</a> was <a href=\"/wiki/Sequencing\" title=\"Sequencing\">sequenced</a> in 1977,<sup class=\"reference\" id=\"cite_ref-pmid870828_22-0\"><a href=\"#cite_note-pmid870828-22\">[22]</a></sup> the <a class=\"mw-redirect\" href=\"/wiki/DNA_sequence\" title=\"DNA sequence\">DNA sequences</a> of thousands of organisms have been decoded and stored in databases. This sequence information is analyzed to determine genes that encode <a href=\"/wiki/Protein\" title=\"Protein\">proteins</a>, RNA genes, regulatory sequences, structural motifs, and repetitive sequences. A comparison of genes within a <a href=\"/wiki/Species\" title=\"Species\">species</a> or between different species can show similarities between protein functions, or relations between species (the use of <a class=\"mw-redirect\" href=\"/wiki/Molecular_systematics\" title=\"Molecular systematics\">molecular systematics</a> to construct <a href=\"/wiki/Phylogenetic_tree\" title=\"Phylogenetic tree\">phylogenetic trees</a>). With the growing amount of data, it long ago became impractical to analyze DNA sequences manually. <a href=\"/wiki/Computer_program\" title=\"Computer program\">Computer programs</a> such as <a class=\"mw-redirect\" href=\"/wiki/BLAST\" title=\"BLAST\">BLAST</a> are used routinely to search sequencesas of 2008, from more than 260,000 organisms, containing over 190 billion <a href=\"/wiki/Nucleotide\" title=\"Nucleotide\">nucleotides</a>.<sup class=\"reference\" id=\"cite_ref-pmid18073190_23-0\"><a href=\"#cite_note-pmid18073190-23\">[23]</a></sup>\n",
       " </p>,\n",
       " <p>Before sequences can be analyzed, they are obtained from a data storage bank, such as Genbank. <a href=\"/wiki/DNA_sequencing\" title=\"DNA sequencing\">DNA sequencing</a> is still a non-trivial problem as the raw data may be noisy or affected by weak signals. <a href=\"/wiki/Algorithm\" title=\"Algorithm\">Algorithms</a> have been developed for <a href=\"/wiki/Base_calling\" title=\"Base calling\">base calling</a> for the various experimental approaches to DNA sequencing.\n",
       " </p>,\n",
       " <p>Most DNA sequencing techniques produce short fragments of sequence that need to be assembled to obtain complete gene or genome sequences. The <a href=\"/wiki/Shotgun_sequencing\" title=\"Shotgun sequencing\">shotgun sequencing</a> technique (used by <a class=\"mw-redirect\" href=\"/wiki/The_Institute_for_Genomic_Research\" title=\"The Institute for Genomic Research\">The Institute for Genomic Research</a> (TIGR) to sequence the first bacterial genome, <i><a href=\"/wiki/Haemophilus_influenzae\" title=\"Haemophilus influenzae\">Haemophilus influenzae</a></i>)<sup class=\"reference\" id=\"cite_ref-pmid7542800_24-0\"><a href=\"#cite_note-pmid7542800-24\">[24]</a></sup> generates the sequences of many thousands of small DNA fragments (ranging from 35 to 900 nucleotides long, depending on the sequencing technology). The ends of these fragments overlap and, when aligned properly by a genome assembly program, can be used to reconstruct the complete genome. Shotgun sequencing yields sequence data quickly, but the task of assembling the fragments can be quite complicated for larger genomes. For a genome as large as the <a href=\"/wiki/Human_genome\" title=\"Human genome\">human genome</a>, it may take many days of CPU time on large-memory, multiprocessor computers to assemble the fragments, and the resulting assembly usually contains numerous gaps that must be filled in later. Shotgun sequencing is the method of choice for virtually all genomes sequenced (rather than chain-termination or chemical degradation methods), and genome assembly algorithms are a critical area of bioinformatics research.\n",
       " </p>,\n",
       " <p>In <a href=\"/wiki/Genomics\" title=\"Genomics\">genomics</a>, <a href=\"/wiki/Genome_project#Genome_annotation\" title=\"Genome project\">annotation</a> refers to the process of marking the stop and start regions of genes and other biological features in a sequenced DNA sequence. Many genomes are too large to be annotated by hand. As the rate of <a href=\"/wiki/DNA_sequencing\" title=\"DNA sequencing\">sequencing</a> exceeds the rate of genome annotation, genome annotation has become the new bottleneck in bioinformatics<sup class=\"noprint Inline-Template\" style=\"white-space:nowrap;\">[<i><a href=\"/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Chronological_items\" title=\"Wikipedia:Manual of Style/Dates and numbers\"><span title=\"The time period mentioned near this tag is ambiguous. (June 2023)\">when?</span></a></i>]</sup>. \n",
       " </p>,\n",
       " <p>Genome annotation can be classified into three levels: the nucleotide, protein, and process levels.\n",
       " </p>,\n",
       " <p>Gene finding is a chief aspect of nucleotide-level annotation. For complex genomes, a combination of <a href=\"/wiki/Ab_initio\" title=\"Ab initio\">ab initio</a> gene prediction and sequence comparison with expressed sequence databases and other organisms can be successful. Nucleotide-level annotation also allows the integration of genome sequence with other genetic and physical maps of the genome.\n",
       " </p>,\n",
       " <p>The principal aim of protein-level annotation is to assign function to the <a href=\"/wiki/Protein\" title=\"Protein\">protein</a> products of the genome. Databases of protein sequences and functional domains and motifs are used for this type of annotation. About half of the predicted proteins in a new genome sequence tend to have no obvious function.\n",
       " </p>,\n",
       " <p>Understanding the function of genes and their products in the context of cellular and organismal physiology is the goal of process-level annotation. An obstacle of process-level annotation has been the inconsistency of terms used by different model systems. The Gene Ontology Consortium is helping to solve this problem.<sup class=\"reference\" id=\"cite_ref-25\"><a href=\"#cite_note-25\">[25]</a></sup>\n",
       " </p>,\n",
       " <p>The first description of a comprehensive annotation system was published in 1995<sup class=\"reference\" id=\"cite_ref-pmid7542800_24-1\"><a href=\"#cite_note-pmid7542800-24\">[24]</a></sup> by the <a class=\"mw-redirect\" href=\"/wiki/The_Institute_for_Genomic_Research\" title=\"The Institute for Genomic Research\">The Institute for Genomic Research</a>, which performed the first complete sequencing and analysis of the genome of a <a class=\"mw-disambig\" href=\"/wiki/Free-living\" title=\"Free-living\">free-living</a> organism, the bacterium <i><a href=\"/wiki/Haemophilus_influenzae\" title=\"Haemophilus influenzae\">Haemophilus influenzae</a></i>.<sup class=\"reference\" id=\"cite_ref-pmid7542800_24-2\"><a href=\"#cite_note-pmid7542800-24\">[24]</a></sup> The system identifies the genes encoding all proteins, transfer RNAs, ribosomal RNAs, in order to make initial functional assignments. The <a href=\"/wiki/GeneMark\" title=\"GeneMark\">GeneMark</a> program trained to find protein-coding genes in <i><a href=\"/wiki/Haemophilus_influenzae\" title=\"Haemophilus influenzae\">Haemophilus influenzae</a></i> is constantly changing and improving.\n",
       " </p>,\n",
       " <p>Following the goals that the Human Genome Project left to achieve after its closure in 2003, the <a href=\"/wiki/ENCODE\" title=\"ENCODE\">ENCODE</a> project was developed by the <a href=\"/wiki/National_Human_Genome_Research_Institute\" title=\"National Human Genome Research Institute\">National Human Genome Research Institute</a>. This project is a collaborative data collection of the functional elements of the human genome that uses next-generation DNA-sequencing technologies and genomic tiling arrays, technologies able to automatically generate large amounts of data at a dramatically reduced per-base cost but with the same accuracy (base call error) and fidelity (assembly error).\n",
       " </p>,\n",
       " <p>While genome annotation is primarily based on sequence similarity (and thus <a href=\"/wiki/Homology_(biology)\" title=\"Homology (biology)\">homology</a>), other properties of sequences can be used to predict the function of genes. In fact, most <i>gene</i> function prediction methods focus on <i>protein</i> sequences as they are more informative and more feature-rich. For instance, the distribution of hydrophobic <a href=\"/wiki/Amino_acid\" title=\"Amino acid\">amino acids</a> predicts <a href=\"/wiki/Transmembrane_domain\" title=\"Transmembrane domain\">transmembrane segments</a> in proteins. However, protein function prediction can also use external information such as gene (or protein) <a href=\"/wiki/Gene_expression\" title=\"Gene expression\">expression</a> data, <a href=\"/wiki/Protein_structure\" title=\"Protein structure\">protein structure</a>, or <a href=\"/wiki/Protein%E2%80%93protein_interaction\" title=\"Proteinprotein interaction\">protein-protein interactions</a>.<sup class=\"reference\" id=\"cite_ref-26\"><a href=\"#cite_note-26\">[26]</a></sup>\n",
       " </p>,\n",
       " <p><a href=\"/wiki/Evolutionary_biology\" title=\"Evolutionary biology\">Evolutionary biology</a> is the study of the origin and descent of <a href=\"/wiki/Species\" title=\"Species\">species</a>, as well as their change over time. <a class=\"mw-redirect\" href=\"/wiki/Informatics_(academic_field)\" title=\"Informatics (academic field)\">Informatics</a> has assisted evolutionary biologists by enabling researchers to:\n",
       " </p>,\n",
       " <p>Future work endeavours to reconstruct the now more complex <a class=\"mw-redirect\" href=\"/wiki/Evolutionary_tree\" title=\"Evolutionary tree\">tree of life</a>.<sup class=\"noprint Inline-Template\" style=\"margin-left:0.1em; white-space:nowrap;\">[<i><a href=\"/wiki/Wikipedia:Manual_of_Style/Words_to_watch#Unsupported_attributions\" title=\"Wikipedia:Manual of Style/Words to watch\"><span title=\"The material near this tag may use weasel words or too-vague attribution. (June 2020)\">according to whom?</span></a></i>]</sup>\n",
       " </p>,\n",
       " <p>The core of comparative genome analysis is the establishment of the correspondence between <a class=\"mw-redirect\" href=\"/wiki/Genes\" title=\"Genes\">genes</a> (<a href=\"/wiki/Homology_(biology)#Orthology\" title=\"Homology (biology)\">orthology</a> analysis) or other genomic features in different organisms. Intergenomic maps are made to trace the evolutionary processes responsible for the divergence of two genomes. A multitude of evolutionary events acting at various organizational levels shape genome evolution. At the lowest level, point mutations affect individual nucleotides. At a higher level, large chromosomal segments undergo duplication, lateral transfer, inversion, transposition, deletion and insertion.<sup class=\"reference\" id=\"cite_ref-28\"><a href=\"#cite_note-28\">[28]</a></sup> Entire genomes are involved in processes of hybridization, polyploidization and <a class=\"mw-redirect\" href=\"/wiki/Endosymbiosis\" title=\"Endosymbiosis\">endosymbiosis</a> that lead to rapid speciation. The complexity of genome evolution poses many exciting challenges to developers of mathematical models and algorithms, who have recourse to a spectrum of algorithmic, statistical and mathematical techniques, ranging from exact, <a class=\"mw-redirect\" href=\"/wiki/Heuristics\" title=\"Heuristics\">heuristics</a>, fixed parameter and <a class=\"mw-redirect\" href=\"/wiki/Approximation_algorithms\" title=\"Approximation algorithms\">approximation algorithms</a> for problems based on parsimony models to <a href=\"/wiki/Markov_chain_Monte_Carlo\" title=\"Markov chain Monte Carlo\">Markov chain Monte Carlo</a> algorithms for <a class=\"mw-redirect\" href=\"/wiki/Bayesian_analysis\" title=\"Bayesian analysis\">Bayesian analysis</a> of problems based on probabilistic models.\n",
       " </p>,\n",
       " <p>Many of these studies are based on the detection of <a href=\"/wiki/Sequence_homology\" title=\"Sequence homology\">sequence homology</a> to assign sequences to <a href=\"/wiki/Protein_family\" title=\"Protein family\">protein families</a>.<sup class=\"reference\" id=\"cite_ref-29\"><a href=\"#cite_note-29\">[29]</a></sup>\n",
       " </p>,\n",
       " <p>Pan genomics is a concept introduced in 2005 by Tettelin and Medini. Pan genome is the complete gene repertoire of a particular <a href=\"/wiki/Monophyly\" title=\"Monophyly\">monophyletic</a> taxonomic group. Although initially applied to closely related strains of a species, it can be applied to a larger context like genus, phylum, etc. It is divided in two parts: the Core genome, a set of genes common to all the genomes under study (often housekeeping genes vital for survival), and the Dispensable/Flexible genome: a set of genes not present in all but one or some genomes under study. A bioinformatics tool BPGA can be used to characterize the Pan Genome of bacterial species.<sup class=\"reference\" id=\"cite_ref-30\"><a href=\"#cite_note-30\">[30]</a></sup>\n",
       " </p>,\n",
       " <p>As of 2013, the existence of efficient high-throughput next-generation sequencing technology allows for the identification of cause many different human disorders. Simple <a href=\"/wiki/Mendelian_inheritance\" title=\"Mendelian inheritance\">Mendelian inheritance</a> has been observed for over 3,000 disorders that have been identified at the <a href=\"/wiki/Online_Mendelian_Inheritance_in_Man\" title=\"Online Mendelian Inheritance in Man\">Online Mendelian Inheritance in Man</a> database, but complex diseases are more difficult. Association studies have found many individual genetic regions that individually are weakly associated with complex diseases (such as <a href=\"/wiki/Infertility\" title=\"Infertility\">infertility</a>,<sup class=\"reference\" id=\"cite_ref-Demerec1945_31-0\"><a href=\"#cite_note-Demerec1945-31\">[31]</a></sup> <a href=\"/wiki/Breast_cancer\" title=\"Breast cancer\">breast cancer</a><sup class=\"reference\" id=\"cite_ref-Vron2013_32-0\"><a href=\"#cite_note-Vron2013-32\">[32]</a></sup> and <a href=\"/wiki/Alzheimer%27s_disease\" title=\"Alzheimer's disease\">Alzheimer's disease</a><sup class=\"reference\" id=\"cite_ref-Tosto2013_33-0\"><a href=\"#cite_note-Tosto2013-33\">[33]</a></sup>), rather than a single cause.<sup class=\"reference\" id=\"cite_ref-Londin2013_34-0\"><a href=\"#cite_note-Londin2013-34\">[34]</a></sup><sup class=\"reference\" id=\"cite_ref-35\"><a href=\"#cite_note-35\">[35]</a></sup> There are currently many challenges to using genes for diagnosis and treatment, such as how we don't know which genes are important, or how stable the choices an algorithm provides. <sup class=\"reference\" id=\"cite_ref-36\"><a href=\"#cite_note-36\">[36]</a></sup>\n",
       " </p>,\n",
       " <p>Genome-wide association studies have successfully identified thousands of common genetic variants for complex diseases and traits; however, these common variants only explain a small fraction of heritability.<sup class=\"reference\" id=\"cite_ref-37\"><a href=\"#cite_note-37\">[37]</a></sup> <a class=\"mw-redirect\" href=\"/wiki/Rare_functional_variant\" title=\"Rare functional variant\">Rare variants</a> may account for some of the <a href=\"/wiki/Missing_heritability_problem\" title=\"Missing heritability problem\">missing heritability</a>.<sup class=\"reference\" id=\"cite_ref-38\"><a href=\"#cite_note-38\">[38]</a></sup> Large-scale <a href=\"/wiki/Whole_genome_sequencing\" title=\"Whole genome sequencing\">whole genome sequencing</a> studies have rapidly sequenced millions of whole genomes, and such studies have identified hundreds of millions of <a class=\"mw-redirect\" href=\"/wiki/Rare_functional_variant\" title=\"Rare functional variant\">rare variants</a>.<sup class=\"reference\" id=\"cite_ref-39\"><a href=\"#cite_note-39\">[39]</a></sup> <a href=\"/wiki/SNP_annotation\" title=\"SNP annotation\">Functional annotations</a> predict the effect or function of a genetic variant and help to prioritize rare functional variants, and incorporating these annotations can effectively boost the power of genetic association of rare variants analysis of whole genome sequencing studies.<sup class=\"reference\" id=\"cite_ref-40\"><a href=\"#cite_note-40\">[40]</a></sup> Some tools have been developed to provide all-in-one rare variant association analysis for whole-genome sequencing data, including integration of genotype data and their functional annotations, association analysis, result summary and visualization.<sup class=\"reference\" id=\"cite_ref-41\"><a href=\"#cite_note-41\">[41]</a></sup><sup class=\"reference\" id=\"cite_ref-42\"><a href=\"#cite_note-42\">[42]</a></sup> Meta-analysis of whole genome sequencing studies provides an attractive solution to the problem of collecting large sample sizes for discovering rare variants associated with complex phenotypes.<sup class=\"reference\" id=\"cite_ref-43\"><a href=\"#cite_note-43\">[43]</a></sup>\n",
       " </p>,\n",
       " <p>In <a href=\"/wiki/Cancer\" title=\"Cancer\">cancer</a>, the genomes of affected cells are rearranged in complex or unpredictable ways. In addition to <a href=\"/wiki/Single-nucleotide_polymorphism\" title=\"Single-nucleotide polymorphism\">single-nucleotide polymorphism</a> arrays identifying <a href=\"/wiki/Point_mutation\" title=\"Point mutation\">point mutations</a> that cause cancer, <a href=\"/wiki/Oligonucleotide\" title=\"Oligonucleotide\">oligonucleotide</a> microarrays can be used to identify chromosomal gains and losses (called <a href=\"/wiki/Comparative_genomic_hybridization\" title=\"Comparative genomic hybridization\">comparative genomic hybridization</a>). These detection methods generate <a class=\"mw-redirect\" href=\"/wiki/Terabyte\" title=\"Terabyte\">terabytes</a> of data per experiment. The data is often found to contain considerable variability, or <a href=\"/wiki/Noise\" title=\"Noise\">noise</a>, and thus <a href=\"/wiki/Hidden_Markov_model\" title=\"Hidden Markov model\">Hidden Markov model</a> and change-point analysis methods are being developed to infer real <a href=\"/wiki/Copy_number_variation\" title=\"Copy number variation\">copy number</a> changes.<sup class=\"noprint Inline-Template Template-Fact\" style=\"white-space:nowrap;\">[<i><a href=\"/wiki/Wikipedia:Citation_needed\" title=\"Wikipedia:Citation needed\"><span title=\"This claim needs references to reliable sources. (June 2023)\">citation needed</span></a></i>]</sup>\n",
       " </p>,\n",
       " <p>Two important principles can be used to identify cancer by mutations in the <a href=\"/wiki/Exome\" title=\"Exome\">exome</a>. First, cancer is a disease of accumulated somatic mutations in genes. Second, cancer contains driver mutations which need to be distinguished from passengers.<sup class=\"reference\" id=\"cite_ref-44\"><a href=\"#cite_note-44\">[44]</a></sup>\n",
       " </p>,\n",
       " <p>Further improvements in bioinformatics could allow for classifying types of cancer by analysis of cancer driven mutations in the genome. Furthermore, tracking of patients while the disease progresses may be possible in the future with the sequence of cancer samples. Another type of data that requires novel informatics development is the analysis of <a href=\"/wiki/Lesion\" title=\"Lesion\">lesions</a> found to be recurrent among many tumors.<sup class=\"reference\" id=\"cite_ref-45\"><a href=\"#cite_note-45\">[45]</a></sup>\n",
       " </p>,\n",
       " <p>The <a href=\"/wiki/Gene_expression\" title=\"Gene expression\">expression</a> of many genes can be determined by measuring <a href=\"/wiki/Messenger_RNA\" title=\"Messenger RNA\">mRNA</a> levels with multiple techniques including <a href=\"/wiki/DNA_microarray\" title=\"DNA microarray\">microarrays</a>, <a href=\"/wiki/Expressed_sequence_tag\" title=\"Expressed sequence tag\">expressed cDNA sequence tag</a> (EST) sequencing, <a href=\"/wiki/Serial_analysis_of_gene_expression\" title=\"Serial analysis of gene expression\">serial analysis of gene expression</a> (SAGE) tag sequencing, <a href=\"/wiki/Massively_parallel_signature_sequencing\" title=\"Massively parallel signature sequencing\">massively parallel signature sequencing</a> (MPSS), <a href=\"/wiki/RNA-Seq\" title=\"RNA-Seq\">RNA-Seq</a>, also known as \"Whole Transcriptome Shotgun Sequencing\" (WTSS), or various applications of multiplexed in-situ hybridization. All of these techniques are extremely noise-prone and/or subject to bias in the biological measurement, and a major research area in computational biology involves developing statistical tools to separate <a class=\"mw-redirect\" href=\"/wiki/Signal_(information_theory)\" title=\"Signal (information theory)\">signal</a> from <a href=\"/wiki/Noise\" title=\"Noise\">noise</a> in high-throughput gene expression studies.<sup class=\"reference\" id=\"cite_ref-46\"><a href=\"#cite_note-46\">[46]</a></sup> Such studies are often used to determine the genes implicated in a disorder: one might compare microarray data from cancerous <a class=\"mw-redirect\" href=\"/wiki/Epithelial\" title=\"Epithelial\">epithelial</a> cells to data from non-cancerous cells to determine the transcripts that are up-regulated and down-regulated in a particular population of cancer cells.\n",
       " </p>,\n",
       " <p><a href=\"/wiki/Protein_microarray\" title=\"Protein microarray\">Protein microarrays</a> and high throughput (HT) <a href=\"/wiki/Mass_spectrometry\" title=\"Mass spectrometry\">mass spectrometry</a> (MS) can provide a snapshot of the proteins present in a biological sample. The former approach faces similar problems as with microarrays targeted at mRNA, the latter involves the problem of matching large amounts of mass data against predicted masses from protein sequence databases, and the complicated statistical analysis of samples when multiple incomplete peptides from each protein are detected. Cellular protein localization in a tissue context can be achieved through affinity <a href=\"/wiki/Proteomics\" title=\"Proteomics\">proteomics</a> displayed as spatial data based on <a href=\"/wiki/Immunohistochemistry\" title=\"Immunohistochemistry\">immunohistochemistry</a> and <a href=\"/wiki/Tissue_microarray\" title=\"Tissue microarray\">tissue microarrays</a>.<sup class=\"reference\" id=\"cite_ref-47\"><a href=\"#cite_note-47\">[47]</a></sup>\n",
       " </p>,\n",
       " <p><a href=\"/wiki/Regulation_of_gene_expression\" title=\"Regulation of gene expression\">Gene regulation</a> is a complex process where a signal, such as an extracellular signal such as a <a href=\"/wiki/Hormone\" title=\"Hormone\">hormone</a>, eventually leads to an increase or decrease in the activity of one or more <a href=\"/wiki/Protein\" title=\"Protein\">proteins</a>. Bioinformatics techniques have been applied to explore various steps in this process.\n",
       " </p>,\n",
       " <p>For example, gene expression can be regulated by nearby elements in the genome. Promoter analysis involves the identification and study of <a href=\"/wiki/Sequence_motif\" title=\"Sequence motif\">sequence motifs</a> in the DNA surrounding the protein-coding region of a gene. These motifs influence the extent to which that region is transcribed into mRNA. <a href=\"/wiki/Enhancer_(genetics)\" title=\"Enhancer (genetics)\">Enhancer</a> elements far away from the promoter can also regulate gene expression, through three-dimensional looping interactions. These interactions can be determined by bioinformatic analysis of <a href=\"/wiki/Chromosome_conformation_capture\" title=\"Chromosome conformation capture\">chromosome conformation capture</a> experiments.\n",
       " </p>,\n",
       " <p>Expression data can be used to infer gene regulation: one might compare <a href=\"/wiki/Microarray\" title=\"Microarray\">microarray</a> data from a wide variety of states of an organism to form hypotheses about the genes involved in each state. In a single-cell organism, one might compare stages of the <a href=\"/wiki/Cell_cycle\" title=\"Cell cycle\">cell cycle</a>, along with various stress conditions (heat shock, starvation, etc.). <a href=\"/wiki/Cluster_analysis\" title=\"Cluster analysis\">Clustering algorithms</a> can be then applied to expression data to determine which genes are co-expressed. For example, the upstream regions (promoters) of co-expressed genes can be searched for over-represented <a class=\"mw-redirect\" href=\"/wiki/Regulatory_elements\" title=\"Regulatory elements\">regulatory elements</a>. Examples of clustering algorithms applied in gene clustering are <a href=\"/wiki/K-means_clustering\" title=\"K-means clustering\">k-means clustering</a>, <a href=\"/wiki/Self-organizing_map\" title=\"Self-organizing map\">self-organizing maps</a> (SOMs), <a href=\"/wiki/Hierarchical_clustering\" title=\"Hierarchical clustering\">hierarchical clustering</a>, and <a href=\"/wiki/Consensus_clustering\" title=\"Consensus clustering\">consensus clustering</a> methods.\n",
       " </p>,\n",
       " <p>Several approaches have been developed to analyze the location of organelles, genes, proteins, and other components within cells. A <a class=\"mw-redirect\" href=\"/wiki/Gene_ontology\" title=\"Gene ontology\">gene ontology</a> category, <i>cellular component</i>, has been devised to capture subcellular localization in many <a href=\"/wiki/Biological_database\" title=\"Biological database\">biological databases</a>.\n",
       " </p>,\n",
       " <p>Microscopic pictures allow for the location of <a href=\"/wiki/Organelle\" title=\"Organelle\">organelles</a> as well as molecules, which may be the source of abnormalities in diseases.\n",
       " </p>,\n",
       " <p>Finding the location of proteins allows us to predict what they do. This is called <a href=\"/wiki/Protein_function_prediction\" title=\"Protein function prediction\">protein function prediction</a>. For instance, if a protein is found in the <a href=\"/wiki/Cell_nucleus\" title=\"Cell nucleus\">nucleus</a> it may be involved in <a href=\"/wiki/Regulation_of_gene_expression\" title=\"Regulation of gene expression\">gene regulation</a> or <a href=\"/wiki/RNA_splicing\" title=\"RNA splicing\">splicing</a>. By contrast, if a protein is found in <a href=\"/wiki/Mitochondrion\" title=\"Mitochondrion\">mitochondria</a>, it may be involved in <a href=\"/wiki/Cellular_respiration\" title=\"Cellular respiration\">respiration</a> or other <a href=\"/wiki/Metabolism\" title=\"Metabolism\">metabolic processes</a>. There are well developed <a href=\"/wiki/Protein_subcellular_localization_prediction\" title=\"Protein subcellular localization prediction\">protein subcellular localization prediction</a> resources available, including protein subcellular location databases, and prediction tools.<sup class=\"reference\" id=\"cite_ref-48\"><a href=\"#cite_note-48\">[48]</a></sup><sup class=\"reference\" id=\"cite_ref-49\"><a href=\"#cite_note-49\">[49]</a></sup>\n",
       " </p>,\n",
       " <p>Data from high-throughput <a href=\"/wiki/Chromosome_conformation_capture\" title=\"Chromosome conformation capture\">chromosome conformation capture</a> experiments, such as <a href=\"/wiki/Hi-C_(genomic_analysis_technique)\" title=\"Hi-C (genomic analysis technique)\">Hi-C (experiment)</a> and <a href=\"/wiki/ChIA-PET\" title=\"ChIA-PET\">ChIA-PET</a>, can provide information on the three-dimensional structure and <a href=\"/wiki/Nuclear_organization\" title=\"Nuclear organization\">nuclear organization</a> of <a href=\"/wiki/Chromatin\" title=\"Chromatin\">chromatin</a>. Bioinformatic challenges in this field include partitioning the genome into domains, such as <a class=\"mw-redirect\" href=\"/wiki/Topologically_Associating_Domain\" title=\"Topologically Associating Domain\">Topologically Associating Domains</a> (TADs), that are organised together in three-dimensional space.<sup class=\"reference\" id=\"cite_ref-50\"><a href=\"#cite_note-50\">[50]</a></sup>\n",
       " </p>,\n",
       " <p>Finding the structure of proteins important application of bioinformatics. The Critical Assessment of Protein Structure Prediction (CASP) is an open competition where worldwide research groups submit protein models for evaluating unknown protein models.<sup class=\"reference\" id=\"cite_ref-51\"><a href=\"#cite_note-51\">[51]</a></sup><sup class=\"reference\" id=\"cite_ref-52\"><a href=\"#cite_note-52\">[52]</a></sup>\n",
       " </p>,\n",
       " <p>The linear <a href=\"/wiki/Amino_acid\" title=\"Amino acid\">amino acid</a> sequence of a protein is called the <a class=\"mw-redirect\" href=\"/wiki/Primary_structure\" title=\"Primary structure\">primary structure</a>, can be easily determined from the sequence of <a class=\"mw-redirect\" href=\"/wiki/Codons\" title=\"Codons\">codons</a> on the DNA gene that codes for it. In most proteins, the primary structure uniquely determines the 3-dimensional structure of a protein in its native environment. An exception is the <a href=\"/wiki/Prion\" title=\"Prion\">misfolded protein</a> involved in <a href=\"/wiki/Bovine_spongiform_encephalopathy\" title=\"Bovine spongiform encephalopathy\">bovine spongiform encephalopathy</a>. This structure is linked to the function of the protein. Additional structural information includes the <i><a class=\"mw-redirect\" href=\"/wiki/Secondary_structure\" title=\"Secondary structure\">secondary</a></i>, <i><a class=\"mw-redirect\" href=\"/wiki/Tertiary_structure\" title=\"Tertiary structure\">tertiary</a></i> and <i><a class=\"mw-redirect\" href=\"/wiki/Quaternary_structure\" title=\"Quaternary structure\">quaternary</a></i> structure. A viable general solution to the prediction of the function of a protein remains an open problem. Most efforts have so far been directed towards heuristics that work most of the time.<sup class=\"noprint Inline-Template Template-Fact\" style=\"white-space:nowrap;\">[<i><a href=\"/wiki/Wikipedia:Citation_needed\" title=\"Wikipedia:Citation needed\"><span title=\"This claim needs references to reliable sources. (July 2015)\">citation needed</span></a></i>]</sup>\n",
       " </p>,\n",
       " <p>In the genomic branch of bioinformatics, homology is used to predict the function of a gene: if the sequence of gene <i>A</i>, whose function is known, is homologous to the sequence of gene <i>B,</i> whose function is unknown, one could infer that B may share A's function. In structural bioinformatics, homology is used to determine which parts of a protein are important in structure formation and interaction with other proteins. <a href=\"/wiki/Homology_modeling\" title=\"Homology modeling\">Homology modeling</a> is used to predict the structure of an unknown protein from existing homologous proteins. \n",
       " </p>,\n",
       " <p>One example of this is hemoglobin in humans and the hemoglobin in legumes (<a href=\"/wiki/Leghemoglobin\" title=\"Leghemoglobin\">leghemoglobin</a>), which are distant relatives from the same <a href=\"/wiki/Protein_superfamily\" title=\"Protein superfamily\">protein superfamily</a>. Both serve the same purpose of transporting oxygen in the organism. Although both of these proteins have completely different amino acid sequences, their protein structures are virtually identical, which reflects their near identical purposes and shared ancestor.<sup class=\"reference\" id=\"cite_ref-53\"><a href=\"#cite_note-53\">[53]</a></sup>\n",
       " </p>,\n",
       " <p>Other techniques for predicting protein structure include protein threading and <i>de novo</i> (from scratch) physics-based modeling.\n",
       " </p>,\n",
       " <p>Another aspect of structural bioinformatics include the use of protein structures for <a href=\"/wiki/Virtual_screening\" title=\"Virtual screening\">Virtual Screening</a> models such as <a class=\"mw-redirect\" href=\"/wiki/QSAR\" title=\"QSAR\">Quantitative Structure-Activity Relationship</a> models and proteochemometric models (PCM). Furthermore, a protein's crystal structure can be used in simulation of for example ligand-binding studies and <i>in silico</i> mutagenesis studies.\n",
       " </p>,\n",
       " <p>A 2021 <a class=\"mw-redirect\" href=\"/wiki/Deep-learning\" title=\"Deep-learning\">deep-learning</a> algorithms-based software called <a href=\"/wiki/AlphaFold\" title=\"AlphaFold\">AlphaFold</a>, developed by Google's <a class=\"mw-redirect\" href=\"/wiki/DeepMind\" title=\"DeepMind\">DeepMind</a>, greatly outperforms all other prediction software methods<sup class=\"reference\" id=\"cite_ref-54\"><a href=\"#cite_note-54\">[54]</a></sup><sup class=\"noprint Inline-Template\" style=\"white-space:nowrap;\">[<i><a href=\"/wiki/Wikipedia:Please_clarify\" title=\"Wikipedia:Please clarify\"><span title=\"Please clarify the preceding statement or statements with a good explanation from a reliable source. (June 2023)\">how?</span></a></i>]</sup>, and has released predicted structures for hundreds of millions of proteins in the AlphaFold protein structure database.<sup class=\"reference\" id=\"cite_ref-55\"><a href=\"#cite_note-55\">[55]</a></sup>\n",
       " </p>,\n",
       " <p><i>Network analysis</i> seeks to understand the relationships within <a href=\"/wiki/Biological_network\" title=\"Biological network\">biological networks</a> such as <a href=\"/wiki/Metabolic_network\" title=\"Metabolic network\">metabolic</a> or <a href=\"/wiki/Interactome\" title=\"Interactome\">proteinprotein interaction networks</a>. Although biological networks can be constructed from a single type of molecule or entity (such as genes), network biology often attempts to integrate many different data types, such as proteins, small molecules, gene expression data, and others, which are all connected physically, functionally, or both.\n",
       " </p>,\n",
       " <p><i>Systems biology</i> involves the use of <a href=\"/wiki/Computer_simulation\" title=\"Computer simulation\">computer simulations</a> of <a href=\"/wiki/Cell_(biology)\" title=\"Cell (biology)\">cellular</a> subsystems (such as the <a href=\"/wiki/Metabolic_network\" title=\"Metabolic network\">networks of metabolites</a> and <a href=\"/wiki/Enzyme\" title=\"Enzyme\">enzymes</a> that comprise <a href=\"/wiki/Metabolism\" title=\"Metabolism\">metabolism</a>, <a href=\"/wiki/Signal_transduction\" title=\"Signal transduction\">signal transduction</a> pathways and <a href=\"/wiki/Gene_regulatory_network\" title=\"Gene regulatory network\">gene regulatory networks</a>) to both analyze and visualize the complex connections of these cellular processes. <a href=\"/wiki/Artificial_life\" title=\"Artificial life\">Artificial life</a> or virtual evolution attempts to understand evolutionary processes via the computer simulation of simple (artificial) life forms.\n",
       " </p>,\n",
       " <p>Tens of thousands of three-dimensional protein structures have been determined by <a href=\"/wiki/X-ray_crystallography\" title=\"X-ray crystallography\">X-ray crystallography</a> and <a class=\"mw-redirect\" href=\"/wiki/Protein_nuclear_magnetic_resonance_spectroscopy\" title=\"Protein nuclear magnetic resonance spectroscopy\">protein nuclear magnetic resonance spectroscopy</a> (protein NMR) and a central question in structural bioinformatics is whether it is practical to predict possible proteinprotein interactions only based on these 3D shapes, without performing <a href=\"/wiki/Protein%E2%80%93protein_interaction\" title=\"Proteinprotein interaction\">proteinprotein interaction</a> experiments. A variety of methods have been developed to tackle the <a class=\"mw-redirect\" href=\"/wiki/Protein%E2%80%93protein_docking\" title=\"Proteinprotein docking\">proteinprotein docking</a> problem, though it seems that there is still much work to be done in this field.\n",
       " </p>,\n",
       " <p>Other interactions encountered in the field include Proteinligand (including drug) and <a class=\"new\" href=\"/w/index.php?title=Protein%E2%80%93peptide&amp;action=edit&amp;redlink=1\" title=\"Proteinpeptide (page does not exist)\">proteinpeptide</a>. Molecular dynamic simulation of movement of atoms about rotatable bonds is the fundamental principle behind computational <a href=\"/wiki/Algorithm\" title=\"Algorithm\">algorithms</a>, termed docking algorithms, for studying <a href=\"/wiki/Interactome\" title=\"Interactome\">molecular interactions</a>.\n",
       " </p>,\n",
       " <p>The enormous number of published literature makes it virtually impossible for individuals to read every paper, resulting in disjointed sub-fields of research. Literature analysis aims to employ computational and statistical linguistics to mine this growing library of text resources. For example:\n",
       " </p>,\n",
       " <p>The area of research draws from <a href=\"/wiki/Statistics\" title=\"Statistics\">statistics</a> and <a href=\"/wiki/Computational_linguistics\" title=\"Computational linguistics\">computational linguistics</a>.\n",
       " </p>,\n",
       " <p>Computational technologies are used to automate the processing, quantification and analysis of large amounts of high-information-content <a href=\"/wiki/Medical_imaging\" title=\"Medical imaging\">biomedical imagery</a>. Modern <a href=\"/wiki/Image_analysis\" title=\"Image analysis\">image analysis</a> systems can improve an observer's <a class=\"mw-redirect\" href=\"/wiki/Accuracy\" title=\"Accuracy\">accuracy</a>, <a href=\"/wiki/Objectivity_(science)\" title=\"Objectivity (science)\">objectivity</a>, or speed. Image analysis is important for both <a class=\"mw-redirect\" href=\"/wiki/Diagnostics\" title=\"Diagnostics\">diagnostics</a> and research. Some examples are:\n",
       " </p>,\n",
       " <p>Computational techniques are used to analyse high-throughput, low-measurement single cell data, such as that obtained from <a href=\"/wiki/Flow_cytometry\" title=\"Flow cytometry\">flow cytometry</a>. These methods typically involve finding populations of cells that are relevant to a particular disease state or experimental condition.\n",
       " </p>,\n",
       " <p>Biodiversity informatics deals with the collection and analysis of <a href=\"/wiki/Biodiversity\" title=\"Biodiversity\">biodiversity</a> data, such as <a href=\"/wiki/Taxonomic_database\" title=\"Taxonomic database\">taxonomic databases</a>, or <a href=\"/wiki/Microbiome\" title=\"Microbiome\">microbiome</a> data. Examples of such analyses include <a href=\"/wiki/Phylogenetics\" title=\"Phylogenetics\">phylogenetics</a>, <a class=\"mw-redirect\" href=\"/wiki/Niche_modelling\" title=\"Niche modelling\">niche modelling</a>, <a href=\"/wiki/Species_richness\" title=\"Species richness\">species richness</a> mapping, <a href=\"/wiki/DNA_barcoding\" title=\"DNA barcoding\">DNA barcoding</a>, or <a href=\"/wiki/Speciesism\" title=\"Speciesism\">species</a> identification tools.\n",
       " </p>,\n",
       " <p>Biological ontologies are <a href=\"/wiki/Directed_acyclic_graph\" title=\"Directed acyclic graph\">directed acyclic graphs</a> of <a href=\"/wiki/Controlled_vocabulary\" title=\"Controlled vocabulary\">controlled vocabularies</a>. They create categories for biological concepts and descriptions so they can be easily analyzed with computers. When categorised in this way, it is possible to gain added value from holistic and integrated analysis.<sup class=\"noprint Inline-Template Template-Fact\" style=\"white-space:nowrap;\">[<i><a href=\"/wiki/Wikipedia:Citation_needed\" title=\"Wikipedia:Citation needed\"><span title=\"This claim needs references to reliable sources. (June 2023)\">citation needed</span></a></i>]</sup>\n",
       " </p>,\n",
       " <p>The <a href=\"/wiki/OBO_Foundry\" title=\"OBO Foundry\">OBO Foundry</a> was an effort to standardise certain ontologies. One of the most widespread is the <a class=\"mw-redirect\" href=\"/wiki/Gene_ontology\" title=\"Gene ontology\">Gene ontology</a> which describes gene function. There are also ontologies which describe phenotypes.\n",
       " </p>,\n",
       " <p>Databases are essential for bioinformatics research and applications. Databases exist for many different information types, including DNA and protein sequences, molecular structures, phenotypes and biodiversity. Databases can contain both empirical data (obtained directly from experiments) and predicted data (obtained from analysis of existing data). They may be specific to a particular organism, pathway or molecule of interest. Alternatively, they can incorporate data compiled from multiple other databases. Databases can have different formats, access mechanisms, and be public or private.\n",
       " </p>,\n",
       " <p>Some of the most commonly used databases are listed below:\n",
       " </p>,\n",
       " <p><a href=\"/wiki/List_of_bioinformatics_software\" title=\"List of bioinformatics software\">Software tools for bioinformatics</a> include simple command-line tools, more complex graphical programs, and standalone web-services. They are made by <a href=\"/wiki/List_of_bioinformatics_companies\" title=\"List of bioinformatics companies\">bioinformatics companies</a> or by public institutions.\n",
       " </p>,\n",
       " <p>Many <a href=\"/wiki/Free_and_open-source_software\" title=\"Free and open-source software\">free and open-source software</a> tools have existed and continued to grow since the 1980s.<sup class=\"reference\" id=\"cite_ref-obf-main_57-0\"><a href=\"#cite_note-obf-main-57\">[57]</a></sup> The combination of a continued need for new <a href=\"/wiki/Algorithm\" title=\"Algorithm\">algorithms</a> for the analysis of emerging types of biological readouts, the potential for innovative <i><a href=\"/wiki/In_silico\" title=\"In silico\">in silico</a></i> experiments, and freely available <a class=\"mw-redirect\" href=\"/wiki/Open_code\" title=\"Open code\">open code</a> bases have created opportunities for research groups to contribute to both bioinformatics regardless of <a href=\"/wiki/Funding_of_science\" title=\"Funding of science\">funding</a>. The open source tools often act as incubators of ideas, or community-supported <a href=\"/wiki/Plug-in_(computing)\" title=\"Plug-in (computing)\">plug-ins</a> in commercial applications. They may also provide <i><a href=\"/wiki/De_facto\" title=\"De facto\">de facto</a></i> standards and shared object models for assisting with the challenge of bioinformation integration.\n",
       " </p>,\n",
       " <p>Open-source bioinformatics software includes <a href=\"/wiki/Bioconductor\" title=\"Bioconductor\">Bioconductor</a>, <a href=\"/wiki/BioPerl\" title=\"BioPerl\">BioPerl</a>, <a href=\"/wiki/Biopython\" title=\"Biopython\">Biopython</a>, <a href=\"/wiki/BioJava\" title=\"BioJava\">BioJava</a>, <a href=\"/wiki/BioJS\" title=\"BioJS\">BioJS</a>, <a href=\"/wiki/BioRuby\" title=\"BioRuby\">BioRuby</a>, <a href=\"/wiki/Bioclipse\" title=\"Bioclipse\">Bioclipse</a>, <a href=\"/wiki/EMBOSS\" title=\"EMBOSS\">EMBOSS</a>, <a href=\"/wiki/.NET_Bio\" title=\".NET Bio\">.NET Bio</a>, <a href=\"/wiki/Orange_(software)\" title=\"Orange (software)\">Orange</a> with its bioinformatics add-on, <a href=\"/wiki/Apache_Taverna\" title=\"Apache Taverna\">Apache Taverna</a>, <a href=\"/wiki/UGENE\" title=\"UGENE\">UGENE</a> and <a href=\"/wiki/GenoCAD\" title=\"GenoCAD\">GenoCAD</a>. \n",
       " </p>,\n",
       " <p>The non-profit <a href=\"/wiki/Open_Bioinformatics_Foundation\" title=\"Open Bioinformatics Foundation\">Open Bioinformatics Foundation</a><sup class=\"reference\" id=\"cite_ref-obf-main_57-1\"><a href=\"#cite_note-obf-main-57\">[57]</a></sup> and the annual <a href=\"/wiki/Bioinformatics_Open_Source_Conference\" title=\"Bioinformatics Open Source Conference\">Bioinformatics Open Source Conference</a> promote open-source bioinformatics software.<sup class=\"reference\" id=\"cite_ref-obf-bosc_58-0\"><a href=\"#cite_note-obf-bosc-58\">[58]</a></sup>\n",
       " </p>,\n",
       " <p><a href=\"/wiki/SOAP\" title=\"SOAP\">SOAP</a>- and <a class=\"mw-redirect\" href=\"/wiki/REST\" title=\"REST\">REST</a>-based interfaces have been developed to allow client computers to use algorithms, data and computing resources from servers in other parts of the world. The main advantage are that end users do not have to deal with software and database maintenance overheads.\n",
       " </p>,\n",
       " <p>Basic bioinformatics services are classified by the <a href=\"/wiki/European_Bioinformatics_Institute\" title=\"European Bioinformatics Institute\">EBI</a> into three categories: <a class=\"mw-redirect\" href=\"/wiki/Sequence_alignment_software\" title=\"Sequence alignment software\">SSS</a> (Sequence Search Services), <a href=\"/wiki/Multiple_sequence_alignment\" title=\"Multiple sequence alignment\">MSA</a> (Multiple Sequence Alignment), and <a href=\"#Sequence_analysis\">BSA</a> (Biological Sequence Analysis).<sup class=\"reference\" id=\"cite_ref-59\"><a href=\"#cite_note-59\">[59]</a></sup> The availability of these <a href=\"/wiki/Service-orientation\" title=\"Service-orientation\">service-oriented</a> bioinformatics resources demonstrate the applicability of web-based bioinformatics solutions, and range from a collection of standalone tools with a common data format under a single web-based interface, to integrative, distributed and extensible <a class=\"mw-redirect\" href=\"/wiki/Bioinformatics_workflow_management_systems\" title=\"Bioinformatics workflow management systems\">bioinformatics workflow management systems</a>.\n",
       " </p>,\n",
       " <p>A <a class=\"mw-redirect\" href=\"/wiki/Bioinformatics_workflow_management_systems\" title=\"Bioinformatics workflow management systems\">bioinformatics workflow management system</a> is a specialized form of a <a href=\"/wiki/Workflow_management_system\" title=\"Workflow management system\">workflow management system</a> designed specifically to compose and execute a series of computational or data manipulation steps, or a workflow, in a Bioinformatics application. Such systems are designed to\n",
       " </p>,\n",
       " <p>Some of the platforms giving this service: <a href=\"/wiki/Galaxy_(computational_biology)\" title=\"Galaxy (computational biology)\">Galaxy</a>, <a href=\"/wiki/Kepler_scientific_workflow_system\" title=\"Kepler scientific workflow system\">Kepler</a>, <a href=\"/wiki/Apache_Taverna\" title=\"Apache Taverna\">Taverna</a>, <a href=\"/wiki/UGENE\" title=\"UGENE\">UGENE</a>, <a href=\"/wiki/Anduril_(workflow_engine)\" title=\"Anduril (workflow engine)\">Anduril</a>, <a href=\"/wiki/High-performance_Integrated_Virtual_Environment\" title=\"High-performance Integrated Virtual Environment\">HIVE</a>.\n",
       " </p>,\n",
       " <p>In 2014, the <a href=\"/wiki/Food_and_Drug_Administration\" title=\"Food and Drug Administration\">US Food and Drug Administration</a> sponsored a conference held at the <a href=\"/wiki/National_Institutes_of_Health\" title=\"National Institutes of Health\">National Institutes of Health</a> Bethesda Campus to discuss reproducibility in bioinformatics.<sup class=\"reference\" id=\"cite_ref-60\"><a href=\"#cite_note-60\">[60]</a></sup> Over the next three years, a consortium of stakeholders met regularly to discuss what would become BioCompute paradigm.<sup class=\"reference\" id=\"cite_ref-61\"><a href=\"#cite_note-61\">[61]</a></sup> These stakeholders included representatives from government, industry, and academic entities. Session leaders represented numerous branches of the FDA and NIH Institutes and Centers, non-profit entities including the <a href=\"/wiki/Human_Variome_Project\" title=\"Human Variome Project\">Human Variome Project</a> and the <a href=\"/wiki/European_Federation_for_Medical_Informatics\" title=\"European Federation for Medical Informatics\">European Federation for Medical Informatics</a>, and research institutions including <a href=\"/wiki/Stanford_University\" title=\"Stanford University\">Stanford</a>, the <a href=\"/wiki/New_York_Genome_Center\" title=\"New York Genome Center\">New York Genome Center</a>, and the <a href=\"/wiki/George_Washington_University\" title=\"George Washington University\">George Washington University</a>.\n",
       " </p>,\n",
       " <p>It was decided that the BioCompute paradigm would be in the form of digital 'lab notebooks' which allow for the reproducibility, replication, review, and reuse, of bioinformatics protocols. This was proposed to enable greater continuity within a research group over the course of normal personnel flux while furthering the exchange of ideas between groups. The US FDA funded this work so that information on pipelines would be more transparent and accessible to their regulatory staff.<sup class=\"reference\" id=\"cite_ref-62\"><a href=\"#cite_note-62\">[62]</a></sup>\n",
       " </p>,\n",
       " <p>In 2016, the group reconvened at the NIH in Bethesda and discussed the potential for a <a href=\"/wiki/BioCompute_Object\" title=\"BioCompute Object\">BioCompute Object</a>, an instance of the BioCompute paradigm. This work was copied as both a \"standard trial use\" document and a preprint paper uploaded to bioRxiv. The BioCompute object allows for the JSON-ized record to be shared among employees, collaborators, and regulators.<sup class=\"reference\" id=\"cite_ref-63\"><a href=\"#cite_note-63\">[63]</a></sup><sup class=\"reference\" id=\"cite_ref-64\"><a href=\"#cite_note-64\">[64]</a></sup>\n",
       " </p>,\n",
       " <p>Bioinformatics is not only taught as in-person <a href=\"/wiki/Master%27s_degree\" title=\"Master's degree\">masters degree</a> at many universities. The computational nature of bioinformatics lends it to <a href=\"/wiki/Educational_technology\" title=\"Educational technology\">computer-aided and online learning</a>.<sup class=\"reference\" id=\"cite_ref-65\"><a href=\"#cite_note-65\">[65]</a></sup><sup class=\"reference\" id=\"cite_ref-66\"><a href=\"#cite_note-66\">[66]</a></sup> Software platforms designed to teach bioinformatics concepts and methods include <a href=\"/wiki/Rosalind_(education_platform)\" title=\"Rosalind (education platform)\">Rosalind</a> and online courses offered through the <a href=\"/wiki/Swiss_Institute_of_Bioinformatics\" title=\"Swiss Institute of Bioinformatics\">Swiss Institute of Bioinformatics</a> Training Portal. The <a href=\"/wiki/Canadian_Bioinformatics_Workshops\" title=\"Canadian Bioinformatics Workshops\">Canadian Bioinformatics Workshops</a> provides videos and slides from training workshops on their website under a <a href=\"/wiki/Creative_Commons\" title=\"Creative Commons\">Creative Commons</a> license. The 4273 project or 4273pi project<sup class=\"reference\" id=\"cite_ref-67\"><a href=\"#cite_note-67\">[67]</a></sup> also offers open source educational materials for free. The course runs on low cost <a href=\"/wiki/Raspberry_Pi\" title=\"Raspberry Pi\">Raspberry Pi</a> computers and has been used to teach adults and school pupils.<sup class=\"reference\" id=\"cite_ref-68\"><a href=\"#cite_note-68\">[68]</a></sup><sup class=\"reference\" id=\"cite_ref-69\"><a href=\"#cite_note-69\">[69]</a></sup> 4283 is actively developed by a consortium of academics and research staff who have run research level bioinformatics using Raspberry Pi computers and the 4283 operating system.<sup class=\"reference\" id=\"cite_ref-70\"><a href=\"#cite_note-70\">[70]</a></sup><sup class=\"reference\" id=\"cite_ref-71\"><a href=\"#cite_note-71\">[71]</a></sup>\n",
       " </p>,\n",
       " <p><a href=\"/wiki/Massive_open_online_course\" title=\"Massive open online course\">MOOC</a> platforms also provide online certifications in bioinformatics and related disciplines, including <a href=\"/wiki/Coursera\" title=\"Coursera\">Coursera</a>'s Bioinformatics Specialization (<a href=\"/wiki/University_of_California,_San_Diego\" title=\"University of California, San Diego\">UC San Diego</a>) and Genomic Data Science Specialization (<a href=\"/wiki/Johns_Hopkins_University\" title=\"Johns Hopkins University\">Johns Hopkins</a>) as well as <a href=\"/wiki/EdX\" title=\"EdX\">EdX</a>'s Data Analysis for Life Sciences XSeries (<a href=\"/wiki/Harvard_University\" title=\"Harvard University\">Harvard</a>).\n",
       " </p>,\n",
       " <p>There are several large conferences that are concerned with bioinformatics. Some of the most notable examples are <a href=\"/wiki/Intelligent_Systems_for_Molecular_Biology\" title=\"Intelligent Systems for Molecular Biology\">Intelligent Systems for Molecular Biology</a> (ISMB), <a href=\"/wiki/European_Conference_on_Computational_Biology\" title=\"European Conference on Computational Biology\">European Conference on Computational Biology</a> (ECCB), and <a href=\"/wiki/Research_in_Computational_Molecular_Biology\" title=\"Research in Computational Molecular Biology\">Research in Computational Molecular Biology</a> (RECOMB).\n",
       " </p>]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "paragraphs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "article_text = \"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "for p in paragraphs:\n",
    "    article_text += p.text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Bioinformatics (/ba.onfrmtks/ (listen)) is an interdisciplinary field of science that develops methods and software tools for understanding biological data, especially when the data sets are large and complex. Bioinformatics uses biology, chemistry, physics, computer science, computer programming, information engineering, mathematics and statistics to analyze and interpret biological data. The subsequent process of analyzing and interpreting data is referred to as computational biology. \n",
      "Computational, statistical, and computer programming techniques have been used  for computer simulation analyses of biological queries. They include reused specific analysis \"pipelines\", particularly in the field of genomics, such as by the identification of genes and single nucleotide polymorphisms (SNPs). These pipelines are used to better understand the genetic basis of disease, unique adaptations, desirable properties (esp. in agricultural species), or differences between populations. Bioinformatics also includes proteomics, which tries to understand the organizational principles within nucleic acid and protein sequences.[1]\n",
      "Image and signal processing allow extraction of useful results from large amounts of raw data. In the field of genetics, it aids in sequencing and annotating genomes and their observed mutations. Bioinformatics includes text mining of biological literature and the development of biological and gene ontologies to organize and query biological data. It also plays a role in the analysis of gene and protein expression and regulation. Bioinformatics tools aid in comparing, analyzing and interpreting genetic and genomic data and more generally in the understanding of evolutionary aspects of molecular biology. At a more integrative level, it helps analyze and catalogue the biological pathways and networks that are an important part of systems biology. In structural biology, it aids in the simulation and modeling of DNA,[2] RNA,[2][3] proteins[4] as well as biomolecular interactions.[5][6][7][8]\n",
      "The first definition of the term bioinformatics was coined by Paulien Hogeweg and Ben Hesper in 1970, to refer to the study of information processes in biotic systems.[9][10][11][12][13] This definition placed bioinformatics as a field parallel to biochemistry (the study of chemical processes in biological systems).[10]\n",
      "Bioinformatics and computational biology involved the analysis of biological data, particularly DNA, RNA, and protein sequences. The field of bioinformatics experienced explosive growth starting in the mid-1990s, driven largely by the Human Genome Project and by rapid advances in DNA sequencing technology.\n",
      "Analyzing biological data to produce meaningful information involves writing and running software programs that use algorithms from graph theory, artificial intelligence, soft computing, data mining, image processing, and computer simulation. The algorithms in turn depend on theoretical foundations such as discrete mathematics, control theory, system theory, information theory, and statistics.\n",
      "There has been a tremendous advance in speed and cost reduction since the completion of the Human Genome Project, with some labs able to sequence over 100,000 billion bases each year, and a full genome can be sequenced for $1,000 or less.[14]\n",
      "Computers became essential in molecular biology when protein sequences became available after Frederick Sanger determined the sequence of insulin in the early 1950s. Comparing multiple sequences manually turned out to be impractical. Margaret Oakley Dayhoff, a pioneer in the field,[15] compiled one of the first protein sequence databases, initially published as books[16] as well as methods of sequence alignment and molecular evolution.[17] Another early contributor to bioinformatics was Elvin A. Kabat, who pioneered biological sequence analysis in 1970 with his comprehensive volumes of antibody sequences released online with Tai Te Wu between 1980 and 1991.[18]\n",
      "In the 1970s, new techniques for sequencing DNA were applied to bacteriophage MS2 and X174, and the extended nucleotide sequences were then parsed with informational and statistical algorithms.  These studies illustrated that well known features, such as the coding segments and the triplet code, are revealed in straightforward statistical analyses and were the proof of the concept that bioinformatics would be insightful.[19][20]\n",
      "In order to study how normal cellular activities are altered in different disease states, raw biological data must be combined to form a comprehensive picture of these activities. Therefore[when?], the field of bioinformatics has evolved such that the most pressing task now involves the analysis and interpretation of various types of data. This also includes nucleotide and amino acid sequences, protein domains, and protein structures.[21]\n",
      "Important sub-disciplines within bioinformatics and computational biology include:\n",
      "The primary goal of bioinformatics is to increase the understanding of biological processes. What sets it apart from other approaches is its focus on developing and applying computationally intensive techniques to achieve this goal. Examples include: pattern recognition, data mining, machine learning algorithms, and visualization. Major research efforts in the field include sequence alignment, gene finding, genome assembly, drug design, drug discovery, protein structure alignment, protein structure prediction, prediction of gene expression and proteinprotein interactions, genome-wide association studies, the modeling of evolution and cell division/mitosis.\n",
      "Bioinformatics entails the creation and advancement of databases, algorithms, computational and statistical techniques, and theory to solve formal and practical problems arising from the management and analysis of biological data.\n",
      "Over the past few decades, rapid developments in genomic and other molecular research technologies and developments in information technologies have combined to produce a tremendous amount of information related to molecular biology. Bioinformatics is the name given to these mathematical and computing approaches used to glean understanding of biological processes.\n",
      "Common activities in bioinformatics include mapping and analyzing DNA and protein sequences, aligning DNA and protein sequences to compare them, and creating and viewing 3-D models of protein structures.\n",
      "Since the bacteriophage Phage -X174 was sequenced in 1977,[22] the DNA sequences of thousands of organisms have been decoded and stored in databases. This sequence information is analyzed to determine genes that encode proteins, RNA genes, regulatory sequences, structural motifs, and repetitive sequences. A comparison of genes within a species or between different species can show similarities between protein functions, or relations between species (the use of molecular systematics to construct phylogenetic trees). With the growing amount of data, it long ago became impractical to analyze DNA sequences manually. Computer programs such as BLAST are used routinely to search sequencesas of 2008, from more than 260,000 organisms, containing over 190 billion nucleotides.[23]\n",
      "Before sequences can be analyzed, they are obtained from a data storage bank, such as Genbank. DNA sequencing is still a non-trivial problem as the raw data may be noisy or affected by weak signals. Algorithms have been developed for base calling for the various experimental approaches to DNA sequencing.\n",
      "Most DNA sequencing techniques produce short fragments of sequence that need to be assembled to obtain complete gene or genome sequences. The shotgun sequencing technique (used by The Institute for Genomic Research (TIGR) to sequence the first bacterial genome, Haemophilus influenzae)[24] generates the sequences of many thousands of small DNA fragments (ranging from 35 to 900 nucleotides long, depending on the sequencing technology). The ends of these fragments overlap and, when aligned properly by a genome assembly program, can be used to reconstruct the complete genome. Shotgun sequencing yields sequence data quickly, but the task of assembling the fragments can be quite complicated for larger genomes. For a genome as large as the human genome, it may take many days of CPU time on large-memory, multiprocessor computers to assemble the fragments, and the resulting assembly usually contains numerous gaps that must be filled in later. Shotgun sequencing is the method of choice for virtually all genomes sequenced (rather than chain-termination or chemical degradation methods), and genome assembly algorithms are a critical area of bioinformatics research.\n",
      "In genomics, annotation refers to the process of marking the stop and start regions of genes and other biological features in a sequenced DNA sequence. Many genomes are too large to be annotated by hand. As the rate of sequencing exceeds the rate of genome annotation, genome annotation has become the new bottleneck in bioinformatics[when?]. \n",
      "Genome annotation can be classified into three levels: the nucleotide, protein, and process levels.\n",
      "Gene finding is a chief aspect of nucleotide-level annotation. For complex genomes, a combination of ab initio gene prediction and sequence comparison with expressed sequence databases and other organisms can be successful. Nucleotide-level annotation also allows the integration of genome sequence with other genetic and physical maps of the genome.\n",
      "The principal aim of protein-level annotation is to assign function to the protein products of the genome. Databases of protein sequences and functional domains and motifs are used for this type of annotation. About half of the predicted proteins in a new genome sequence tend to have no obvious function.\n",
      "Understanding the function of genes and their products in the context of cellular and organismal physiology is the goal of process-level annotation. An obstacle of process-level annotation has been the inconsistency of terms used by different model systems. The Gene Ontology Consortium is helping to solve this problem.[25]\n",
      "The first description of a comprehensive annotation system was published in 1995[24] by the The Institute for Genomic Research, which performed the first complete sequencing and analysis of the genome of a free-living organism, the bacterium Haemophilus influenzae.[24] The system identifies the genes encoding all proteins, transfer RNAs, ribosomal RNAs, in order to make initial functional assignments. The GeneMark program trained to find protein-coding genes in Haemophilus influenzae is constantly changing and improving.\n",
      "Following the goals that the Human Genome Project left to achieve after its closure in 2003, the ENCODE project was developed by the National Human Genome Research Institute. This project is a collaborative data collection of the functional elements of the human genome that uses next-generation DNA-sequencing technologies and genomic tiling arrays, technologies able to automatically generate large amounts of data at a dramatically reduced per-base cost but with the same accuracy (base call error) and fidelity (assembly error).\n",
      "While genome annotation is primarily based on sequence similarity (and thus homology), other properties of sequences can be used to predict the function of genes. In fact, most gene function prediction methods focus on protein sequences as they are more informative and more feature-rich. For instance, the distribution of hydrophobic amino acids predicts transmembrane segments in proteins. However, protein function prediction can also use external information such as gene (or protein) expression data, protein structure, or protein-protein interactions.[26]\n",
      "Evolutionary biology is the study of the origin and descent of species, as well as their change over time. Informatics has assisted evolutionary biologists by enabling researchers to:\n",
      "Future work endeavours to reconstruct the now more complex tree of life.[according to whom?]\n",
      "The core of comparative genome analysis is the establishment of the correspondence between genes (orthology analysis) or other genomic features in different organisms. Intergenomic maps are made to trace the evolutionary processes responsible for the divergence of two genomes. A multitude of evolutionary events acting at various organizational levels shape genome evolution. At the lowest level, point mutations affect individual nucleotides. At a higher level, large chromosomal segments undergo duplication, lateral transfer, inversion, transposition, deletion and insertion.[28] Entire genomes are involved in processes of hybridization, polyploidization and endosymbiosis that lead to rapid speciation. The complexity of genome evolution poses many exciting challenges to developers of mathematical models and algorithms, who have recourse to a spectrum of algorithmic, statistical and mathematical techniques, ranging from exact, heuristics, fixed parameter and approximation algorithms for problems based on parsimony models to Markov chain Monte Carlo algorithms for Bayesian analysis of problems based on probabilistic models.\n",
      "Many of these studies are based on the detection of sequence homology to assign sequences to protein families.[29]\n",
      "Pan genomics is a concept introduced in 2005 by Tettelin and Medini. Pan genome is the complete gene repertoire of a particular monophyletic taxonomic group. Although initially applied to closely related strains of a species, it can be applied to a larger context like genus, phylum, etc. It is divided in two parts: the Core genome, a set of genes common to all the genomes under study (often housekeeping genes vital for survival), and the Dispensable/Flexible genome: a set of genes not present in all but one or some genomes under study. A bioinformatics tool BPGA can be used to characterize the Pan Genome of bacterial species.[30]\n",
      "As of 2013, the existence of efficient high-throughput next-generation sequencing technology allows for the identification of cause many different human disorders. Simple Mendelian inheritance has been observed for over 3,000 disorders that have been identified at the Online Mendelian Inheritance in Man database, but complex diseases are more difficult. Association studies have found many individual genetic regions that individually are weakly associated with complex diseases (such as infertility,[31] breast cancer[32] and Alzheimer's disease[33]), rather than a single cause.[34][35] There are currently many challenges to using genes for diagnosis and treatment, such as how we don't know which genes are important, or how stable the choices an algorithm provides. [36]\n",
      "Genome-wide association studies have successfully identified thousands of common genetic variants for complex diseases and traits; however, these common variants only explain a small fraction of heritability.[37] Rare variants may account for some of the missing heritability.[38] Large-scale whole genome sequencing studies have rapidly sequenced millions of whole genomes, and such studies have identified hundreds of millions of rare variants.[39] Functional annotations predict the effect or function of a genetic variant and help to prioritize rare functional variants, and incorporating these annotations can effectively boost the power of genetic association of rare variants analysis of whole genome sequencing studies.[40] Some tools have been developed to provide all-in-one rare variant association analysis for whole-genome sequencing data, including integration of genotype data and their functional annotations, association analysis, result summary and visualization.[41][42] Meta-analysis of whole genome sequencing studies provides an attractive solution to the problem of collecting large sample sizes for discovering rare variants associated with complex phenotypes.[43]\n",
      "In cancer, the genomes of affected cells are rearranged in complex or unpredictable ways. In addition to single-nucleotide polymorphism arrays identifying point mutations that cause cancer, oligonucleotide microarrays can be used to identify chromosomal gains and losses (called comparative genomic hybridization). These detection methods generate terabytes of data per experiment. The data is often found to contain considerable variability, or noise, and thus Hidden Markov model and change-point analysis methods are being developed to infer real copy number changes.[citation needed]\n",
      "Two important principles can be used to identify cancer by mutations in the exome. First, cancer is a disease of accumulated somatic mutations in genes. Second, cancer contains driver mutations which need to be distinguished from passengers.[44]\n",
      "Further improvements in bioinformatics could allow for classifying types of cancer by analysis of cancer driven mutations in the genome. Furthermore, tracking of patients while the disease progresses may be possible in the future with the sequence of cancer samples. Another type of data that requires novel informatics development is the analysis of lesions found to be recurrent among many tumors.[45]\n",
      "The expression of many genes can be determined by measuring mRNA levels with multiple techniques including microarrays, expressed cDNA sequence tag (EST) sequencing, serial analysis of gene expression (SAGE) tag sequencing, massively parallel signature sequencing (MPSS), RNA-Seq, also known as \"Whole Transcriptome Shotgun Sequencing\" (WTSS), or various applications of multiplexed in-situ hybridization. All of these techniques are extremely noise-prone and/or subject to bias in the biological measurement, and a major research area in computational biology involves developing statistical tools to separate signal from noise in high-throughput gene expression studies.[46] Such studies are often used to determine the genes implicated in a disorder: one might compare microarray data from cancerous epithelial cells to data from non-cancerous cells to determine the transcripts that are up-regulated and down-regulated in a particular population of cancer cells.\n",
      "Protein microarrays and high throughput (HT) mass spectrometry (MS) can provide a snapshot of the proteins present in a biological sample. The former approach faces similar problems as with microarrays targeted at mRNA, the latter involves the problem of matching large amounts of mass data against predicted masses from protein sequence databases, and the complicated statistical analysis of samples when multiple incomplete peptides from each protein are detected. Cellular protein localization in a tissue context can be achieved through affinity proteomics displayed as spatial data based on immunohistochemistry and tissue microarrays.[47]\n",
      "Gene regulation is a complex process where a signal, such as an extracellular signal such as a hormone, eventually leads to an increase or decrease in the activity of one or more proteins. Bioinformatics techniques have been applied to explore various steps in this process.\n",
      "For example, gene expression can be regulated by nearby elements in the genome. Promoter analysis involves the identification and study of sequence motifs in the DNA surrounding the protein-coding region of a gene. These motifs influence the extent to which that region is transcribed into mRNA. Enhancer elements far away from the promoter can also regulate gene expression, through three-dimensional looping interactions. These interactions can be determined by bioinformatic analysis of chromosome conformation capture experiments.\n",
      "Expression data can be used to infer gene regulation: one might compare microarray data from a wide variety of states of an organism to form hypotheses about the genes involved in each state. In a single-cell organism, one might compare stages of the cell cycle, along with various stress conditions (heat shock, starvation, etc.). Clustering algorithms can be then applied to expression data to determine which genes are co-expressed. For example, the upstream regions (promoters) of co-expressed genes can be searched for over-represented regulatory elements. Examples of clustering algorithms applied in gene clustering are k-means clustering, self-organizing maps (SOMs), hierarchical clustering, and consensus clustering methods.\n",
      "Several approaches have been developed to analyze the location of organelles, genes, proteins, and other components within cells. A gene ontology category, cellular component, has been devised to capture subcellular localization in many biological databases.\n",
      "Microscopic pictures allow for the location of organelles as well as molecules, which may be the source of abnormalities in diseases.\n",
      "Finding the location of proteins allows us to predict what they do. This is called protein function prediction. For instance, if a protein is found in the nucleus it may be involved in gene regulation or splicing. By contrast, if a protein is found in mitochondria, it may be involved in respiration or other metabolic processes. There are well developed protein subcellular localization prediction resources available, including protein subcellular location databases, and prediction tools.[48][49]\n",
      "Data from high-throughput chromosome conformation capture experiments, such as Hi-C (experiment) and ChIA-PET, can provide information on the three-dimensional structure and nuclear organization of chromatin. Bioinformatic challenges in this field include partitioning the genome into domains, such as Topologically Associating Domains (TADs), that are organised together in three-dimensional space.[50]\n",
      "Finding the structure of proteins important application of bioinformatics. The Critical Assessment of Protein Structure Prediction (CASP) is an open competition where worldwide research groups submit protein models for evaluating unknown protein models.[51][52]\n",
      "The linear amino acid sequence of a protein is called the primary structure, can be easily determined from the sequence of codons on the DNA gene that codes for it. In most proteins, the primary structure uniquely determines the 3-dimensional structure of a protein in its native environment. An exception is the misfolded protein involved in bovine spongiform encephalopathy. This structure is linked to the function of the protein. Additional structural information includes the secondary, tertiary and quaternary structure. A viable general solution to the prediction of the function of a protein remains an open problem. Most efforts have so far been directed towards heuristics that work most of the time.[citation needed]\n",
      "In the genomic branch of bioinformatics, homology is used to predict the function of a gene: if the sequence of gene A, whose function is known, is homologous to the sequence of gene B, whose function is unknown, one could infer that B may share A's function. In structural bioinformatics, homology is used to determine which parts of a protein are important in structure formation and interaction with other proteins. Homology modeling is used to predict the structure of an unknown protein from existing homologous proteins. \n",
      "One example of this is hemoglobin in humans and the hemoglobin in legumes (leghemoglobin), which are distant relatives from the same protein superfamily. Both serve the same purpose of transporting oxygen in the organism. Although both of these proteins have completely different amino acid sequences, their protein structures are virtually identical, which reflects their near identical purposes and shared ancestor.[53]\n",
      "Other techniques for predicting protein structure include protein threading and de novo (from scratch) physics-based modeling.\n",
      "Another aspect of structural bioinformatics include the use of protein structures for Virtual Screening models such as Quantitative Structure-Activity Relationship models and proteochemometric models (PCM). Furthermore, a protein's crystal structure can be used in simulation of for example ligand-binding studies and in silico mutagenesis studies.\n",
      "A 2021 deep-learning algorithms-based software called AlphaFold, developed by Google's DeepMind, greatly outperforms all other prediction software methods[54][how?], and has released predicted structures for hundreds of millions of proteins in the AlphaFold protein structure database.[55]\n",
      "Network analysis seeks to understand the relationships within biological networks such as metabolic or proteinprotein interaction networks. Although biological networks can be constructed from a single type of molecule or entity (such as genes), network biology often attempts to integrate many different data types, such as proteins, small molecules, gene expression data, and others, which are all connected physically, functionally, or both.\n",
      "Systems biology involves the use of computer simulations of cellular subsystems (such as the networks of metabolites and enzymes that comprise metabolism, signal transduction pathways and gene regulatory networks) to both analyze and visualize the complex connections of these cellular processes. Artificial life or virtual evolution attempts to understand evolutionary processes via the computer simulation of simple (artificial) life forms.\n",
      "Tens of thousands of three-dimensional protein structures have been determined by X-ray crystallography and protein nuclear magnetic resonance spectroscopy (protein NMR) and a central question in structural bioinformatics is whether it is practical to predict possible proteinprotein interactions only based on these 3D shapes, without performing proteinprotein interaction experiments. A variety of methods have been developed to tackle the proteinprotein docking problem, though it seems that there is still much work to be done in this field.\n",
      "Other interactions encountered in the field include Proteinligand (including drug) and proteinpeptide. Molecular dynamic simulation of movement of atoms about rotatable bonds is the fundamental principle behind computational algorithms, termed docking algorithms, for studying molecular interactions.\n",
      "The enormous number of published literature makes it virtually impossible for individuals to read every paper, resulting in disjointed sub-fields of research. Literature analysis aims to employ computational and statistical linguistics to mine this growing library of text resources. For example:\n",
      "The area of research draws from statistics and computational linguistics.\n",
      "Computational technologies are used to automate the processing, quantification and analysis of large amounts of high-information-content biomedical imagery. Modern image analysis systems can improve an observer's accuracy, objectivity, or speed. Image analysis is important for both diagnostics and research. Some examples are:\n",
      "Computational techniques are used to analyse high-throughput, low-measurement single cell data, such as that obtained from flow cytometry. These methods typically involve finding populations of cells that are relevant to a particular disease state or experimental condition.\n",
      "Biodiversity informatics deals with the collection and analysis of biodiversity data, such as taxonomic databases, or microbiome data. Examples of such analyses include phylogenetics, niche modelling, species richness mapping, DNA barcoding, or species identification tools.\n",
      "Biological ontologies are directed acyclic graphs of controlled vocabularies. They create categories for biological concepts and descriptions so they can be easily analyzed with computers. When categorised in this way, it is possible to gain added value from holistic and integrated analysis.[citation needed]\n",
      "The OBO Foundry was an effort to standardise certain ontologies. One of the most widespread is the Gene ontology which describes gene function. There are also ontologies which describe phenotypes.\n",
      "Databases are essential for bioinformatics research and applications. Databases exist for many different information types, including DNA and protein sequences, molecular structures, phenotypes and biodiversity. Databases can contain both empirical data (obtained directly from experiments) and predicted data (obtained from analysis of existing data). They may be specific to a particular organism, pathway or molecule of interest. Alternatively, they can incorporate data compiled from multiple other databases. Databases can have different formats, access mechanisms, and be public or private.\n",
      "Some of the most commonly used databases are listed below:\n",
      "Software tools for bioinformatics include simple command-line tools, more complex graphical programs, and standalone web-services. They are made by bioinformatics companies or by public institutions.\n",
      "Many free and open-source software tools have existed and continued to grow since the 1980s.[57] The combination of a continued need for new algorithms for the analysis of emerging types of biological readouts, the potential for innovative in silico experiments, and freely available open code bases have created opportunities for research groups to contribute to both bioinformatics regardless of funding. The open source tools often act as incubators of ideas, or community-supported plug-ins in commercial applications. They may also provide de facto standards and shared object models for assisting with the challenge of bioinformation integration.\n",
      "Open-source bioinformatics software includes Bioconductor, BioPerl, Biopython, BioJava, BioJS, BioRuby, Bioclipse, EMBOSS, .NET Bio, Orange with its bioinformatics add-on, Apache Taverna, UGENE and GenoCAD. \n",
      "The non-profit Open Bioinformatics Foundation[57] and the annual Bioinformatics Open Source Conference promote open-source bioinformatics software.[58]\n",
      "SOAP- and REST-based interfaces have been developed to allow client computers to use algorithms, data and computing resources from servers in other parts of the world. The main advantage are that end users do not have to deal with software and database maintenance overheads.\n",
      "Basic bioinformatics services are classified by the EBI into three categories: SSS (Sequence Search Services), MSA (Multiple Sequence Alignment), and BSA (Biological Sequence Analysis).[59] The availability of these service-oriented bioinformatics resources demonstrate the applicability of web-based bioinformatics solutions, and range from a collection of standalone tools with a common data format under a single web-based interface, to integrative, distributed and extensible bioinformatics workflow management systems.\n",
      "A bioinformatics workflow management system is a specialized form of a workflow management system designed specifically to compose and execute a series of computational or data manipulation steps, or a workflow, in a Bioinformatics application. Such systems are designed to\n",
      "Some of the platforms giving this service: Galaxy, Kepler, Taverna, UGENE, Anduril, HIVE.\n",
      "In 2014, the US Food and Drug Administration sponsored a conference held at the National Institutes of Health Bethesda Campus to discuss reproducibility in bioinformatics.[60] Over the next three years, a consortium of stakeholders met regularly to discuss what would become BioCompute paradigm.[61] These stakeholders included representatives from government, industry, and academic entities. Session leaders represented numerous branches of the FDA and NIH Institutes and Centers, non-profit entities including the Human Variome Project and the European Federation for Medical Informatics, and research institutions including Stanford, the New York Genome Center, and the George Washington University.\n",
      "It was decided that the BioCompute paradigm would be in the form of digital 'lab notebooks' which allow for the reproducibility, replication, review, and reuse, of bioinformatics protocols. This was proposed to enable greater continuity within a research group over the course of normal personnel flux while furthering the exchange of ideas between groups. The US FDA funded this work so that information on pipelines would be more transparent and accessible to their regulatory staff.[62]\n",
      "In 2016, the group reconvened at the NIH in Bethesda and discussed the potential for a BioCompute Object, an instance of the BioCompute paradigm. This work was copied as both a \"standard trial use\" document and a preprint paper uploaded to bioRxiv. The BioCompute object allows for the JSON-ized record to be shared among employees, collaborators, and regulators.[63][64]\n",
      "Bioinformatics is not only taught as in-person masters degree at many universities. The computational nature of bioinformatics lends it to computer-aided and online learning.[65][66] Software platforms designed to teach bioinformatics concepts and methods include Rosalind and online courses offered through the Swiss Institute of Bioinformatics Training Portal. The Canadian Bioinformatics Workshops provides videos and slides from training workshops on their website under a Creative Commons license. The 4273 project or 4273pi project[67] also offers open source educational materials for free. The course runs on low cost Raspberry Pi computers and has been used to teach adults and school pupils.[68][69] 4283 is actively developed by a consortium of academics and research staff who have run research level bioinformatics using Raspberry Pi computers and the 4283 operating system.[70][71]\n",
      "MOOC platforms also provide online certifications in bioinformatics and related disciplines, including Coursera's Bioinformatics Specialization (UC San Diego) and Genomic Data Science Specialization (Johns Hopkins) as well as EdX's Data Analysis for Life Sciences XSeries (Harvard).\n",
      "There are several large conferences that are concerned with bioinformatics. Some of the most notable examples are Intelligent Systems for Molecular Biology (ISMB), European Conference on Computational Biology (ECCB), and Research in Computational Molecular Biology (RECOMB).\n",
      "\n"
     ]
    }
   ],
   "source": [
    "print(article_text)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "step2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "try:\n",
    "    import string\n",
    "    from nltk.corpus import stopwords\n",
    "    import nltk\n",
    "except Exception as e:\n",
    "    print(e)    \n",
    "\n",
    "\n",
    "class PreProcessText(object):\n",
    "    def __init__(self):\n",
    "        pass\n",
    "    def __remove_punctuation(self, text):\n",
    "        '''\n",
    "        Takes a string \n",
    "        return : Return a string\n",
    "        '''\n",
    "        message = []\n",
    "        for x in text:\n",
    "            if x in string.punctuation:\n",
    "                pass\n",
    "            else:\n",
    "                message.append(x)\n",
    "        message = ''.join(message)\n",
    "\n",
    "        return message\n",
    "    \n",
    "    def __remove_stopwords(self, text):\n",
    "        '''\n",
    "        Takes a String\n",
    "        return List\n",
    "        '''\n",
    "        words = []\n",
    "        for x in text.split():\n",
    "\n",
    "            if x.lower() in stopwords.words('english'):\n",
    "                pass\n",
    "            else:\n",
    "                words.append(x)\n",
    "\n",
    "        return words\n",
    "    \n",
    "    def token_words(self, text):\n",
    "        '''\n",
    "        Takes String\n",
    "        Return Token also called list of words that is used to train \n",
    "        Train the Model\n",
    "        '''\n",
    "        message = self.__remove_punctuation(text)\n",
    "        words = self.__remove_stopwords(message)\n",
    "        return words"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package stopwords to\n",
      "[nltk_data]     C:\\Users\\mohdalwaz\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package stopwords is already up-to-date!\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Download Stop Words........\n"
     ]
    }
   ],
   "source": [
    "import nltk\n",
    "flag = nltk.download(\"stopwords\")\n",
    "\n",
    "if (flag == \"False\" or flag == False):\n",
    "    print(\"Failed to download Stop words\")\n",
    "else:\n",
    "    print(\"Download Stop Words........\")\n",
    "    helper = PreProcessText()\n",
    "    #words = helper.taken_words(text=txt)\n",
    "    words = helper.token_words(text=article_text)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Bioinformatics',\n",
       " 'baonfrmtks',\n",
       " 'listen',\n",
       " 'interdisciplinary',\n",
       " 'field',\n",
       " 'science',\n",
       " 'develops',\n",
       " 'methods',\n",
       " 'software',\n",
       " 'tools',\n",
       " 'understanding',\n",
       " 'biological',\n",
       " 'data',\n",
       " 'especially',\n",
       " 'data',\n",
       " 'sets',\n",
       " 'large',\n",
       " 'complex',\n",
       " 'Bioinformatics',\n",
       " 'uses',\n",
       " 'biology',\n",
       " 'chemistry',\n",
       " 'physics',\n",
       " 'computer',\n",
       " 'science',\n",
       " 'computer',\n",
       " 'programming',\n",
       " 'information',\n",
       " 'engineering',\n",
       " 'mathematics',\n",
       " 'statistics',\n",
       " 'analyze',\n",
       " 'interpret',\n",
       " 'biological',\n",
       " 'data',\n",
       " 'subsequent',\n",
       " 'process',\n",
       " 'analyzing',\n",
       " 'interpreting',\n",
       " 'data',\n",
       " 'referred',\n",
       " 'computational',\n",
       " 'biology',\n",
       " 'Computational',\n",
       " 'statistical',\n",
       " 'computer',\n",
       " 'programming',\n",
       " 'techniques',\n",
       " 'used',\n",
       " 'computer',\n",
       " 'simulation',\n",
       " 'analyses',\n",
       " 'biological',\n",
       " 'queries',\n",
       " 'include',\n",
       " 'reused',\n",
       " 'specific',\n",
       " 'analysis',\n",
       " 'pipelines',\n",
       " 'particularly',\n",
       " 'field',\n",
       " 'genomics',\n",
       " 'identification',\n",
       " 'genes',\n",
       " 'single',\n",
       " 'nucleotide',\n",
       " 'polymorphisms',\n",
       " 'SNPs',\n",
       " 'pipelines',\n",
       " 'used',\n",
       " 'better',\n",
       " 'understand',\n",
       " 'genetic',\n",
       " 'basis',\n",
       " 'disease',\n",
       " 'unique',\n",
       " 'adaptations',\n",
       " 'desirable',\n",
       " 'properties',\n",
       " 'esp',\n",
       " 'agricultural',\n",
       " 'species',\n",
       " 'differences',\n",
       " 'populations',\n",
       " 'Bioinformatics',\n",
       " 'also',\n",
       " 'includes',\n",
       " 'proteomics',\n",
       " 'tries',\n",
       " 'understand',\n",
       " 'organizational',\n",
       " 'principles',\n",
       " 'within',\n",
       " 'nucleic',\n",
       " 'acid',\n",
       " 'protein',\n",
       " 'sequences1',\n",
       " 'Image',\n",
       " 'signal',\n",
       " 'processing',\n",
       " 'allow',\n",
       " 'extraction',\n",
       " 'useful',\n",
       " 'results',\n",
       " 'large',\n",
       " 'amounts',\n",
       " 'raw',\n",
       " 'data',\n",
       " 'field',\n",
       " 'genetics',\n",
       " 'aids',\n",
       " 'sequencing',\n",
       " 'annotating',\n",
       " 'genomes',\n",
       " 'observed',\n",
       " 'mutations',\n",
       " 'Bioinformatics',\n",
       " 'includes',\n",
       " 'text',\n",
       " 'mining',\n",
       " 'biological',\n",
       " 'literature',\n",
       " 'development',\n",
       " 'biological',\n",
       " 'gene',\n",
       " 'ontologies',\n",
       " 'organize',\n",
       " 'query',\n",
       " 'biological',\n",
       " 'data',\n",
       " 'also',\n",
       " 'plays',\n",
       " 'role',\n",
       " 'analysis',\n",
       " 'gene',\n",
       " 'protein',\n",
       " 'expression',\n",
       " 'regulation',\n",
       " 'Bioinformatics',\n",
       " 'tools',\n",
       " 'aid',\n",
       " 'comparing',\n",
       " 'analyzing',\n",
       " 'interpreting',\n",
       " 'genetic',\n",
       " 'genomic',\n",
       " 'data',\n",
       " 'generally',\n",
       " 'understanding',\n",
       " 'evolutionary',\n",
       " 'aspects',\n",
       " 'molecular',\n",
       " 'biology',\n",
       " 'integrative',\n",
       " 'level',\n",
       " 'helps',\n",
       " 'analyze',\n",
       " 'catalogue',\n",
       " 'biological',\n",
       " 'pathways',\n",
       " 'networks',\n",
       " 'important',\n",
       " 'part',\n",
       " 'systems',\n",
       " 'biology',\n",
       " 'structural',\n",
       " 'biology',\n",
       " 'aids',\n",
       " 'simulation',\n",
       " 'modeling',\n",
       " 'DNA2',\n",
       " 'RNA23',\n",
       " 'proteins4',\n",
       " 'well',\n",
       " 'biomolecular',\n",
       " 'interactions5678',\n",
       " 'first',\n",
       " 'definition',\n",
       " 'term',\n",
       " 'bioinformatics',\n",
       " 'coined',\n",
       " 'Paulien',\n",
       " 'Hogeweg',\n",
       " 'Ben',\n",
       " 'Hesper',\n",
       " '1970',\n",
       " 'refer',\n",
       " 'study',\n",
       " 'information',\n",
       " 'processes',\n",
       " 'biotic',\n",
       " 'systems910111213',\n",
       " 'definition',\n",
       " 'placed',\n",
       " 'bioinformatics',\n",
       " 'field',\n",
       " 'parallel',\n",
       " 'biochemistry',\n",
       " 'study',\n",
       " 'chemical',\n",
       " 'processes',\n",
       " 'biological',\n",
       " 'systems10',\n",
       " 'Bioinformatics',\n",
       " 'computational',\n",
       " 'biology',\n",
       " 'involved',\n",
       " 'analysis',\n",
       " 'biological',\n",
       " 'data',\n",
       " 'particularly',\n",
       " 'DNA',\n",
       " 'RNA',\n",
       " 'protein',\n",
       " 'sequences',\n",
       " 'field',\n",
       " 'bioinformatics',\n",
       " 'experienced',\n",
       " 'explosive',\n",
       " 'growth',\n",
       " 'starting',\n",
       " 'mid1990s',\n",
       " 'driven',\n",
       " 'largely',\n",
       " 'Human',\n",
       " 'Genome',\n",
       " 'Project',\n",
       " 'rapid',\n",
       " 'advances',\n",
       " 'DNA',\n",
       " 'sequencing',\n",
       " 'technology',\n",
       " 'Analyzing',\n",
       " 'biological',\n",
       " 'data',\n",
       " 'produce',\n",
       " 'meaningful',\n",
       " 'information',\n",
       " 'involves',\n",
       " 'writing',\n",
       " 'running',\n",
       " 'software',\n",
       " 'programs',\n",
       " 'use',\n",
       " 'algorithms',\n",
       " 'graph',\n",
       " 'theory',\n",
       " 'artificial',\n",
       " 'intelligence',\n",
       " 'soft',\n",
       " 'computing',\n",
       " 'data',\n",
       " 'mining',\n",
       " 'image',\n",
       " 'processing',\n",
       " 'computer',\n",
       " 'simulation',\n",
       " 'algorithms',\n",
       " 'turn',\n",
       " 'depend',\n",
       " 'theoretical',\n",
       " 'foundations',\n",
       " 'discrete',\n",
       " 'mathematics',\n",
       " 'control',\n",
       " 'theory',\n",
       " 'system',\n",
       " 'theory',\n",
       " 'information',\n",
       " 'theory',\n",
       " 'statistics',\n",
       " 'tremendous',\n",
       " 'advance',\n",
       " 'speed',\n",
       " 'cost',\n",
       " 'reduction',\n",
       " 'since',\n",
       " 'completion',\n",
       " 'Human',\n",
       " 'Genome',\n",
       " 'Project',\n",
       " 'labs',\n",
       " 'able',\n",
       " 'sequence',\n",
       " '100000',\n",
       " 'billion',\n",
       " 'bases',\n",
       " 'year',\n",
       " 'full',\n",
       " 'genome',\n",
       " 'sequenced',\n",
       " '1000',\n",
       " 'less14',\n",
       " 'Computers',\n",
       " 'became',\n",
       " 'essential',\n",
       " 'molecular',\n",
       " 'biology',\n",
       " 'protein',\n",
       " 'sequences',\n",
       " 'became',\n",
       " 'available',\n",
       " 'Frederick',\n",
       " 'Sanger',\n",
       " 'determined',\n",
       " 'sequence',\n",
       " 'insulin',\n",
       " 'early',\n",
       " '1950s',\n",
       " 'Comparing',\n",
       " 'multiple',\n",
       " 'sequences',\n",
       " 'manually',\n",
       " 'turned',\n",
       " 'impractical',\n",
       " 'Margaret',\n",
       " 'Oakley',\n",
       " 'Dayhoff',\n",
       " 'pioneer',\n",
       " 'field15',\n",
       " 'compiled',\n",
       " 'one',\n",
       " 'first',\n",
       " 'protein',\n",
       " 'sequence',\n",
       " 'databases',\n",
       " 'initially',\n",
       " 'published',\n",
       " 'books16',\n",
       " 'well',\n",
       " 'methods',\n",
       " 'sequence',\n",
       " 'alignment',\n",
       " 'molecular',\n",
       " 'evolution17',\n",
       " 'Another',\n",
       " 'early',\n",
       " 'contributor',\n",
       " 'bioinformatics',\n",
       " 'Elvin',\n",
       " 'Kabat',\n",
       " 'pioneered',\n",
       " 'biological',\n",
       " 'sequence',\n",
       " 'analysis',\n",
       " '1970',\n",
       " 'comprehensive',\n",
       " 'volumes',\n",
       " 'antibody',\n",
       " 'sequences',\n",
       " 'released',\n",
       " 'online',\n",
       " 'Tai',\n",
       " 'Te',\n",
       " 'Wu',\n",
       " '1980',\n",
       " '199118',\n",
       " '1970s',\n",
       " 'new',\n",
       " 'techniques',\n",
       " 'sequencing',\n",
       " 'DNA',\n",
       " 'applied',\n",
       " 'bacteriophage',\n",
       " 'MS2',\n",
       " 'X174',\n",
       " 'extended',\n",
       " 'nucleotide',\n",
       " 'sequences',\n",
       " 'parsed',\n",
       " 'informational',\n",
       " 'statistical',\n",
       " 'algorithms',\n",
       " 'studies',\n",
       " 'illustrated',\n",
       " 'well',\n",
       " 'known',\n",
       " 'features',\n",
       " 'coding',\n",
       " 'segments',\n",
       " 'triplet',\n",
       " 'code',\n",
       " 'revealed',\n",
       " 'straightforward',\n",
       " 'statistical',\n",
       " 'analyses',\n",
       " 'proof',\n",
       " 'concept',\n",
       " 'bioinformatics',\n",
       " 'would',\n",
       " 'insightful1920',\n",
       " 'order',\n",
       " 'study',\n",
       " 'normal',\n",
       " 'cellular',\n",
       " 'activities',\n",
       " 'altered',\n",
       " 'different',\n",
       " 'disease',\n",
       " 'states',\n",
       " 'raw',\n",
       " 'biological',\n",
       " 'data',\n",
       " 'must',\n",
       " 'combined',\n",
       " 'form',\n",
       " 'comprehensive',\n",
       " 'picture',\n",
       " 'activities',\n",
       " 'Thereforewhen',\n",
       " 'field',\n",
       " 'bioinformatics',\n",
       " 'evolved',\n",
       " 'pressing',\n",
       " 'task',\n",
       " 'involves',\n",
       " 'analysis',\n",
       " 'interpretation',\n",
       " 'various',\n",
       " 'types',\n",
       " 'data',\n",
       " 'also',\n",
       " 'includes',\n",
       " 'nucleotide',\n",
       " 'amino',\n",
       " 'acid',\n",
       " 'sequences',\n",
       " 'protein',\n",
       " 'domains',\n",
       " 'protein',\n",
       " 'structures21',\n",
       " 'Important',\n",
       " 'subdisciplines',\n",
       " 'within',\n",
       " 'bioinformatics',\n",
       " 'computational',\n",
       " 'biology',\n",
       " 'include',\n",
       " 'primary',\n",
       " 'goal',\n",
       " 'bioinformatics',\n",
       " 'increase',\n",
       " 'understanding',\n",
       " 'biological',\n",
       " 'processes',\n",
       " 'sets',\n",
       " 'apart',\n",
       " 'approaches',\n",
       " 'focus',\n",
       " 'developing',\n",
       " 'applying',\n",
       " 'computationally',\n",
       " 'intensive',\n",
       " 'techniques',\n",
       " 'achieve',\n",
       " 'goal',\n",
       " 'Examples',\n",
       " 'include',\n",
       " 'pattern',\n",
       " 'recognition',\n",
       " 'data',\n",
       " 'mining',\n",
       " 'machine',\n",
       " 'learning',\n",
       " 'algorithms',\n",
       " 'visualization',\n",
       " 'Major',\n",
       " 'research',\n",
       " 'efforts',\n",
       " 'field',\n",
       " 'include',\n",
       " 'sequence',\n",
       " 'alignment',\n",
       " 'gene',\n",
       " 'finding',\n",
       " 'genome',\n",
       " 'assembly',\n",
       " 'drug',\n",
       " 'design',\n",
       " 'drug',\n",
       " 'discovery',\n",
       " 'protein',\n",
       " 'structure',\n",
       " 'alignment',\n",
       " 'protein',\n",
       " 'structure',\n",
       " 'prediction',\n",
       " 'prediction',\n",
       " 'gene',\n",
       " 'expression',\n",
       " 'proteinprotein',\n",
       " 'interactions',\n",
       " 'genomewide',\n",
       " 'association',\n",
       " 'studies',\n",
       " 'modeling',\n",
       " 'evolution',\n",
       " 'cell',\n",
       " 'divisionmitosis',\n",
       " 'Bioinformatics',\n",
       " 'entails',\n",
       " 'creation',\n",
       " 'advancement',\n",
       " 'databases',\n",
       " 'algorithms',\n",
       " 'computational',\n",
       " 'statistical',\n",
       " 'techniques',\n",
       " 'theory',\n",
       " 'solve',\n",
       " 'formal',\n",
       " 'practical',\n",
       " 'problems',\n",
       " 'arising',\n",
       " 'management',\n",
       " 'analysis',\n",
       " 'biological',\n",
       " 'data',\n",
       " 'past',\n",
       " 'decades',\n",
       " 'rapid',\n",
       " 'developments',\n",
       " 'genomic',\n",
       " 'molecular',\n",
       " 'research',\n",
       " 'technologies',\n",
       " 'developments',\n",
       " 'information',\n",
       " 'technologies',\n",
       " 'combined',\n",
       " 'produce',\n",
       " 'tremendous',\n",
       " 'amount',\n",
       " 'information',\n",
       " 'related',\n",
       " 'molecular',\n",
       " 'biology',\n",
       " 'Bioinformatics',\n",
       " 'name',\n",
       " 'given',\n",
       " 'mathematical',\n",
       " 'computing',\n",
       " 'approaches',\n",
       " 'used',\n",
       " 'glean',\n",
       " 'understanding',\n",
       " 'biological',\n",
       " 'processes',\n",
       " 'Common',\n",
       " 'activities',\n",
       " 'bioinformatics',\n",
       " 'include',\n",
       " 'mapping',\n",
       " 'analyzing',\n",
       " 'DNA',\n",
       " 'protein',\n",
       " 'sequences',\n",
       " 'aligning',\n",
       " 'DNA',\n",
       " 'protein',\n",
       " 'sequences',\n",
       " 'compare',\n",
       " 'creating',\n",
       " 'viewing',\n",
       " '3D',\n",
       " 'models',\n",
       " 'protein',\n",
       " 'structures',\n",
       " 'Since',\n",
       " 'bacteriophage',\n",
       " 'Phage',\n",
       " 'X174',\n",
       " 'sequenced',\n",
       " '197722',\n",
       " 'DNA',\n",
       " 'sequences',\n",
       " 'thousands',\n",
       " 'organisms',\n",
       " 'decoded',\n",
       " 'stored',\n",
       " 'databases',\n",
       " 'sequence',\n",
       " 'information',\n",
       " 'analyzed',\n",
       " 'determine',\n",
       " 'genes',\n",
       " 'encode',\n",
       " 'proteins',\n",
       " 'RNA',\n",
       " 'genes',\n",
       " 'regulatory',\n",
       " 'sequences',\n",
       " 'structural',\n",
       " 'motifs',\n",
       " 'repetitive',\n",
       " 'sequences',\n",
       " 'comparison',\n",
       " 'genes',\n",
       " 'within',\n",
       " 'species',\n",
       " 'different',\n",
       " 'species',\n",
       " 'show',\n",
       " 'similarities',\n",
       " 'protein',\n",
       " 'functions',\n",
       " 'relations',\n",
       " 'species',\n",
       " 'use',\n",
       " 'molecular',\n",
       " 'systematics',\n",
       " 'construct',\n",
       " 'phylogenetic',\n",
       " 'trees',\n",
       " 'growing',\n",
       " 'amount',\n",
       " 'data',\n",
       " 'long',\n",
       " 'ago',\n",
       " 'became',\n",
       " 'impractical',\n",
       " 'analyze',\n",
       " 'DNA',\n",
       " 'sequences',\n",
       " 'manually',\n",
       " 'Computer',\n",
       " 'programs',\n",
       " 'BLAST',\n",
       " 'used',\n",
       " 'routinely',\n",
       " 'search',\n",
       " 'sequencesas',\n",
       " '2008',\n",
       " '260000',\n",
       " 'organisms',\n",
       " 'containing',\n",
       " '190',\n",
       " 'billion',\n",
       " 'nucleotides23',\n",
       " 'sequences',\n",
       " 'analyzed',\n",
       " 'obtained',\n",
       " 'data',\n",
       " 'storage',\n",
       " 'bank',\n",
       " 'Genbank',\n",
       " 'DNA',\n",
       " 'sequencing',\n",
       " 'still',\n",
       " 'nontrivial',\n",
       " 'problem',\n",
       " 'raw',\n",
       " 'data',\n",
       " 'may',\n",
       " 'noisy',\n",
       " 'affected',\n",
       " 'weak',\n",
       " 'signals',\n",
       " 'Algorithms',\n",
       " 'developed',\n",
       " 'base',\n",
       " 'calling',\n",
       " 'various',\n",
       " 'experimental',\n",
       " 'approaches',\n",
       " 'DNA',\n",
       " 'sequencing',\n",
       " 'DNA',\n",
       " 'sequencing',\n",
       " 'techniques',\n",
       " 'produce',\n",
       " 'short',\n",
       " 'fragments',\n",
       " 'sequence',\n",
       " 'need',\n",
       " 'assembled',\n",
       " 'obtain',\n",
       " 'complete',\n",
       " 'gene',\n",
       " 'genome',\n",
       " 'sequences',\n",
       " 'shotgun',\n",
       " 'sequencing',\n",
       " 'technique',\n",
       " 'used',\n",
       " 'Institute',\n",
       " 'Genomic',\n",
       " 'Research',\n",
       " 'TIGR',\n",
       " 'sequence',\n",
       " 'first',\n",
       " 'bacterial',\n",
       " 'genome',\n",
       " 'Haemophilus',\n",
       " 'influenzae24',\n",
       " 'generates',\n",
       " 'sequences',\n",
       " 'many',\n",
       " 'thousands',\n",
       " 'small',\n",
       " 'DNA',\n",
       " 'fragments',\n",
       " 'ranging',\n",
       " '35',\n",
       " '900',\n",
       " 'nucleotides',\n",
       " 'long',\n",
       " 'depending',\n",
       " 'sequencing',\n",
       " 'technology',\n",
       " 'ends',\n",
       " 'fragments',\n",
       " 'overlap',\n",
       " 'aligned',\n",
       " 'properly',\n",
       " 'genome',\n",
       " 'assembly',\n",
       " 'program',\n",
       " 'used',\n",
       " 'reconstruct',\n",
       " 'complete',\n",
       " 'genome',\n",
       " 'Shotgun',\n",
       " 'sequencing',\n",
       " 'yields',\n",
       " 'sequence',\n",
       " 'data',\n",
       " 'quickly',\n",
       " 'task',\n",
       " 'assembling',\n",
       " 'fragments',\n",
       " 'quite',\n",
       " 'complicated',\n",
       " 'larger',\n",
       " 'genomes',\n",
       " 'genome',\n",
       " 'large',\n",
       " 'human',\n",
       " 'genome',\n",
       " 'may',\n",
       " 'take',\n",
       " 'many',\n",
       " 'days',\n",
       " 'CPU',\n",
       " 'time',\n",
       " 'largememory',\n",
       " 'multiprocessor',\n",
       " 'computers',\n",
       " 'assemble',\n",
       " 'fragments',\n",
       " 'resulting',\n",
       " 'assembly',\n",
       " 'usually',\n",
       " 'contains',\n",
       " 'numerous',\n",
       " 'gaps',\n",
       " 'must',\n",
       " 'filled',\n",
       " 'later',\n",
       " 'Shotgun',\n",
       " 'sequencing',\n",
       " 'method',\n",
       " 'choice',\n",
       " 'virtually',\n",
       " 'genomes',\n",
       " 'sequenced',\n",
       " 'rather',\n",
       " 'chaintermination',\n",
       " 'chemical',\n",
       " 'degradation',\n",
       " 'methods',\n",
       " 'genome',\n",
       " 'assembly',\n",
       " 'algorithms',\n",
       " 'critical',\n",
       " 'area',\n",
       " 'bioinformatics',\n",
       " 'research',\n",
       " 'genomics',\n",
       " 'annotation',\n",
       " 'refers',\n",
       " 'process',\n",
       " 'marking',\n",
       " 'stop',\n",
       " 'start',\n",
       " 'regions',\n",
       " 'genes',\n",
       " 'biological',\n",
       " 'features',\n",
       " 'sequenced',\n",
       " 'DNA',\n",
       " 'sequence',\n",
       " 'Many',\n",
       " 'genomes',\n",
       " 'large',\n",
       " 'annotated',\n",
       " 'hand',\n",
       " 'rate',\n",
       " 'sequencing',\n",
       " 'exceeds',\n",
       " 'rate',\n",
       " 'genome',\n",
       " 'annotation',\n",
       " 'genome',\n",
       " 'annotation',\n",
       " 'become',\n",
       " 'new',\n",
       " 'bottleneck',\n",
       " 'bioinformaticswhen',\n",
       " 'Genome',\n",
       " 'annotation',\n",
       " 'classified',\n",
       " 'three',\n",
       " 'levels',\n",
       " 'nucleotide',\n",
       " 'protein',\n",
       " 'process',\n",
       " 'levels',\n",
       " 'Gene',\n",
       " 'finding',\n",
       " 'chief',\n",
       " 'aspect',\n",
       " 'nucleotidelevel',\n",
       " 'annotation',\n",
       " 'complex',\n",
       " 'genomes',\n",
       " 'combination',\n",
       " 'ab',\n",
       " 'initio',\n",
       " 'gene',\n",
       " 'prediction',\n",
       " 'sequence',\n",
       " 'comparison',\n",
       " 'expressed',\n",
       " 'sequence',\n",
       " 'databases',\n",
       " 'organisms',\n",
       " 'successful',\n",
       " 'Nucleotidelevel',\n",
       " 'annotation',\n",
       " 'also',\n",
       " 'allows',\n",
       " 'integration',\n",
       " 'genome',\n",
       " 'sequence',\n",
       " 'genetic',\n",
       " 'physical',\n",
       " 'maps',\n",
       " 'genome',\n",
       " 'principal',\n",
       " 'aim',\n",
       " 'proteinlevel',\n",
       " 'annotation',\n",
       " 'assign',\n",
       " 'function',\n",
       " 'protein',\n",
       " 'products',\n",
       " 'genome',\n",
       " 'Databases',\n",
       " 'protein',\n",
       " 'sequences',\n",
       " 'functional',\n",
       " 'domains',\n",
       " 'motifs',\n",
       " 'used',\n",
       " 'type',\n",
       " 'annotation',\n",
       " 'half',\n",
       " 'predicted',\n",
       " 'proteins',\n",
       " 'new',\n",
       " 'genome',\n",
       " 'sequence',\n",
       " 'tend',\n",
       " 'obvious',\n",
       " 'function',\n",
       " 'Understanding',\n",
       " 'function',\n",
       " 'genes',\n",
       " 'products',\n",
       " 'context',\n",
       " 'cellular',\n",
       " 'organismal',\n",
       " 'physiology',\n",
       " 'goal',\n",
       " 'processlevel',\n",
       " 'annotation',\n",
       " 'obstacle',\n",
       " 'processlevel',\n",
       " 'annotation',\n",
       " 'inconsistency',\n",
       " 'terms',\n",
       " 'used',\n",
       " 'different',\n",
       " 'model',\n",
       " 'systems',\n",
       " 'Gene',\n",
       " 'Ontology',\n",
       " 'Consortium',\n",
       " 'helping',\n",
       " 'solve',\n",
       " 'problem25',\n",
       " 'first',\n",
       " 'description',\n",
       " 'comprehensive',\n",
       " 'annotation',\n",
       " 'system',\n",
       " 'published',\n",
       " '199524',\n",
       " 'Institute',\n",
       " 'Genomic',\n",
       " 'Research',\n",
       " 'performed',\n",
       " 'first',\n",
       " 'complete',\n",
       " 'sequencing',\n",
       " 'analysis',\n",
       " 'genome',\n",
       " 'freeliving',\n",
       " 'organism',\n",
       " 'bacterium',\n",
       " 'Haemophilus',\n",
       " 'influenzae24',\n",
       " 'system',\n",
       " 'identifies',\n",
       " 'genes',\n",
       " 'encoding',\n",
       " 'proteins',\n",
       " 'transfer',\n",
       " 'RNAs',\n",
       " 'ribosomal',\n",
       " 'RNAs',\n",
       " 'order',\n",
       " 'make',\n",
       " 'initial',\n",
       " 'functional',\n",
       " 'assignments',\n",
       " 'GeneMark',\n",
       " 'program',\n",
       " 'trained',\n",
       " 'find',\n",
       " 'proteincoding',\n",
       " 'genes',\n",
       " 'Haemophilus',\n",
       " 'influenzae',\n",
       " 'constantly',\n",
       " 'changing',\n",
       " 'improving',\n",
       " 'Following',\n",
       " 'goals',\n",
       " 'Human',\n",
       " 'Genome',\n",
       " 'Project',\n",
       " 'left',\n",
       " 'achieve',\n",
       " 'closure',\n",
       " '2003',\n",
       " 'ENCODE',\n",
       " 'project',\n",
       " 'developed',\n",
       " 'National',\n",
       " 'Human',\n",
       " 'Genome',\n",
       " 'Research',\n",
       " 'Institute',\n",
       " 'project',\n",
       " 'collaborative',\n",
       " 'data',\n",
       " 'collection',\n",
       " 'functional',\n",
       " 'elements',\n",
       " 'human',\n",
       " 'genome',\n",
       " 'uses',\n",
       " 'nextgeneration',\n",
       " 'DNAsequencing',\n",
       " 'technologies',\n",
       " 'genomic',\n",
       " 'tiling',\n",
       " 'arrays',\n",
       " 'technologies',\n",
       " 'able',\n",
       " 'automatically',\n",
       " 'generate',\n",
       " 'large',\n",
       " 'amounts',\n",
       " 'data',\n",
       " 'dramatically',\n",
       " 'reduced',\n",
       " 'perbase',\n",
       " 'cost',\n",
       " 'accuracy',\n",
       " 'base',\n",
       " 'call',\n",
       " 'error',\n",
       " 'fidelity',\n",
       " 'assembly',\n",
       " 'error',\n",
       " 'genome',\n",
       " 'annotation',\n",
       " ...]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "words"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Step3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gensim.models import Word2Vec"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "model = Word2Vec([words],window=5,min_count=1,workers=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "vocabulary = model.wv.index_to_key"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['protein',\n",
       " 'data',\n",
       " 'bioinformatics',\n",
       " 'genome',\n",
       " 'analysis',\n",
       " 'gene',\n",
       " 'sequence',\n",
       " 'genes',\n",
       " 'biological',\n",
       " 'used',\n",
       " 'sequences',\n",
       " 'sequencing',\n",
       " 'Bioinformatics',\n",
       " 'DNA',\n",
       " 'proteins',\n",
       " 'structure',\n",
       " 'function',\n",
       " 'algorithms',\n",
       " 'research',\n",
       " 'biology',\n",
       " 'include',\n",
       " 'annotation',\n",
       " 'information',\n",
       " 'many',\n",
       " 'techniques',\n",
       " 'genomes',\n",
       " 'also',\n",
       " 'studies',\n",
       " 'complex',\n",
       " 'may',\n",
       " 'developed',\n",
       " 'databases',\n",
       " 'large',\n",
       " 'prediction',\n",
       " 'tools',\n",
       " 'expression',\n",
       " 'methods',\n",
       " 'computational',\n",
       " 'field',\n",
       " 'different',\n",
       " 'cancer',\n",
       " 'processes',\n",
       " 'models',\n",
       " 'statistical',\n",
       " 'including',\n",
       " 'molecular',\n",
       " 'species',\n",
       " 'study',\n",
       " 'Genome',\n",
       " 'use',\n",
       " 'one',\n",
       " 'genetic',\n",
       " 'well',\n",
       " 'mutations',\n",
       " 'computer',\n",
       " 'software',\n",
       " 'various',\n",
       " 'applied',\n",
       " 'cells',\n",
       " 'predict',\n",
       " 'interactions',\n",
       " 'involves',\n",
       " 'involved',\n",
       " 'structures',\n",
       " 'simulation',\n",
       " 'based',\n",
       " 'variants',\n",
       " 'within',\n",
       " 'important',\n",
       " 'networks',\n",
       " 'genomic',\n",
       " 'structural',\n",
       " 'process',\n",
       " 'Databases',\n",
       " 'clustering',\n",
       " 'disease',\n",
       " 'computers',\n",
       " 'cellular',\n",
       " 'Computational',\n",
       " 'rare',\n",
       " 'types',\n",
       " 'fragments',\n",
       " 'technologies',\n",
       " 'determine',\n",
       " 'problem',\n",
       " 'functional',\n",
       " 'assembly',\n",
       " 'allow',\n",
       " 'includes',\n",
       " 'open',\n",
       " 'proteinprotein',\n",
       " 'association',\n",
       " 'provide',\n",
       " 'analyze',\n",
       " 'first',\n",
       " 'system',\n",
       " 'sequenced',\n",
       " 'BioCompute',\n",
       " 'Human',\n",
       " 'systems',\n",
       " 'often',\n",
       " 'work',\n",
       " 'single',\n",
       " 'theory',\n",
       " 'found',\n",
       " 'experiments',\n",
       " 'example',\n",
       " 'signal',\n",
       " 'organism',\n",
       " 'evolutionary',\n",
       " 'determined',\n",
       " 'threedimensional',\n",
       " 'particular',\n",
       " 'thousands',\n",
       " 'microarrays',\n",
       " 'organisms',\n",
       " 'called',\n",
       " 'regulatory',\n",
       " 'workflow',\n",
       " 'highthroughput',\n",
       " 'allows',\n",
       " 'common',\n",
       " 'motifs',\n",
       " 'location',\n",
       " 'diseases',\n",
       " 'obtained',\n",
       " 'resources',\n",
       " 'elements',\n",
       " 'Gene',\n",
       " 'levels',\n",
       " 'whole',\n",
       " 'complete',\n",
       " 'Institute',\n",
       " 'homology',\n",
       " 'Research',\n",
       " 'predicted',\n",
       " 'approaches',\n",
       " 'compare',\n",
       " 'multiple',\n",
       " 'understand',\n",
       " 'nucleotide',\n",
       " 'new',\n",
       " 'amounts',\n",
       " 'online',\n",
       " 'would',\n",
       " 'ontologies',\n",
       " 'identification',\n",
       " 'regulation',\n",
       " 'form',\n",
       " 'level',\n",
       " 'amino',\n",
       " 'modeling',\n",
       " 'understanding',\n",
       " 'management',\n",
       " 'problems',\n",
       " 'evolution',\n",
       " 'acid',\n",
       " 'Project',\n",
       " 'technology',\n",
       " 'rapid',\n",
       " 'mining',\n",
       " 'identified',\n",
       " 'raw',\n",
       " 'parts',\n",
       " 'provides',\n",
       " 'Although',\n",
       " 'processing',\n",
       " 'group',\n",
       " 'annotations',\n",
       " 'possible',\n",
       " 'infer',\n",
       " 'needed',\n",
       " 'interaction',\n",
       " 'unknown',\n",
       " 'groups',\n",
       " 'Data',\n",
       " 'statistics',\n",
       " 'source',\n",
       " 'subcellular',\n",
       " 'analyzing',\n",
       " 'analyses',\n",
       " 'pipelines',\n",
       " 'capture',\n",
       " 'genomics',\n",
       " 'localization',\n",
       " 'might',\n",
       " 'applications',\n",
       " 'mRNA',\n",
       " 'Pan',\n",
       " 'produce',\n",
       " 'computing',\n",
       " 'challenges',\n",
       " 'hybridization',\n",
       " 'area',\n",
       " 'virtually',\n",
       " 'human',\n",
       " 'Shotgun',\n",
       " 'small',\n",
       " 'activities',\n",
       " 'Haemophilus',\n",
       " 'Genomic',\n",
       " 'domains',\n",
       " 'need',\n",
       " 'primary',\n",
       " 'goal',\n",
       " 'Molecular',\n",
       " 'Examples',\n",
       " 'finding',\n",
       " 'drug',\n",
       " 'cell',\n",
       " 'analyzed',\n",
       " 'related',\n",
       " 'segments',\n",
       " 'regions',\n",
       " 'Many',\n",
       " 'context',\n",
       " 'programs',\n",
       " 'instance',\n",
       " 'mathematical',\n",
       " 'cost',\n",
       " 'became',\n",
       " 'available',\n",
       " 'collection',\n",
       " 'project',\n",
       " 'published',\n",
       " 'three',\n",
       " 'type',\n",
       " 'alignment',\n",
       " 'Another',\n",
       " 'comprehensive',\n",
       " 'maps',\n",
       " 'integration',\n",
       " 'known',\n",
       " 'features',\n",
       " 'shared',\n",
       " 'millions',\n",
       " 'Biology',\n",
       " 'platforms',\n",
       " 'designed',\n",
       " 'Sequence',\n",
       " '1970',\n",
       " 'states',\n",
       " 'cause',\n",
       " 'Systems',\n",
       " 'disorders',\n",
       " 'rate',\n",
       " 'paradigm',\n",
       " 'Mendelian',\n",
       " 'become',\n",
       " 'parallel',\n",
       " 'attempts',\n",
       " 'chemical',\n",
       " 'molecule',\n",
       " 'de',\n",
       " 'RNA',\n",
       " 'must',\n",
       " 'coexpressed',\n",
       " 'database',\n",
       " 'normal',\n",
       " 'order',\n",
       " 'AlphaFold',\n",
       " 'driven',\n",
       " 'concept',\n",
       " 'associated',\n",
       " 'silico',\n",
       " 'number',\n",
       " 'code',\n",
       " 'classified',\n",
       " 'aspect',\n",
       " 'combined',\n",
       " 'using',\n",
       " 'Furthermore',\n",
       " 'taxonomic',\n",
       " 'products',\n",
       " 'assign',\n",
       " 'could',\n",
       " 'integrative',\n",
       " 'chromosome',\n",
       " 'docking',\n",
       " 'conformation',\n",
       " 'sample',\n",
       " 'pathways',\n",
       " 'etc',\n",
       " 'task',\n",
       " 'set',\n",
       " 'teach',\n",
       " 'growing',\n",
       " 'variety',\n",
       " 'expressed',\n",
       " 'long',\n",
       " 'state',\n",
       " 'combination',\n",
       " 'simple',\n",
       " 'present',\n",
       " 'definition',\n",
       " 'life',\n",
       " 'rather',\n",
       " 'organelles',\n",
       " 'Open',\n",
       " 'noise',\n",
       " 'affected',\n",
       " 'billion',\n",
       " 'bases',\n",
       " 'reconstruct',\n",
       " 'hundreds',\n",
       " 'program',\n",
       " 'metabolic',\n",
       " 'variant',\n",
       " 'essential',\n",
       " 'Institutes',\n",
       " 'nucleotides',\n",
       " 'base',\n",
       " 'released',\n",
       " 'ranging',\n",
       " 'early',\n",
       " 'nuclear',\n",
       " 'easily',\n",
       " 'experimental',\n",
       " 'manually',\n",
       " 'influenzae24',\n",
       " 'impractical',\n",
       " 'US',\n",
       " 'bacterial',\n",
       " 'compiled',\n",
       " 'initially',\n",
       " 'course',\n",
       " 'application',\n",
       " 'directed',\n",
       " 'Conference',\n",
       " 'able',\n",
       " 'Bethesda',\n",
       " 'ontology',\n",
       " 'numerous',\n",
       " 'contains',\n",
       " 'artificial',\n",
       " 'resulting',\n",
       " 'contain',\n",
       " 'experiment',\n",
       " 'identical',\n",
       " 'image',\n",
       " 'still',\n",
       " 'time',\n",
       " 'molecules',\n",
       " 'Finding',\n",
       " 'identify',\n",
       " 'tremendous',\n",
       " 'larger',\n",
       " 'speed',\n",
       " 'hemoglobin',\n",
       " 'One',\n",
       " 'existing',\n",
       " 'complicated',\n",
       " 'bacteriophage',\n",
       " 'since',\n",
       " 'B',\n",
       " 'homologous',\n",
       " 'whose',\n",
       " 'far',\n",
       " 'solution',\n",
       " 'individual',\n",
       " 'concepts',\n",
       " 'NIH',\n",
       " 'point',\n",
       " 'consortium',\n",
       " 'continued',\n",
       " 'uses',\n",
       " 'Informatics',\n",
       " 'accuracy',\n",
       " 'chromosomal',\n",
       " 'potential',\n",
       " 'sets',\n",
       " 'categories',\n",
       " 'FDA',\n",
       " 'proteincoding',\n",
       " 'Taverna',\n",
       " 'object',\n",
       " 'tissue',\n",
       " 'error',\n",
       " 'Biological',\n",
       " 'mapping',\n",
       " 'European',\n",
       " 'efforts',\n",
       " 'Software',\n",
       " 'particularly',\n",
       " 'stakeholders',\n",
       " 'properties',\n",
       " 'National',\n",
       " 'mathematics',\n",
       " 'solve',\n",
       " 'public',\n",
       " 'mass',\n",
       " 'nextgeneration',\n",
       " 'programming',\n",
       " 'comparative',\n",
       " '3D',\n",
       " 'interpreting',\n",
       " 'standalone',\n",
       " 'generate',\n",
       " 'specific',\n",
       " 'made',\n",
       " 'practical',\n",
       " 'phenotypes',\n",
       " 'Specialization',\n",
       " 'entities',\n",
       " 'institutions',\n",
       " 'free',\n",
       " 'opensource',\n",
       " 'two',\n",
       " 'Protein',\n",
       " 'developments',\n",
       " 'arrays',\n",
       " 'RNAs',\n",
       " 'Image',\n",
       " 'informatics',\n",
       " 'heuristics',\n",
       " 'comparison',\n",
       " 'model',\n",
       " 'developing',\n",
       " 'linguistics',\n",
       " 'focus',\n",
       " 'Markov',\n",
       " 'aids',\n",
       " 'processlevel',\n",
       " 'paper',\n",
       " 'region',\n",
       " 'observed',\n",
       " 'samples',\n",
       " 'text',\n",
       " 'webbased',\n",
       " 'literature',\n",
       " 'microarray',\n",
       " 'development',\n",
       " 'detection',\n",
       " 'increase',\n",
       " 'UGENE',\n",
       " 'amount',\n",
       " 'biodiversity',\n",
       " 'steps',\n",
       " 'principles',\n",
       " 'proteomics',\n",
       " 'Raspberry',\n",
       " 'organizational',\n",
       " 'Pi',\n",
       " 'populations',\n",
       " 'examples',\n",
       " 'science',\n",
       " 'reproducibility',\n",
       " 'ideas',\n",
       " 'tag',\n",
       " 'among',\n",
       " 'transfer',\n",
       " 'thus',\n",
       " 'discuss',\n",
       " 'achieve',\n",
       " 'nonprofit',\n",
       " 'creating',\n",
       " 'calling',\n",
       " 'glean',\n",
       " 'included',\n",
       " 'aligning',\n",
       " 'Algorithms',\n",
       " 'obtain',\n",
       " 'assembled',\n",
       " 'short',\n",
       " 'held',\n",
       " 'weak',\n",
       " 'conference',\n",
       " 'paradigm61',\n",
       " 'signals',\n",
       " 'Common',\n",
       " 'similarities',\n",
       " 'noisy',\n",
       " 'BLAST',\n",
       " 'years',\n",
       " 'next',\n",
       " 'sequencesas',\n",
       " 'search',\n",
       " 'routinely',\n",
       " 'encode',\n",
       " 'bioinformatics60',\n",
       " 'Computer',\n",
       " '260000',\n",
       " 'ago',\n",
       " 'repetitive',\n",
       " 'trees',\n",
       " 'show',\n",
       " 'construct',\n",
       " 'systematics',\n",
       " 'relations',\n",
       " '2008',\n",
       " 'stored',\n",
       " 'viewing',\n",
       " '197722',\n",
       " 'regularly',\n",
       " 'met',\n",
       " 'Health',\n",
       " 'nontrivial',\n",
       " 'Since',\n",
       " 'Phage',\n",
       " 'X174',\n",
       " 'Genbank',\n",
       " 'containing',\n",
       " 'bank',\n",
       " 'storage',\n",
       " 'functions',\n",
       " 'Campus',\n",
       " 'nucleotides23',\n",
       " 'decoded',\n",
       " '190',\n",
       " 'phylogenetic',\n",
       " 'assisting',\n",
       " 'sponsored',\n",
       " 'GeneMark',\n",
       " 'initial',\n",
       " 'make',\n",
       " 'ribosomal',\n",
       " 'encoding',\n",
       " 'identifies',\n",
       " 'bacterium',\n",
       " 'applicability',\n",
       " 'freeliving',\n",
       " 'performed',\n",
       " '199524',\n",
       " 'description',\n",
       " 'problem25',\n",
       " 'helping',\n",
       " 'Consortium',\n",
       " 'Ontology',\n",
       " 'terms',\n",
       " 'inconsistency',\n",
       " 'obstacle',\n",
       " 'physiology',\n",
       " 'organismal',\n",
       " 'Understanding',\n",
       " 'obvious',\n",
       " 'tend',\n",
       " 'solutions',\n",
       " 'half',\n",
       " 'range',\n",
       " 'format',\n",
       " 'interface',\n",
       " 'distributed',\n",
       " 'assignments',\n",
       " 'trained',\n",
       " 'aim',\n",
       " 'find',\n",
       " 'informative',\n",
       " 'fact',\n",
       " 'Alignment',\n",
       " 'BSA',\n",
       " 'similarity',\n",
       " 'Analysis59',\n",
       " 'primarily',\n",
       " 'fidelity',\n",
       " 'call',\n",
       " 'perbase',\n",
       " 'reduced',\n",
       " 'dramatically',\n",
       " 'automatically',\n",
       " 'tiling',\n",
       " 'DNAsequencing',\n",
       " 'availability',\n",
       " 'serviceoriented',\n",
       " 'collaborative',\n",
       " 'demonstrate',\n",
       " 'ENCODE',\n",
       " '2003',\n",
       " 'closure',\n",
       " 'left',\n",
       " 'goals',\n",
       " 'Following',\n",
       " 'improving',\n",
       " 'changing',\n",
       " 'constantly',\n",
       " 'influenzae',\n",
       " 'proteinlevel',\n",
       " 'principal',\n",
       " 'shotgun',\n",
       " 'gaps',\n",
       " 'usually',\n",
       " 'assemble',\n",
       " 'Galaxy',\n",
       " 'multiprocessor',\n",
       " 'largememory',\n",
       " 'CPU',\n",
       " 'days',\n",
       " 'take',\n",
       " 'Kepler',\n",
       " 'quite',\n",
       " 'assembling',\n",
       " 'quickly',\n",
       " 'yields',\n",
       " 'Anduril',\n",
       " 'properly',\n",
       " 'aligned',\n",
       " 'overlap',\n",
       " 'ends',\n",
       " 'depending',\n",
       " '900',\n",
       " '35',\n",
       " 'HIVE',\n",
       " '2014',\n",
       " 'generates',\n",
       " 'TIGR',\n",
       " 'Food',\n",
       " 'Drug',\n",
       " 'Administration',\n",
       " 'technique',\n",
       " 'name',\n",
       " 'filled',\n",
       " 'extensible',\n",
       " 'later',\n",
       " 'physical',\n",
       " 'specialized',\n",
       " 'Nucleotidelevel',\n",
       " 'successful',\n",
       " 'initio',\n",
       " 'ab',\n",
       " 'nucleotidelevel',\n",
       " 'chief',\n",
       " 'specifically',\n",
       " 'compose',\n",
       " 'execute',\n",
       " 'bioinformaticswhen',\n",
       " 'bottleneck',\n",
       " 'exceeds',\n",
       " 'hand',\n",
       " 'annotated',\n",
       " 'series',\n",
       " 'manipulation',\n",
       " 'start',\n",
       " 'stop',\n",
       " 'marking',\n",
       " 'refers',\n",
       " 'giving',\n",
       " 'critical',\n",
       " 'degradation',\n",
       " 'chaintermination',\n",
       " 'service',\n",
       " 'choice',\n",
       " 'method',\n",
       " 'given',\n",
       " 'Washington',\n",
       " 'representatives',\n",
       " 'catalogue',\n",
       " 'generally',\n",
       " 'Canadian',\n",
       " 'aspects',\n",
       " 'Portal',\n",
       " 'Training',\n",
       " 'helps',\n",
       " 'Swiss',\n",
       " 'learning6566',\n",
       " 'offered',\n",
       " 'part',\n",
       " 'courses',\n",
       " 'Rosalind',\n",
       " 'DNA2',\n",
       " 'RNA23',\n",
       " 'Workshops',\n",
       " 'comparing',\n",
       " 'aid',\n",
       " 'videos',\n",
       " 'slides',\n",
       " 'role',\n",
       " 'plays',\n",
       " 'query',\n",
       " 'organize',\n",
       " 'training',\n",
       " 'workshops',\n",
       " 'website',\n",
       " 'Creative',\n",
       " 'Commons',\n",
       " 'annotating',\n",
       " 'license',\n",
       " 'genetics',\n",
       " 'proteins4',\n",
       " 'biomolecular',\n",
       " '4273pi',\n",
       " 'mid1990s',\n",
       " 'masters',\n",
       " 'inperson',\n",
       " 'experienced',\n",
       " 'explosive',\n",
       " 'growth',\n",
       " 'starting',\n",
       " 'largely',\n",
       " 'interactions5678',\n",
       " 'taught',\n",
       " 'regulators6364',\n",
       " 'collaborators',\n",
       " 'employees',\n",
       " 'advances',\n",
       " 'record',\n",
       " 'degree',\n",
       " 'systems10',\n",
       " 'biochemistry',\n",
       " 'placed',\n",
       " 'systems910111213',\n",
       " 'biotic',\n",
       " 'universities',\n",
       " 'nature',\n",
       " 'refer',\n",
       " 'Hesper',\n",
       " 'Ben',\n",
       " 'Hogeweg',\n",
       " 'Paulien',\n",
       " 'coined',\n",
       " 'lends',\n",
       " 'term',\n",
       " 'computeraided',\n",
       " '4273',\n",
       " 'results',\n",
       " 'government',\n",
       " 'Johns',\n",
       " 'Analysis',\n",
       " 'engineering',\n",
       " 'EdXs',\n",
       " 'Hopkins',\n",
       " 'interpret',\n",
       " 'subsequent',\n",
       " 'Science',\n",
       " 'queries',\n",
       " 'referred',\n",
       " 'Diego',\n",
       " 'San',\n",
       " 'UC',\n",
       " 'Courseras',\n",
       " 'disciplines',\n",
       " 'Life',\n",
       " 'physics',\n",
       " 'chemistry',\n",
       " 'Sciences',\n",
       " 'XSeries',\n",
       " 'Harvard',\n",
       " 'especially',\n",
       " 'several',\n",
       " 'conferences',\n",
       " 'concerned',\n",
       " 'notable',\n",
       " 'Intelligent',\n",
       " 'develops',\n",
       " 'ISMB',\n",
       " 'interdisciplinary',\n",
       " 'listen',\n",
       " 'baonfrmtks',\n",
       " 'certifications',\n",
       " 'MOOC',\n",
       " 'useful',\n",
       " 'nucleic',\n",
       " 'esp',\n",
       " 'agricultural',\n",
       " 'adults',\n",
       " 'differences',\n",
       " 'tries',\n",
       " 'low',\n",
       " 'runs',\n",
       " 'reused',\n",
       " 'materials',\n",
       " 'sequences1',\n",
       " 'educational',\n",
       " 'offers',\n",
       " 'project67',\n",
       " 'extraction',\n",
       " 'desirable',\n",
       " 'adaptations',\n",
       " 'unique',\n",
       " 'school',\n",
       " 'basis',\n",
       " 'pupils6869',\n",
       " '4283',\n",
       " 'better',\n",
       " 'SNPs',\n",
       " 'polymorphisms',\n",
       " 'actively',\n",
       " 'academics',\n",
       " 'staff',\n",
       " 'run',\n",
       " '4283',\n",
       " 'operating',\n",
       " 'system7071',\n",
       " 'Analyzing',\n",
       " 'JSONized',\n",
       " 'meaningful',\n",
       " 'pressing',\n",
       " 'digital',\n",
       " 'altered',\n",
       " 'decided',\n",
       " 'picture',\n",
       " 'Thereforewhen',\n",
       " 'evolved',\n",
       " 'interpretation',\n",
       " 'York',\n",
       " 'University',\n",
       " 'Multiple',\n",
       " 'George',\n",
       " 'Center',\n",
       " 'structures21',\n",
       " 'Important',\n",
       " 'lab',\n",
       " 'insightful1920',\n",
       " 'notebooks',\n",
       " 'proof',\n",
       " 'straightforward',\n",
       " 'revealed',\n",
       " 'triplet',\n",
       " 'replication',\n",
       " 'coding',\n",
       " 'review',\n",
       " 'reuse',\n",
       " 'illustrated',\n",
       " 'protocols',\n",
       " 'informational',\n",
       " 'parsed',\n",
       " 'extended',\n",
       " 'X174',\n",
       " 'subdisciplines',\n",
       " 'New',\n",
       " 'bioRxiv',\n",
       " 'creation',\n",
       " 'represented',\n",
       " 'genomewide',\n",
       " 'leaders',\n",
       " 'Session',\n",
       " 'divisionmitosis',\n",
       " 'entails',\n",
       " 'advancement',\n",
       " 'apart',\n",
       " 'formal',\n",
       " 'academic',\n",
       " 'arising',\n",
       " 'industry',\n",
       " 'past',\n",
       " 'decades',\n",
       " 'branches',\n",
       " 'discovery',\n",
       " 'design',\n",
       " 'Centers',\n",
       " 'Variome',\n",
       " 'Federation',\n",
       " 'Major',\n",
       " 'visualization',\n",
       " 'learning',\n",
       " 'machine',\n",
       " 'recognition',\n",
       " 'pattern',\n",
       " 'Medical',\n",
       " 'intensive',\n",
       " 'computationally',\n",
       " 'applying',\n",
       " 'Stanford',\n",
       " 'MS2',\n",
       " 'proposed',\n",
       " 'enable',\n",
       " '100000',\n",
       " 'advance',\n",
       " 'Object',\n",
       " 'reduction',\n",
       " 'completion',\n",
       " 'labs',\n",
       " 'discussed',\n",
       " 'year',\n",
       " '1970s',\n",
       " 'full',\n",
       " 'reconvened',\n",
       " '2016',\n",
       " '1000',\n",
       " 'less14',\n",
       " 'Computers',\n",
       " 'copied',\n",
       " 'control',\n",
       " 'discrete',\n",
       " 'foundations',\n",
       " 'theoretical',\n",
       " 'depend',\n",
       " 'turn',\n",
       " 'standard',\n",
       " 'soft',\n",
       " 'intelligence',\n",
       " 'trial',\n",
       " 'graph',\n",
       " 'document',\n",
       " 'preprint',\n",
       " 'uploaded',\n",
       " 'running',\n",
       " 'writing',\n",
       " 'staff62',\n",
       " 'accessible',\n",
       " 'Frederick',\n",
       " 'personnel',\n",
       " '199118',\n",
       " '1980',\n",
       " 'Wu',\n",
       " 'Te',\n",
       " 'Tai',\n",
       " 'greater',\n",
       " 'antibody',\n",
       " 'volumes',\n",
       " 'continuity',\n",
       " 'pioneered',\n",
       " 'Kabat',\n",
       " 'Elvin',\n",
       " 'contributor',\n",
       " 'evolution17',\n",
       " 'books16',\n",
       " 'Sanger',\n",
       " 'flux',\n",
       " 'furthering',\n",
       " 'exchange',\n",
       " 'field15',\n",
       " 'pioneer',\n",
       " 'Dayhoff',\n",
       " 'Oakley',\n",
       " 'Margaret',\n",
       " 'turned',\n",
       " 'funded',\n",
       " 'Comparing',\n",
       " '1950s',\n",
       " 'insulin',\n",
       " 'transparent',\n",
       " 'featurerich',\n",
       " 'inversion',\n",
       " 'distribution',\n",
       " 'reflects',\n",
       " 'branch',\n",
       " 'share',\n",
       " 'formation',\n",
       " 'Opensource',\n",
       " 'Homology',\n",
       " 'humans',\n",
       " 'legumes',\n",
       " 'leghemoglobin',\n",
       " 'distant',\n",
       " 'relatives',\n",
       " 'superfamily',\n",
       " 'serve',\n",
       " 'purpose',\n",
       " 'transporting',\n",
       " 'oxygen',\n",
       " 'timecitation',\n",
       " 'towards',\n",
       " 'remains',\n",
       " 'spongiform',\n",
       " '3dimensional',\n",
       " 'native',\n",
       " 'environment',\n",
       " 'exception',\n",
       " 'misfolded',\n",
       " 'bovine',\n",
       " 'encephalopathy',\n",
       " 'general',\n",
       " 'linked',\n",
       " 'Additional',\n",
       " 'secondary',\n",
       " 'tertiary',\n",
       " 'quaternary',\n",
       " 'viable',\n",
       " 'completely',\n",
       " 'near',\n",
       " 'uniquely',\n",
       " 'purposes',\n",
       " 'deeplearning',\n",
       " 'algorithmsbased',\n",
       " 'Googles',\n",
       " 'DeepMind',\n",
       " 'greatly',\n",
       " 'outperforms',\n",
       " 'methods54how',\n",
       " 'database55',\n",
       " 'Network',\n",
       " 'seeks',\n",
       " 'relationships',\n",
       " 'constructed',\n",
       " 'entity',\n",
       " 'network',\n",
       " 'integrate',\n",
       " '2021',\n",
       " 'mutagenesis',\n",
       " 'ligandbinding',\n",
       " 'physicsbased',\n",
       " 'bioinformation',\n",
       " 'ancestor53',\n",
       " 'predicting',\n",
       " 'threading',\n",
       " 'novo',\n",
       " 'scratch',\n",
       " 'Virtual',\n",
       " 'crystal',\n",
       " 'Screening',\n",
       " 'Quantitative',\n",
       " 'StructureActivity',\n",
       " 'Relationship',\n",
       " 'proteochemometric',\n",
       " 'PCM',\n",
       " 'determines',\n",
       " 'codes',\n",
       " 'hydrophobic',\n",
       " 'us',\n",
       " 'kmeans',\n",
       " 'selforganizing',\n",
       " ...]"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vocabulary"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "sim_words = model.wv.most_similar('learning')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('detection', 0.3454124927520752),\n",
       " ('effect', 0.33981311321258545),\n",
       " ('aim', 0.3367534577846527),\n",
       " ('trained', 0.3200497627258301),\n",
       " ('practical', 0.29747986793518066),\n",
       " ('text', 0.2875995337963104),\n",
       " ('developers', 0.27530112862586975),\n",
       " ('exact', 0.2713748812675476),\n",
       " ('Federation', 0.266357421875),\n",
       " ('signature', 0.260215699672699)]"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sim_words"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.16"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
